{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "import numpy as np\n",
    "\n",
    "import dendropy\n",
    "\n",
    "import os\n",
    "\n",
    "import pysam\n",
    "import vamb\n",
    "\n",
    "import glob\n",
    "\n",
    "import pickle"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Purpose of this Notebook\n",
    "\n",
    "In order to test the relevant impact of tuning the beta parameter for out disentangled autoencoder, I want to set up some compelling experiments. These will consist of a small grid search methodology where we modify both the number of genomes in a simulation as well as the phylogenetic distance between those genomes. That strategy would look something like this:\n",
    "\n",
    "\n",
    "\n",
    "|     | 2   |  10 | 50  |\n",
    "|-----|-----|-----|-----|\n",
    "| 5   |     |     |     |\n",
    "| 50  |     |     |     |\n",
    "| 250 |     |     |     |\n",
    "\n",
    "\n",
    "In order do do this, I'm going to set up these simulations first by deciding how I'll collect input reference genomes of varying levels of phylogenetic similarity."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 1. Gathering Genomes and Adjusting Similarity\n",
    "\n",
    "We'll start by obtaining the complete list of bacterial genomes present in the NCBI Genome database:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--2021-03-14 00:32:06--  ftp://ftp.ncbi.nlm.nih.gov/genomes/genbank/bacteria/assembly_summary.txt\n",
      "           => ‘assembly_summary.txt’\n",
      "Resolving ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)... 130.14.250.13, 130.14.250.12, 2607:f220:41e:250::10, ...\n",
      "Connecting to ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)|130.14.250.13|:21... connected.\n",
      "Logging in as anonymous ... Logged in!\n",
      "==> SYST ... done.    ==> PWD ... done.\n",
      "==> TYPE I ... done.  ==> CWD (1) /genomes/genbank/bacteria ... done.\n",
      "==> SIZE assembly_summary.txt ... 276853487\n",
      "==> PASV ... done.    ==> RETR assembly_summary.txt ... done.\n",
      "Length: 276853487 (264M) (unauthoritative)\n",
      "\n",
      "assembly_summary.tx 100%[===================>] 264.03M  23.2MB/s    in 12s     \n",
      "\n",
      "2021-03-14 00:32:19 (21.4 MB/s) - ‘assembly_summary.txt’ saved [276853487]\n",
      "\n"
     ]
    }
   ],
   "source": [
    "!wget ftp://ftp.ncbi.nlm.nih.gov/genomes/genbank/bacteria/assembly_summary.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/pathinformatics/miniconda3/envs/vamb_env/lib/python3.8/site-packages/IPython/core/interactiveshell.py:3146: DtypeWarning: Columns (21) have mixed types.Specify dtype option on import or set low_memory=False.\n",
      "  has_raised = await self.run_ast_nodes(code_ast.body, cell_name,\n"
     ]
    }
   ],
   "source": [
    "all_ncbi_genomes = pd.read_csv('assembly_summary.txt', sep='\\t', skiprows=1)\n",
    "all_complete_genomes = all_ncbi_genomes\n",
    "#all_complete_genomes = all_ncbi_genomes[all_ncbi_genomes['assembly_level']=='Complete Genome']\n",
    "all_complete_genomes = all_complete_genomes[(~all_complete_genomes['organism_name'].str.contains('sp.')) & \n",
    "                                            (~all_complete_genomes['organism_name'].str.contains('endosymbiont')) & \n",
    "                                            (~all_complete_genomes['organism_name'].str.contains('candidate'))]\n",
    "all_complete_genomes = all_complete_genomes[pd.notna(all_complete_genomes['excluded_from_refseq'])]\n",
    "\n",
    "all_complete_genomes = all_complete_genomes[['# assembly_accession','wgs_master','taxid','organism_name',\n",
    "                                             'infraspecific_name','ftp_path']].drop_duplicates()\n",
    "\n",
    "all_complete_genomes = all_complete_genomes.groupby('organism_name').first().reset_index()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>organism_name</th>\n",
       "      <th># assembly_accession</th>\n",
       "      <th>wgs_master</th>\n",
       "      <th>taxid</th>\n",
       "      <th>infraspecific_name</th>\n",
       "      <th>ftp_path</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>'Brassica napus' phytoplasma</td>\n",
       "      <td>GCA_003181115.1</td>\n",
       "      <td>QGKT00000000.1</td>\n",
       "      <td>469009</td>\n",
       "      <td>None</td>\n",
       "      <td>ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/003...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>'Candidatus Kapabacteria' thiocyanatum</td>\n",
       "      <td>GCA_001899175.1</td>\n",
       "      <td>MKVH00000000.1</td>\n",
       "      <td>1895771</td>\n",
       "      <td>None</td>\n",
       "      <td>ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/001...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                            organism_name # assembly_accession  \\\n",
       "0            'Brassica napus' phytoplasma      GCA_003181115.1   \n",
       "1  'Candidatus Kapabacteria' thiocyanatum      GCA_001899175.1   \n",
       "\n",
       "       wgs_master    taxid infraspecific_name  \\\n",
       "0  QGKT00000000.1   469009               None   \n",
       "1  MKVH00000000.1  1895771               None   \n",
       "\n",
       "                                            ftp_path  \n",
       "0  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/003...  \n",
       "1  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/001...  "
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "all_complete_genomes.head(2)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Having this, I've also obtained a reference phylogenetic tree which we can use to group genome by differing levels of phylogenetic similarity. The particular tree I'll be using is one we obtained from the [Web Of Life (WOL)](https://biocore.github.io/wol/start) project. I read this in with dendropy and then convert this into a phylogenetic distance matrix `pdm`. This is calculated using the patristic distances between items in the tree and, thus, necessitates a pairwise calculation which can take quite a bit. Moreover, this creates a pretty big pickle file (several GB)."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "# with open('astral.cons.nid.e5p50.nwk', 'r') as treefile:\n",
    "#     tree = dendropy.Tree.get(\n",
    "#             data=treefile.read(),\n",
    "#             schema=\"newick\",\n",
    "#             )\n",
    "    \n",
    "#     pdm = tree.phylogenetic_distance_matrix()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "#with open('bacteria_pdm.pickle', 'wb') as f:\n",
    "#    pickle.dump(pdm, f, pickle.HIGHEST_PROTOCOL)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "I will then use this distance matrix to grab a collection of genomes relevant to each of our experiments. This will be a collection of genomes from 1 genus, from 5 genera, and from 10 genera. I'll first select which genera these are and then I'll create three specimens for each genera set: one with 25 genomes, one with 50 genomes, and one with 250 genomes genomes.\n",
    "\n",
    "So we start with our genera list which represent both common bacterial genera and things that are not totally unique giving us the ability to demonstrate similar vs un-similar genomes."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "genus_set_1 = 'Escherichia'\n",
    "genus_set_2, genus_set_3, genus_set_4, genus_set_5 = 'Vibrio', 'Bacteroides', 'Clostridium', 'Listeria'\n",
    "genus_set_6, genus_set_7, genus_set_8, genus_set_9, genus_set_10 = 'Lactobacillus', 'Staphylococcus', 'Shigella', 'Pseudomonas', 'Bifidobacterium'"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "So finally, I can use these genera labels to create the actual genome lists to use for sample simulation."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Escherichia 2.5 2\n",
      "--2021-03-14 01:23:23--  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000/397/385/GCA_000397385.1_Ec13_E2F4_TOP293-4_120714/GCA_000397385.1_Ec13_E2F4_TOP293-4_120714_genomic.fna.gz\n",
      "           => ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_000397385.1_Ec13_E2F4_TOP293-4_120714_genomic.fna.gz’\n",
      "Resolving ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)... 130.14.250.12, 130.14.250.13, 2607:f220:41e:250::10, ...\n",
      "Connecting to ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)|130.14.250.12|:21... connected.\n",
      "Logging in as anonymous ... Logged in!\n",
      "==> SYST ... done.    ==> PWD ... done.\n",
      "==> TYPE I ... done.  ==> CWD (1) /genomes/all/GCA/000/397/385/GCA_000397385.1_Ec13_E2F4_TOP293-4_120714 ... done.\n",
      "==> SIZE GCA_000397385.1_Ec13_E2F4_TOP293-4_120714_genomic.fna.gz ... 1626284\n",
      "==> PASV ... done.    ==> RETR GCA_000397385.1_Ec13_E2F4_TOP293-4_120714_genomic.fna.gz ... done.\n",
      "Length: 1626284 (1.6M) (unauthoritative)\n",
      "\n",
      "GCA_000397385.1_Ec1 100%[===================>]   1.55M  3.33MB/s    in 0.5s    \n",
      "\n",
      "2021-03-14 01:23:24 (3.33 MB/s) - ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_000397385.1_Ec13_E2F4_TOP293-4_120714_genomic.fna.gz’ saved [1626284]\n",
      "\n",
      "--2021-03-14 01:23:24--  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/011/855/365/GCA_011855365.1_PDT000259510.3/GCA_011855365.1_PDT000259510.3_genomic.fna.gz\n",
      "           => ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_011855365.1_PDT000259510.3_genomic.fna.gz’\n",
      "Resolving ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)... 130.14.250.12, 130.14.250.13, 2607:f220:41e:250::10, ...\n",
      "Connecting to ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)|130.14.250.12|:21... connected.\n",
      "Logging in as anonymous ... Logged in!\n",
      "==> SYST ... done.    ==> PWD ... done.\n",
      "==> TYPE I ... done.  ==> CWD (1) /genomes/all/GCA/011/855/365/GCA_011855365.1_PDT000259510.3 ... done.\n",
      "==> SIZE GCA_011855365.1_PDT000259510.3_genomic.fna.gz ... 1574940\n",
      "==> PASV ... done.    ==> RETR GCA_011855365.1_PDT000259510.3_genomic.fna.gz ... done.\n",
      "Length: 1574940 (1.5M) (unauthoritative)\n",
      "\n",
      "GCA_011855365.1_PDT 100%[===================>]   1.50M  2.71MB/s    in 0.6s    \n",
      "\n",
      "2021-03-14 01:23:25 (2.71 MB/s) - ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_011855365.1_PDT000259510.3_genomic.fna.gz’ saved [1574940]\n",
      "\n",
      "Vibrio 2.5 2\n",
      "--2021-03-14 01:23:26--  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/001/597/655/GCA_001597655.1_ASM159765v1/GCA_001597655.1_ASM159765v1_genomic.fna.gz\n",
      "           => ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_001597655.1_ASM159765v1_genomic.fna.gz’\n",
      "Resolving ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)... 130.14.250.12, 130.14.250.13, 2607:f220:41e:250::10, ...\n",
      "Connecting to ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)|130.14.250.12|:21... connected.\n",
      "Logging in as anonymous ... Logged in!\n",
      "==> SYST ... done.    ==> PWD ... done.\n",
      "==> TYPE I ... done.  ==> CWD (1) /genomes/all/GCA/001/597/655/GCA_001597655.1_ASM159765v1 ... done.\n",
      "==> SIZE GCA_001597655.1_ASM159765v1_genomic.fna.gz ... 1399787\n",
      "==> PASV ... done.    ==> RETR GCA_001597655.1_ASM159765v1_genomic.fna.gz ... done.\n",
      "Length: 1399787 (1.3M) (unauthoritative)\n",
      "\n",
      "GCA_001597655.1_ASM 100%[===================>]   1.33M  3.51MB/s    in 0.4s    \n",
      "\n",
      "2021-03-14 01:23:27 (3.51 MB/s) - ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_001597655.1_ASM159765v1_genomic.fna.gz’ saved [1399787]\n",
      "\n",
      "--2021-03-14 01:23:27--  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000/269/765/GCA_000269765.1_ASM26976v1/GCA_000269765.1_ASM26976v1_genomic.fna.gz\n",
      "           => ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_000269765.1_ASM26976v1_genomic.fna.gz’\n",
      "Resolving ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)... 130.14.250.12, 130.14.250.13, 2607:f220:41e:250::10, ...\n",
      "Connecting to ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)|130.14.250.12|:21... connected.\n",
      "Logging in as anonymous ... Logged in!\n",
      "==> SYST ... done.    ==> PWD ... done.\n",
      "==> TYPE I ... done.  ==> CWD (1) /genomes/all/GCA/000/269/765/GCA_000269765.1_ASM26976v1 ... done.\n",
      "==> SIZE GCA_000269765.1_ASM26976v1_genomic.fna.gz ... 1481534\n",
      "==> PASV ... done.    ==> RETR GCA_000269765.1_ASM26976v1_genomic.fna.gz ... done.\n",
      "Length: 1481534 (1.4M) (unauthoritative)\n",
      "\n",
      "GCA_000269765.1_ASM 100%[===================>]   1.41M  3.66MB/s    in 0.4s    \n",
      "\n",
      "2021-03-14 01:23:28 (3.66 MB/s) - ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_000269765.1_ASM26976v1_genomic.fna.gz’ saved [1481534]\n",
      "\n",
      "Bacteroides 2.5 2\n",
      "--2021-03-14 01:23:28--  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000/438/035/GCA_000438035.1_MGS6/GCA_000438035.1_MGS6_genomic.fna.gz\n",
      "           => ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_000438035.1_MGS6_genomic.fna.gz’\n",
      "Resolving ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)... 130.14.250.12, 130.14.250.13, 2607:f220:41e:250::10, ...\n",
      "Connecting to ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)|130.14.250.12|:21... connected.\n",
      "Logging in as anonymous ... Logged in!\n",
      "==> SYST ... done.    ==> PWD ... done.\n",
      "==> TYPE I ... done.  ==> CWD (1) /genomes/all/GCA/000/438/035/GCA_000438035.1_MGS6 ... done.\n",
      "==> SIZE GCA_000438035.1_MGS6_genomic.fna.gz ... 1151242\n",
      "==> PASV ... done.    ==> RETR GCA_000438035.1_MGS6_genomic.fna.gz ... done.\n",
      "Length: 1151242 (1.1M) (unauthoritative)\n",
      "\n",
      "GCA_000438035.1_MGS 100%[===================>]   1.10M  2.56MB/s    in 0.4s    \n",
      "\n",
      "2021-03-14 01:23:29 (2.56 MB/s) - ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_000438035.1_MGS6_genomic.fna.gz’ saved [1151242]\n",
      "\n",
      "--2021-03-14 01:23:29--  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000/433/375/GCA_000433375.1_MGS32/GCA_000433375.1_MGS32_genomic.fna.gz\n",
      "           => ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_000433375.1_MGS32_genomic.fna.gz’\n",
      "Resolving ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)... 130.14.250.12, 130.14.250.13, 2607:f220:41e:250::10, ...\n",
      "Connecting to ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)|130.14.250.12|:21... connected.\n",
      "Logging in as anonymous ... Logged in!\n",
      "==> SYST ... done.    ==> PWD ... done.\n",
      "==> TYPE I ... done.  ==> CWD (1) /genomes/all/GCA/000/433/375/GCA_000433375.1_MGS32 ... done.\n",
      "==> SIZE GCA_000433375.1_MGS32_genomic.fna.gz ... 1430411\n",
      "==> PASV ... done.    ==> RETR GCA_000433375.1_MGS32_genomic.fna.gz ... done.\n",
      "Length: 1430411 (1.4M) (unauthoritative)\n",
      "\n",
      "GCA_000433375.1_MGS 100%[===================>]   1.36M  3.23MB/s    in 0.4s    \n",
      "\n",
      "2021-03-14 01:23:30 (3.23 MB/s) - ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_000433375.1_MGS32_genomic.fna.gz’ saved [1430411]\n",
      "\n",
      "Clostridium 2.5 2\n",
      "--2021-03-14 01:23:30--  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000/436/455/GCA_000436455.1_MGS132/GCA_000436455.1_MGS132_genomic.fna.gz\n",
      "           => ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_000436455.1_MGS132_genomic.fna.gz’\n",
      "Resolving ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)... 130.14.250.12, 130.14.250.13, 2607:f220:41e:250::10, ...\n",
      "Connecting to ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)|130.14.250.12|:21... connected.\n",
      "Logging in as anonymous ... Logged in!\n",
      "==> SYST ... done.    ==> PWD ... done.\n",
      "==> TYPE I ... done.  ==> CWD (1) /genomes/all/GCA/000/436/455/GCA_000436455.1_MGS132 ... done.\n",
      "==> SIZE GCA_000436455.1_MGS132_genomic.fna.gz ... 1342123\n",
      "==> PASV ... done.    ==> RETR GCA_000436455.1_MGS132_genomic.fna.gz ... done.\n",
      "Length: 1342123 (1.3M) (unauthoritative)\n",
      "\n",
      "GCA_000436455.1_MGS 100%[===================>]   1.28M  3.01MB/s    in 0.4s    \n",
      "\n",
      "2021-03-14 01:23:31 (3.01 MB/s) - ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_000436455.1_MGS132_genomic.fna.gz’ saved [1342123]\n",
      "\n",
      "--2021-03-14 01:23:31--  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/016/902/235/GCA_016902235.1_ASM1690223v1/GCA_016902235.1_ASM1690223v1_genomic.fna.gz\n",
      "           => ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_016902235.1_ASM1690223v1_genomic.fna.gz’\n",
      "Resolving ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)... 130.14.250.12, 130.14.250.13, 2607:f220:41e:250::10, ...\n",
      "Connecting to ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)|130.14.250.12|:21... connected.\n",
      "Logging in as anonymous ... Logged in!\n",
      "==> SYST ... done.    ==> PWD ... done.\n",
      "==> TYPE I ... done.  ==> CWD (1) /genomes/all/GCA/016/902/235/GCA_016902235.1_ASM1690223v1 ... done.\n",
      "==> SIZE GCA_016902235.1_ASM1690223v1_genomic.fna.gz ... 678653\n",
      "==> PASV ... done.    ==> RETR GCA_016902235.1_ASM1690223v1_genomic.fna.gz ... done.\n",
      "Length: 678653 (663K) (unauthoritative)\n",
      "\n",
      "GCA_016902235.1_ASM 100%[===================>] 662.75K  2.04MB/s    in 0.3s    \n",
      "\n",
      "2021-03-14 01:23:32 (2.04 MB/s) - ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_016902235.1_ASM1690223v1_genomic.fna.gz’ saved [678653]\n",
      "\n",
      "Listeria 2.5 2\n",
      "--2021-03-14 01:23:33--  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/004/710/765/GCA_004710765.1_PDT000030589.4/GCA_004710765.1_PDT000030589.4_genomic.fna.gz\n",
      "           => ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_004710765.1_PDT000030589.4_genomic.fna.gz’\n",
      "Resolving ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)... 130.14.250.12, 130.14.250.13, 2607:f220:41e:250::10, ...\n",
      "Connecting to ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)|130.14.250.12|:21... connected.\n",
      "Logging in as anonymous ... Logged in!\n",
      "==> SYST ... done.    ==> PWD ... done.\n",
      "==> TYPE I ... done.  ==> CWD (1) /genomes/all/GCA/004/710/765/GCA_004710765.1_PDT000030589.4 ... done.\n",
      "==> SIZE GCA_004710765.1_PDT000030589.4_genomic.fna.gz ... 858215\n",
      "==> PASV ... done.    ==> RETR GCA_004710765.1_PDT000030589.4_genomic.fna.gz ... done.\n",
      "Length: 858215 (838K) (unauthoritative)\n",
      "\n",
      "GCA_004710765.1_PDT 100%[===================>] 838.10K  2.49MB/s    in 0.3s    \n",
      "\n",
      "2021-03-14 01:23:33 (2.49 MB/s) - ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_004710765.1_PDT000030589.4_genomic.fna.gz’ saved [858215]\n",
      "\n",
      "--2021-03-14 01:23:34--  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000/567/965/GCA_000567965.1_ASM56796v1/GCA_000567965.1_ASM56796v1_genomic.fna.gz\n",
      "           => ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_000567965.1_ASM56796v1_genomic.fna.gz’\n",
      "Resolving ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)... 130.14.250.12, 130.14.250.13, 2607:f220:41e:250::10, ...\n",
      "Connecting to ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)|130.14.250.12|:21... connected.\n",
      "Logging in as anonymous ... Logged in!\n",
      "==> SYST ... done.    ==> PWD ... done.\n",
      "==> TYPE I ... done.  ==> CWD (1) /genomes/all/GCA/000/567/965/GCA_000567965.1_ASM56796v1 ... done.\n",
      "==> SIZE GCA_000567965.1_ASM56796v1_genomic.fna.gz ... 879223\n",
      "==> PASV ... done.    ==> RETR GCA_000567965.1_ASM56796v1_genomic.fna.gz ... done.\n",
      "Length: 879223 (859K) (unauthoritative)\n",
      "\n",
      "GCA_000567965.1_ASM 100%[===================>] 858.62K  2.39MB/s    in 0.4s    \n",
      "\n",
      "2021-03-14 01:23:35 (2.39 MB/s) - ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_000567965.1_ASM56796v1_genomic.fna.gz’ saved [879223]\n",
      "\n",
      "Lactobacillus 2.5 2\n",
      "--2021-03-14 01:23:35--  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/001/311/525/GCA_001311525.1_ASM131152v1/GCA_001311525.1_ASM131152v1_genomic.fna.gz\n",
      "           => ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_001311525.1_ASM131152v1_genomic.fna.gz’\n",
      "Resolving ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)... 130.14.250.12, 130.14.250.13, 2607:f220:41e:250::10, ...\n",
      "Connecting to ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)|130.14.250.12|:21... connected.\n",
      "Logging in as anonymous ... Logged in!\n",
      "==> SYST ... done.    ==> PWD ... done.\n",
      "==> TYPE I ... done.  ==> CWD (1) /genomes/all/GCA/001/311/525/GCA_001311525.1_ASM131152v1 ... done.\n",
      "==> SIZE GCA_001311525.1_ASM131152v1_genomic.fna.gz ... 554526\n",
      "==> PASV ... done.    ==> RETR GCA_001311525.1_ASM131152v1_genomic.fna.gz ... done.\n",
      "Length: 554526 (542K) (unauthoritative)\n",
      "\n",
      "GCA_001311525.1_ASM 100%[===================>] 541.53K  1.68MB/s    in 0.3s    \n",
      "\n",
      "2021-03-14 01:23:36 (1.68 MB/s) - ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_001311525.1_ASM131152v1_genomic.fna.gz’ saved [554526]\n",
      "\n",
      "--2021-03-14 01:23:36--  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000/165/775/GCA_000165775.3_ASM16577v3/GCA_000165775.3_ASM16577v3_genomic.fna.gz\n",
      "           => ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_000165775.3_ASM16577v3_genomic.fna.gz’\n",
      "Resolving ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)... 130.14.250.12, 130.14.250.13, 2607:f220:41e:250::10, ...\n",
      "Connecting to ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)|130.14.250.12|:21... connected.\n",
      "Logging in as anonymous ... Logged in!\n",
      "==> SYST ... done.    ==> PWD ... done.\n",
      "==> TYPE I ... done.  ==> CWD (1) /genomes/all/GCA/000/165/775/GCA_000165775.3_ASM16577v3 ... done.\n",
      "==> SIZE GCA_000165775.3_ASM16577v3_genomic.fna.gz ... 623427\n",
      "==> PASV ... done.    ==> RETR GCA_000165775.3_ASM16577v3_genomic.fna.gz ... done.\n",
      "Length: 623427 (609K) (unauthoritative)\n",
      "\n",
      "GCA_000165775.3_ASM 100%[===================>] 608.82K  1.82MB/s    in 0.3s    \n",
      "\n",
      "2021-03-14 01:23:37 (1.82 MB/s) - ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_000165775.3_ASM16577v3_genomic.fna.gz’ saved [623427]\n",
      "\n",
      "Staphylococcus 2.5 2\n",
      "--2021-03-14 01:23:37--  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/002/902/575/GCA_002902575.1_ASM290257v1/GCA_002902575.1_ASM290257v1_genomic.fna.gz\n",
      "           => ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_002902575.1_ASM290257v1_genomic.fna.gz’\n",
      "Resolving ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)... 130.14.250.12, 130.14.250.13, 2607:f220:41e:250::10, ...\n",
      "Connecting to ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)|130.14.250.12|:21... connected.\n",
      "Logging in as anonymous ... Logged in!\n",
      "==> SYST ... done.    ==> PWD ... done.\n",
      "==> TYPE I ... done.  ==> CWD (1) /genomes/all/GCA/002/902/575/GCA_002902575.1_ASM290257v1 ... done.\n",
      "==> SIZE GCA_002902575.1_ASM290257v1_genomic.fna.gz ... 704198\n",
      "==> PASV ... done.    ==> RETR GCA_002902575.1_ASM290257v1_genomic.fna.gz ... done.\n",
      "Length: 704198 (688K) (unauthoritative)\n",
      "\n",
      "GCA_002902575.1_ASM 100%[===================>] 687.69K  1.93MB/s    in 0.3s    \n",
      "\n",
      "2021-03-14 01:23:38 (1.93 MB/s) - ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_002902575.1_ASM290257v1_genomic.fna.gz’ saved [704198]\n",
      "\n",
      "--2021-03-14 01:23:38--  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000/555/645/GCA_000555645.1_Stap_aure_M0406_V1/GCA_000555645.1_Stap_aure_M0406_V1_genomic.fna.gz\n",
      "           => ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_000555645.1_Stap_aure_M0406_V1_genomic.fna.gz’\n",
      "Resolving ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)... 130.14.250.12, 130.14.250.13, 2607:f220:41e:250::10, ...\n",
      "Connecting to ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)|130.14.250.12|:21... connected.\n",
      "Logging in as anonymous ... Logged in!\n",
      "==> SYST ... done.    ==> PWD ... done.\n",
      "==> TYPE I ... done.  ==> CWD (1) /genomes/all/GCA/000/555/645/GCA_000555645.1_Stap_aure_M0406_V1 ... done.\n",
      "==> SIZE GCA_000555645.1_Stap_aure_M0406_V1_genomic.fna.gz ... 919251\n",
      "==> PASV ... done.    ==> RETR GCA_000555645.1_Stap_aure_M0406_V1_genomic.fna.gz ... done.\n",
      "Length: 919251 (898K) (unauthoritative)\n",
      "\n",
      "GCA_000555645.1_Sta 100%[===================>] 897.71K  2.20MB/s    in 0.4s    \n",
      "\n",
      "2021-03-14 01:23:39 (2.20 MB/s) - ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_000555645.1_Stap_aure_M0406_V1_genomic.fna.gz’ saved [919251]\n",
      "\n",
      "Shigella 2.5 2\n",
      "--2021-03-14 01:23:39--  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000/268/085/GCA_000268085.1_ASM26808v1/GCA_000268085.1_ASM26808v1_genomic.fna.gz\n",
      "           => ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_000268085.1_ASM26808v1_genomic.fna.gz’\n",
      "Resolving ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)... 130.14.250.12, 130.14.250.13, 2607:f220:41e:250::10, ...\n",
      "Connecting to ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)|130.14.250.12|:21... connected.\n",
      "Logging in as anonymous ... Logged in!\n",
      "==> SYST ... done.    ==> PWD ... done.\n",
      "==> TYPE I ... done.  ==> CWD (1) /genomes/all/GCA/000/268/085/GCA_000268085.1_ASM26808v1 ... done.\n",
      "==> SIZE GCA_000268085.1_ASM26808v1_genomic.fna.gz ... 1415441\n",
      "==> PASV ... done.    ==> RETR GCA_000268085.1_ASM26808v1_genomic.fna.gz ... done.\n",
      "Length: 1415441 (1.3M) (unauthoritative)\n",
      "\n",
      "GCA_000268085.1_ASM 100%[===================>]   1.35M  3.04MB/s    in 0.4s    \n",
      "\n",
      "2021-03-14 01:23:40 (3.04 MB/s) - ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_000268085.1_ASM26808v1_genomic.fna.gz’ saved [1415441]\n",
      "\n",
      "--2021-03-14 01:23:41--  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000/268/165/GCA_000268165.1_ASM26816v1/GCA_000268165.1_ASM26816v1_genomic.fna.gz\n",
      "           => ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_000268165.1_ASM26816v1_genomic.fna.gz’\n",
      "Resolving ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)... 130.14.250.12, 130.14.250.13, 2607:f220:41e:250::10, ...\n",
      "Connecting to ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)|130.14.250.12|:21... connected.\n",
      "Logging in as anonymous ... Logged in!\n",
      "==> SYST ... done.    ==> PWD ... done.\n",
      "==> TYPE I ... done.  ==> CWD (1) /genomes/all/GCA/000/268/165/GCA_000268165.1_ASM26816v1 ... done.\n",
      "==> SIZE GCA_000268165.1_ASM26816v1_genomic.fna.gz ... 1325670\n",
      "==> PASV ... done.    ==> RETR GCA_000268165.1_ASM26816v1_genomic.fna.gz ... done.\n",
      "Length: 1325670 (1.3M) (unauthoritative)\n",
      "\n",
      "GCA_000268165.1_ASM 100%[===================>]   1.26M  2.67MB/s    in 0.5s    \n",
      "\n",
      "2021-03-14 01:23:42 (2.67 MB/s) - ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_000268165.1_ASM26816v1_genomic.fna.gz’ saved [1325670]\n",
      "\n",
      "Pseudomonas 2.5 2\n",
      "--2021-03-14 01:23:42--  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000/953/455/GCA_000953455.1_Ppseudo_Pac/GCA_000953455.1_Ppseudo_Pac_genomic.fna.gz\n",
      "           => ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_000953455.1_Ppseudo_Pac_genomic.fna.gz’\n",
      "Resolving ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)... 130.14.250.12, 130.14.250.13, 2607:f220:41e:250::10, ...\n",
      "Connecting to ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)|130.14.250.12|:21... connected.\n",
      "Logging in as anonymous ... Logged in!\n",
      "==> SYST ... done.    ==> PWD ... done.\n",
      "==> TYPE I ... done.  ==> CWD (1) /genomes/all/GCA/000/953/455/GCA_000953455.1_Ppseudo_Pac ... done.\n",
      "==> SIZE GCA_000953455.1_Ppseudo_Pac_genomic.fna.gz ... 1366143\n",
      "==> PASV ... done.    ==> RETR GCA_000953455.1_Ppseudo_Pac_genomic.fna.gz ... done.\n",
      "Length: 1366143 (1.3M) (unauthoritative)\n",
      "\n",
      "GCA_000953455.1_Pps 100%[===================>]   1.30M  2.99MB/s    in 0.4s    \n",
      "\n",
      "2021-03-14 01:23:43 (2.99 MB/s) - ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_000953455.1_Ppseudo_Pac_genomic.fna.gz’ saved [1366143]\n",
      "\n",
      "--2021-03-14 01:23:43--  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/009/911/885/GCA_009911885.1_ASM991188v1/GCA_009911885.1_ASM991188v1_genomic.fna.gz\n",
      "           => ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_009911885.1_ASM991188v1_genomic.fna.gz’\n",
      "Resolving ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)... 130.14.250.12, 130.14.250.13, 2607:f220:41e:250::10, ...\n",
      "Connecting to ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)|130.14.250.12|:21... connected.\n",
      "Logging in as anonymous ... Logged in!\n",
      "==> SYST ... done.    ==> PWD ... done.\n",
      "==> TYPE I ... done.  ==> CWD (1) /genomes/all/GCA/009/911/885/GCA_009911885.1_ASM991188v1 ... done.\n",
      "==> SIZE GCA_009911885.1_ASM991188v1_genomic.fna.gz ... 104893\n",
      "==> PASV ... done.    ==> RETR GCA_009911885.1_ASM991188v1_genomic.fna.gz ... done.\n",
      "Length: 104893 (102K) (unauthoritative)\n",
      "\n",
      "GCA_009911885.1_ASM 100%[===================>] 102.43K  --.-KB/s    in 0.1s    \n",
      "\n",
      "2021-03-14 01:23:44 (742 KB/s) - ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_009911885.1_ASM991188v1_genomic.fna.gz’ saved [104893]\n",
      "\n",
      "Bifidobacterium 2.5 2\n",
      "--2021-03-14 01:23:44--  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/002/311/215/GCA_002311215.1_ASM231121v1/GCA_002311215.1_ASM231121v1_genomic.fna.gz\n",
      "           => ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_002311215.1_ASM231121v1_genomic.fna.gz’\n",
      "Resolving ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)... 130.14.250.12, 130.14.250.13, 2607:f220:41e:250::10, ...\n",
      "Connecting to ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)|130.14.250.12|:21... connected.\n",
      "Logging in as anonymous ... Logged in!\n",
      "==> SYST ... done.    ==> PWD ... done.\n",
      "==> TYPE I ... done.  ==> CWD (1) /genomes/all/GCA/002/311/215/GCA_002311215.1_ASM231121v1 ... done.\n",
      "==> SIZE GCA_002311215.1_ASM231121v1_genomic.fna.gz ... 523383\n",
      "==> PASV ... done.    ==> RETR GCA_002311215.1_ASM231121v1_genomic.fna.gz ... done.\n",
      "Length: 523383 (511K) (unauthoritative)\n",
      "\n",
      "GCA_002311215.1_ASM 100%[===================>] 511.12K  1.57MB/s    in 0.3s    \n",
      "\n",
      "2021-03-14 01:23:45 (1.57 MB/s) - ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_002311215.1_ASM231121v1_genomic.fna.gz’ saved [523383]\n",
      "\n",
      "--2021-03-14 01:23:45--  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000/435/235/GCA_000435235.1_MGS119/GCA_000435235.1_MGS119_genomic.fna.gz\n",
      "           => ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_000435235.1_MGS119_genomic.fna.gz’\n",
      "Resolving ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)... 130.14.250.12, 130.14.250.13, 2607:f220:41e:250::10, ...\n",
      "Connecting to ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)|130.14.250.12|:21... connected.\n",
      "Logging in as anonymous ... Logged in!\n",
      "==> SYST ... done.    ==> PWD ... done.\n",
      "==> TYPE I ... done.  ==> CWD (1) /genomes/all/GCA/000/435/235/GCA_000435235.1_MGS119 ... done.\n",
      "==> SIZE GCA_000435235.1_MGS119_genomic.fna.gz ... 570236\n",
      "==> PASV ... done.    ==> RETR GCA_000435235.1_MGS119_genomic.fna.gz ... done.\n",
      "Length: 570236 (557K) (unauthoritative)\n",
      "\n",
      "GCA_000435235.1_MGS 100%[===================>] 556.87K  1.74MB/s    in 0.3s    \n",
      "\n",
      "2021-03-14 01:23:46 (1.74 MB/s) - ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_000435235.1_MGS119_genomic.fna.gz’ saved [570236]\n",
      "\n"
     ]
    }
   ],
   "source": [
    "for idx, genus_set in enumerate([[genus_set_1], \n",
    "                  [genus_set_1, genus_set_2, genus_set_3, genus_set_4, genus_set_5], \n",
    "                  [genus_set_1, genus_set_2, genus_set_3, genus_set_4, genus_set_5, genus_set_6, genus_set_7, genus_set_8, genus_set_9, genus_set_10]]):\n",
    "    \n",
    "    for num_genomes in [25, 50, 250]:\n",
    "        if idx == 0:\n",
    "            num_genera = 1\n",
    "        else:\n",
    "            num_genera = idx*5\n",
    "        complexity_sim_folder = os.path.join(os.getcwd(), f\"example_input_data/new_simulations/complexity_sim_{num_genera}_genera_{num_genomes}_genomes\")\n",
    "        if not os.path.exists(complexity_sim_folder):\n",
    "            os.mkdir(complexity_sim_folder)\n",
    "            os.mkdir(os.path.join(complexity_sim_folder, 'genomes'))\n",
    "\n",
    "            for genus in genus_set:\n",
    "                genus_subset = all_complete_genomes[all_complete_genomes['organism_name'].str.startswith(genus)]\n",
    "\n",
    "                per_genus_sample_size = num_genomes/len(genus_set)\n",
    "                genomes = genus_subset.sample(int(per_genus_sample_size))\n",
    "                print(genus, per_genus_sample_size, genomes['ftp_path'].nunique())\n",
    "\n",
    "                for ftp_path in genomes['ftp_path'].values:\n",
    "                    genome_path = ftp_path + f\"/{os.path.basename(ftp_path)}_genomic.fna.gz\"\n",
    "                    genome_folder = os.path.join(complexity_sim_folder, 'genomes')\n",
    "                    !wget $genome_path -P $genome_folder"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "I'll also just verify that we got the right number of genomes downloaded and the right number of genera represented in each of these experiments."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1_genera_25_genomes\n",
      "25\n",
      "1_genera_50_genomes\n",
      "50\n",
      "1_genera_250_genomes\n",
      "250\n",
      "5_genera_25_genomes\n",
      "25\n",
      "5_genera_50_genomes\n",
      "50\n",
      "5_genera_250_genomes\n",
      "250\n",
      "10_genera_25_genomes\n",
      "25\n",
      "10_genera_50_genomes\n",
      "50\n",
      "10_genera_250_genomes\n",
      "250\n"
     ]
    }
   ],
   "source": [
    "for idx, genus_set in enumerate([[genus_set_1], \n",
    "                  [genus_set_1, genus_set_2, genus_set_3, genus_set_4, genus_set_5], \n",
    "                  [genus_set_1, genus_set_2, genus_set_3, genus_set_4, genus_set_5, genus_set_6, genus_set_7, genus_set_8, genus_set_9, genus_set_10]]):\n",
    "    \n",
    "    for num_genomes in [25, 50, 250]:\n",
    "        param_thing = f\"{len(genus_set)}_genera_{num_genomes}_genomes\"\n",
    "        print(param_thing)\n",
    "\n",
    "        print(len(glob.glob(f\"example_input_data/new_simulations/complexity_sim_{param_thing}/genomes/*\")) )\n",
    "\n",
    "        targets = ['_'.join(i.split('/')[-1].strip('.fna.gz').split('_')[:2]) for \n",
    "                   i in glob.glob(f\"example_input_data/new_simulations/complexity_sim_{param_thing}/genomes/*\")]\n",
    "        \n",
    "        assert len(targets) == num_genomes\n",
    "\n",
    "        all_complete_genomes[all_complete_genomes['# assembly_accession'].isin(targets)]['organism_name'].str.contains(genus_set_1).all()\n",
    "        \n",
    "        assert len(set(genus_set).intersection(\n",
    "            set([i.split(' ')[0] for \n",
    "                 i in all_complete_genomes[all_complete_genomes['# assembly_accession'].isin(targets)]['organism_name'].values]))) == len(genus_set)    "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "We also have to set up the genomes and metadata tsv files for simulation"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [],
   "source": [
    "for complexity_experiment in glob.glob('example_input_data/new_simulations/complexity*'):\n",
    "    experiment_name = os.path.basename(complexity_experiment)\n",
    "    \n",
    "    for fna_file in glob.glob(f\"example_input_data/new_simulations/{experiment_name}/genomes/*.gz\"):\n",
    "        !gzip -d $fna_file\n",
    "    \n",
    "    genomes = glob.glob(os.path.join(complexity_experiment,'genomes','*'))\n",
    "    genomes_mod = [os.path.join('/input/genomes',os.path.basename(i)) for i in genomes]\n",
    "    \n",
    "    df = pd.DataFrame(genomes_mod)\n",
    "    df['genome_ID'] = [ f\"Genome{float(i+1)}\" for i in range(len(df))]\n",
    "    df = df[['genome_ID',0]]\n",
    "    df.to_csv(os.path.join(complexity_experiment,'genome_to_id.tsv'), sep='\\t', header=False, index=False)\n",
    "    \n",
    "    metadata_df = df\n",
    "    metadata_df['novelty_category'] = 'known_strain'\n",
    "    metadata_df['NCBI_ID'] = metadata_df[0].str.split('/', expand=True)[3].str.split('_', expand=True)[[0,1]].agg('_'.join, axis=1)\n",
    "    \n",
    "    metadata_df = metadata_df.merge(all_complete_genomes, left_on='NCBI_ID', \n",
    "                      right_on='# assembly_accession', \n",
    "                      how='inner')\n",
    "    metadata_df['NCBI_ID'] = metadata_df['ftp_path'].str.split('/', expand=True)[9].str.strip('.fna.gz')\n",
    "    \n",
    "    metadata_df = metadata_df[['genome_ID','NCBI_ID','taxid','novelty_category']].rename(columns={'NCBI_ID':'OTU','taxid':'NCBI_ID',})\n",
    "    metadata_df['NCBI_ID'] = 2\n",
    "    \n",
    "    metadata_df.to_csv(os.path.join(complexity_experiment,'metadata.tsv'), sep='\\t', index=False)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "And now I can run this to create the simulated samples"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {
    "collapsed": true,
    "jupyter": {
     "outputs_hidden": true
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "simulating complexity_sim_10_genera_250_genomes\n",
      "[sudo] password for pathinformatics: 2021-03-14 07:34:03 INFO: [MetagenomeSimulationPipeline] Metagenome simulation starting\n",
      "2021-03-14 07:34:03 INFO: [MetagenomeSimulationPipeline] Validating Genomes\n",
      "2021-03-14 07:34:03 INFO: [MetadataReader] Reading file: '/input/genome_to_id.tsv'\n",
      "2021-03-14 07:37:36 INFO: [MetagenomeSimulationPipeline] Design Communities\n",
      "2021-03-14 07:37:36 INFO: [CommunityDesign] Drawing strains.\n",
      "2021-03-14 07:37:36 INFO: [MetadataReader 82380310288] Reading file: '/input/metadata.tsv'\n",
      "2021-03-14 07:37:36 INFO: [MetadataReader 28782960663] Reading file: '/input/genome_to_id.tsv'\n",
      "2021-03-14 07:37:36 INFO: [CommunityDesign] Validating raw sequence files!\n",
      "2021-03-14 07:37:55 INFO: [NcbiTaxonomy] Building taxonomy tree...\n",
      "2021-03-14 07:37:55 INFO: [NcbiTaxonomy] Reading 'nodes' file:\t'/tmp/tmp6Yt7pz/NCBI/nodes.dmp'\n",
      "2021-03-14 07:38:06 INFO: [NcbiTaxonomy] Reading 'names' file:\t'/tmp/tmp6Yt7pz/NCBI/names.dmp'\n",
      "2021-03-14 07:38:08 INFO: [NcbiTaxonomy] Reading 'merged' file:\t'/tmp/tmp6Yt7pz/NCBI/merged.dmp'\n",
      "2021-03-14 07:38:08 INFO: [NcbiTaxonomy] Done (13.0s)\n",
      "2021-03-14 07:38:08 INFO: [MetagenomeSimulationPipeline] Move Genomes\n",
      "2021-03-14 07:38:09 INFO: [MetagenomeSimulationPipeline] Read simulation\n",
      "2021-03-14 07:38:09 INFO: [GenomePreparation 82430254076] Using 'mbarc' error profile.\n",
      "2021-03-14 07:38:09 INFO: [GenomePreparation 82430254076] Reading distribution file\n",
      "2021-03-14 07:38:09 INFO: [GenomePreparation 82430254076] Reading genome location file\n",
      "2021-03-14 07:38:12 INFO: [GenomePreparation 82430254076] Genome 'Genome68.0' has sequences below minimum, creating filtered copy.\n",
      "2021-03-14 07:38:16 INFO: [GenomePreparation 82430254076] Genome 'Genome152.0' has sequences below minimum, creating filtered copy.\n",
      "2021-03-14 07:38:27 INFO: [GenomePreparation 82430254076] Genome 'Genome23.0' has sequences below minimum, creating filtered copy.\n",
      "2021-03-14 07:38:28 INFO: [GenomePreparation 82430254076] Simulating reads using ReadSimulationArtIllumina readsimulator...\n",
      "2021-03-14 07:38:28 INFO: [GenomePreparation 82430254076] Simulating reads from Genome201.0: '/output/source_genomes/GCA_001915595.1_ASM191559v1_genomic.fna'\n",
      "2021-03-14 07:38:28 INFO: [GenomePreparation 82430254076] Simulating reads from Genome14.0: '/output/source_genomes/GCA_002741635.1_ASM274163v1_genomic.fna'\n",
      "2021-03-14 07:38:28 INFO: [GenomePreparation 82430254076] Simulating reads from Genome68.0: '/tmp/tmpo1x7ji/tmpHykssw'\n",
      "2021-03-14 07:38:28 INFO: [GenomePreparation 82430254076] Simulating reads from Genome34.0: '/output/source_genomes/GCA_007813835.1_ASM781383v1_genomic.fna'\n",
      "2021-03-14 07:38:28 INFO: [GenomePreparation 82430254076] Simulating reads from Genome25.0: '/output/source_genomes/GCA_000168435.1_ASM16843v1_genomic.fna'\n",
      "2021-03-14 07:38:28 INFO: [GenomePreparation 82430254076] Simulating reads from Genome77.0: '/output/source_genomes/GCA_003573955.1_ASM357395v1_genomic.fna'\n",
      "2021-03-14 07:38:28 INFO: [GenomePreparation 82430254076] Simulating reads from Genome67.0: '/output/source_genomes/GCA_004710765.1_PDT000030589.4_genomic.fna'\n",
      "2021-03-14 07:38:28 INFO: [GenomePreparation 82430254076] Simulating reads from Genome136.0: '/output/source_genomes/GCA_000309585.1_ASM30958v1_genomic.fna'\n",
      "2021-03-14 07:38:28 INFO: [GenomePreparation 82430254076] Simulating reads from Genome152.0: '/tmp/tmpo1x7ji/tmpql5fYZ'\n",
      "2021-03-14 07:38:28 INFO: [GenomePreparation 82430254076] Simulating reads from Genome149.0: '/output/source_genomes/GCA_003433455.1_ASM343345v1_genomic.fna'\n",
      "2021-03-14 07:38:28 INFO: [GenomePreparation 82430254076] Simulating reads from Genome210.0: '/output/source_genomes/GCA_012247505.1_PDT000290375.2_genomic.fna'\n",
      "2021-03-14 07:38:28 INFO: [GenomePreparation 82430254076] Simulating reads from Genome174.0: '/output/source_genomes/GCA_012247165.1_PDT000290409.2_genomic.fna'\n",
      "2021-03-14 07:38:28 INFO: [GenomePreparation 82430254076] Simulating reads from Genome196.0: '/output/source_genomes/GCA_000439655.1_CFSAN002369_1.0_genomic.fna'\n",
      "2021-03-14 07:38:28 INFO: [GenomePreparation 82430254076] Simulating reads from Genome145.0: '/output/source_genomes/GCA_905202165.1_ERR1190858-mag-bin.7_genomic.fna'\n",
      "2021-03-14 07:38:28 INFO: [GenomePreparation 82430254076] Simulating reads from Genome62.0: '/output/source_genomes/GCA_000747465.1_O157_08BKT61141_454_genomic.fna'\n",
      "2021-03-14 07:38:28 INFO: [GenomePreparation 82430254076] Simulating reads from Genome2.0: '/output/source_genomes/GCA_002339425.1_ASM233942v1_genomic.fna'\n",
      "2021-03-14 07:38:28 INFO: [GenomePreparation 82430254076] Simulating reads from Genome221.0: '/output/source_genomes/GCA_004711065.1_PDT000030599.4_genomic.fna'\n",
      "2021-03-14 07:38:28 INFO: [GenomePreparation 82430254076] Simulating reads from Genome70.0: '/output/source_genomes/GCA_000431755.1_MGS69_genomic.fna'\n",
      "2021-03-14 07:38:28 INFO: [GenomePreparation 82430254076] Simulating reads from Genome232.0: '/output/source_genomes/GCA_004728485.1_PDT000353211.2_genomic.fna'\n",
      "2021-03-14 07:38:28 INFO: [GenomePreparation 82430254076] Simulating reads from Genome48.0: '/output/source_genomes/GCA_000092345.1_ASM9234v1_genomic.fna'\n",
      "2021-03-14 07:38:28 INFO: [GenomePreparation 82430254076] Simulating reads from Genome49.0: '/output/source_genomes/GCA_016956135.1_PDT000971140.1_genomic.fna'\n",
      "2021-03-14 07:38:28 INFO: [GenomePreparation 82430254076] Simulating reads from Genome5.0: '/output/source_genomes/GCA_000237745.2_ASM23774v2_genomic.fna'\n",
      "2021-03-14 07:38:28 INFO: [GenomePreparation 82430254076] Simulating reads from Genome23.0: '/tmp/tmpo1x7ji/tmpGsEdPX'\n",
      "2021-03-14 07:38:28 INFO: [GenomePreparation 82430254076] Simulating reads from Genome115.0: '/output/source_genomes/GCA_000382945.1_ASM38294v1_genomic.fna'\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.06\n",
      "\n",
      "The random seed for the run: 50971974\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0129508X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome67.01.fq\n",
      "\t the 2nd reads: /tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome67.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome67.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.09\n",
      "\n",
      "The random seed for the run: 963059643\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0964866X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome34.01.fq\n",
      "\t the 2nd reads: /tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome34.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome34.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.11\n",
      "\n",
      "The random seed for the run: 507207848\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0250506X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome201.01.fq\n",
      "\t the 2nd reads: /tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome201.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome201.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.16\n",
      "\n",
      "The random seed for the run: 393287511\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.388771X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome25.01.fq\n",
      "\t the 2nd reads: /tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome25.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome25.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.1\n",
      "\n",
      "The random seed for the run: 1414977469\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0199176X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome210.01.fq\n",
      "\t the 2nd reads: /tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome210.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome210.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.15\n",
      "\n",
      "The random seed for the run: 1937464905\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.13405X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome196.01.fq\n",
      "\t the 2nd reads: /tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome196.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome196.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.05\n",
      "\n",
      "The random seed for the run: 411957455\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0447741X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome145.01.fq\n",
      "\t the 2nd reads: /tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome145.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome145.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.46\n",
      "\n",
      "The random seed for the run: 1027999508\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.844945X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome149.01.fq\n",
      "\t the 2nd reads: /tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome149.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome149.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.58\n",
      "\n",
      "The random seed for the run: 2130432890\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.562927X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome174.01.fq\n",
      "\t the 2nd reads: /tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome174.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome174.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.86\n",
      "\n",
      "The random seed for the run: 1594031057\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            1.57076X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome136.01.fq\n",
      "\t the 2nd reads: /tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome136.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome136.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.88\n",
      "\n",
      "The random seed for the run: 1044513605\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            2.35566X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome77.01.fq\n",
      "\t the 2nd reads: /tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome77.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome77.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.22\n",
      "\n",
      "The random seed for the run: 460212766\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.213342X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome2.01.fq\n",
      "\t the 2nd reads: /tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome2.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome2.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.09\n",
      "\n",
      "The random seed for the run: 1097061061\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0567769X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome221.01.fq\n",
      "\t the 2nd reads: /tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome221.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome221.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.98\n",
      "\n",
      "The random seed for the run: 1093218188\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            2.29332X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome68.01.fq\n",
      "\t the 2nd reads: /tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome68.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome68.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.06\n",
      "\n",
      "The random seed for the run: 1669488642\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0619974X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome70.01.fq\n",
      "\t the 2nd reads: /tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome70.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome70.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.08\n",
      "\n",
      "The random seed for the run: 2029296162\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0315956X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome232.01.fq\n",
      "\t the 2nd reads: /tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome232.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome232.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.08\n",
      "\n",
      "The random seed for the run: 642284040\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0652236X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome115.01.fq\n",
      "\t the 2nd reads: /tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome115.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome115.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.17\n",
      "\n",
      "The random seed for the run: 215267604\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.094545X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome49.01.fq\n",
      "\t the 2nd reads: /tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome49.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome49.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.36\n",
      "\n",
      "The random seed for the run: 263232151\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.479189X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome48.01.fq\n",
      "\t the 2nd reads: /tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome48.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome48.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.39\n",
      "\n",
      "The random seed for the run: 370300383\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.511411X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome5.01.fq\n",
      "\t the 2nd reads: /tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome5.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome5.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.76\n",
      "\n",
      "The random seed for the run: 670345468\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.815831X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome62.01.fq\n",
      "\t the 2nd reads: /tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome62.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome62.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.43\n",
      "\n",
      "The random seed for the run: 1940362427\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.586427X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome23.01.fq\n",
      "\t the 2nd reads: /tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome23.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome23.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 1.73\n",
      "\n",
      "The random seed for the run: 1795828534\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            2.17651X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome14.01.fq\n",
      "\t the 2nd reads: /tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome14.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome14.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 6.98\n",
      "\n",
      "The random seed for the run: 941171800\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            9.58414X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome152.01.fq\n",
      "\t the 2nd reads: /tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome152.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpo1x7ji/2021.03.14_07.34.03_sample_0/reads/Genome152.0.sam\n",
      "\n",
      "2021-03-14 07:38:35 INFO: [GenomePreparation 82430254076] Simulating reads finished\n",
      "2021-03-14 07:38:46 INFO: [MetagenomeSimulationPipeline] Generate gold standard assembly\n",
      "2021-03-14 07:38:46 INFO: [MetadataReader 87205974322] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-14 07:39:10 INFO: [MetagenomeSimulationPipeline] Generate pooled strains gold standard assembly\n",
      "2021-03-14 07:39:10 INFO: [MetadataReader 114087710] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-14 07:39:10 INFO: [MetagenomeSimulationPipeline] Samples used for pooled assembly: '2021.03.14_07.34.03_sample_0'\n",
      "2021-03-14 07:39:10 INFO: [Validator 10795378910] Creating pooled gold standard\n",
      "2021-03-14 07:39:10 INFO: [Validator 10795378910] Merging bam files.\n",
      "2021-03-14 07:39:10 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_07.34.03_sample_0/bam/Genome201.0.bam'\n",
      "2021-03-14 07:39:10 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_07.34.03_sample_0/bam/Genome14.0.bam'\n",
      "2021-03-14 07:39:10 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_07.34.03_sample_0/bam/Genome68.0.bam'\n",
      "2021-03-14 07:39:10 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_07.34.03_sample_0/bam/Genome34.0.bam'\n",
      "2021-03-14 07:39:10 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_07.34.03_sample_0/bam/Genome25.0.bam'\n",
      "2021-03-14 07:39:10 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_07.34.03_sample_0/bam/Genome77.0.bam'\n",
      "2021-03-14 07:39:10 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_07.34.03_sample_0/bam/Genome136.0.bam'\n",
      "2021-03-14 07:39:10 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_07.34.03_sample_0/bam/Genome67.0.bam'\n",
      "2021-03-14 07:39:10 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_07.34.03_sample_0/bam/Genome152.0.bam'\n",
      "2021-03-14 07:39:10 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_07.34.03_sample_0/bam/Genome149.0.bam'\n",
      "2021-03-14 07:39:10 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_07.34.03_sample_0/bam/Genome210.0.bam'\n",
      "2021-03-14 07:39:10 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_07.34.03_sample_0/bam/Genome174.0.bam'\n",
      "2021-03-14 07:39:10 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_07.34.03_sample_0/bam/Genome196.0.bam'\n",
      "2021-03-14 07:39:10 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_07.34.03_sample_0/bam/Genome145.0.bam'\n",
      "2021-03-14 07:39:10 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_07.34.03_sample_0/bam/Genome62.0.bam'\n",
      "2021-03-14 07:39:10 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_07.34.03_sample_0/bam/Genome2.0.bam'\n",
      "2021-03-14 07:39:10 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_07.34.03_sample_0/bam/Genome221.0.bam'\n",
      "2021-03-14 07:39:10 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_07.34.03_sample_0/bam/Genome70.0.bam'\n",
      "2021-03-14 07:39:10 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_07.34.03_sample_0/bam/Genome232.0.bam'\n",
      "2021-03-14 07:39:10 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_07.34.03_sample_0/bam/Genome48.0.bam'\n",
      "2021-03-14 07:39:10 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_07.34.03_sample_0/bam/Genome49.0.bam'\n",
      "2021-03-14 07:39:10 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_07.34.03_sample_0/bam/Genome5.0.bam'\n",
      "2021-03-14 07:39:10 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_07.34.03_sample_0/bam/Genome23.0.bam'\n",
      "2021-03-14 07:39:10 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_07.34.03_sample_0/bam/Genome115.0.bam'\n",
      "2021-03-14 07:39:33 INFO: [MetagenomeSimulationPipeline] Anonymize Data\n",
      "2021-03-14 07:39:33 INFO: [FastaAnonymizer] Interweave shuffle and anonymize\n",
      "2021-03-14 07:40:40 INFO: [MetadataReader 4631107874] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-14 07:40:41 INFO: [MetadataReader 14300761082] Reading file: '/output/internal/meta_data.tsv'\n",
      "2021-03-14 07:40:41 INFO: [MetadataReader 87652392539] Reading file: '/tmp/tmpo1x7ji/tmpih1NHm'\n",
      "2021-03-14 07:40:43 INFO: [FastaAnonymizer 66694448886] Shuffle and anonymize '/tmp/tmpo1x7ji/tmpT15BqK'\n",
      "2021-03-14 07:40:50 INFO: [MetadataReader 4079751341] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-14 07:40:51 INFO: [MetadataReader 84063950941] Reading file: '/output/internal/meta_data.tsv'\n",
      "2021-03-14 07:40:51 INFO: [MetadataReader 48704713275] Reading file: '/tmp/tmpo1x7ji/read_start_positionsXEkuBO'\n",
      "2021-03-14 07:40:53 INFO: [MetadataReader 43119039112] Reading file: '/tmp/tmpo1x7ji/tmpeq8s6m'\n",
      "2021-03-14 07:41:34 INFO: [FastaAnonymizer 64368966291] Shuffle and anonymize '/tmp/tmpo1x7ji/tmp1MRVoK'\n",
      "2021-03-14 07:41:42 INFO: [MetadataReader 78717221707] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-14 07:41:42 INFO: [MetadataReader 89765374142] Reading file: '/output/internal/meta_data.tsv'\n",
      "2021-03-14 07:41:42 INFO: [MetadataReader 98541404663] Reading file: '/tmp/tmpo1x7ji/read_start_positionsrT8kJ6'\n",
      "2021-03-14 07:41:44 INFO: [MetadataReader 99062647065] Reading file: '/tmp/tmpo1x7ji/tmpxuCrDp'\n",
      "2021-03-14 07:42:25 INFO: [MetagenomeSimulationPipeline] Compress Data\n",
      "2021-03-14 07:42:26 INFO: [Compress 66056670849] Compressing 'tmpJRuYbX'\n",
      "2021-03-14 07:42:26 INFO: [Compress 7407710481] Compressing 'anonymous_gs_mappingfbKiIM'\n",
      "2021-03-14 07:42:26 INFO: [Compress 53471816818] Compressing 'tmpr3U_gc'\n",
      "2021-03-14 07:42:26 INFO: [Compress 51358739710] Compressing 'anonymous_gsa_mappingu6nTTC'\n",
      "2021-03-14 07:42:26 INFO: [Compress 53768138361] Compressing 'tmpz06rbC'\n",
      "2021-03-14 07:42:26 INFO: [Compress 44067189498] Compressing 'anonymous_gsa_pooled_mappingBaPQrG'\n",
      "2021-03-14 07:42:26 INFO: [Compress 44067189498] Done compressing 'gsa_pooled_mapping.tsv.gz' in 0:00:00s.\n",
      "2021-03-14 07:42:26 INFO: [Compress 51358739710] Done compressing 'gsa_mapping.tsv.gz' in 0:00:00s.\n",
      "2021-03-14 07:42:27 INFO: [Compress 7407710481] Done compressing 'reads_mapping.tsv.gz' in 0:00:02s.\n",
      "2021-03-14 07:42:27 INFO: [Compress 53768138361] Done compressing 'anonymous_gsa_pooled.fasta.gz' in 0:00:02s.\n",
      "2021-03-14 07:42:27 INFO: [Compress 53471816818] Done compressing 'anonymous_gsa.fasta.gz' in 0:00:02s.\n",
      "2021-03-14 07:42:34 INFO: [Compress 66056670849] Done compressing 'anonymous_reads.fq.gz' in 0:00:09s.\n",
      "2021-03-14 07:42:34 INFO: [MetagenomeSimulationPipeline] Metagenome simulation finished\n",
      "simulating complexity_sim_1_genera_250_genomes\n",
      "[sudo] password for pathinformatics: 2021-03-14 07:42:37 INFO: [MetagenomeSimulationPipeline] Metagenome simulation starting\n",
      "2021-03-14 07:42:37 INFO: [MetagenomeSimulationPipeline] Validating Genomes\n",
      "2021-03-14 07:42:37 INFO: [MetadataReader] Reading file: '/input/genome_to_id.tsv'\n",
      "2021-03-14 07:47:20 INFO: [MetagenomeSimulationPipeline] Design Communities\n",
      "2021-03-14 07:47:20 INFO: [CommunityDesign] Drawing strains.\n",
      "2021-03-14 07:47:20 INFO: [MetadataReader 82380310288] Reading file: '/input/metadata.tsv'\n",
      "2021-03-14 07:47:20 INFO: [MetadataReader 28782960663] Reading file: '/input/genome_to_id.tsv'\n",
      "2021-03-14 07:47:20 INFO: [CommunityDesign] Validating raw sequence files!\n",
      "2021-03-14 07:47:52 INFO: [NcbiTaxonomy] Building taxonomy tree...\n",
      "2021-03-14 07:47:52 INFO: [NcbiTaxonomy] Reading 'nodes' file:\t'/tmp/tmpoma2lH/NCBI/nodes.dmp'\n",
      "2021-03-14 07:48:03 INFO: [NcbiTaxonomy] Reading 'names' file:\t'/tmp/tmpoma2lH/NCBI/names.dmp'\n",
      "2021-03-14 07:48:05 INFO: [NcbiTaxonomy] Reading 'merged' file:\t'/tmp/tmpoma2lH/NCBI/merged.dmp'\n",
      "2021-03-14 07:48:05 INFO: [NcbiTaxonomy] Done (13.0s)\n",
      "2021-03-14 07:48:05 INFO: [MetagenomeSimulationPipeline] Move Genomes\n",
      "2021-03-14 07:48:06 INFO: [MetagenomeSimulationPipeline] Read simulation\n",
      "2021-03-14 07:48:06 INFO: [GenomePreparation 82430254076] Using 'mbarc' error profile.\n",
      "2021-03-14 07:48:06 INFO: [GenomePreparation 82430254076] Reading distribution file\n",
      "2021-03-14 07:48:06 INFO: [GenomePreparation 82430254076] Reading genome location file\n",
      "2021-03-14 07:48:29 INFO: [GenomePreparation 82430254076] Genome 'Genome142.0' has sequences below minimum, creating filtered copy.\n",
      "2021-03-14 07:48:37 INFO: [GenomePreparation 82430254076] Genome 'Genome22.0' has sequences below minimum, creating filtered copy.\n",
      "2021-03-14 07:48:37 INFO: [GenomePreparation 82430254076] Simulating reads using ReadSimulationArtIllumina readsimulator...\n",
      "2021-03-14 07:48:37 INFO: [GenomePreparation 82430254076] Simulating reads from Genome172.0: '/output/source_genomes/GCA_000508685.1_Ecoli_DORA_A_5_14_21_velvet_1.1_genomic.fna'\n",
      "2021-03-14 07:48:37 INFO: [GenomePreparation 82430254076] Simulating reads from Genome223.0: '/output/source_genomes/GCA_012251505.1_PDT000468252.1_genomic.fna'\n",
      "2021-03-14 07:48:37 INFO: [GenomePreparation 82430254076] Simulating reads from Genome29.0: '/output/source_genomes/GCA_012229025.1_PDT000300831.2_genomic.fna'\n",
      "2021-03-14 07:48:37 INFO: [GenomePreparation 82430254076] Simulating reads from Genome34.0: '/output/source_genomes/GCA_013049715.1_PDT000290410.4_genomic.fna'\n",
      "2021-03-14 07:48:37 INFO: [GenomePreparation 82430254076] Simulating reads from Genome25.0: '/output/source_genomes/GCA_902729235.2_easoj011_ecoli_genomic_umi_ont_r10_g324hac_genomic.fna'\n",
      "2021-03-14 07:48:37 INFO: [GenomePreparation 82430254076] Simulating reads from Genome128.0: '/output/source_genomes/GCA_012248085.1_PDT000290413.2_genomic.fna'\n",
      "2021-03-14 07:48:37 INFO: [GenomePreparation 82430254076] Simulating reads from Genome59.0: '/output/source_genomes/GCA_013039825.1_PDT000415768.1_genomic.fna'\n",
      "2021-03-14 07:48:37 INFO: [GenomePreparation 82430254076] Simulating reads from Genome133.0: '/output/source_genomes/GCA_016955435.1_PDT000971175.1_genomic.fna'\n",
      "2021-03-14 07:48:37 INFO: [GenomePreparation 82430254076] Simulating reads from Genome64.0: '/output/source_genomes/GCA_016954975.1_PDT000971222.1_genomic.fna'\n",
      "2021-03-14 07:48:37 INFO: [GenomePreparation 82430254076] Simulating reads from Genome136.0: '/output/source_genomes/GCA_000432955.1_MGS4_genomic.fna'\n",
      "2021-03-14 07:48:37 INFO: [GenomePreparation 82430254076] Simulating reads from Genome134.0: '/output/source_genomes/GCA_000614625.1_ASM61462v1_genomic.fna'\n",
      "2021-03-14 07:48:37 INFO: [GenomePreparation 82430254076] Simulating reads from Genome23.0: '/output/source_genomes/GCA_000627905.1_ASM62790v1_genomic.fna'\n",
      "2021-03-14 07:48:37 INFO: [GenomePreparation 82430254076] Simulating reads from Genome107.0: '/output/source_genomes/GCA_012306995.1_PDT000275040.2_genomic.fna'\n",
      "2021-03-14 07:48:37 INFO: [GenomePreparation 82430254076] Simulating reads from Genome197.0: '/output/source_genomes/GCA_016955235.1_PDT000971187.1_genomic.fna'\n",
      "2021-03-14 07:48:37 INFO: [GenomePreparation 82430254076] Simulating reads from Genome205.0: '/output/source_genomes/GCA_016955515.1_PDT000971169.1_genomic.fna'\n",
      "2021-03-14 07:48:37 INFO: [GenomePreparation 82430254076] Simulating reads from Genome147.0: '/output/source_genomes/GCA_000249395.2_ASM24939v2_genomic.fna'\n",
      "2021-03-14 07:48:37 INFO: [GenomePreparation 82430254076] Simulating reads from Genome142.0: '/tmp/tmpb8rc2y/tmpsxa0uH'\n",
      "2021-03-14 07:48:37 INFO: [GenomePreparation 82430254076] Simulating reads from Genome5.0: '/output/source_genomes/GCA_000249355.2_ASM24935v2_genomic.fna'\n",
      "2021-03-14 07:48:37 INFO: [GenomePreparation 82430254076] Simulating reads from Genome245.0: '/output/source_genomes/GCA_012215445.1_PDT000305442.2_genomic.fna'\n",
      "2021-03-14 07:48:37 INFO: [GenomePreparation 82430254076] Simulating reads from Genome226.0: '/output/source_genomes/GCA_000601175.1_ASM60117v1_genomic.fna'\n",
      "2021-03-14 07:48:37 INFO: [GenomePreparation 82430254076] Simulating reads from Genome182.0: '/output/source_genomes/GCA_012166075.1_PDT000336566.2_genomic.fna'\n",
      "2021-03-14 07:48:37 INFO: [GenomePreparation 82430254076] Simulating reads from Genome10.0: '/output/source_genomes/GCA_016954795.1_PDT000971232.1_genomic.fna'\n",
      "2021-03-14 07:48:37 INFO: [GenomePreparation 82430254076] Simulating reads from Genome228.0: '/output/source_genomes/GCA_016955755.1_PDT000971147.1_genomic.fna'\n",
      "2021-03-14 07:48:37 INFO: [GenomePreparation 82430254076] Simulating reads from Genome22.0: '/tmp/tmpb8rc2y/tmpminJBk'\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.09\n",
      "\n",
      "The random seed for the run: 1044513605\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0202189X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome128.01.fq\n",
      "\t the 2nd reads: /tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome128.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome128.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.12\n",
      "\n",
      "The random seed for the run: 507207848\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0255015X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome172.01.fq\n",
      "\t the 2nd reads: /tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome172.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome172.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.15\n",
      "\n",
      "The random seed for the run: 1795828534\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0763095X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome223.01.fq\n",
      "\t the 2nd reads: /tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome223.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome223.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.16\n",
      "\n",
      "The random seed for the run: 1594031057\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.108195X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome133.01.fq\n",
      "\t the 2nd reads: /tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome133.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome133.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.13\n",
      "\n",
      "The random seed for the run: 1027999508\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0526435X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome136.01.fq\n",
      "\t the 2nd reads: /tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome136.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome136.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.35\n",
      "\n",
      "The random seed for the run: 50971974\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.313786X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome59.01.fq\n",
      "\t the 2nd reads: /tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome59.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome59.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.48\n",
      "\n",
      "The random seed for the run: 1093218188\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.47332X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome29.01.fq\n",
      "\t the 2nd reads: /tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome29.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome29.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.11\n",
      "\n",
      "The random seed for the run: 411957455\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0104529X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome197.01.fq\n",
      "\t the 2nd reads: /tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome197.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome197.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.14\n",
      "\n",
      "The random seed for the run: 670345468\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0778767X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome205.01.fq\n",
      "\t the 2nd reads: /tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome205.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome205.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.14\n",
      "\n",
      "The random seed for the run: 460212766\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0500396X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome147.01.fq\n",
      "\t the 2nd reads: /tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome147.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome147.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.66\n",
      "\n",
      "The random seed for the run: 941171800\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.681975X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome64.01.fq\n",
      "\t the 2nd reads: /tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome64.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome64.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.14\n",
      "\n",
      "The random seed for the run: 1669488642\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.045826X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome5.01.fq\n",
      "\t the 2nd reads: /tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome5.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome5.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.12\n",
      "\n",
      "The random seed for the run: 2029296162\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0361382X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome245.01.fq\n",
      "\t the 2nd reads: /tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome245.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome245.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.11\n",
      "\n",
      "The random seed for the run: 263232151\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0160759X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome226.01.fq\n",
      "\t the 2nd reads: /tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome226.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome226.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.24\n",
      "\n",
      "The random seed for the run: 215267604\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.172194X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome182.01.fq\n",
      "\t the 2nd reads: /tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome182.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome182.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 1.35\n",
      "\n",
      "The random seed for the run: 393287511\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.386765X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome25.01.fq\n",
      "\t the 2nd reads: /tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome25.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome25.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 1.13\n",
      "\n",
      "The random seed for the run: 2130432890\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            1.2678X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome23.01.fq\n",
      "\t the 2nd reads: /tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome23.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome23.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.64\n",
      "\n",
      "The random seed for the run: 1097061061\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.658476X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome142.01.fq\n",
      "\t the 2nd reads: /tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome142.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome142.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 1.57\n",
      "\n",
      "The random seed for the run: 963059643\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            1.851X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome34.01.fq\n",
      "\t the 2nd reads: /tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome34.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome34.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.46\n",
      "\n",
      "The random seed for the run: 370300383\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.454352X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome10.01.fq\n",
      "\t the 2nd reads: /tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome10.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome10.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.44\n",
      "\n",
      "The random seed for the run: 1940362427\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.412772X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome228.01.fq\n",
      "\t the 2nd reads: /tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome228.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome228.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 1.46\n",
      "\n",
      "The random seed for the run: 1937464905\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            1.75671X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome107.01.fq\n",
      "\t the 2nd reads: /tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome107.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome107.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 1.42\n",
      "\n",
      "The random seed for the run: 642284040\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            1.90131X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome22.01.fq\n",
      "\t the 2nd reads: /tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome22.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome22.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 5.4\n",
      "\n",
      "The random seed for the run: 1414977469\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            7.73559X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome134.01.fq\n",
      "\t the 2nd reads: /tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome134.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpb8rc2y/2021.03.14_07.42.37_sample_0/reads/Genome134.0.sam\n",
      "\n",
      "2021-03-14 07:48:43 INFO: [GenomePreparation 82430254076] Simulating reads finished\n",
      "2021-03-14 07:48:52 INFO: [MetagenomeSimulationPipeline] Generate gold standard assembly\n",
      "2021-03-14 07:48:52 INFO: [MetadataReader 87205974322] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-14 07:49:22 INFO: [MetagenomeSimulationPipeline] Generate pooled strains gold standard assembly\n",
      "2021-03-14 07:49:22 INFO: [MetadataReader 114087710] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-14 07:49:22 INFO: [MetagenomeSimulationPipeline] Samples used for pooled assembly: '2021.03.14_07.42.37_sample_0'\n",
      "2021-03-14 07:49:22 INFO: [Validator 10795378910] Creating pooled gold standard\n",
      "2021-03-14 07:49:22 INFO: [Validator 10795378910] Merging bam files.\n",
      "2021-03-14 07:49:22 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_07.42.37_sample_0/bam/Genome172.0.bam'\n",
      "2021-03-14 07:49:22 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_07.42.37_sample_0/bam/Genome223.0.bam'\n",
      "2021-03-14 07:49:22 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_07.42.37_sample_0/bam/Genome29.0.bam'\n",
      "2021-03-14 07:49:22 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_07.42.37_sample_0/bam/Genome34.0.bam'\n",
      "2021-03-14 07:49:22 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_07.42.37_sample_0/bam/Genome25.0.bam'\n",
      "2021-03-14 07:49:22 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_07.42.37_sample_0/bam/Genome128.0.bam'\n",
      "2021-03-14 07:49:22 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_07.42.37_sample_0/bam/Genome59.0.bam'\n",
      "2021-03-14 07:49:22 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_07.42.37_sample_0/bam/Genome133.0.bam'\n",
      "2021-03-14 07:49:22 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_07.42.37_sample_0/bam/Genome64.0.bam'\n",
      "2021-03-14 07:49:22 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_07.42.37_sample_0/bam/Genome136.0.bam'\n",
      "2021-03-14 07:49:22 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_07.42.37_sample_0/bam/Genome134.0.bam'\n",
      "2021-03-14 07:49:22 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_07.42.37_sample_0/bam/Genome226.0.bam'\n",
      "2021-03-14 07:49:22 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_07.42.37_sample_0/bam/Genome107.0.bam'\n",
      "2021-03-14 07:49:22 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_07.42.37_sample_0/bam/Genome197.0.bam'\n",
      "2021-03-14 07:49:22 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_07.42.37_sample_0/bam/Genome205.0.bam'\n",
      "2021-03-14 07:49:22 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_07.42.37_sample_0/bam/Genome147.0.bam'\n",
      "2021-03-14 07:49:22 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_07.42.37_sample_0/bam/Genome142.0.bam'\n",
      "2021-03-14 07:49:22 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_07.42.37_sample_0/bam/Genome5.0.bam'\n",
      "2021-03-14 07:49:22 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_07.42.37_sample_0/bam/Genome245.0.bam'\n",
      "2021-03-14 07:49:22 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_07.42.37_sample_0/bam/Genome23.0.bam'\n",
      "2021-03-14 07:49:22 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_07.42.37_sample_0/bam/Genome182.0.bam'\n",
      "2021-03-14 07:49:22 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_07.42.37_sample_0/bam/Genome10.0.bam'\n",
      "2021-03-14 07:49:22 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_07.42.37_sample_0/bam/Genome228.0.bam'\n",
      "2021-03-14 07:49:22 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_07.42.37_sample_0/bam/Genome22.0.bam'\n",
      "2021-03-14 07:49:52 INFO: [MetagenomeSimulationPipeline] Anonymize Data\n",
      "2021-03-14 07:49:52 INFO: [FastaAnonymizer] Interweave shuffle and anonymize\n",
      "2021-03-14 07:50:59 INFO: [MetadataReader 4631107874] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-14 07:51:00 INFO: [MetadataReader 14300761082] Reading file: '/output/internal/meta_data.tsv'\n",
      "2021-03-14 07:51:00 INFO: [MetadataReader 87652392539] Reading file: '/tmp/tmpb8rc2y/tmpC8AZJR'\n",
      "2021-03-14 07:51:03 INFO: [FastaAnonymizer 66694448886] Shuffle and anonymize '/tmp/tmpb8rc2y/tmpkTzzKD'\n",
      "2021-03-14 07:51:12 INFO: [MetadataReader 4079751341] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-14 07:51:13 INFO: [MetadataReader 84063950941] Reading file: '/output/internal/meta_data.tsv'\n",
      "2021-03-14 07:51:13 INFO: [MetadataReader 48704713275] Reading file: '/tmp/tmpb8rc2y/read_start_positionsXerRQQ'\n",
      "2021-03-14 07:51:14 INFO: [MetadataReader 43119039112] Reading file: '/tmp/tmpb8rc2y/tmpC85zPh'\n",
      "2021-03-14 07:51:19 INFO: [FastaAnonymizer 64368966291] Shuffle and anonymize '/tmp/tmpb8rc2y/tmp_0d25N'\n",
      "2021-03-14 07:51:28 INFO: [MetadataReader 78717221707] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-14 07:51:29 INFO: [MetadataReader 89765374142] Reading file: '/output/internal/meta_data.tsv'\n",
      "2021-03-14 07:51:29 INFO: [MetadataReader 98541404663] Reading file: '/tmp/tmpb8rc2y/read_start_positionsCv_bfo'\n",
      "2021-03-14 07:51:31 INFO: [MetadataReader 99062647065] Reading file: '/tmp/tmpb8rc2y/tmpanVJmC'\n",
      "2021-03-14 07:51:35 INFO: [MetagenomeSimulationPipeline] Compress Data\n",
      "2021-03-14 07:51:36 INFO: [Compress 32234350125] Compressing 'tmpHHhyBc'\n",
      "2021-03-14 07:51:36 INFO: [Compress 29782785963] Compressing 'anonymous_gs_mappinghN8sYe'\n",
      "2021-03-14 07:51:36 INFO: [Compress 56054085991] Compressing 'tmp3oTRNf'\n",
      "2021-03-14 07:51:36 INFO: [Compress 70270261540] Compressing 'anonymous_gsa_mappingAnrPQu'\n",
      "2021-03-14 07:51:36 INFO: [Compress 73179684442] Compressing 'tmpvOTZiM'\n",
      "2021-03-14 07:51:36 INFO: [Compress 35416206579] Compressing 'anonymous_gsa_pooled_mappingMpfZRp'\n",
      "2021-03-14 07:51:36 INFO: [Compress 35416206579] Done compressing 'gsa_pooled_mapping.tsv.gz' in 0:00:00s.\n",
      "2021-03-14 07:51:36 INFO: [Compress 70270261540] Done compressing 'gsa_mapping.tsv.gz' in 0:00:00s.\n",
      "2021-03-14 07:51:37 INFO: [Compress 29782785963] Done compressing 'reads_mapping.tsv.gz' in 0:00:02s.\n",
      "2021-03-14 07:51:38 INFO: [Compress 56054085991] Done compressing 'anonymous_gsa.fasta.gz' in 0:00:02s.\n",
      "2021-03-14 07:51:38 INFO: [Compress 73179684442] Done compressing 'anonymous_gsa_pooled.fasta.gz' in 0:00:02s.\n",
      "2021-03-14 07:51:45 INFO: [Compress 32234350125] Done compressing 'anonymous_reads.fq.gz' in 0:00:09s.\n",
      "2021-03-14 07:51:45 INFO: [MetagenomeSimulationPipeline] Metagenome simulation finished\n",
      "simulating complexity_sim_1_genera_25_genomes\n",
      "[sudo] password for pathinformatics: 2021-03-14 07:51:47 INFO: [MetagenomeSimulationPipeline] Metagenome simulation starting\n",
      "2021-03-14 07:51:47 INFO: [MetagenomeSimulationPipeline] Validating Genomes\n",
      "2021-03-14 07:51:47 INFO: [MetadataReader] Reading file: '/input/genome_to_id.tsv'\n",
      "2021-03-14 07:52:17 INFO: [MetagenomeSimulationPipeline] Design Communities\n",
      "2021-03-14 07:52:17 INFO: [CommunityDesign] Drawing strains.\n",
      "2021-03-14 07:52:17 INFO: [MetadataReader 82380310288] Reading file: '/input/metadata.tsv'\n",
      "2021-03-14 07:52:17 INFO: [MetadataReader 25595780401] Reading file: '/input/genome_to_id.tsv'\n",
      "2021-03-14 07:52:17 INFO: [CommunityDesign] Validating raw sequence files!\n",
      "2021-03-14 07:52:47 INFO: [NcbiTaxonomy] Building taxonomy tree...\n",
      "2021-03-14 07:52:47 INFO: [NcbiTaxonomy] Reading 'nodes' file:\t'/tmp/tmpvywn4z/NCBI/nodes.dmp'\n",
      "2021-03-14 07:52:59 INFO: [NcbiTaxonomy] Reading 'names' file:\t'/tmp/tmpvywn4z/NCBI/names.dmp'\n",
      "2021-03-14 07:53:00 INFO: [NcbiTaxonomy] Reading 'merged' file:\t'/tmp/tmpvywn4z/NCBI/merged.dmp'\n",
      "2021-03-14 07:53:00 INFO: [NcbiTaxonomy] Done (13.0s)\n",
      "2021-03-14 07:53:00 INFO: [MetagenomeSimulationPipeline] Move Genomes\n",
      "2021-03-14 07:53:02 INFO: [MetagenomeSimulationPipeline] Read simulation\n",
      "2021-03-14 07:53:02 INFO: [GenomePreparation 70514612869] Using 'mbarc' error profile.\n",
      "2021-03-14 07:53:02 INFO: [GenomePreparation 70514612869] Reading distribution file\n",
      "2021-03-14 07:53:02 INFO: [GenomePreparation 70514612869] Reading genome location file\n",
      "2021-03-14 07:53:07 INFO: [GenomePreparation 70514612869] Genome 'Genome24.0' has sequences below minimum, creating filtered copy.\n",
      "2021-03-14 07:53:11 INFO: [GenomePreparation 70514612869] Genome 'Genome18.0' has sequences below minimum, creating filtered copy.\n",
      "2021-03-14 07:53:33 INFO: [GenomePreparation 70514612869] Simulating reads using ReadSimulationArtIllumina readsimulator...\n",
      "2021-03-14 07:53:33 INFO: [GenomePreparation 70514612869] Simulating reads from Genome15.0: '/output/source_genomes/GCA_000800845.2_ASM80084v2_genomic.fna'\n",
      "2021-03-14 07:53:33 INFO: [GenomePreparation 70514612869] Simulating reads from Genome14.0: '/output/source_genomes/GCA_016955605.1_PDT000971167.1_genomic.fna'\n",
      "2021-03-14 07:53:33 INFO: [GenomePreparation 70514612869] Simulating reads from Genome11.0: '/output/source_genomes/GCA_016955655.1_PDT000971153.1_genomic.fna'\n",
      "2021-03-14 07:53:33 INFO: [GenomePreparation 70514612869] Simulating reads from Genome24.0: '/tmp/tmpnNUlkA/tmpeRuTOp'\n",
      "2021-03-14 07:53:33 INFO: [GenomePreparation 70514612869] Simulating reads from Genome12.0: '/output/source_genomes/GCA_013013995.1_PDT000430796.1_genomic.fna'\n",
      "2021-03-14 07:53:33 INFO: [GenomePreparation 70514612869] Simulating reads from Genome25.0: '/output/source_genomes/GCA_000155005.1_ASM15500v1_genomic.fna'\n",
      "2021-03-14 07:53:33 INFO: [GenomePreparation 70514612869] Simulating reads from Genome18.0: '/tmp/tmpnNUlkA/tmpZBpTA0'\n",
      "2021-03-14 07:53:33 INFO: [GenomePreparation 70514612869] Simulating reads from Genome17.0: '/output/source_genomes/GCA_013014205.1_PDT000430767.1_genomic.fna'\n",
      "2021-03-14 07:53:33 INFO: [GenomePreparation 70514612869] Simulating reads from Genome16.0: '/output/source_genomes/GCA_011845705.1_PDT000606249.2_genomic.fna'\n",
      "2021-03-14 07:53:33 INFO: [GenomePreparation 70514612869] Simulating reads from Genome2.0: '/output/source_genomes/GCA_012265765.1_PDT000281731.2_genomic.fna'\n",
      "2021-03-14 07:53:33 INFO: [GenomePreparation 70514612869] Simulating reads from Genome1.0: '/output/source_genomes/GCA_012247525.1_PDT000290367.2_genomic.fna'\n",
      "2021-03-14 07:53:33 INFO: [GenomePreparation 70514612869] Simulating reads from Genome21.0: '/output/source_genomes/GCA_013040725.1_PDT000415607.1_genomic.fna'\n",
      "2021-03-14 07:53:33 INFO: [GenomePreparation 70514612869] Simulating reads from Genome4.0: '/output/source_genomes/GCA_013040605.1_PDT000415656.1_genomic.fna'\n",
      "2021-03-14 07:53:33 INFO: [GenomePreparation 70514612869] Simulating reads from Genome5.0: '/output/source_genomes/GCA_012247405.1_PDT000290386.2_genomic.fna'\n",
      "2021-03-14 07:53:33 INFO: [GenomePreparation 70514612869] Simulating reads from Genome3.0: '/output/source_genomes/GCA_012307065.1_PDT000275037.3_genomic.fna'\n",
      "2021-03-14 07:53:33 INFO: [GenomePreparation 70514612869] Simulating reads from Genome20.0: '/output/source_genomes/GCA_012247165.1_PDT000290409.2_genomic.fna'\n",
      "2021-03-14 07:53:33 INFO: [GenomePreparation 70514612869] Simulating reads from Genome6.0: '/output/source_genomes/GCA_016955115.1_PDT000971190.1_genomic.fna'\n",
      "2021-03-14 07:53:33 INFO: [GenomePreparation 70514612869] Simulating reads from Genome23.0: '/output/source_genomes/GCA_012211845.1_PDT000307441.2_genomic.fna'\n",
      "2021-03-14 07:53:33 INFO: [GenomePreparation 70514612869] Simulating reads from Genome9.0: '/output/source_genomes/GCA_013014285.1_PDT000430776.1_genomic.fna'\n",
      "2021-03-14 07:53:33 INFO: [GenomePreparation 70514612869] Simulating reads from Genome7.0: '/output/source_genomes/GCA_016954835.1_PDT000971228.1_genomic.fna'\n",
      "2021-03-14 07:53:33 INFO: [GenomePreparation 70514612869] Simulating reads from Genome13.0: '/output/source_genomes/GCA_012247305.1_PDT000290405.2_genomic.fna'\n",
      "2021-03-14 07:53:33 INFO: [GenomePreparation 70514612869] Simulating reads from Genome10.0: '/output/source_genomes/GCA_000350185.1_ASM35018v1_genomic.fna'\n",
      "2021-03-14 07:53:33 INFO: [GenomePreparation 70514612869] Simulating reads from Genome8.0: '/output/source_genomes/GCA_000188755.2_ASM18875v2_genomic.fna'\n",
      "2021-03-14 07:53:33 INFO: [GenomePreparation 70514612869] Simulating reads from Genome22.0: '/output/source_genomes/GCA_013012765.1_PDT000430806.1_genomic.fna'\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.1\n",
      "\n",
      "The random seed for the run: 170568346\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.00286187X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome18.01.fq\n",
      "\t the 2nd reads: /tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome18.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome18.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.29\n",
      "\n",
      "The random seed for the run: 65634987\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.261613X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome11.01.fq\n",
      "\t the 2nd reads: /tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome11.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome11.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.35\n",
      "\n",
      "The random seed for the run: 2129338277\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0899223X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome25.01.fq\n",
      "\t the 2nd reads: /tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome25.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome25.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.1\n",
      "\n",
      "The random seed for the run: 1577639001\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0215184X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome2.01.fq\n",
      "\t the 2nd reads: /tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome2.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome2.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.14\n",
      "\n",
      "The random seed for the run: 1713385206\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0536326X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome1.01.fq\n",
      "\t the 2nd reads: /tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome1.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome1.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.62\n",
      "\n",
      "The random seed for the run: 540968737\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.61836X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome15.01.fq\n",
      "\t the 2nd reads: /tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome15.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome15.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.63\n",
      "\n",
      "The random seed for the run: 1294107793\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.639196X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome24.01.fq\n",
      "\t the 2nd reads: /tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome24.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome24.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.17\n",
      "\n",
      "The random seed for the run: 49814198\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0938536X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome21.01.fq\n",
      "\t the 2nd reads: /tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome21.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome21.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.08\n",
      "\n",
      "The random seed for the run: 1557933274\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0020573X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome4.01.fq\n",
      "\t the 2nd reads: /tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome4.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome4.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.09\n",
      "\n",
      "The random seed for the run: 1329514041\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.00242505X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome5.01.fq\n",
      "\t the 2nd reads: /tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome5.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome5.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.17\n",
      "\n",
      "The random seed for the run: 1168871082\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.103938X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome3.01.fq\n",
      "\t the 2nd reads: /tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome3.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome3.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.08\n",
      "\n",
      "The random seed for the run: 923704085\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.00951369X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome6.01.fq\n",
      "\t the 2nd reads: /tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome6.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome6.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 1\n",
      "\n",
      "The random seed for the run: 1703660943\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            1.14594X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome17.01.fq\n",
      "\t the 2nd reads: /tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome17.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome17.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.13\n",
      "\n",
      "The random seed for the run: 1923521921\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0464062X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome7.01.fq\n",
      "\t the 2nd reads: /tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome7.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome7.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.1\n",
      "\n",
      "The random seed for the run: 854832072\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.015815X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome13.01.fq\n",
      "\t the 2nd reads: /tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome13.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome13.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.38\n",
      "\n",
      "The random seed for the run: 1550755105\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.355331X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome23.01.fq\n",
      "\t the 2nd reads: /tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome23.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome23.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.11\n",
      "\n",
      "The random seed for the run: 689939719\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0510678X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome10.01.fq\n",
      "\t the 2nd reads: /tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome10.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome10.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.11\n",
      "\n",
      "The random seed for the run: 2102807139\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.030329X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome8.01.fq\n",
      "\t the 2nd reads: /tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome8.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome8.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.12\n",
      "\n",
      "The random seed for the run: 1940270192\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0417548X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome22.01.fq\n",
      "\t the 2nd reads: /tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome22.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome22.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 1.28\n",
      "\n",
      "The random seed for the run: 1961570421\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            1.43421X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome20.01.fq\n",
      "\t the 2nd reads: /tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome20.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome20.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 2.01\n",
      "\n",
      "The random seed for the run: 1661055960\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            2.30539X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome16.01.fq\n",
      "\t the 2nd reads: /tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome16.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome16.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 1.86\n",
      "\n",
      "The random seed for the run: 1992431107\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            2.1414X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome9.01.fq\n",
      "\t the 2nd reads: /tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome9.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome9.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 3.24\n",
      "\n",
      "The random seed for the run: 1847505585\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            4.10578X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome14.01.fq\n",
      "\t the 2nd reads: /tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome14.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome14.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 4.05\n",
      "\n",
      "The random seed for the run: 573380147\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            5.73688X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome12.01.fq\n",
      "\t the 2nd reads: /tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome12.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpnNUlkA/2021.03.14_07.51.47_sample_0/reads/Genome12.0.sam\n",
      "\n",
      "2021-03-14 07:53:37 INFO: [GenomePreparation 70514612869] Simulating reads finished\n",
      "2021-03-14 07:53:43 INFO: [MetagenomeSimulationPipeline] Generate gold standard assembly\n",
      "2021-03-14 07:53:43 INFO: [MetadataReader 64136302938] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-14 07:54:11 INFO: [MetagenomeSimulationPipeline] Generate pooled strains gold standard assembly\n",
      "2021-03-14 07:54:11 INFO: [MetadataReader 58498923446] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-14 07:54:11 INFO: [MetagenomeSimulationPipeline] Samples used for pooled assembly: '2021.03.14_07.51.47_sample_0'\n",
      "2021-03-14 07:54:11 INFO: [Validator 48443846240] Creating pooled gold standard\n",
      "2021-03-14 07:54:11 INFO: [Validator 48443846240] Merging bam files.\n",
      "2021-03-14 07:54:11 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_07.51.47_sample_0/bam/Genome15.0.bam'\n",
      "2021-03-14 07:54:11 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_07.51.47_sample_0/bam/Genome14.0.bam'\n",
      "2021-03-14 07:54:11 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_07.51.47_sample_0/bam/Genome11.0.bam'\n",
      "2021-03-14 07:54:11 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_07.51.47_sample_0/bam/Genome24.0.bam'\n",
      "2021-03-14 07:54:11 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_07.51.47_sample_0/bam/Genome12.0.bam'\n",
      "2021-03-14 07:54:11 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_07.51.47_sample_0/bam/Genome25.0.bam'\n",
      "2021-03-14 07:54:11 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_07.51.47_sample_0/bam/Genome18.0.bam'\n",
      "2021-03-14 07:54:11 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_07.51.47_sample_0/bam/Genome17.0.bam'\n",
      "2021-03-14 07:54:11 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_07.51.47_sample_0/bam/Genome16.0.bam'\n",
      "2021-03-14 07:54:11 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_07.51.47_sample_0/bam/Genome2.0.bam'\n",
      "2021-03-14 07:54:11 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_07.51.47_sample_0/bam/Genome1.0.bam'\n",
      "2021-03-14 07:54:11 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_07.51.47_sample_0/bam/Genome21.0.bam'\n",
      "2021-03-14 07:54:11 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_07.51.47_sample_0/bam/Genome4.0.bam'\n",
      "2021-03-14 07:54:11 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_07.51.47_sample_0/bam/Genome5.0.bam'\n",
      "2021-03-14 07:54:11 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_07.51.47_sample_0/bam/Genome3.0.bam'\n",
      "2021-03-14 07:54:11 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_07.51.47_sample_0/bam/Genome20.0.bam'\n",
      "2021-03-14 07:54:11 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_07.51.47_sample_0/bam/Genome6.0.bam'\n",
      "2021-03-14 07:54:11 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_07.51.47_sample_0/bam/Genome23.0.bam'\n",
      "2021-03-14 07:54:11 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_07.51.47_sample_0/bam/Genome9.0.bam'\n",
      "2021-03-14 07:54:11 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_07.51.47_sample_0/bam/Genome7.0.bam'\n",
      "2021-03-14 07:54:11 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_07.51.47_sample_0/bam/Genome13.0.bam'\n",
      "2021-03-14 07:54:11 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_07.51.47_sample_0/bam/Genome10.0.bam'\n",
      "2021-03-14 07:54:11 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_07.51.47_sample_0/bam/Genome8.0.bam'\n",
      "2021-03-14 07:54:11 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_07.51.47_sample_0/bam/Genome22.0.bam'\n",
      "2021-03-14 07:54:39 INFO: [MetagenomeSimulationPipeline] Anonymize Data\n",
      "2021-03-14 07:54:39 INFO: [FastaAnonymizer] Interweave shuffle and anonymize\n",
      "2021-03-14 07:55:45 INFO: [MetadataReader 72802808290] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-14 07:55:46 INFO: [MetadataReader 47315449936] Reading file: '/output/internal/meta_data.tsv'\n",
      "2021-03-14 07:55:46 INFO: [MetadataReader 65300095466] Reading file: '/tmp/tmpnNUlkA/tmppzUTdU'\n",
      "2021-03-14 07:55:49 INFO: [FastaAnonymizer 43075827506] Shuffle and anonymize '/tmp/tmpnNUlkA/tmpk6MzNy'\n",
      "2021-03-14 07:55:56 INFO: [MetadataReader 71873526544] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-14 07:55:57 INFO: [MetadataReader 45105173502] Reading file: '/output/internal/meta_data.tsv'\n",
      "2021-03-14 07:55:57 INFO: [MetadataReader 84613406616] Reading file: '/tmp/tmpnNUlkA/read_start_positions8Aji3d'\n",
      "2021-03-14 07:55:59 INFO: [MetadataReader 10674074437] Reading file: '/tmp/tmpnNUlkA/tmpzNu2hl'\n",
      "2021-03-14 07:56:13 INFO: [FastaAnonymizer 86623674311] Shuffle and anonymize '/tmp/tmpnNUlkA/tmp4xLM6y'\n",
      "2021-03-14 07:56:21 INFO: [MetadataReader 15345066727] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-14 07:56:22 INFO: [MetadataReader 77753400540] Reading file: '/output/internal/meta_data.tsv'\n",
      "2021-03-14 07:56:22 INFO: [MetadataReader 12266665971] Reading file: '/tmp/tmpnNUlkA/read_start_positionsteOO0d'\n",
      "2021-03-14 07:56:24 INFO: [MetadataReader 42355547664] Reading file: '/tmp/tmpnNUlkA/tmpzhzXCa'\n",
      "2021-03-14 07:56:38 INFO: [MetagenomeSimulationPipeline] Compress Data\n",
      "2021-03-14 07:56:38 INFO: [Compress 19537999510] Compressing 'tmp4fe2AC'\n",
      "2021-03-14 07:56:38 INFO: [Compress 17206328422] Compressing 'anonymous_gs_mappinglIZNk7'\n",
      "2021-03-14 07:56:38 INFO: [Compress 49404659626] Compressing 'tmpIQUuCA'\n",
      "2021-03-14 07:56:38 INFO: [Compress 78620986601] Compressing 'anonymous_gsa_mappingqd20yh'\n",
      "2021-03-14 07:56:38 INFO: [Compress 28775991571] Compressing 'tmpTSFPHi'\n",
      "2021-03-14 07:56:38 INFO: [Compress 79674513343] Compressing 'anonymous_gsa_pooled_mappingzmteja'\n",
      "2021-03-14 07:56:38 INFO: [Compress 79674513343] Done compressing 'gsa_pooled_mapping.tsv.gz' in 0:00:00s.\n",
      "2021-03-14 07:56:38 INFO: [Compress 78620986601] Done compressing 'gsa_mapping.tsv.gz' in 0:00:00s.\n",
      "2021-03-14 07:56:40 INFO: [Compress 17206328422] Done compressing 'reads_mapping.tsv.gz' in 0:00:02s.\n",
      "2021-03-14 07:56:40 INFO: [Compress 49404659626] Done compressing 'anonymous_gsa.fasta.gz' in 0:00:02s.\n",
      "2021-03-14 07:56:40 INFO: [Compress 28775991571] Done compressing 'anonymous_gsa_pooled.fasta.gz' in 0:00:02s.\n",
      "2021-03-14 07:56:47 INFO: [Compress 19537999510] Done compressing 'anonymous_reads.fq.gz' in 0:00:09s.\n",
      "2021-03-14 07:56:47 INFO: [MetagenomeSimulationPipeline] Metagenome simulation finished\n",
      "simulating complexity_sim_5_genera_50_genomes\n",
      "[sudo] password for pathinformatics: 2021-03-14 07:56:49 INFO: [MetagenomeSimulationPipeline] Metagenome simulation starting\n",
      "2021-03-14 07:56:49 INFO: [MetagenomeSimulationPipeline] Validating Genomes\n",
      "2021-03-14 07:56:49 INFO: [MetadataReader] Reading file: '/input/genome_to_id.tsv'\n",
      "2021-03-14 07:57:37 INFO: [MetagenomeSimulationPipeline] Design Communities\n",
      "2021-03-14 07:57:37 INFO: [CommunityDesign] Drawing strains.\n",
      "2021-03-14 07:57:37 INFO: [MetadataReader 82380310288] Reading file: '/input/metadata.tsv'\n",
      "2021-03-14 07:57:37 INFO: [MetadataReader 89885674421] Reading file: '/input/genome_to_id.tsv'\n",
      "2021-03-14 07:57:37 INFO: [CommunityDesign] Validating raw sequence files!\n",
      "2021-03-14 07:58:01 INFO: [NcbiTaxonomy] Building taxonomy tree...\n",
      "2021-03-14 07:58:01 INFO: [NcbiTaxonomy] Reading 'nodes' file:\t'/tmp/tmpiu0Yca/NCBI/nodes.dmp'\n",
      "2021-03-14 07:58:12 INFO: [NcbiTaxonomy] Reading 'names' file:\t'/tmp/tmpiu0Yca/NCBI/names.dmp'\n",
      "2021-03-14 07:58:14 INFO: [NcbiTaxonomy] Reading 'merged' file:\t'/tmp/tmpiu0Yca/NCBI/merged.dmp'\n",
      "2021-03-14 07:58:14 INFO: [NcbiTaxonomy] Done (13.0s)\n",
      "2021-03-14 07:58:14 INFO: [MetagenomeSimulationPipeline] Move Genomes\n",
      "2021-03-14 07:58:15 INFO: [MetagenomeSimulationPipeline] Read simulation\n",
      "2021-03-14 07:58:15 INFO: [GenomePreparation 64152656348] Using 'mbarc' error profile.\n",
      "2021-03-14 07:58:15 INFO: [GenomePreparation 64152656348] Reading distribution file\n",
      "2021-03-14 07:58:15 INFO: [GenomePreparation 64152656348] Reading genome location file\n",
      "2021-03-14 07:58:17 INFO: [GenomePreparation 64152656348] Genome 'Genome14.0' has sequences below minimum, creating filtered copy.\n",
      "2021-03-14 07:58:28 INFO: [GenomePreparation 64152656348] Genome 'Genome3.0' has sequences below minimum, creating filtered copy.\n",
      "2021-03-14 07:58:34 INFO: [GenomePreparation 64152656348] Genome 'Genome45.0' has sequences below minimum, creating filtered copy.\n",
      "2021-03-14 07:58:37 INFO: [GenomePreparation 64152656348] Genome 'Genome26.0' has sequences below minimum, creating filtered copy.\n",
      "2021-03-14 07:58:38 INFO: [GenomePreparation 64152656348] Simulating reads using ReadSimulationArtIllumina readsimulator...\n",
      "2021-03-14 07:58:38 INFO: [GenomePreparation 64152656348] Simulating reads from Genome15.0: '/output/source_genomes/GCA_905212815.1_SRR7721869-mag-bin.13_genomic.fna'\n",
      "2021-03-14 07:58:38 INFO: [GenomePreparation 64152656348] Simulating reads from Genome14.0: '/tmp/tmpNYO7x_/tmpEi8tdL'\n",
      "2021-03-14 07:58:38 INFO: [GenomePreparation 64152656348] Simulating reads from Genome37.0: '/output/source_genomes/GCA_004710095.1_PDT000000914.4_genomic.fna'\n",
      "2021-03-14 07:58:38 INFO: [GenomePreparation 64152656348] Simulating reads from Genome33.0: '/output/source_genomes/GCA_000183845.1_ASM18384v1_genomic.fna'\n",
      "2021-03-14 07:58:38 INFO: [GenomePreparation 64152656348] Simulating reads from Genome34.0: '/output/source_genomes/GCA_905206855.1_ERR1600702-mag-bin.11_genomic.fna'\n",
      "2021-03-14 07:58:38 INFO: [GenomePreparation 64152656348] Simulating reads from Genome43.0: '/output/source_genomes/GCA_000168455.1_ASM16845v1_genomic.fna'\n",
      "2021-03-14 07:58:38 INFO: [GenomePreparation 64152656348] Simulating reads from Genome46.0: '/output/source_genomes/GCA_002018185.1_ASM201818v1_genomic.fna'\n",
      "2021-03-14 07:58:38 INFO: [GenomePreparation 64152656348] Simulating reads from Genome17.0: '/output/source_genomes/GCA_012214265.1_PDT000306620.2_genomic.fna'\n",
      "2021-03-14 07:58:38 INFO: [GenomePreparation 64152656348] Simulating reads from Genome28.0: '/output/source_genomes/GCA_012265825.1_PDT000281741.2_genomic.fna'\n",
      "2021-03-14 07:58:38 INFO: [GenomePreparation 64152656348] Simulating reads from Genome16.0: '/output/source_genomes/GCA_003683855.1_PDT000362377.1_genomic.fna'\n",
      "2021-03-14 07:58:38 INFO: [GenomePreparation 64152656348] Simulating reads from Genome23.0: '/output/source_genomes/GCA_012229505.1_PDT000300822.2_genomic.fna'\n",
      "2021-03-14 07:58:38 INFO: [GenomePreparation 64152656348] Simulating reads from Genome1.0: '/output/source_genomes/GCA_000436855.1_MGS203_genomic.fna'\n",
      "2021-03-14 07:58:38 INFO: [GenomePreparation 64152656348] Simulating reads from Genome4.0: '/output/source_genomes/GCA_000830415.1_ASM83041v1_genomic.fna'\n",
      "2021-03-14 07:58:38 INFO: [GenomePreparation 64152656348] Simulating reads from Genome48.0: '/output/source_genomes/GCA_000508465.1_CloBut_DORA_1_velvet_1.1_genomic.fna'\n",
      "2021-03-14 07:58:38 INFO: [GenomePreparation 64152656348] Simulating reads from Genome3.0: '/tmp/tmpNYO7x_/tmp242Iaf'\n",
      "2021-03-14 07:58:38 INFO: [GenomePreparation 64152656348] Simulating reads from Genome39.0: '/output/source_genomes/GCA_012211845.1_PDT000307441.2_genomic.fna'\n",
      "2021-03-14 07:58:38 INFO: [GenomePreparation 64152656348] Simulating reads from Genome20.0: '/output/source_genomes/GCA_015058435.1_ASM1505843v1_genomic.fna'\n",
      "2021-03-14 07:58:38 INFO: [GenomePreparation 64152656348] Simulating reads from Genome38.0: '/output/source_genomes/GCA_013012765.1_PDT000430806.1_genomic.fna'\n",
      "2021-03-14 07:58:38 INFO: [GenomePreparation 64152656348] Simulating reads from Genome6.0: '/output/source_genomes/GCA_015803995.1_PDT000350817.2_genomic.fna'\n",
      "2021-03-14 07:58:38 INFO: [GenomePreparation 64152656348] Simulating reads from Genome45.0: '/tmp/tmpNYO7x_/tmpMPIq2w'\n",
      "2021-03-14 07:58:38 INFO: [GenomePreparation 64152656348] Simulating reads from Genome7.0: '/output/source_genomes/GCA_013039825.1_PDT000415768.1_genomic.fna'\n",
      "2021-03-14 07:58:38 INFO: [GenomePreparation 64152656348] Simulating reads from Genome10.0: '/output/source_genomes/GCA_004534905.1_PDT000481402.1_genomic.fna'\n",
      "2021-03-14 07:58:38 INFO: [GenomePreparation 64152656348] Simulating reads from Genome26.0: '/tmp/tmpNYO7x_/tmp1oVHK4'\n",
      "2021-03-14 07:58:38 INFO: [GenomePreparation 64152656348] Simulating reads from Genome44.0: '/output/source_genomes/GCA_011854965.1_PDT000281620.4_genomic.fna'\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.09\n",
      "\n",
      "The random seed for the run: 854832072\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0795003X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome37.01.fq\n",
      "\t the 2nd reads: /tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome37.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome37.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.11\n",
      "\n",
      "The random seed for the run: 1992431107\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0500155X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome15.01.fq\n",
      "\t the 2nd reads: /tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome15.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome15.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.27\n",
      "\n",
      "The random seed for the run: 1923521921\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.210836X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome14.01.fq\n",
      "\t the 2nd reads: /tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome14.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome14.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.27\n",
      "\n",
      "The random seed for the run: 689939719\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.171611X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome33.01.fq\n",
      "\t the 2nd reads: /tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome33.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome33.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.31\n",
      "\n",
      "The random seed for the run: 2102807139\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.347739X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome34.01.fq\n",
      "\t the 2nd reads: /tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome34.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome34.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.27\n",
      "\n",
      "The random seed for the run: 1168110496\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.210783X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome28.01.fq\n",
      "\t the 2nd reads: /tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome28.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome28.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.48\n",
      "\n",
      "The random seed for the run: 199492375\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.508933X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome17.01.fq\n",
      "\t the 2nd reads: /tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome17.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome17.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.1\n",
      "\n",
      "The random seed for the run: 1208022829\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0704941X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome1.01.fq\n",
      "\t the 2nd reads: /tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome1.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome1.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.17\n",
      "\n",
      "The random seed for the run: 840935803\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0970512X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome23.01.fq\n",
      "\t the 2nd reads: /tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome23.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome23.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.07\n",
      "\n",
      "The random seed for the run: 387153331\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0047818X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome3.01.fq\n",
      "\t the 2nd reads: /tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome3.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome3.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.27\n",
      "\n",
      "The random seed for the run: 1937252460\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.241583X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome4.01.fq\n",
      "\t the 2nd reads: /tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome4.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome4.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.18\n",
      "\n",
      "The random seed for the run: 458448008\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.169059X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome48.01.fq\n",
      "\t the 2nd reads: /tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome48.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome48.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.09\n",
      "\n",
      "The random seed for the run: 565794829\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0216062X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome45.01.fq\n",
      "\t the 2nd reads: /tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome45.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome45.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.28\n",
      "\n",
      "The random seed for the run: 526849290\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.208097X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome39.01.fq\n",
      "\t the 2nd reads: /tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome39.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome39.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.06\n",
      "\n",
      "The random seed for the run: 2139276349\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.00322151X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome10.01.fq\n",
      "\t the 2nd reads: /tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome10.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome10.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 1.05\n",
      "\n",
      "The random seed for the run: 1940270192\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            2.32318X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome43.01.fq\n",
      "\t the 2nd reads: /tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome43.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome43.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.22\n",
      "\n",
      "The random seed for the run: 2090645473\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.153118X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome7.01.fq\n",
      "\t the 2nd reads: /tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome7.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome7.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.18\n",
      "\n",
      "The random seed for the run: 1203576411\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.123406X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome26.01.fq\n",
      "\t the 2nd reads: /tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome26.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome26.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.76\n",
      "\n",
      "The random seed for the run: 918751087\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.832193X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome38.01.fq\n",
      "\t the 2nd reads: /tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome38.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome38.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 1.54\n",
      "\n",
      "The random seed for the run: 783731559\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            3.70663X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome46.01.fq\n",
      "\t the 2nd reads: /tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome46.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome46.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 1.34\n",
      "\n",
      "The random seed for the run: 1268568432\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            2.66351X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome16.01.fq\n",
      "\t the 2nd reads: /tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome16.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome16.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 1.46\n",
      "\n",
      "The random seed for the run: 597928115\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            2.0247X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome6.01.fq\n",
      "\t the 2nd reads: /tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome6.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome6.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 3.37\n",
      "\n",
      "The random seed for the run: 26380989\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            5.18065X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome20.01.fq\n",
      "\t the 2nd reads: /tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome20.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome20.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 3.67\n",
      "\n",
      "The random seed for the run: 2029410300\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            4.97728X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome44.01.fq\n",
      "\t the 2nd reads: /tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome44.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpNYO7x_/2021.03.14_07.56.49_sample_0/reads/Genome44.0.sam\n",
      "\n",
      "2021-03-14 07:58:43 INFO: [GenomePreparation 64152656348] Simulating reads finished\n",
      "2021-03-14 07:58:49 INFO: [MetagenomeSimulationPipeline] Generate gold standard assembly\n",
      "2021-03-14 07:58:49 INFO: [MetadataReader 38994325909] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-14 07:59:16 INFO: [MetagenomeSimulationPipeline] Generate pooled strains gold standard assembly\n",
      "2021-03-14 07:59:16 INFO: [MetadataReader 34289848075] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-14 07:59:16 INFO: [MetagenomeSimulationPipeline] Samples used for pooled assembly: '2021.03.14_07.56.49_sample_0'\n",
      "2021-03-14 07:59:16 INFO: [Validator 5358345304] Creating pooled gold standard\n",
      "2021-03-14 07:59:16 INFO: [Validator 5358345304] Merging bam files.\n",
      "2021-03-14 07:59:16 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_07.56.49_sample_0/bam/Genome15.0.bam'\n",
      "2021-03-14 07:59:16 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_07.56.49_sample_0/bam/Genome14.0.bam'\n",
      "2021-03-14 07:59:16 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_07.56.49_sample_0/bam/Genome37.0.bam'\n",
      "2021-03-14 07:59:16 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_07.56.49_sample_0/bam/Genome33.0.bam'\n",
      "2021-03-14 07:59:16 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_07.56.49_sample_0/bam/Genome34.0.bam'\n",
      "2021-03-14 07:59:16 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_07.56.49_sample_0/bam/Genome43.0.bam'\n",
      "2021-03-14 07:59:16 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_07.56.49_sample_0/bam/Genome46.0.bam'\n",
      "2021-03-14 07:59:16 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_07.56.49_sample_0/bam/Genome17.0.bam'\n",
      "2021-03-14 07:59:16 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_07.56.49_sample_0/bam/Genome28.0.bam'\n",
      "2021-03-14 07:59:16 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_07.56.49_sample_0/bam/Genome16.0.bam'\n",
      "2021-03-14 07:59:16 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_07.56.49_sample_0/bam/Genome45.0.bam'\n",
      "2021-03-14 07:59:16 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_07.56.49_sample_0/bam/Genome1.0.bam'\n",
      "2021-03-14 07:59:16 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_07.56.49_sample_0/bam/Genome4.0.bam'\n",
      "2021-03-14 07:59:16 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_07.56.49_sample_0/bam/Genome48.0.bam'\n",
      "2021-03-14 07:59:16 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_07.56.49_sample_0/bam/Genome3.0.bam'\n",
      "2021-03-14 07:59:16 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_07.56.49_sample_0/bam/Genome39.0.bam'\n",
      "2021-03-14 07:59:16 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_07.56.49_sample_0/bam/Genome20.0.bam'\n",
      "2021-03-14 07:59:16 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_07.56.49_sample_0/bam/Genome38.0.bam'\n",
      "2021-03-14 07:59:16 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_07.56.49_sample_0/bam/Genome6.0.bam'\n",
      "2021-03-14 07:59:16 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_07.56.49_sample_0/bam/Genome23.0.bam'\n",
      "2021-03-14 07:59:16 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_07.56.49_sample_0/bam/Genome7.0.bam'\n",
      "2021-03-14 07:59:16 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_07.56.49_sample_0/bam/Genome10.0.bam'\n",
      "2021-03-14 07:59:16 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_07.56.49_sample_0/bam/Genome26.0.bam'\n",
      "2021-03-14 07:59:16 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_07.56.49_sample_0/bam/Genome44.0.bam'\n",
      "2021-03-14 07:59:42 INFO: [MetagenomeSimulationPipeline] Anonymize Data\n",
      "2021-03-14 07:59:42 INFO: [FastaAnonymizer] Interweave shuffle and anonymize\n",
      "2021-03-14 08:00:49 INFO: [MetadataReader 55801910510] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-14 08:00:49 INFO: [MetadataReader 35298217545] Reading file: '/output/internal/meta_data.tsv'\n",
      "2021-03-14 08:00:49 INFO: [MetadataReader 50273713801] Reading file: '/tmp/tmpNYO7x_/tmpBxSwyl'\n",
      "2021-03-14 08:00:52 INFO: [FastaAnonymizer 20754120427] Shuffle and anonymize '/tmp/tmpNYO7x_/tmpAcl5u1'\n",
      "2021-03-14 08:00:59 INFO: [MetadataReader 64547429207] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-14 08:01:00 INFO: [MetadataReader 10989772577] Reading file: '/output/internal/meta_data.tsv'\n",
      "2021-03-14 08:01:00 INFO: [MetadataReader 53809671431] Reading file: '/tmp/tmpNYO7x_/read_start_positionsji0wJz'\n",
      "2021-03-14 08:01:01 INFO: [MetadataReader 18168782768] Reading file: '/tmp/tmpNYO7x_/tmpjYs3o2'\n",
      "2021-03-14 08:01:09 INFO: [FastaAnonymizer 92324153505] Shuffle and anonymize '/tmp/tmpNYO7x_/tmpjGsHDN'\n",
      "2021-03-14 08:01:16 INFO: [MetadataReader 83768748887] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-14 08:01:17 INFO: [MetadataReader 11249978774] Reading file: '/output/internal/meta_data.tsv'\n",
      "2021-03-14 08:01:17 INFO: [MetadataReader 66861377693] Reading file: '/tmp/tmpNYO7x_/read_start_positionskTC9p_'\n",
      "2021-03-14 08:01:19 INFO: [MetadataReader 8017839927] Reading file: '/tmp/tmpNYO7x_/tmpQdFlWs'\n",
      "2021-03-14 08:01:27 INFO: [MetagenomeSimulationPipeline] Compress Data\n",
      "2021-03-14 08:01:27 INFO: [Compress 59297227130] Compressing 'tmpIs0GCm'\n",
      "2021-03-14 08:01:27 INFO: [Compress 16475349584] Compressing 'anonymous_gs_mappingxnwpfp'\n",
      "2021-03-14 08:01:27 INFO: [Compress 99275954537] Compressing 'tmpo4KEbk'\n",
      "2021-03-14 08:01:27 INFO: [Compress 18355602539] Compressing 'anonymous_gsa_mappingo1E3xY'\n",
      "2021-03-14 08:01:27 INFO: [Compress 45959397057] Compressing 'tmp1SXUj1'\n",
      "2021-03-14 08:01:27 INFO: [Compress 2875833809] Compressing 'anonymous_gsa_pooled_mappingl16lyu'\n",
      "2021-03-14 08:01:27 INFO: [Compress 18355602539] Done compressing 'gsa_mapping.tsv.gz' in 0:00:00s.\n",
      "2021-03-14 08:01:27 INFO: [Compress 2875833809] Done compressing 'gsa_pooled_mapping.tsv.gz' in 0:00:00s.\n",
      "2021-03-14 08:01:28 INFO: [Compress 16475349584] Done compressing 'reads_mapping.tsv.gz' in 0:00:02s.\n",
      "2021-03-14 08:01:29 INFO: [Compress 45959397057] Done compressing 'anonymous_gsa_pooled.fasta.gz' in 0:00:02s.\n",
      "2021-03-14 08:01:29 INFO: [Compress 99275954537] Done compressing 'anonymous_gsa.fasta.gz' in 0:00:02s.\n",
      "2021-03-14 08:01:36 INFO: [Compress 59297227130] Done compressing 'anonymous_reads.fq.gz' in 0:00:09s.\n",
      "2021-03-14 08:01:36 INFO: [MetagenomeSimulationPipeline] Metagenome simulation finished\n",
      "simulating complexity_sim_10_genera_25_genomes\n",
      "[sudo] password for pathinformatics: 2021-03-14 08:01:38 INFO: [MetagenomeSimulationPipeline] Metagenome simulation starting\n",
      "2021-03-14 08:01:38 INFO: [MetagenomeSimulationPipeline] Validating Genomes\n",
      "2021-03-14 08:01:38 INFO: [MetadataReader] Reading file: '/input/genome_to_id.tsv'\n",
      "2021-03-14 08:01:58 INFO: [MetagenomeSimulationPipeline] Design Communities\n",
      "2021-03-14 08:01:58 INFO: [CommunityDesign] Drawing strains.\n",
      "2021-03-14 08:01:58 INFO: [MetadataReader 82380310288] Reading file: '/input/metadata.tsv'\n",
      "2021-03-14 08:01:58 INFO: [MetadataReader 25595780401] Reading file: '/input/genome_to_id.tsv'\n",
      "2021-03-14 08:01:59 INFO: [CommunityDesign] Validating raw sequence files!\n",
      "2021-03-14 08:02:20 INFO: [NcbiTaxonomy] Building taxonomy tree...\n",
      "2021-03-14 08:02:20 INFO: [NcbiTaxonomy] Reading 'nodes' file:\t'/tmp/tmpDMJub8/NCBI/nodes.dmp'\n",
      "2021-03-14 08:02:32 INFO: [NcbiTaxonomy] Reading 'names' file:\t'/tmp/tmpDMJub8/NCBI/names.dmp'\n",
      "2021-03-14 08:02:33 INFO: [NcbiTaxonomy] Reading 'merged' file:\t'/tmp/tmpDMJub8/NCBI/merged.dmp'\n",
      "2021-03-14 08:02:33 INFO: [NcbiTaxonomy] Done (13.0s)\n",
      "2021-03-14 08:02:34 INFO: [MetagenomeSimulationPipeline] Move Genomes\n",
      "2021-03-14 08:02:35 INFO: [MetagenomeSimulationPipeline] Read simulation\n",
      "2021-03-14 08:02:35 INFO: [GenomePreparation 70514612869] Using 'mbarc' error profile.\n",
      "2021-03-14 08:02:35 INFO: [GenomePreparation 70514612869] Reading distribution file\n",
      "2021-03-14 08:02:35 INFO: [GenomePreparation 70514612869] Reading genome location file\n",
      "2021-03-14 08:02:39 INFO: [GenomePreparation 70514612869] Genome 'Genome25.0' has sequences below minimum, creating filtered copy.\n",
      "2021-03-14 08:02:41 INFO: [GenomePreparation 70514612869] Genome 'Genome16.0' has sequences below minimum, creating filtered copy.\n",
      "2021-03-14 08:02:42 INFO: [GenomePreparation 70514612869] Genome 'Genome19.0' has sequences below minimum, creating filtered copy.\n",
      "2021-03-14 08:02:54 INFO: [GenomePreparation 70514612869] Genome 'Genome10.0' has sequences below minimum, creating filtered copy.\n",
      "2021-03-14 08:02:56 INFO: [GenomePreparation 70514612869] Simulating reads using ReadSimulationArtIllumina readsimulator...\n",
      "2021-03-14 08:02:56 INFO: [GenomePreparation 70514612869] Simulating reads from Genome15.0: '/output/source_genomes/GCA_001597655.1_ASM159765v1_genomic.fna'\n",
      "2021-03-14 08:02:56 INFO: [GenomePreparation 70514612869] Simulating reads from Genome14.0: '/output/source_genomes/GCA_000435235.1_MGS119_genomic.fna'\n",
      "2021-03-14 08:02:56 INFO: [GenomePreparation 70514612869] Simulating reads from Genome11.0: '/output/source_genomes/GCA_000268165.1_ASM26816v1_genomic.fna'\n",
      "2021-03-14 08:02:56 INFO: [GenomePreparation 70514612869] Simulating reads from Genome24.0: '/output/source_genomes/GCA_000555645.1_Stap_aure_M0406_V1_genomic.fna'\n",
      "2021-03-14 08:02:56 INFO: [GenomePreparation 70514612869] Simulating reads from Genome12.0: '/output/source_genomes/GCA_004710765.1_PDT000030589.4_genomic.fna'\n",
      "2021-03-14 08:02:56 INFO: [GenomePreparation 70514612869] Simulating reads from Genome25.0: '/tmp/tmpCQoimW/tmpWlH9Lr'\n",
      "2021-03-14 08:02:56 INFO: [GenomePreparation 70514612869] Simulating reads from Genome18.0: '/output/source_genomes/GCA_013013995.1_PDT000430796.1_genomic.fna'\n",
      "2021-03-14 08:02:56 INFO: [GenomePreparation 70514612869] Simulating reads from Genome16.0: '/tmp/tmpCQoimW/tmph6Ua6T'\n",
      "2021-03-14 08:02:56 INFO: [GenomePreparation 70514612869] Simulating reads from Genome19.0: '/tmp/tmpCQoimW/tmp2PhzHb'\n",
      "2021-03-14 08:02:56 INFO: [GenomePreparation 70514612869] Simulating reads from Genome2.0: '/output/source_genomes/GCA_000567965.1_ASM56796v1_genomic.fna'\n",
      "2021-03-14 08:02:56 INFO: [GenomePreparation 70514612869] Simulating reads from Genome1.0: '/output/source_genomes/GCA_000953455.1_Ppseudo_Pac_genomic.fna'\n",
      "2021-03-14 08:02:56 INFO: [GenomePreparation 70514612869] Simulating reads from Genome21.0: '/output/source_genomes/GCA_000438035.1_MGS6_genomic.fna'\n",
      "2021-03-14 08:02:56 INFO: [GenomePreparation 70514612869] Simulating reads from Genome4.0: '/output/source_genomes/GCA_002311215.1_ASM231121v1_genomic.fna'\n",
      "2021-03-14 08:02:56 INFO: [GenomePreparation 70514612869] Simulating reads from Genome5.0: '/output/source_genomes/GCA_000433375.1_MGS32_genomic.fna'\n",
      "2021-03-14 08:02:56 INFO: [GenomePreparation 70514612869] Simulating reads from Genome3.0: '/output/source_genomes/GCA_000268085.1_ASM26808v1_genomic.fna'\n",
      "2021-03-14 08:02:56 INFO: [GenomePreparation 70514612869] Simulating reads from Genome20.0: '/output/source_genomes/GCA_011845705.1_PDT000606249.2_genomic.fna'\n",
      "2021-03-14 08:02:56 INFO: [GenomePreparation 70514612869] Simulating reads from Genome6.0: '/output/source_genomes/GCA_000436455.1_MGS132_genomic.fna'\n",
      "2021-03-14 08:02:56 INFO: [GenomePreparation 70514612869] Simulating reads from Genome23.0: '/output/source_genomes/GCA_011855365.1_PDT000259510.3_genomic.fna'\n",
      "2021-03-14 08:02:56 INFO: [GenomePreparation 70514612869] Simulating reads from Genome9.0: '/output/source_genomes/GCA_013014285.1_PDT000430776.1_genomic.fna'\n",
      "2021-03-14 08:02:56 INFO: [GenomePreparation 70514612869] Simulating reads from Genome7.0: '/output/source_genomes/GCA_016955115.1_PDT000971190.1_genomic.fna'\n",
      "2021-03-14 08:02:56 INFO: [GenomePreparation 70514612869] Simulating reads from Genome13.0: '/output/source_genomes/GCA_000165775.3_ASM16577v3_genomic.fna'\n",
      "2021-03-14 08:02:56 INFO: [GenomePreparation 70514612869] Simulating reads from Genome10.0: '/tmp/tmpCQoimW/tmpxmQ2mg'\n",
      "2021-03-14 08:02:56 INFO: [GenomePreparation 70514612869] Simulating reads from Genome8.0: '/output/source_genomes/GCA_016954835.1_PDT000971228.1_genomic.fna'\n",
      "2021-03-14 08:02:56 INFO: [GenomePreparation 70514612869] Simulating reads from Genome22.0: '/output/source_genomes/GCA_001311525.1_ASM131152v1_genomic.fna'\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.07\n",
      "\n",
      "The random seed for the run: 573380147\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0545426X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome12.01.fq\n",
      "\t the 2nd reads: /tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome12.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome12.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.1\n",
      "\n",
      "The random seed for the run: 65634987\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0490757X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome11.01.fq\n",
      "\t the 2nd reads: /tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome11.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome11.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.13\n",
      "\n",
      "The random seed for the run: 1847505585\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.307482X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome14.01.fq\n",
      "\t the 2nd reads: /tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome14.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome14.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.12\n",
      "\n",
      "The random seed for the run: 1294107793\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.122161X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome24.01.fq\n",
      "\t the 2nd reads: /tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome24.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome24.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.16\n",
      "\n",
      "The random seed for the run: 1703660943\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0630359X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome16.01.fq\n",
      "\t the 2nd reads: /tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome16.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome16.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.08\n",
      "\n",
      "The random seed for the run: 1661055960\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.00285024X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome19.01.fq\n",
      "\t the 2nd reads: /tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome19.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome19.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.09\n",
      "\n",
      "The random seed for the run: 49814198\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0185878X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome21.01.fq\n",
      "\t the 2nd reads: /tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome21.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome21.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.07\n",
      "\n",
      "The random seed for the run: 1557933274\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.105688X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome4.01.fq\n",
      "\t the 2nd reads: /tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome4.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome4.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.31\n",
      "\n",
      "The random seed for the run: 2129338277\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.726777X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome25.01.fq\n",
      "\t the 2nd reads: /tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome25.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome25.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.09\n",
      "\n",
      "The random seed for the run: 1168871082\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.00336364X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome3.01.fq\n",
      "\t the 2nd reads: /tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome3.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome3.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.26\n",
      "\n",
      "The random seed for the run: 1577639001\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.417631X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome2.01.fq\n",
      "\t the 2nd reads: /tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome2.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome2.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.09\n",
      "\n",
      "The random seed for the run: 1961570421\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.002418X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome20.01.fq\n",
      "\t the 2nd reads: /tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome20.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome20.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.17\n",
      "\n",
      "The random seed for the run: 923704085\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.110309X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome6.01.fq\n",
      "\t the 2nd reads: /tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome6.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome6.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.66\n",
      "\n",
      "The random seed for the run: 540968737\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.751266X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome15.01.fq\n",
      "\t the 2nd reads: /tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome15.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome15.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.14\n",
      "\n",
      "The random seed for the run: 1550755105\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0600215X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome23.01.fq\n",
      "\t the 2nd reads: /tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome23.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome23.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.06\n",
      "\n",
      "The random seed for the run: 854832072\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0111817X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome13.01.fq\n",
      "\t the 2nd reads: /tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome13.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome13.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.06\n",
      "\n",
      "The random seed for the run: 1940270192\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0356466X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome22.01.fq\n",
      "\t the 2nd reads: /tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome22.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome22.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.11\n",
      "\n",
      "The random seed for the run: 2102807139\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0252913X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome8.01.fq\n",
      "\t the 2nd reads: /tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome8.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome8.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 1.07\n",
      "\n",
      "The random seed for the run: 1329514041\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            1.34685X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome5.01.fq\n",
      "\t the 2nd reads: /tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome5.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome5.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.86\n",
      "\n",
      "The random seed for the run: 689939719\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            2.51685X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome10.01.fq\n",
      "\t the 2nd reads: /tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome10.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome10.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 1.52\n",
      "\n",
      "The random seed for the run: 1992431107\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            1.68567X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome9.01.fq\n",
      "\t the 2nd reads: /tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome9.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome9.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 2.1\n",
      "\n",
      "The random seed for the run: 1923521921\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            2.70959X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome7.01.fq\n",
      "\t the 2nd reads: /tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome7.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome7.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 3.32\n",
      "\n",
      "The random seed for the run: 1713385206\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            4.82565X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome1.01.fq\n",
      "\t the 2nd reads: /tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome1.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome1.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 4.59\n",
      "\n",
      "The random seed for the run: 170568346\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            6.74273X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome18.01.fq\n",
      "\t the 2nd reads: /tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome18.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpCQoimW/2021.03.14_08.01.38_sample_0/reads/Genome18.0.sam\n",
      "\n",
      "2021-03-14 08:03:01 INFO: [GenomePreparation 70514612869] Simulating reads finished\n",
      "2021-03-14 08:03:08 INFO: [MetagenomeSimulationPipeline] Generate gold standard assembly\n",
      "2021-03-14 08:03:08 INFO: [MetadataReader 64136302938] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-14 08:03:32 INFO: [MetagenomeSimulationPipeline] Generate pooled strains gold standard assembly\n",
      "2021-03-14 08:03:32 INFO: [MetadataReader 58498923446] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-14 08:03:32 INFO: [MetagenomeSimulationPipeline] Samples used for pooled assembly: '2021.03.14_08.01.38_sample_0'\n",
      "2021-03-14 08:03:32 INFO: [Validator 48443846240] Creating pooled gold standard\n",
      "2021-03-14 08:03:32 INFO: [Validator 48443846240] Merging bam files.\n",
      "2021-03-14 08:03:32 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_08.01.38_sample_0/bam/Genome15.0.bam'\n",
      "2021-03-14 08:03:32 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_08.01.38_sample_0/bam/Genome14.0.bam'\n",
      "2021-03-14 08:03:32 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_08.01.38_sample_0/bam/Genome11.0.bam'\n",
      "2021-03-14 08:03:32 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_08.01.38_sample_0/bam/Genome24.0.bam'\n",
      "2021-03-14 08:03:32 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_08.01.38_sample_0/bam/Genome12.0.bam'\n",
      "2021-03-14 08:03:32 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_08.01.38_sample_0/bam/Genome25.0.bam'\n",
      "2021-03-14 08:03:32 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_08.01.38_sample_0/bam/Genome18.0.bam'\n",
      "2021-03-14 08:03:32 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_08.01.38_sample_0/bam/Genome16.0.bam'\n",
      "2021-03-14 08:03:32 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_08.01.38_sample_0/bam/Genome19.0.bam'\n",
      "2021-03-14 08:03:32 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_08.01.38_sample_0/bam/Genome2.0.bam'\n",
      "2021-03-14 08:03:32 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_08.01.38_sample_0/bam/Genome1.0.bam'\n",
      "2021-03-14 08:03:32 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_08.01.38_sample_0/bam/Genome21.0.bam'\n",
      "2021-03-14 08:03:32 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_08.01.38_sample_0/bam/Genome4.0.bam'\n",
      "2021-03-14 08:03:32 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_08.01.38_sample_0/bam/Genome5.0.bam'\n",
      "2021-03-14 08:03:32 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_08.01.38_sample_0/bam/Genome3.0.bam'\n",
      "2021-03-14 08:03:32 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_08.01.38_sample_0/bam/Genome20.0.bam'\n",
      "2021-03-14 08:03:32 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_08.01.38_sample_0/bam/Genome6.0.bam'\n",
      "2021-03-14 08:03:32 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_08.01.38_sample_0/bam/Genome23.0.bam'\n",
      "2021-03-14 08:03:32 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_08.01.38_sample_0/bam/Genome9.0.bam'\n",
      "2021-03-14 08:03:32 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_08.01.38_sample_0/bam/Genome7.0.bam'\n",
      "2021-03-14 08:03:32 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_08.01.38_sample_0/bam/Genome13.0.bam'\n",
      "2021-03-14 08:03:32 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_08.01.38_sample_0/bam/Genome10.0.bam'\n",
      "2021-03-14 08:03:32 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_08.01.38_sample_0/bam/Genome8.0.bam'\n",
      "2021-03-14 08:03:32 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_08.01.38_sample_0/bam/Genome22.0.bam'\n",
      "2021-03-14 08:03:56 INFO: [MetagenomeSimulationPipeline] Anonymize Data\n",
      "2021-03-14 08:03:56 INFO: [FastaAnonymizer] Interweave shuffle and anonymize\n",
      "2021-03-14 08:05:04 INFO: [MetadataReader 72802808290] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-14 08:05:05 INFO: [MetadataReader 47315449936] Reading file: '/output/internal/meta_data.tsv'\n",
      "2021-03-14 08:05:05 INFO: [MetadataReader 65300095466] Reading file: '/tmp/tmpCQoimW/tmpo80c9z'\n",
      "2021-03-14 08:05:08 INFO: [FastaAnonymizer 43075827506] Shuffle and anonymize '/tmp/tmpCQoimW/tmpWIPhaf'\n",
      "2021-03-14 08:05:14 INFO: [MetadataReader 71873526544] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-14 08:05:15 INFO: [MetadataReader 45105173502] Reading file: '/output/internal/meta_data.tsv'\n",
      "2021-03-14 08:05:15 INFO: [MetadataReader 84613406616] Reading file: '/tmp/tmpCQoimW/read_start_positions1KmCf2'\n",
      "2021-03-14 08:05:17 INFO: [MetadataReader 10674074437] Reading file: '/tmp/tmpCQoimW/tmpCBjMip'\n",
      "2021-03-14 08:05:40 INFO: [FastaAnonymizer 86623674311] Shuffle and anonymize '/tmp/tmpCQoimW/tmpyKb7bH'\n",
      "2021-03-14 08:05:46 INFO: [MetadataReader 15345066727] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-14 08:05:47 INFO: [MetadataReader 77753400540] Reading file: '/output/internal/meta_data.tsv'\n",
      "2021-03-14 08:05:47 INFO: [MetadataReader 12266665971] Reading file: '/tmp/tmpCQoimW/read_start_positionsgdoU8f'\n",
      "2021-03-14 08:05:49 INFO: [MetadataReader 42355547664] Reading file: '/tmp/tmpCQoimW/tmp7VR8IQ'\n",
      "2021-03-14 08:06:12 INFO: [MetagenomeSimulationPipeline] Compress Data\n",
      "2021-03-14 08:06:12 INFO: [Compress 67194535592] Compressing 'tmp3FBuSe'\n",
      "2021-03-14 08:06:12 INFO: [Compress 26123705258] Compressing 'anonymous_gs_mappingZ5UYcA'\n",
      "2021-03-14 08:06:12 INFO: [Compress 63106805185] Compressing 'tmpHA190Y'\n",
      "2021-03-14 08:06:12 INFO: [Compress 88581561846] Compressing 'anonymous_gsa_mappingOaqXUU'\n",
      "2021-03-14 08:06:12 INFO: [Compress 57327298662] Compressing 'tmpaeoJdm'\n",
      "2021-03-14 08:06:12 INFO: [Compress 4707750833] Compressing 'anonymous_gsa_pooled_mappingwOvBpX'\n",
      "2021-03-14 08:06:12 INFO: [Compress 4707750833] Done compressing 'gsa_pooled_mapping.tsv.gz' in 0:00:00s.\n",
      "2021-03-14 08:06:12 INFO: [Compress 88581561846] Done compressing 'gsa_mapping.tsv.gz' in 0:00:00s.\n",
      "2021-03-14 08:06:14 INFO: [Compress 26123705258] Done compressing 'reads_mapping.tsv.gz' in 0:00:02s.\n",
      "2021-03-14 08:06:14 INFO: [Compress 63106805185] Done compressing 'anonymous_gsa.fasta.gz' in 0:00:02s.\n",
      "2021-03-14 08:06:14 INFO: [Compress 57327298662] Done compressing 'anonymous_gsa_pooled.fasta.gz' in 0:00:02s.\n",
      "2021-03-14 08:06:21 INFO: [Compress 67194535592] Done compressing 'anonymous_reads.fq.gz' in 0:00:09s.\n",
      "2021-03-14 08:06:21 INFO: [MetagenomeSimulationPipeline] Metagenome simulation finished\n",
      "simulating complexity_sim_1_genera_50_genomes\n",
      "[sudo] password for pathinformatics: 2021-03-14 08:06:23 INFO: [MetagenomeSimulationPipeline] Metagenome simulation starting\n",
      "2021-03-14 08:06:23 INFO: [MetagenomeSimulationPipeline] Validating Genomes\n",
      "2021-03-14 08:06:23 INFO: [MetadataReader] Reading file: '/input/genome_to_id.tsv'\n",
      "2021-03-14 08:07:17 INFO: [MetagenomeSimulationPipeline] Design Communities\n",
      "2021-03-14 08:07:17 INFO: [CommunityDesign] Drawing strains.\n",
      "2021-03-14 08:07:17 INFO: [MetadataReader 82380310288] Reading file: '/input/metadata.tsv'\n",
      "2021-03-14 08:07:17 INFO: [MetadataReader 89885674421] Reading file: '/input/genome_to_id.tsv'\n",
      "2021-03-14 08:07:17 INFO: [CommunityDesign] Validating raw sequence files!\n",
      "2021-03-14 08:07:44 INFO: [NcbiTaxonomy] Building taxonomy tree...\n",
      "2021-03-14 08:07:44 INFO: [NcbiTaxonomy] Reading 'nodes' file:\t'/tmp/tmpvw_iSI/NCBI/nodes.dmp'\n",
      "2021-03-14 08:07:55 INFO: [NcbiTaxonomy] Reading 'names' file:\t'/tmp/tmpvw_iSI/NCBI/names.dmp'\n",
      "2021-03-14 08:07:56 INFO: [NcbiTaxonomy] Reading 'merged' file:\t'/tmp/tmpvw_iSI/NCBI/merged.dmp'\n",
      "2021-03-14 08:07:56 INFO: [NcbiTaxonomy] Done (13.0s)\n",
      "2021-03-14 08:07:56 INFO: [MetagenomeSimulationPipeline] Move Genomes\n",
      "2021-03-14 08:07:58 INFO: [MetagenomeSimulationPipeline] Read simulation\n",
      "2021-03-14 08:07:58 INFO: [GenomePreparation 64152656348] Using 'mbarc' error profile.\n",
      "2021-03-14 08:07:58 INFO: [GenomePreparation 64152656348] Reading distribution file\n",
      "2021-03-14 08:07:58 INFO: [GenomePreparation 64152656348] Reading genome location file\n",
      "2021-03-14 08:08:01 INFO: [GenomePreparation 64152656348] Genome 'Genome29.0' has sequences below minimum, creating filtered copy.\n",
      "2021-03-14 08:08:25 INFO: [GenomePreparation 64152656348] Genome 'Genome22.0' has sequences below minimum, creating filtered copy.\n",
      "2021-03-14 08:08:25 INFO: [GenomePreparation 64152656348] Simulating reads using ReadSimulationArtIllumina readsimulator...\n",
      "2021-03-14 08:08:25 INFO: [GenomePreparation 64152656348] Simulating reads from Genome15.0: '/output/source_genomes/GCA_012215085.1_PDT000305495.2_genomic.fna'\n",
      "2021-03-14 08:08:25 INFO: [GenomePreparation 64152656348] Simulating reads from Genome33.0: '/output/source_genomes/GCA_013049785.1_PDT000281772.4_genomic.fna'\n",
      "2021-03-14 08:08:25 INFO: [GenomePreparation 64152656348] Simulating reads from Genome29.0: '/tmp/tmpA3XNm0/tmp6aWjcF'\n",
      "2021-03-14 08:08:25 INFO: [GenomePreparation 64152656348] Simulating reads from Genome42.0: '/output/source_genomes/GCA_012211765.1_PDT000307443.2_genomic.fna'\n",
      "2021-03-14 08:08:25 INFO: [GenomePreparation 64152656348] Simulating reads from Genome12.0: '/output/source_genomes/GCA_013014505.1_PDT000430688.1_genomic.fna'\n",
      "2021-03-14 08:08:25 INFO: [GenomePreparation 64152656348] Simulating reads from Genome25.0: '/output/source_genomes/GCA_012226925.1_PDT000312635.2_genomic.fna'\n",
      "2021-03-14 08:08:25 INFO: [GenomePreparation 64152656348] Simulating reads from Genome46.0: '/output/source_genomes/GCA_012213965.1_PDT000306473.2_genomic.fna'\n",
      "2021-03-14 08:08:25 INFO: [GenomePreparation 64152656348] Simulating reads from Genome18.0: '/output/source_genomes/GCA_016955555.1_PDT000971166.1_genomic.fna'\n",
      "2021-03-14 08:08:25 INFO: [GenomePreparation 64152656348] Simulating reads from Genome28.0: '/output/source_genomes/GCA_013049965.1_PDT000275054.4_genomic.fna'\n",
      "2021-03-14 08:08:25 INFO: [GenomePreparation 64152656348] Simulating reads from Genome30.0: '/output/source_genomes/GCA_013014205.1_PDT000430767.1_genomic.fna'\n",
      "2021-03-14 08:08:25 INFO: [GenomePreparation 64152656348] Simulating reads from Genome19.0: '/output/source_genomes/GCA_000754045.1_ASM75404v1_genomic.fna'\n",
      "2021-03-14 08:08:25 INFO: [GenomePreparation 64152656348] Simulating reads from Genome23.0: '/output/source_genomes/GCA_012247845.1_PDT000289861.2_genomic.fna'\n",
      "2021-03-14 08:08:25 INFO: [GenomePreparation 64152656348] Simulating reads from Genome43.0: '/output/source_genomes/GCA_013040725.1_PDT000415607.1_genomic.fna'\n",
      "2021-03-14 08:08:25 INFO: [GenomePreparation 64152656348] Simulating reads from Genome21.0: '/output/source_genomes/GCA_012224975.1_PDT000299829.2_genomic.fna'\n",
      "2021-03-14 08:08:25 INFO: [GenomePreparation 64152656348] Simulating reads from Genome48.0: '/output/source_genomes/GCA_016955755.1_PDT000971147.1_genomic.fna'\n",
      "2021-03-14 08:08:25 INFO: [GenomePreparation 64152656348] Simulating reads from Genome3.0: '/output/source_genomes/GCA_016955535.1_PDT000971170.1_genomic.fna'\n",
      "2021-03-14 08:08:25 INFO: [GenomePreparation 64152656348] Simulating reads from Genome39.0: '/output/source_genomes/GCA_012429225.1_PDT000234933.3_genomic.fna'\n",
      "2021-03-14 08:08:25 INFO: [GenomePreparation 64152656348] Simulating reads from Genome20.0: '/output/source_genomes/GCA_000215265.2_ASM21526v2_genomic.fna'\n",
      "2021-03-14 08:08:25 INFO: [GenomePreparation 64152656348] Simulating reads from Genome45.0: '/output/source_genomes/GCA_004231925.2_PDT000273066.5_genomic.fna'\n",
      "2021-03-14 08:08:25 INFO: [GenomePreparation 64152656348] Simulating reads from Genome7.0: '/output/source_genomes/GCA_016955185.1_PDT000971185.1_genomic.fna'\n",
      "2021-03-14 08:08:25 INFO: [GenomePreparation 64152656348] Simulating reads from Genome10.0: '/output/source_genomes/GCA_000529555.1_ISC41v1_genomic.fna'\n",
      "2021-03-14 08:08:25 INFO: [GenomePreparation 64152656348] Simulating reads from Genome44.0: '/output/source_genomes/GCA_013014345.1_PDT000430715.1_genomic.fna'\n",
      "2021-03-14 08:08:25 INFO: [GenomePreparation 64152656348] Simulating reads from Genome27.0: '/output/source_genomes/GCA_011845705.1_PDT000606249.2_genomic.fna'\n",
      "2021-03-14 08:08:25 INFO: [GenomePreparation 64152656348] Simulating reads from Genome22.0: '/tmp/tmpA3XNm0/tmpAFwHEN'\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.13\n",
      "\n",
      "The random seed for the run: 1940270192\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0646535X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome25.01.fq\n",
      "\t the 2nd reads: /tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome25.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome25.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.13\n",
      "\n",
      "The random seed for the run: 2102807139\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0573292X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome12.01.fq\n",
      "\t the 2nd reads: /tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome12.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome12.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.19\n",
      "\n",
      "The random seed for the run: 1992431107\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.124523X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome15.01.fq\n",
      "\t the 2nd reads: /tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome15.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome15.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.2\n",
      "\n",
      "The random seed for the run: 689939719\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.169234X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome42.01.fq\n",
      "\t the 2nd reads: /tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome42.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome42.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.23\n",
      "\n",
      "The random seed for the run: 783731559\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.171462X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome46.01.fq\n",
      "\t the 2nd reads: /tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome46.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome46.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.31\n",
      "\n",
      "The random seed for the run: 199492375\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.282798X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome18.01.fq\n",
      "\t the 2nd reads: /tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome18.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome18.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.15\n",
      "\n",
      "The random seed for the run: 840935803\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0406751X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome19.01.fq\n",
      "\t the 2nd reads: /tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome19.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome19.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.16\n",
      "\n",
      "The random seed for the run: 458448008\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0789267X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome21.01.fq\n",
      "\t the 2nd reads: /tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome21.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome21.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.64\n",
      "\n",
      "The random seed for the run: 854832072\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.67678X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome29.01.fq\n",
      "\t the 2nd reads: /tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome29.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome29.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.23\n",
      "\n",
      "The random seed for the run: 526849290\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.171419X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome3.01.fq\n",
      "\t the 2nd reads: /tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome3.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome3.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.45\n",
      "\n",
      "The random seed for the run: 387153331\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.413889X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome48.01.fq\n",
      "\t the 2nd reads: /tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome48.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome48.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.2\n",
      "\n",
      "The random seed for the run: 918751087\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.10036X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome20.01.fq\n",
      "\t the 2nd reads: /tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome20.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome20.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.21\n",
      "\n",
      "The random seed for the run: 597928115\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.137487X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome45.01.fq\n",
      "\t the 2nd reads: /tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome45.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome45.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.1\n",
      "\n",
      "The random seed for the run: 565794829\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.00261989X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome7.01.fq\n",
      "\t the 2nd reads: /tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome7.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome7.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.1\n",
      "\n",
      "The random seed for the run: 2090645473\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.00388879X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome10.01.fq\n",
      "\t the 2nd reads: /tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome10.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome10.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 1.48\n",
      "\n",
      "The random seed for the run: 1923521921\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            1.88932X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome33.01.fq\n",
      "\t the 2nd reads: /tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome33.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome33.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.19\n",
      "\n",
      "The random seed for the run: 2139276349\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.139562X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome44.01.fq\n",
      "\t the 2nd reads: /tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome44.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome44.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.11\n",
      "\n",
      "The random seed for the run: 1203576411\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0175712X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome27.01.fq\n",
      "\t the 2nd reads: /tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome27.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome27.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.25\n",
      "\n",
      "The random seed for the run: 2029410300\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.196467X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome22.01.fq\n",
      "\t the 2nd reads: /tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome22.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome22.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 1.7\n",
      "\n",
      "The random seed for the run: 1168110496\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            2.1661X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome28.01.fq\n",
      "\t the 2nd reads: /tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome28.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome28.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 1.35\n",
      "\n",
      "The random seed for the run: 26380989\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            1.64658X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome39.01.fq\n",
      "\t the 2nd reads: /tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome39.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome39.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 2.4\n",
      "\n",
      "The random seed for the run: 1208022829\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            3.01442X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome23.01.fq\n",
      "\t the 2nd reads: /tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome23.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome23.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 3.11\n",
      "\n",
      "The random seed for the run: 1268568432\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            4.04777X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome30.01.fq\n",
      "\t the 2nd reads: /tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome30.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome30.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 3.21\n",
      "\n",
      "The random seed for the run: 1937252460\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            4.21316X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome43.01.fq\n",
      "\t the 2nd reads: /tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome43.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpA3XNm0/2021.03.14_08.06.23_sample_0/reads/Genome43.0.sam\n",
      "\n",
      "2021-03-14 08:08:29 INFO: [GenomePreparation 64152656348] Simulating reads finished\n",
      "2021-03-14 08:08:34 INFO: [MetagenomeSimulationPipeline] Generate gold standard assembly\n",
      "2021-03-14 08:08:34 INFO: [MetadataReader 38994325909] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-14 08:09:03 INFO: [MetagenomeSimulationPipeline] Generate pooled strains gold standard assembly\n",
      "2021-03-14 08:09:03 INFO: [MetadataReader 34289848075] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-14 08:09:03 INFO: [MetagenomeSimulationPipeline] Samples used for pooled assembly: '2021.03.14_08.06.23_sample_0'\n",
      "2021-03-14 08:09:03 INFO: [Validator 5358345304] Creating pooled gold standard\n",
      "2021-03-14 08:09:03 INFO: [Validator 5358345304] Merging bam files.\n",
      "2021-03-14 08:09:03 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_08.06.23_sample_0/bam/Genome15.0.bam'\n",
      "2021-03-14 08:09:03 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_08.06.23_sample_0/bam/Genome33.0.bam'\n",
      "2021-03-14 08:09:03 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_08.06.23_sample_0/bam/Genome29.0.bam'\n",
      "2021-03-14 08:09:03 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_08.06.23_sample_0/bam/Genome42.0.bam'\n",
      "2021-03-14 08:09:03 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_08.06.23_sample_0/bam/Genome12.0.bam'\n",
      "2021-03-14 08:09:03 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_08.06.23_sample_0/bam/Genome43.0.bam'\n",
      "2021-03-14 08:09:03 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_08.06.23_sample_0/bam/Genome46.0.bam'\n",
      "2021-03-14 08:09:03 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_08.06.23_sample_0/bam/Genome18.0.bam'\n",
      "2021-03-14 08:09:03 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_08.06.23_sample_0/bam/Genome28.0.bam'\n",
      "2021-03-14 08:09:03 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_08.06.23_sample_0/bam/Genome30.0.bam'\n",
      "2021-03-14 08:09:03 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_08.06.23_sample_0/bam/Genome19.0.bam'\n",
      "2021-03-14 08:09:03 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_08.06.23_sample_0/bam/Genome23.0.bam'\n",
      "2021-03-14 08:09:03 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_08.06.23_sample_0/bam/Genome25.0.bam'\n",
      "2021-03-14 08:09:03 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_08.06.23_sample_0/bam/Genome21.0.bam'\n",
      "2021-03-14 08:09:03 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_08.06.23_sample_0/bam/Genome48.0.bam'\n",
      "2021-03-14 08:09:03 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_08.06.23_sample_0/bam/Genome3.0.bam'\n",
      "2021-03-14 08:09:03 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_08.06.23_sample_0/bam/Genome39.0.bam'\n",
      "2021-03-14 08:09:03 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_08.06.23_sample_0/bam/Genome20.0.bam'\n",
      "2021-03-14 08:09:03 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_08.06.23_sample_0/bam/Genome45.0.bam'\n",
      "2021-03-14 08:09:03 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_08.06.23_sample_0/bam/Genome7.0.bam'\n",
      "2021-03-14 08:09:03 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_08.06.23_sample_0/bam/Genome10.0.bam'\n",
      "2021-03-14 08:09:03 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_08.06.23_sample_0/bam/Genome44.0.bam'\n",
      "2021-03-14 08:09:03 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_08.06.23_sample_0/bam/Genome27.0.bam'\n",
      "2021-03-14 08:09:03 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_08.06.23_sample_0/bam/Genome22.0.bam'\n",
      "2021-03-14 08:09:32 INFO: [MetagenomeSimulationPipeline] Anonymize Data\n",
      "2021-03-14 08:09:32 INFO: [FastaAnonymizer] Interweave shuffle and anonymize\n",
      "2021-03-14 08:10:39 INFO: [MetadataReader 55801910510] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-14 08:10:40 INFO: [MetadataReader 35298217545] Reading file: '/output/internal/meta_data.tsv'\n",
      "2021-03-14 08:10:40 INFO: [MetadataReader 50273713801] Reading file: '/tmp/tmpA3XNm0/tmpRC2pYE'\n",
      "2021-03-14 08:10:42 INFO: [FastaAnonymizer 20754120427] Shuffle and anonymize '/tmp/tmpA3XNm0/tmp5Xp7a2'\n",
      "2021-03-14 08:10:50 INFO: [MetadataReader 64547429207] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-14 08:10:51 INFO: [MetadataReader 10989772577] Reading file: '/output/internal/meta_data.tsv'\n",
      "2021-03-14 08:10:51 INFO: [MetadataReader 53809671431] Reading file: '/tmp/tmpA3XNm0/read_start_positionsYw0ypl'\n",
      "2021-03-14 08:10:53 INFO: [MetadataReader 18168782768] Reading file: '/tmp/tmpA3XNm0/tmp_TbbJL'\n",
      "2021-03-14 08:10:59 INFO: [FastaAnonymizer 92324153505] Shuffle and anonymize '/tmp/tmpA3XNm0/tmpMcT_h8'\n",
      "2021-03-14 08:11:07 INFO: [MetadataReader 83768748887] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-14 08:11:08 INFO: [MetadataReader 11249978774] Reading file: '/output/internal/meta_data.tsv'\n",
      "2021-03-14 08:11:08 INFO: [MetadataReader 66861377693] Reading file: '/tmp/tmpA3XNm0/read_start_positionsAzyLPb'\n",
      "2021-03-14 08:11:09 INFO: [MetadataReader 8017839927] Reading file: '/tmp/tmpA3XNm0/tmprb7qXd'\n",
      "2021-03-14 08:11:16 INFO: [MetagenomeSimulationPipeline] Compress Data\n",
      "2021-03-14 08:11:16 INFO: [Compress 35959976255] Compressing 'tmpZ3OlIH'\n",
      "2021-03-14 08:11:16 INFO: [Compress 89569036172] Compressing 'anonymous_gs_mappingv0BOqD'\n",
      "2021-03-14 08:11:16 INFO: [Compress 63555624946] Compressing 'tmp_CoQAk'\n",
      "2021-03-14 08:11:16 INFO: [Compress 47720839895] Compressing 'anonymous_gsa_mappingsjywoQ'\n",
      "2021-03-14 08:11:16 INFO: [Compress 84506752462] Compressing 'tmpAoEOnz'\n",
      "2021-03-14 08:11:16 INFO: [Compress 93364898019] Compressing 'anonymous_gsa_pooled_mappingFTAGKX'\n",
      "2021-03-14 08:11:16 INFO: [Compress 93364898019] Done compressing 'gsa_pooled_mapping.tsv.gz' in 0:00:00s.\n",
      "2021-03-14 08:11:16 INFO: [Compress 47720839895] Done compressing 'gsa_mapping.tsv.gz' in 0:00:00s.\n",
      "2021-03-14 08:11:18 INFO: [Compress 89569036172] Done compressing 'reads_mapping.tsv.gz' in 0:00:02s.\n",
      "2021-03-14 08:11:18 INFO: [Compress 63555624946] Done compressing 'anonymous_gsa.fasta.gz' in 0:00:02s.\n",
      "2021-03-14 08:11:18 INFO: [Compress 84506752462] Done compressing 'anonymous_gsa_pooled.fasta.gz' in 0:00:02s.\n",
      "2021-03-14 08:11:25 INFO: [Compress 35959976255] Done compressing 'anonymous_reads.fq.gz' in 0:00:09s.\n",
      "2021-03-14 08:11:25 INFO: [MetagenomeSimulationPipeline] Metagenome simulation finished\n",
      "simulating complexity_sim_5_genera_250_genomes\n",
      "[sudo] password for pathinformatics: 2021-03-14 08:11:27 INFO: [MetagenomeSimulationPipeline] Metagenome simulation starting\n",
      "2021-03-14 08:11:27 INFO: [MetagenomeSimulationPipeline] Validating Genomes\n",
      "2021-03-14 08:11:27 INFO: [MetadataReader] Reading file: '/input/genome_to_id.tsv'\n",
      "2021-03-14 08:15:22 INFO: [MetagenomeSimulationPipeline] Design Communities\n",
      "2021-03-14 08:15:22 INFO: [CommunityDesign] Drawing strains.\n",
      "2021-03-14 08:15:22 INFO: [MetadataReader 82380310288] Reading file: '/input/metadata.tsv'\n",
      "2021-03-14 08:15:22 INFO: [MetadataReader 28782960663] Reading file: '/input/genome_to_id.tsv'\n",
      "2021-03-14 08:15:22 INFO: [CommunityDesign] Validating raw sequence files!\n",
      "2021-03-14 08:15:44 INFO: [NcbiTaxonomy] Building taxonomy tree...\n",
      "2021-03-14 08:15:44 INFO: [NcbiTaxonomy] Reading 'nodes' file:\t'/tmp/tmpVRuuQ4/NCBI/nodes.dmp'\n",
      "2021-03-14 08:15:55 INFO: [NcbiTaxonomy] Reading 'names' file:\t'/tmp/tmpVRuuQ4/NCBI/names.dmp'\n",
      "2021-03-14 08:15:57 INFO: [NcbiTaxonomy] Reading 'merged' file:\t'/tmp/tmpVRuuQ4/NCBI/merged.dmp'\n",
      "2021-03-14 08:15:57 INFO: [NcbiTaxonomy] Done (13.0s)\n",
      "2021-03-14 08:15:57 INFO: [MetagenomeSimulationPipeline] Move Genomes\n",
      "2021-03-14 08:15:58 INFO: [MetagenomeSimulationPipeline] Read simulation\n",
      "2021-03-14 08:15:58 INFO: [GenomePreparation 82430254076] Using 'mbarc' error profile.\n",
      "2021-03-14 08:15:58 INFO: [GenomePreparation 82430254076] Reading distribution file\n",
      "2021-03-14 08:15:58 INFO: [GenomePreparation 82430254076] Reading genome location file\n",
      "2021-03-14 08:16:08 INFO: [GenomePreparation 82430254076] Genome 'Genome189.0' has sequences below minimum, creating filtered copy.\n",
      "2021-03-14 08:16:10 INFO: [GenomePreparation 82430254076] Genome 'Genome108.0' has sequences below minimum, creating filtered copy.\n",
      "2021-03-14 08:16:20 INFO: [GenomePreparation 82430254076] Simulating reads using ReadSimulationArtIllumina readsimulator...\n",
      "2021-03-14 08:16:20 INFO: [GenomePreparation 82430254076] Simulating reads from Genome131.0: '/output/source_genomes/GCA_000176055.1_ASM17605v1_genomic.fna'\n",
      "2021-03-14 08:16:20 INFO: [GenomePreparation 82430254076] Simulating reads from Genome14.0: '/output/source_genomes/GCA_001913905.1_ASM191390v1_genomic.fna'\n",
      "2021-03-14 08:16:20 INFO: [GenomePreparation 82430254076] Simulating reads from Genome65.0: '/output/source_genomes/GCA_012265965.1_PDT000281733.2_genomic.fna'\n",
      "2021-03-14 08:16:20 INFO: [GenomePreparation 82430254076] Simulating reads from Genome188.0: '/output/source_genomes/GCA_000461895.1_OCN008_genomic.fna'\n",
      "2021-03-14 08:16:20 INFO: [GenomePreparation 82430254076] Simulating reads from Genome130.0: '/output/source_genomes/GCA_000154025.1_ASM15402v1_genomic.fna'\n",
      "2021-03-14 08:16:20 INFO: [GenomePreparation 82430254076] Simulating reads from Genome82.0: '/output/source_genomes/GCA_000250235.2_ASM25023v2_genomic.fna'\n",
      "2021-03-14 08:16:20 INFO: [GenomePreparation 82430254076] Simulating reads from Genome215.0: '/output/source_genomes/GCA_905206855.1_ERR1600702-mag-bin.11_genomic.fna'\n",
      "2021-03-14 08:16:20 INFO: [GenomePreparation 82430254076] Simulating reads from Genome17.0: '/output/source_genomes/GCA_000434155.1_MGS437_genomic.fna'\n",
      "2021-03-14 08:16:20 INFO: [GenomePreparation 82430254076] Simulating reads from Genome189.0: '/tmp/tmp3yZQzs/tmpDkqEmL'\n",
      "2021-03-14 08:16:20 INFO: [GenomePreparation 82430254076] Simulating reads from Genome139.0: '/output/source_genomes/GCA_000183885.1_ASM18388v1_genomic.fna'\n",
      "2021-03-14 08:16:20 INFO: [GenomePreparation 82430254076] Simulating reads from Genome108.0: '/tmp/tmp3yZQzs/tmpgHNM5J'\n",
      "2021-03-14 08:16:20 INFO: [GenomePreparation 82430254076] Simulating reads from Genome182.0: '/output/source_genomes/GCA_000382925.1_ASM38292v1_genomic.fna'\n",
      "2021-03-14 08:16:20 INFO: [GenomePreparation 82430254076] Simulating reads from Genome211.0: '/output/source_genomes/GCA_015741235.1_PDT000893871.1_genomic.fna'\n",
      "2021-03-14 08:16:20 INFO: [GenomePreparation 82430254076] Simulating reads from Genome165.0: '/output/source_genomes/GCA_017137375.1_PDT000948274.1_genomic.fna'\n",
      "2021-03-14 08:16:20 INFO: [GenomePreparation 82430254076] Simulating reads from Genome164.0: '/output/source_genomes/GCA_001402175.1_ASM140217v1_genomic.fna'\n",
      "2021-03-14 08:16:20 INFO: [GenomePreparation 82430254076] Simulating reads from Genome204.0: '/output/source_genomes/GCA_013076425.1_PDT000186183.4_genomic.fna'\n",
      "2021-03-14 08:16:20 INFO: [GenomePreparation 82430254076] Simulating reads from Genome233.0: '/output/source_genomes/GCA_004728485.1_PDT000353211.2_genomic.fna'\n",
      "2021-03-14 08:16:20 INFO: [GenomePreparation 82430254076] Simulating reads from Genome93.0: '/output/source_genomes/GCA_000183965.1_ASM18396v1_genomic.fna'\n",
      "2021-03-14 08:16:20 INFO: [GenomePreparation 82430254076] Simulating reads from Genome225.0: '/output/source_genomes/GCA_000753355.2_Clostridium_amazonitimonense_genomic.fna'\n",
      "2021-03-14 08:16:20 INFO: [GenomePreparation 82430254076] Simulating reads from Genome95.0: '/output/source_genomes/GCA_900896415.1_Bacteroides_uniformis_82G1_genomic.fna'\n",
      "2021-03-14 08:16:20 INFO: [GenomePreparation 82430254076] Simulating reads from Genome20.0: '/output/source_genomes/GCA_000433715.1_MGS211_genomic.fna'\n",
      "2021-03-14 08:16:20 INFO: [GenomePreparation 82430254076] Simulating reads from Genome133.0: '/output/source_genomes/GCA_000437995.1_MGS40_genomic.fna'\n",
      "2021-03-14 08:16:20 INFO: [GenomePreparation 82430254076] Simulating reads from Genome36.0: '/output/source_genomes/GCA_000176455.1_ASM17645v1_genomic.fna'\n",
      "2021-03-14 08:16:20 INFO: [GenomePreparation 82430254076] Simulating reads from Genome183.0: '/output/source_genomes/GCA_000582435.1_CDC54085.1_genomic.fna'\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.09\n",
      "\n",
      "The random seed for the run: 50971974\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0108193X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome215.01.fq\n",
      "\t the 2nd reads: /tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome215.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome215.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.12\n",
      "\n",
      "The random seed for the run: 963059643\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0374047X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome188.01.fq\n",
      "\t the 2nd reads: /tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome188.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome188.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.11\n",
      "\n",
      "The random seed for the run: 1795828534\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.111987X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome14.01.fq\n",
      "\t the 2nd reads: /tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome14.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome14.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.24\n",
      "\n",
      "The random seed for the run: 1594031057\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.470275X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome17.01.fq\n",
      "\t the 2nd reads: /tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome17.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome17.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.18\n",
      "\n",
      "The random seed for the run: 1027999508\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0166393X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome139.01.fq\n",
      "\t the 2nd reads: /tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome139.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome139.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.36\n",
      "\n",
      "The random seed for the run: 1044513605\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.324783X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome82.01.fq\n",
      "\t the 2nd reads: /tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome82.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome82.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.22\n",
      "\n",
      "The random seed for the run: 2130432890\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.427238X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome182.01.fq\n",
      "\t the 2nd reads: /tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome182.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome182.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.13\n",
      "\n",
      "The random seed for the run: 411957455\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0517933X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome165.01.fq\n",
      "\t the 2nd reads: /tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome165.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome165.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.13\n",
      "\n",
      "The random seed for the run: 670345468\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0789839X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome164.01.fq\n",
      "\t the 2nd reads: /tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome164.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome164.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.34\n",
      "\n",
      "The random seed for the run: 1937464905\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.400319X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome211.01.fq\n",
      "\t the 2nd reads: /tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome211.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome211.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.14\n",
      "\n",
      "The random seed for the run: 460212766\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0544884X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome204.01.fq\n",
      "\t the 2nd reads: /tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome204.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome204.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.13\n",
      "\n",
      "The random seed for the run: 2029296162\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.080606X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome225.01.fq\n",
      "\t the 2nd reads: /tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome225.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome225.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.15\n",
      "\n",
      "The random seed for the run: 1669488642\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.047432X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome93.01.fq\n",
      "\t the 2nd reads: /tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome93.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome93.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.01\n",
      "\n",
      "The random seed for the run: 263232151\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.705876X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome95.01.fq\n",
      "\t the 2nd reads: /tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome95.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome95.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.28\n",
      "\n",
      "The random seed for the run: 1097061061\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.489908X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome233.01.fq\n",
      "\t the 2nd reads: /tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome233.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome233.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.17\n",
      "\n",
      "The random seed for the run: 215267604\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.178228X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome20.01.fq\n",
      "\t the 2nd reads: /tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome20.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome20.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.08\n",
      "\n",
      "The random seed for the run: 1940362427\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0209275X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome36.01.fq\n",
      "\t the 2nd reads: /tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome36.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome36.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 1.14\n",
      "\n",
      "The random seed for the run: 941171800\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            1.31223X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome189.01.fq\n",
      "\t the 2nd reads: /tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome189.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome189.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.09\n",
      "\n",
      "The random seed for the run: 642284040\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0263953X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome183.01.fq\n",
      "\t the 2nd reads: /tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome183.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome183.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 1.49\n",
      "\n",
      "The random seed for the run: 1093218188\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            1.81828X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome65.01.fq\n",
      "\t the 2nd reads: /tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome65.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome65.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 1.65\n",
      "\n",
      "The random seed for the run: 507207848\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            1.96795X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome131.01.fq\n",
      "\t the 2nd reads: /tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome131.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome131.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 1.69\n",
      "\n",
      "The random seed for the run: 393287511\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            1.91587X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome130.01.fq\n",
      "\t the 2nd reads: /tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome130.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome130.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.63\n",
      "\n",
      "The random seed for the run: 370300383\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.681554X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome133.01.fq\n",
      "\t the 2nd reads: /tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome133.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome133.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 6.65\n",
      "\n",
      "The random seed for the run: 1414977469\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            8.00669X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome108.01.fq\n",
      "\t the 2nd reads: /tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome108.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmp3yZQzs/2021.03.14_08.11.27_sample_0/reads/Genome108.0.sam\n",
      "\n",
      "2021-03-14 08:16:28 INFO: [GenomePreparation 82430254076] Simulating reads finished\n",
      "2021-03-14 08:16:37 INFO: [MetagenomeSimulationPipeline] Generate gold standard assembly\n",
      "2021-03-14 08:16:37 INFO: [MetadataReader 87205974322] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-14 08:17:02 INFO: [MetagenomeSimulationPipeline] Generate pooled strains gold standard assembly\n",
      "2021-03-14 08:17:02 INFO: [MetadataReader 114087710] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-14 08:17:02 INFO: [MetagenomeSimulationPipeline] Samples used for pooled assembly: '2021.03.14_08.11.27_sample_0'\n",
      "2021-03-14 08:17:02 INFO: [Validator 10795378910] Creating pooled gold standard\n",
      "2021-03-14 08:17:02 INFO: [Validator 10795378910] Merging bam files.\n",
      "2021-03-14 08:17:02 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_08.11.27_sample_0/bam/Genome131.0.bam'\n",
      "2021-03-14 08:17:02 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_08.11.27_sample_0/bam/Genome14.0.bam'\n",
      "2021-03-14 08:17:02 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_08.11.27_sample_0/bam/Genome65.0.bam'\n",
      "2021-03-14 08:17:02 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_08.11.27_sample_0/bam/Genome188.0.bam'\n",
      "2021-03-14 08:17:02 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_08.11.27_sample_0/bam/Genome130.0.bam'\n",
      "2021-03-14 08:17:02 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_08.11.27_sample_0/bam/Genome82.0.bam'\n",
      "2021-03-14 08:17:02 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_08.11.27_sample_0/bam/Genome215.0.bam'\n",
      "2021-03-14 08:17:02 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_08.11.27_sample_0/bam/Genome17.0.bam'\n",
      "2021-03-14 08:17:02 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_08.11.27_sample_0/bam/Genome189.0.bam'\n",
      "2021-03-14 08:17:02 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_08.11.27_sample_0/bam/Genome139.0.bam'\n",
      "2021-03-14 08:17:02 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_08.11.27_sample_0/bam/Genome108.0.bam'\n",
      "2021-03-14 08:17:02 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_08.11.27_sample_0/bam/Genome182.0.bam'\n",
      "2021-03-14 08:17:02 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_08.11.27_sample_0/bam/Genome211.0.bam'\n",
      "2021-03-14 08:17:02 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_08.11.27_sample_0/bam/Genome165.0.bam'\n",
      "2021-03-14 08:17:02 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_08.11.27_sample_0/bam/Genome164.0.bam'\n",
      "2021-03-14 08:17:02 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_08.11.27_sample_0/bam/Genome204.0.bam'\n",
      "2021-03-14 08:17:02 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_08.11.27_sample_0/bam/Genome233.0.bam'\n",
      "2021-03-14 08:17:02 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_08.11.27_sample_0/bam/Genome93.0.bam'\n",
      "2021-03-14 08:17:02 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_08.11.27_sample_0/bam/Genome225.0.bam'\n",
      "2021-03-14 08:17:02 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_08.11.27_sample_0/bam/Genome95.0.bam'\n",
      "2021-03-14 08:17:02 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_08.11.27_sample_0/bam/Genome20.0.bam'\n",
      "2021-03-14 08:17:02 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_08.11.27_sample_0/bam/Genome133.0.bam'\n",
      "2021-03-14 08:17:02 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_08.11.27_sample_0/bam/Genome36.0.bam'\n",
      "2021-03-14 08:17:02 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.14_08.11.27_sample_0/bam/Genome183.0.bam'\n",
      "2021-03-14 08:17:27 INFO: [MetagenomeSimulationPipeline] Anonymize Data\n",
      "2021-03-14 08:17:27 INFO: [FastaAnonymizer] Interweave shuffle and anonymize\n",
      "2021-03-14 08:18:34 INFO: [MetadataReader 4631107874] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-14 08:18:35 INFO: [MetadataReader 14300761082] Reading file: '/output/internal/meta_data.tsv'\n",
      "2021-03-14 08:18:35 INFO: [MetadataReader 87652392539] Reading file: '/tmp/tmp3yZQzs/tmpm9RxZb'\n",
      "2021-03-14 08:18:37 INFO: [FastaAnonymizer 66694448886] Shuffle and anonymize '/tmp/tmp3yZQzs/tmpYapW19'\n",
      "2021-03-14 08:18:45 INFO: [MetadataReader 4079751341] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-14 08:18:45 INFO: [MetadataReader 84063950941] Reading file: '/output/internal/meta_data.tsv'\n",
      "2021-03-14 08:18:45 INFO: [MetadataReader 48704713275] Reading file: '/tmp/tmp3yZQzs/read_start_positions0BqbxJ'\n",
      "2021-03-14 08:18:47 INFO: [MetadataReader 43119039112] Reading file: '/tmp/tmp3yZQzs/tmpq_fYlU'\n",
      "2021-03-14 08:18:53 INFO: [FastaAnonymizer 64368966291] Shuffle and anonymize '/tmp/tmp3yZQzs/tmpvQ6w4u'\n",
      "2021-03-14 08:19:00 INFO: [MetadataReader 78717221707] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-14 08:19:01 INFO: [MetadataReader 89765374142] Reading file: '/output/internal/meta_data.tsv'\n",
      "2021-03-14 08:19:01 INFO: [MetadataReader 98541404663] Reading file: '/tmp/tmp3yZQzs/read_start_positionsIBNBlg'\n",
      "2021-03-14 08:19:03 INFO: [MetadataReader 99062647065] Reading file: '/tmp/tmp3yZQzs/tmpBztNFa'\n",
      "2021-03-14 08:19:08 INFO: [MetagenomeSimulationPipeline] Compress Data\n",
      "2021-03-14 08:19:08 INFO: [Compress 31626330865] Compressing 'tmp5NABR0'\n",
      "2021-03-14 08:19:08 INFO: [Compress 34489618644] Compressing 'anonymous_gs_mapping2MKdua'\n",
      "2021-03-14 08:19:08 INFO: [Compress 35797026464] Compressing 'tmpvjkhQY'\n",
      "2021-03-14 08:19:08 INFO: [Compress 10881124434] Compressing 'anonymous_gsa_mappingFcJYYd'\n",
      "2021-03-14 08:19:08 INFO: [Compress 68141777168] Compressing 'tmpbaT0fQ'\n",
      "2021-03-14 08:19:08 INFO: [Compress 31629465763] Compressing 'anonymous_gsa_pooled_mappinglCen5x'\n",
      "2021-03-14 08:19:09 INFO: [Compress 10881124434] Done compressing 'gsa_mapping.tsv.gz' in 0:00:00s.\n",
      "2021-03-14 08:19:09 INFO: [Compress 31629465763] Done compressing 'gsa_pooled_mapping.tsv.gz' in 0:00:00s.\n",
      "2021-03-14 08:19:10 INFO: [Compress 34489618644] Done compressing 'reads_mapping.tsv.gz' in 0:00:02s.\n",
      "2021-03-14 08:19:10 INFO: [Compress 68141777168] Done compressing 'anonymous_gsa_pooled.fasta.gz' in 0:00:02s.\n",
      "2021-03-14 08:19:10 INFO: [Compress 35797026464] Done compressing 'anonymous_gsa.fasta.gz' in 0:00:02s.\n",
      "2021-03-14 08:19:17 INFO: [Compress 31626330865] Done compressing 'anonymous_reads.fq.gz' in 0:00:09s.\n",
      "2021-03-14 08:19:17 INFO: [MetagenomeSimulationPipeline] Metagenome simulation finished\n",
      "simulating complexity_sim_5_genera_25_genomes\n",
      "[sudo] password for pathinformatics: 2021-03-14 08:19:20 INFO: [MetagenomeSimulationPipeline] Metagenome simulation starting\n",
      "2021-03-14 08:19:20 INFO: [MetagenomeSimulationPipeline] Validating Genomes\n",
      "2021-03-14 08:19:20 INFO: [MetadataReader] Reading file: '/input/genome_to_id.tsv'\n",
      "2021-03-14 08:19:44 INFO: [MetagenomeSimulationPipeline] Design Communities\n",
      "2021-03-14 08:19:44 INFO: [CommunityDesign] Drawing strains.\n",
      "2021-03-14 08:19:44 INFO: [MetadataReader 82380310288] Reading file: '/input/metadata.tsv'\n",
      "2021-03-14 08:19:44 INFO: [MetadataReader 25595780401] Reading file: '/input/genome_to_id.tsv'\n",
      "2021-03-14 08:19:44 INFO: [CommunityDesign] Validating raw sequence files!\n",
      "2021-03-14 08:20:08 INFO: [NcbiTaxonomy] Building taxonomy tree...\n",
      "2021-03-14 08:20:08 INFO: [NcbiTaxonomy] Reading 'nodes' file:\t'/tmp/tmpmcSfOl/NCBI/nodes.dmp'\n",
      "2021-03-14 08:20:19 INFO: [NcbiTaxonomy] Reading 'names' file:\t'/tmp/tmpmcSfOl/NCBI/names.dmp'\n",
      "2021-03-14 08:20:21 INFO: [NcbiTaxonomy] Reading 'merged' file:\t'/tmp/tmpmcSfOl/NCBI/merged.dmp'\n",
      "2021-03-14 08:20:21 INFO: [NcbiTaxonomy] Done (13.0s)\n",
      "2021-03-14 08:20:21 INFO: [MetagenomeSimulationPipeline] Move Genomes\n",
      "2021-03-14 08:20:22 INFO: [MetagenomeSimulationPipeline] Read simulation\n",
      "2021-03-14 08:20:22 INFO: [GenomePreparation 70514612869] Using 'mbarc' error profile.\n",
      "2021-03-14 08:20:22 INFO: [GenomePreparation 70514612869] Reading distribution file\n",
      "2021-03-14 08:20:22 INFO: [GenomePreparation 70514612869] Reading genome location file\n",
      "2021-03-14 08:20:25 INFO: [GenomePreparation 70514612869] Genome 'Genome11.0' has sequences below minimum, creating filtered copy.\n",
      "2021-03-14 08:20:30 INFO: [GenomePreparation 70514612869] Genome 'Genome18.0' has sequences below minimum, creating filtered copy.\n",
      "2021-03-14 08:20:32 INFO: [GenomePreparation 70514612869] Genome 'Genome19.0' has sequences below minimum, creating filtered copy.\n",
      "2021-03-14 08:20:36 INFO: [GenomePreparation 70514612869] Genome 'Genome5.0' has sequences below minimum, creating filtered copy.\n",
      "2021-03-14 08:20:42 INFO: [GenomePreparation 70514612869] Genome 'Genome7.0' has sequences below minimum, creating filtered copy.\n",
      "2021-03-14 08:20:45 INFO: [GenomePreparation 70514612869] Genome 'Genome10.0' has sequences below minimum, creating filtered copy.\n",
      "2021-03-14 08:20:46 INFO: [GenomePreparation 70514612869] Simulating reads using ReadSimulationArtIllumina readsimulator...\n",
      "2021-03-14 08:20:46 INFO: [GenomePreparation 70514612869] Simulating reads from Genome15.0: '/output/source_genomes/GCA_012214065.1_PDT000306618.2_genomic.fna'\n",
      "2021-03-14 08:20:46 INFO: [GenomePreparation 70514612869] Simulating reads from Genome14.0: '/output/source_genomes/GCA_016955055.1_PDT000971198.1_genomic.fna'\n",
      "2021-03-14 08:20:46 INFO: [GenomePreparation 70514612869] Simulating reads from Genome11.0: '/tmp/tmpd7gJ4r/tmp8Mgqnb'\n",
      "2021-03-14 08:20:46 INFO: [GenomePreparation 70514612869] Simulating reads from Genome24.0: '/output/source_genomes/GCA_012761095.1_PDT000705083.1_genomic.fna'\n",
      "2021-03-14 08:20:46 INFO: [GenomePreparation 70514612869] Simulating reads from Genome12.0: '/output/source_genomes/GCA_012214265.1_PDT000306620.2_genomic.fna'\n",
      "2021-03-14 08:20:46 INFO: [GenomePreparation 70514612869] Simulating reads from Genome25.0: '/output/source_genomes/GCA_000709105.1_M29_genomic.fna'\n",
      "2021-03-14 08:20:46 INFO: [GenomePreparation 70514612869] Simulating reads from Genome18.0: '/tmp/tmpd7gJ4r/tmpGuLeao'\n",
      "2021-03-14 08:20:46 INFO: [GenomePreparation 70514612869] Simulating reads from Genome16.0: '/output/source_genomes/GCA_000755405.1_ASM75540v1_genomic.fna'\n",
      "2021-03-14 08:20:46 INFO: [GenomePreparation 70514612869] Simulating reads from Genome19.0: '/tmp/tmpd7gJ4r/tmpFe9dOC'\n",
      "2021-03-14 08:20:46 INFO: [GenomePreparation 70514612869] Simulating reads from Genome2.0: '/output/source_genomes/GCA_013407315.1_ASM1340731v1_genomic.fna'\n",
      "2021-03-14 08:20:46 INFO: [GenomePreparation 70514612869] Simulating reads from Genome1.0: '/output/source_genomes/GCA_900576925.1_CLIP201500305T_genomic.fna'\n",
      "2021-03-14 08:20:46 INFO: [GenomePreparation 70514612869] Simulating reads from Genome21.0: '/output/source_genomes/GCA_004728485.1_PDT000353211.2_genomic.fna'\n",
      "2021-03-14 08:20:46 INFO: [GenomePreparation 70514612869] Simulating reads from Genome4.0: '/output/source_genomes/GCA_004710365.1_PDT000000906.5_genomic.fna'\n",
      "2021-03-14 08:20:46 INFO: [GenomePreparation 70514612869] Simulating reads from Genome5.0: '/tmp/tmpd7gJ4r/tmpzpg_zM'\n",
      "2021-03-14 08:20:46 INFO: [GenomePreparation 70514612869] Simulating reads from Genome3.0: '/output/source_genomes/GCA_000436855.1_MGS203_genomic.fna'\n",
      "2021-03-14 08:20:46 INFO: [GenomePreparation 70514612869] Simulating reads from Genome20.0: '/output/source_genomes/GCA_000435315.1_MGS162_genomic.fna'\n",
      "2021-03-14 08:20:46 INFO: [GenomePreparation 70514612869] Simulating reads from Genome6.0: '/output/source_genomes/GCA_002367115.1_ASM236711v1_genomic.fna'\n",
      "2021-03-14 08:20:46 INFO: [GenomePreparation 70514612869] Simulating reads from Genome23.0: '/output/source_genomes/GCA_000435015.1_MGS120_genomic.fna'\n",
      "2021-03-14 08:20:46 INFO: [GenomePreparation 70514612869] Simulating reads from Genome9.0: '/output/source_genomes/GCA_000433375.1_MGS32_genomic.fna'\n",
      "2021-03-14 08:20:46 INFO: [GenomePreparation 70514612869] Simulating reads from Genome7.0: '/tmp/tmpd7gJ4r/tmpQWqJVS'\n",
      "2021-03-14 08:20:46 INFO: [GenomePreparation 70514612869] Simulating reads from Genome13.0: '/output/source_genomes/GCA_000219255.3_CSt2.0_genomic.fna'\n",
      "2021-03-14 08:20:46 INFO: [GenomePreparation 70514612869] Simulating reads from Genome10.0: '/tmp/tmpd7gJ4r/tmpqxhmBY'\n",
      "2021-03-14 08:20:46 INFO: [GenomePreparation 70514612869] Simulating reads from Genome8.0: '/output/source_genomes/GCA_004550345.1_PDT000481416.1_genomic.fna'\n",
      "2021-03-14 08:20:46 INFO: [GenomePreparation 70514612869] Simulating reads from Genome22.0: '/output/source_genomes/GCA_004710035.1_PDT000000903.4_genomic.fna'\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.12\n",
      "\n",
      "The random seed for the run: 540968737\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0467293X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome15.01.fq\n",
      "\t the 2nd reads: /tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome15.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome15.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.11\n",
      "\n",
      "The random seed for the run: 65634987\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.00373638X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome11.01.fq\n",
      "\t the 2nd reads: /tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome11.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome11.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.13\n",
      "\n",
      "The random seed for the run: 2129338277\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0643335X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome25.01.fq\n",
      "\t the 2nd reads: /tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome25.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome25.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.11\n",
      "\n",
      "The random seed for the run: 1294107793\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0243669X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome24.01.fq\n",
      "\t the 2nd reads: /tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome24.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome24.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.12\n",
      "\n",
      "The random seed for the run: 1703660943\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0146582X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome16.01.fq\n",
      "\t the 2nd reads: /tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome16.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome16.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.2\n",
      "\n",
      "The random seed for the run: 573380147\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.144604X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome12.01.fq\n",
      "\t the 2nd reads: /tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome12.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome12.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.08\n",
      "\n",
      "The random seed for the run: 1661055960\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0044094X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome19.01.fq\n",
      "\t the 2nd reads: /tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome19.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome19.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.13\n",
      "\n",
      "The random seed for the run: 1577639001\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0331544X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome2.01.fq\n",
      "\t the 2nd reads: /tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome2.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome2.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.15\n",
      "\n",
      "The random seed for the run: 1557933274\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.160141X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome4.01.fq\n",
      "\t the 2nd reads: /tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome4.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome4.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.22\n",
      "\n",
      "The random seed for the run: 1329514041\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.138547X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome5.01.fq\n",
      "\t the 2nd reads: /tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome5.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome5.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.52\n",
      "\n",
      "The random seed for the run: 170568346\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.547474X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome18.01.fq\n",
      "\t the 2nd reads: /tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome18.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome18.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.14\n",
      "\n",
      "The random seed for the run: 923704085\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0715001X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome6.01.fq\n",
      "\t the 2nd reads: /tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome6.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome6.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.15\n",
      "\n",
      "The random seed for the run: 1992431107\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0786824X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome9.01.fq\n",
      "\t the 2nd reads: /tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome9.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome9.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.14\n",
      "\n",
      "The random seed for the run: 1923521921\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.082634X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome7.01.fq\n",
      "\t the 2nd reads: /tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome7.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome7.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.85\n",
      "\n",
      "The random seed for the run: 1847505585\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.952733X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome14.01.fq\n",
      "\t the 2nd reads: /tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome14.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome14.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.42\n",
      "\n",
      "The random seed for the run: 854832072\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.984836X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome13.01.fq\n",
      "\t the 2nd reads: /tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome13.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome13.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 1.08\n",
      "\n",
      "The random seed for the run: 49814198\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            2.20975X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome21.01.fq\n",
      "\t the 2nd reads: /tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome21.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome21.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.07\n",
      "\n",
      "The random seed for the run: 1940270192\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.00316977X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome22.01.fq\n",
      "\t the 2nd reads: /tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome22.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome22.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.77\n",
      "\n",
      "The random seed for the run: 689939719\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.403078X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome10.01.fq\n",
      "\t the 2nd reads: /tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome10.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome10.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 1.58\n",
      "\n",
      "The random seed for the run: 1713385206\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            3.55201X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome1.01.fq\n",
      "\t the 2nd reads: /tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome1.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome1.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.84\n",
      "\n",
      "The random seed for the run: 2102807139\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            1.76559X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome8.01.fq\n",
      "\t the 2nd reads: /tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome8.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome8.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 1.63\n",
      "\n",
      "The random seed for the run: 1961570421\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            3.29935X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome20.01.fq\n",
      "\t the 2nd reads: /tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome20.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome20.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 3.22\n",
      "\n",
      "The random seed for the run: 1168871082\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            6.32596X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome3.01.fq\n",
      "\t the 2nd reads: /tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome3.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome3.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 3.83\n",
      "\n",
      "The random seed for the run: 1550755105\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            8.83906X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome23.01.fq\n",
      "\t the 2nd reads: /tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome23.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpd7gJ4r/2021.03.14_08.19.20_sample_0/reads/Genome23.0.sam\n",
      "\n",
      "2021-03-14 08:20:51 INFO: [GenomePreparation 70514612869] Simulating reads finished\n",
      "2021-03-14 08:20:58 INFO: [MetagenomeSimulationPipeline] Generate gold standard assembly\n",
      "2021-03-14 08:20:58 INFO: [MetadataReader 64136302938] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-14 08:21:21 INFO: [MetagenomeSimulationPipeline] Generate pooled strains gold standard assembly\n",
      "2021-03-14 08:21:21 INFO: [MetadataReader 58498923446] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-14 08:21:21 INFO: [MetagenomeSimulationPipeline] Samples used for pooled assembly: '2021.03.14_08.19.20_sample_0'\n",
      "2021-03-14 08:21:21 INFO: [Validator 48443846240] Creating pooled gold standard\n",
      "2021-03-14 08:21:21 INFO: [Validator 48443846240] Merging bam files.\n",
      "2021-03-14 08:21:21 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_08.19.20_sample_0/bam/Genome15.0.bam'\n",
      "2021-03-14 08:21:21 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_08.19.20_sample_0/bam/Genome14.0.bam'\n",
      "2021-03-14 08:21:21 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_08.19.20_sample_0/bam/Genome11.0.bam'\n",
      "2021-03-14 08:21:21 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_08.19.20_sample_0/bam/Genome24.0.bam'\n",
      "2021-03-14 08:21:21 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_08.19.20_sample_0/bam/Genome12.0.bam'\n",
      "2021-03-14 08:21:21 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_08.19.20_sample_0/bam/Genome25.0.bam'\n",
      "2021-03-14 08:21:21 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_08.19.20_sample_0/bam/Genome18.0.bam'\n",
      "2021-03-14 08:21:21 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_08.19.20_sample_0/bam/Genome16.0.bam'\n",
      "2021-03-14 08:21:21 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_08.19.20_sample_0/bam/Genome19.0.bam'\n",
      "2021-03-14 08:21:21 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_08.19.20_sample_0/bam/Genome2.0.bam'\n",
      "2021-03-14 08:21:21 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_08.19.20_sample_0/bam/Genome1.0.bam'\n",
      "2021-03-14 08:21:21 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_08.19.20_sample_0/bam/Genome21.0.bam'\n",
      "2021-03-14 08:21:21 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_08.19.20_sample_0/bam/Genome4.0.bam'\n",
      "2021-03-14 08:21:21 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_08.19.20_sample_0/bam/Genome5.0.bam'\n",
      "2021-03-14 08:21:21 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_08.19.20_sample_0/bam/Genome3.0.bam'\n",
      "2021-03-14 08:21:21 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_08.19.20_sample_0/bam/Genome20.0.bam'\n",
      "2021-03-14 08:21:21 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_08.19.20_sample_0/bam/Genome6.0.bam'\n",
      "2021-03-14 08:21:21 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_08.19.20_sample_0/bam/Genome23.0.bam'\n",
      "2021-03-14 08:21:21 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_08.19.20_sample_0/bam/Genome9.0.bam'\n",
      "2021-03-14 08:21:21 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_08.19.20_sample_0/bam/Genome7.0.bam'\n",
      "2021-03-14 08:21:21 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_08.19.20_sample_0/bam/Genome13.0.bam'\n",
      "2021-03-14 08:21:21 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_08.19.20_sample_0/bam/Genome10.0.bam'\n",
      "2021-03-14 08:21:21 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_08.19.20_sample_0/bam/Genome8.0.bam'\n",
      "2021-03-14 08:21:21 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.14_08.19.20_sample_0/bam/Genome22.0.bam'\n",
      "2021-03-14 08:21:44 INFO: [MetagenomeSimulationPipeline] Anonymize Data\n",
      "2021-03-14 08:21:44 INFO: [FastaAnonymizer] Interweave shuffle and anonymize\n",
      "2021-03-14 08:22:51 INFO: [MetadataReader 72802808290] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-14 08:22:52 INFO: [MetadataReader 47315449936] Reading file: '/output/internal/meta_data.tsv'\n",
      "2021-03-14 08:22:52 INFO: [MetadataReader 65300095466] Reading file: '/tmp/tmpd7gJ4r/tmpCi6LUF'\n",
      "2021-03-14 08:22:54 INFO: [FastaAnonymizer 43075827506] Shuffle and anonymize '/tmp/tmpd7gJ4r/tmpwcoXpE'\n",
      "2021-03-14 08:23:01 INFO: [MetadataReader 71873526544] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-14 08:23:02 INFO: [MetadataReader 45105173502] Reading file: '/output/internal/meta_data.tsv'\n",
      "2021-03-14 08:23:02 INFO: [MetadataReader 84613406616] Reading file: '/tmp/tmpd7gJ4r/read_start_positionsCpjDNX'\n",
      "2021-03-14 08:23:03 INFO: [MetadataReader 10674074437] Reading file: '/tmp/tmpd7gJ4r/tmpsHYl0F'\n",
      "2021-03-14 08:23:09 INFO: [FastaAnonymizer 86623674311] Shuffle and anonymize '/tmp/tmpd7gJ4r/tmpup3G_j'\n",
      "2021-03-14 08:23:15 INFO: [MetadataReader 15345066727] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-14 08:23:16 INFO: [MetadataReader 77753400540] Reading file: '/output/internal/meta_data.tsv'\n",
      "2021-03-14 08:23:16 INFO: [MetadataReader 12266665971] Reading file: '/tmp/tmpd7gJ4r/read_start_positions9biBPh'\n",
      "2021-03-14 08:23:18 INFO: [MetadataReader 42355547664] Reading file: '/tmp/tmpd7gJ4r/tmpT3PPPG'\n",
      "2021-03-14 08:23:23 INFO: [MetagenomeSimulationPipeline] Compress Data\n",
      "2021-03-14 08:23:23 INFO: [Compress 1708279566] Compressing 'tmpsNOCKa'\n",
      "2021-03-14 08:23:23 INFO: [Compress 57228035822] Compressing 'anonymous_gs_mappingzYvhwh'\n",
      "2021-03-14 08:23:23 INFO: [Compress 73212408009] Compressing 'tmpIhrA0T'\n",
      "2021-03-14 08:23:23 INFO: [Compress 68500504210] Compressing 'anonymous_gsa_mappingXT1AbB'\n",
      "2021-03-14 08:23:23 INFO: [Compress 88821317864] Compressing 'tmpdS0CEa'\n",
      "2021-03-14 08:23:23 INFO: [Compress 61397146947] Compressing 'anonymous_gsa_pooled_mappingjor1KK'\n",
      "2021-03-14 08:23:23 INFO: [Compress 61397146947] Done compressing 'gsa_pooled_mapping.tsv.gz' in 0:00:00s.\n",
      "2021-03-14 08:23:23 INFO: [Compress 68500504210] Done compressing 'gsa_mapping.tsv.gz' in 0:00:00s.\n",
      "2021-03-14 08:23:25 INFO: [Compress 57228035822] Done compressing 'reads_mapping.tsv.gz' in 0:00:02s.\n",
      "2021-03-14 08:23:25 INFO: [Compress 73212408009] Done compressing 'anonymous_gsa.fasta.gz' in 0:00:02s.\n",
      "2021-03-14 08:23:25 INFO: [Compress 88821317864] Done compressing 'anonymous_gsa_pooled.fasta.gz' in 0:00:02s.\n",
      "2021-03-14 08:23:32 INFO: [Compress 1708279566] Done compressing 'anonymous_reads.fq.gz' in 0:00:09s.\n",
      "2021-03-14 08:23:32 INFO: [MetagenomeSimulationPipeline] Metagenome simulation finished\n",
      "simulating complexity_sim_10_genera_50_genomes\n",
      "[sudo] password for pathinformatics: 2021-03-14 08:23:34 INFO: [MetagenomeSimulationPipeline] Metagenome simulation starting\n",
      "2021-03-14 08:23:34 INFO: [MetagenomeSimulationPipeline] Validating Genomes\n",
      "2021-03-14 08:23:34 INFO: [MetadataReader] Reading file: '/input/genome_to_id.tsv'\n",
      "2021-03-14 08:24:16 INFO: [MetagenomeSimulationPipeline] Design Communities\n",
      "2021-03-14 08:24:16 INFO: [CommunityDesign] Drawing strains.\n",
      "2021-03-14 08:24:16 INFO: [MetadataReader 82380310288] Reading file: '/input/metadata.tsv'\n",
      "2021-03-14 08:24:16 INFO: [MetadataReader 89885674421] Reading file: '/input/genome_to_id.tsv'\n",
      "2021-03-14 08:24:16 INFO: [CommunityDesign] Validating raw sequence files!\n",
      "2021-03-14 08:24:38 INFO: [NcbiTaxonomy] Building taxonomy tree...\n",
      "2021-03-14 08:24:38 INFO: [NcbiTaxonomy] Reading 'nodes' file:\t'/tmp/tmphSwumj/NCBI/nodes.dmp'\n",
      "2021-03-14 08:24:50 INFO: [NcbiTaxonomy] Reading 'names' file:\t'/tmp/tmphSwumj/NCBI/names.dmp'\n",
      "2021-03-14 08:24:51 INFO: [NcbiTaxonomy] Reading 'merged' file:\t'/tmp/tmphSwumj/NCBI/merged.dmp'\n",
      "2021-03-14 08:24:51 INFO: [NcbiTaxonomy] Done (13.0s)\n",
      "2021-03-14 08:24:51 INFO: [MetagenomeSimulationPipeline] Move Genomes\n",
      "2021-03-14 08:24:52 INFO: [MetagenomeSimulationPipeline] Read simulation\n",
      "2021-03-14 08:24:52 INFO: [GenomePreparation 64152656348] Using 'mbarc' error profile.\n",
      "2021-03-14 08:24:52 INFO: [GenomePreparation 64152656348] Reading distribution file\n",
      "2021-03-14 08:24:52 INFO: [GenomePreparation 64152656348] Reading genome location file\n",
      "2021-03-14 08:24:57 INFO: [GenomePreparation 64152656348] Genome 'Genome12.0' has sequences below minimum, creating filtered copy.\n",
      "2021-03-14 08:24:59 INFO: [GenomePreparation 64152656348] Genome 'Genome31.0' has sequences below minimum, creating filtered copy.\n",
      "2021-03-14 08:25:03 INFO: [GenomePreparation 64152656348] Genome 'Genome50.0' has sequences below minimum, creating filtered copy.\n",
      "2021-03-14 08:25:12 INFO: [GenomePreparation 64152656348] Genome 'Genome36.0' has sequences below minimum, creating filtered copy.\n",
      "2021-03-14 08:25:14 INFO: [GenomePreparation 64152656348] Simulating reads using ReadSimulationArtIllumina readsimulator...\n",
      "2021-03-14 08:25:14 INFO: [GenomePreparation 64152656348] Simulating reads from Genome11.0: '/output/source_genomes/GCA_001311115.1_ASM131111v1_genomic.fna'\n",
      "2021-03-14 08:25:14 INFO: [GenomePreparation 64152656348] Simulating reads from Genome29.0: '/output/source_genomes/GCA_000309585.1_ASM30958v1_genomic.fna'\n",
      "2021-03-14 08:25:14 INFO: [GenomePreparation 64152656348] Simulating reads from Genome42.0: '/output/source_genomes/GCA_000705235.1_Pseudomonas_genomic.fna'\n",
      "2021-03-14 08:25:14 INFO: [GenomePreparation 64152656348] Simulating reads from Genome12.0: '/tmp/tmpgTkE_V/tmpJM9l74'\n",
      "2021-03-14 08:25:14 INFO: [GenomePreparation 64152656348] Simulating reads from Genome31.0: '/tmp/tmpgTkE_V/tmpkz22Wm'\n",
      "2021-03-14 08:25:14 INFO: [GenomePreparation 64152656348] Simulating reads from Genome16.0: '/output/source_genomes/GCA_000741445.1_Bifthe_sub.por_genomic.fna'\n",
      "2021-03-14 08:25:14 INFO: [GenomePreparation 64152656348] Simulating reads from Genome24.0: '/output/source_genomes/GCA_000390045.1_ASM39004v1_genomic.fna'\n",
      "2021-03-14 08:25:14 INFO: [GenomePreparation 64152656348] Simulating reads from Genome34.0: '/output/source_genomes/GCA_012211765.1_PDT000307443.2_genomic.fna'\n",
      "2021-03-14 08:25:14 INFO: [GenomePreparation 64152656348] Simulating reads from Genome21.0: '/output/source_genomes/GCA_011856425.1_PDT000595667.2_genomic.fna'\n",
      "2021-03-14 08:25:14 INFO: [GenomePreparation 64152656348] Simulating reads from Genome50.0: '/tmp/tmpgTkE_V/tmpWspY3M'\n",
      "2021-03-14 08:25:14 INFO: [GenomePreparation 64152656348] Simulating reads from Genome47.0: '/output/source_genomes/GCA_002888575.1_ASM288857v1_genomic.fna'\n",
      "2021-03-14 08:25:14 INFO: [GenomePreparation 64152656348] Simulating reads from Genome4.0: '/output/source_genomes/GCA_001311705.1_ASM131170v1_genomic.fna'\n",
      "2021-03-14 08:25:14 INFO: [GenomePreparation 64152656348] Simulating reads from Genome49.0: '/output/source_genomes/GCA_000435015.1_MGS120_genomic.fna'\n",
      "2021-03-14 08:25:14 INFO: [GenomePreparation 64152656348] Simulating reads from Genome5.0: '/output/source_genomes/GCA_000601355.1_ASM60135v1_genomic.fna'\n",
      "2021-03-14 08:25:14 INFO: [GenomePreparation 64152656348] Simulating reads from Genome3.0: '/output/source_genomes/GCA_001311425.1_ASM131142v1_genomic.fna'\n",
      "2021-03-14 08:25:14 INFO: [GenomePreparation 64152656348] Simulating reads from Genome30.0: '/output/source_genomes/GCA_000816945.1_ASM81694v1_genomic.fna'\n",
      "2021-03-14 08:25:14 INFO: [GenomePreparation 64152656348] Simulating reads from Genome39.0: '/output/source_genomes/GCA_000435315.1_MGS162_genomic.fna'\n",
      "2021-03-14 08:25:14 INFO: [GenomePreparation 64152656348] Simulating reads from Genome20.0: '/output/source_genomes/GCA_001578125.1_ASM157812v1_genomic.fna'\n",
      "2021-03-14 08:25:14 INFO: [GenomePreparation 64152656348] Simulating reads from Genome38.0: '/output/source_genomes/GCA_000183865.1_ASM18386v1_genomic.fna'\n",
      "2021-03-14 08:25:14 INFO: [GenomePreparation 64152656348] Simulating reads from Genome23.0: '/output/source_genomes/GCA_000433395.1_MGS224_genomic.fna'\n",
      "2021-03-14 08:25:14 INFO: [GenomePreparation 64152656348] Simulating reads from Genome36.0: '/tmp/tmpgTkE_V/tmprI0zC9'\n",
      "2021-03-14 08:25:14 INFO: [GenomePreparation 64152656348] Simulating reads from Genome40.0: '/output/source_genomes/GCA_016925195.1_ASM1692519v1_genomic.fna'\n",
      "2021-03-14 08:25:14 INFO: [GenomePreparation 64152656348] Simulating reads from Genome8.0: '/output/source_genomes/GCA_000830415.1_ASM83041v1_genomic.fna'\n",
      "2021-03-14 08:25:14 INFO: [GenomePreparation 64152656348] Simulating reads from Genome22.0: '/output/source_genomes/GCA_000972725.1_ASM97272v1_genomic.fna'\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.05\n",
      "\n",
      "The random seed for the run: 1940270192\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.00535866X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome16.01.fq\n",
      "\t the 2nd reads: /tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome16.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome16.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.08\n",
      "\n",
      "The random seed for the run: 1992431107\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.236211X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome11.01.fq\n",
      "\t the 2nd reads: /tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome11.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome11.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.14\n",
      "\n",
      "The random seed for the run: 783731559\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.171589X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome24.01.fq\n",
      "\t the 2nd reads: /tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome24.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome24.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.18\n",
      "\n",
      "The random seed for the run: 854832072\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0890909X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome42.01.fq\n",
      "\t the 2nd reads: /tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome42.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome42.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.2\n",
      "\n",
      "The random seed for the run: 199492375\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.138293X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome34.01.fq\n",
      "\t the 2nd reads: /tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome34.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome34.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0\n",
      "\n",
      "The random seed for the run: 840935803\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.189453X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome47.01.fq\n",
      "\t the 2nd reads: /tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome47.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome47.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.12\n",
      "\n",
      "The random seed for the run: 1268568432\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0560492X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome50.01.fq\n",
      "\t the 2nd reads: /tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome50.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome50.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.3\n",
      "\n",
      "The random seed for the run: 689939719\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0789982X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome12.01.fq\n",
      "\t the 2nd reads: /tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome12.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome12.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.12\n",
      "\n",
      "The random seed for the run: 1208022829\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.233201X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome4.01.fq\n",
      "\t the 2nd reads: /tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome4.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome4.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.31\n",
      "\n",
      "The random seed for the run: 1168110496\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.270727X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome21.01.fq\n",
      "\t the 2nd reads: /tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome21.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome21.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.06\n",
      "\n",
      "The random seed for the run: 26380989\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.00361014X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome39.01.fq\n",
      "\t the 2nd reads: /tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome39.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome39.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.2\n",
      "\n",
      "The random seed for the run: 526849290\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.192313X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome30.01.fq\n",
      "\t the 2nd reads: /tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome30.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome30.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.19\n",
      "\n",
      "The random seed for the run: 565794829\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.108759X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome23.01.fq\n",
      "\t the 2nd reads: /tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome23.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome23.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.07\n",
      "\n",
      "The random seed for the run: 2090645473\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0242127X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome36.01.fq\n",
      "\t the 2nd reads: /tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome36.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome36.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.52\n",
      "\n",
      "The random seed for the run: 918751087\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.570328X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome20.01.fq\n",
      "\t the 2nd reads: /tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome20.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome20.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.91\n",
      "\n",
      "The random seed for the run: 458448008\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.932585X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome5.01.fq\n",
      "\t the 2nd reads: /tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome5.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome5.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 1.16\n",
      "\n",
      "The random seed for the run: 1937252460\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            2.26895X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome49.01.fq\n",
      "\t the 2nd reads: /tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome49.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome49.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.23\n",
      "\n",
      "The random seed for the run: 2029410300\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.389689X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome22.01.fq\n",
      "\t the 2nd reads: /tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome22.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome22.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.64\n",
      "\n",
      "The random seed for the run: 2139276349\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.236271X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome40.01.fq\n",
      "\t the 2nd reads: /tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome40.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome40.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 1.68\n",
      "\n",
      "The random seed for the run: 597928115\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            4.15379X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome38.01.fq\n",
      "\t the 2nd reads: /tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome38.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome38.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 2.63\n",
      "\n",
      "The random seed for the run: 2102807139\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            2.98483X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome31.01.fq\n",
      "\t the 2nd reads: /tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome31.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome31.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 2.88\n",
      "\n",
      "The random seed for the run: 1923521921\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            5.80563X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome29.01.fq\n",
      "\t the 2nd reads: /tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome29.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome29.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 2.59\n",
      "\n",
      "The random seed for the run: 387153331\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            5.57772X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome3.01.fq\n",
      "\t the 2nd reads: /tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome3.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome3.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 1.88\n",
      "\n",
      "The random seed for the run: 1203576411\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            2.60344X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome8.01.fq\n",
      "\t the 2nd reads: /tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome8.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpgTkE_V/2021.03.14_08.23.34_sample_0/reads/Genome8.0.sam\n",
      "\n",
      "2021-03-14 08:25:18 INFO: [GenomePreparation 64152656348] Simulating reads finished\n",
      "2021-03-14 08:25:22 INFO: [MetagenomeSimulationPipeline] Generate gold standard assembly\n",
      "2021-03-14 08:25:22 INFO: [MetadataReader 38994325909] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-14 08:25:48 INFO: [MetagenomeSimulationPipeline] Generate pooled strains gold standard assembly\n",
      "2021-03-14 08:25:48 INFO: [MetadataReader 34289848075] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-14 08:25:48 INFO: [MetagenomeSimulationPipeline] Samples used for pooled assembly: '2021.03.14_08.23.34_sample_0'\n",
      "2021-03-14 08:25:48 INFO: [Validator 5358345304] Creating pooled gold standard\n",
      "2021-03-14 08:25:48 INFO: [Validator 5358345304] Merging bam files.\n",
      "2021-03-14 08:25:48 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_08.23.34_sample_0/bam/Genome11.0.bam'\n",
      "2021-03-14 08:25:48 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_08.23.34_sample_0/bam/Genome29.0.bam'\n",
      "2021-03-14 08:25:48 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_08.23.34_sample_0/bam/Genome42.0.bam'\n",
      "2021-03-14 08:25:48 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_08.23.34_sample_0/bam/Genome34.0.bam'\n",
      "2021-03-14 08:25:48 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_08.23.34_sample_0/bam/Genome31.0.bam'\n",
      "2021-03-14 08:25:48 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_08.23.34_sample_0/bam/Genome16.0.bam'\n",
      "2021-03-14 08:25:48 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_08.23.34_sample_0/bam/Genome24.0.bam'\n",
      "2021-03-14 08:25:48 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_08.23.34_sample_0/bam/Genome12.0.bam'\n",
      "2021-03-14 08:25:48 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_08.23.34_sample_0/bam/Genome47.0.bam'\n",
      "2021-03-14 08:25:48 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_08.23.34_sample_0/bam/Genome50.0.bam'\n",
      "2021-03-14 08:25:48 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_08.23.34_sample_0/bam/Genome21.0.bam'\n",
      "2021-03-14 08:25:48 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_08.23.34_sample_0/bam/Genome4.0.bam'\n",
      "2021-03-14 08:25:48 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_08.23.34_sample_0/bam/Genome49.0.bam'\n",
      "2021-03-14 08:25:48 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_08.23.34_sample_0/bam/Genome5.0.bam'\n",
      "2021-03-14 08:25:48 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_08.23.34_sample_0/bam/Genome3.0.bam'\n",
      "2021-03-14 08:25:48 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_08.23.34_sample_0/bam/Genome30.0.bam'\n",
      "2021-03-14 08:25:48 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_08.23.34_sample_0/bam/Genome39.0.bam'\n",
      "2021-03-14 08:25:48 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_08.23.34_sample_0/bam/Genome20.0.bam'\n",
      "2021-03-14 08:25:48 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_08.23.34_sample_0/bam/Genome38.0.bam'\n",
      "2021-03-14 08:25:48 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_08.23.34_sample_0/bam/Genome23.0.bam'\n",
      "2021-03-14 08:25:48 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_08.23.34_sample_0/bam/Genome36.0.bam'\n",
      "2021-03-14 08:25:48 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_08.23.34_sample_0/bam/Genome40.0.bam'\n",
      "2021-03-14 08:25:48 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_08.23.34_sample_0/bam/Genome8.0.bam'\n",
      "2021-03-14 08:25:48 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.14_08.23.34_sample_0/bam/Genome22.0.bam'\n",
      "2021-03-14 08:26:13 INFO: [MetagenomeSimulationPipeline] Anonymize Data\n",
      "2021-03-14 08:26:13 INFO: [FastaAnonymizer] Interweave shuffle and anonymize\n",
      "2021-03-14 08:27:17 INFO: [MetadataReader 55801910510] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-14 08:27:18 INFO: [MetadataReader 35298217545] Reading file: '/output/internal/meta_data.tsv'\n",
      "2021-03-14 08:27:18 INFO: [MetadataReader 50273713801] Reading file: '/tmp/tmpgTkE_V/tmp8o8GNu'\n",
      "2021-03-14 08:27:20 INFO: [FastaAnonymizer 20754120427] Shuffle and anonymize '/tmp/tmpgTkE_V/tmpToiiJe'\n",
      "2021-03-14 08:27:27 INFO: [MetadataReader 64547429207] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-14 08:27:28 INFO: [MetadataReader 10989772577] Reading file: '/output/internal/meta_data.tsv'\n",
      "2021-03-14 08:27:28 INFO: [MetadataReader 53809671431] Reading file: '/tmp/tmpgTkE_V/read_start_positionsgcyQeQ'\n",
      "2021-03-14 08:27:30 INFO: [MetadataReader 18168782768] Reading file: '/tmp/tmpgTkE_V/tmpM2UB3p'\n",
      "2021-03-14 08:27:51 INFO: [FastaAnonymizer 92324153505] Shuffle and anonymize '/tmp/tmpgTkE_V/tmpa3a3Gg'\n",
      "2021-03-14 08:27:57 INFO: [MetadataReader 83768748887] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-14 08:27:58 INFO: [MetadataReader 11249978774] Reading file: '/output/internal/meta_data.tsv'\n",
      "2021-03-14 08:27:58 INFO: [MetadataReader 66861377693] Reading file: '/tmp/tmpgTkE_V/read_start_positionsh1RzYe'\n",
      "2021-03-14 08:28:00 INFO: [MetadataReader 8017839927] Reading file: '/tmp/tmpgTkE_V/tmpOWujXy'\n",
      "2021-03-14 08:28:21 INFO: [MetagenomeSimulationPipeline] Compress Data\n",
      "2021-03-14 08:28:21 INFO: [Compress 63426972432] Compressing 'tmpTBLREf'\n",
      "2021-03-14 08:28:21 INFO: [Compress 10444597134] Compressing 'anonymous_gs_mappingKsKPoB'\n",
      "2021-03-14 08:28:21 INFO: [Compress 53527434578] Compressing 'tmp5eMbrf'\n",
      "2021-03-14 08:28:21 INFO: [Compress 32725058174] Compressing 'anonymous_gsa_mappingxfiC7Q'\n",
      "2021-03-14 08:28:21 INFO: [Compress 95488907652] Compressing 'tmpPgvcip'\n",
      "2021-03-14 08:28:21 INFO: [Compress 32283605537] Compressing 'anonymous_gsa_pooled_mappingEzNsAY'\n",
      "2021-03-14 08:28:21 INFO: [Compress 32283605537] Done compressing 'gsa_pooled_mapping.tsv.gz' in 0:00:00s.\n",
      "2021-03-14 08:28:21 INFO: [Compress 32725058174] Done compressing 'gsa_mapping.tsv.gz' in 0:00:00s.\n",
      "2021-03-14 08:28:23 INFO: [Compress 10444597134] Done compressing 'reads_mapping.tsv.gz' in 0:00:02s.\n",
      "2021-03-14 08:28:23 INFO: [Compress 95488907652] Done compressing 'anonymous_gsa_pooled.fasta.gz' in 0:00:02s.\n",
      "2021-03-14 08:28:23 INFO: [Compress 53527434578] Done compressing 'anonymous_gsa.fasta.gz' in 0:00:02s.\n",
      "2021-03-14 08:28:30 INFO: [Compress 63426972432] Done compressing 'anonymous_reads.fq.gz' in 0:00:09s.\n",
      "2021-03-14 08:28:30 INFO: [MetagenomeSimulationPipeline] Metagenome simulation finished\n"
     ]
    }
   ],
   "source": [
    "experiments = [os.path.basename(i) for i in glob.glob('example_input_data/new_simulations/complexity*')]\n",
    "\n",
    "for experiment in experiments:\n",
    "    if len(glob.glob(os.path.join('example_input_data/new_simulations',experiment,'*','reads','anonymous*'))) == 0:\n",
    "        print('simulating',experiment)\n",
    "        \n",
    "        input_folder = os.path.join(os.getcwd(), f\"example_input_data/new_simulations/{experiment}\")\n",
    "        output_folder = os.path.join(os.getcwd(), f\"example_input_data/new_simulations/{experiment}\")\n",
    "\n",
    "        input_directory = f\"{input_folder}:/input:rw\"\n",
    "        output_directory = f\"{output_folder}:/output:rw\"\n",
    "\n",
    "        !echo UTMBpath123 | sudo -S docker run --rm -v $input_directory -v $output_directory  \\\n",
    "            cami/camisim:latest  metagenomesimulation.py /input/mini_config.ini"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['example_input_data/new_simulations/complexity_sim_10_genera_250_genomes/2021.03.14_07.34.03_sample_0/reads/anonymous_reads.fq.gz',\n",
       " 'example_input_data/new_simulations/complexity_sim_1_genera_250_genomes/2021.03.14_07.42.37_sample_0/reads/anonymous_reads.fq.gz',\n",
       " 'example_input_data/new_simulations/complexity_sim_1_genera_25_genomes/2021.03.14_07.51.47_sample_0/reads/anonymous_reads.fq.gz',\n",
       " 'example_input_data/new_simulations/complexity_sim_5_genera_50_genomes/2021.03.14_07.56.49_sample_0/reads/anonymous_reads.fq.gz',\n",
       " 'example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/2021.03.14_08.01.38_sample_0/reads/anonymous_reads.fq.gz',\n",
       " 'example_input_data/new_simulations/complexity_sim_1_genera_50_genomes/2021.03.14_08.06.23_sample_0/reads/anonymous_reads.fq.gz',\n",
       " 'example_input_data/new_simulations/complexity_sim_5_genera_250_genomes/2021.03.14_08.11.27_sample_0/reads/anonymous_reads.fq.gz',\n",
       " 'example_input_data/new_simulations/complexity_sim_5_genera_25_genomes/2021.03.14_08.19.20_sample_0/reads/anonymous_reads.fq.gz',\n",
       " 'example_input_data/new_simulations/complexity_sim_10_genera_50_genomes/2021.03.14_08.23.34_sample_0/reads/anonymous_reads.fq.gz']"
      ]
     },
     "execution_count": 21,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "glob.glob('example_input_data/new_simulations/com*/*sample_0*/reads/*anonymous_reads*')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Now we just need to run the assemblies\n",
    "\n",
    "Note that many of these steps are described in greater detail in the `preparing_data` notebook."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [],
   "source": [
    "BASE_DIR = os.getcwd()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<vamb.vambtools.Reader object at 0x7f0e2d159070>\n",
      "<_io.TextIOWrapper name='/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_250_genomes/2021.03.14_07.34.03_sample_0/contigs/anonymous_gsa_filtered.fasta.gz' mode='w' encoding='UTF-8'>\n",
      "<vamb.vambtools.Reader object at 0x7f0e2e6c3cd0>\n",
      "<_io.TextIOWrapper name='/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_1_genera_250_genomes/2021.03.14_07.42.37_sample_0/contigs/anonymous_gsa_filtered.fasta.gz' mode='w' encoding='UTF-8'>\n",
      "<vamb.vambtools.Reader object at 0x7f0e2d1598e0>\n",
      "<_io.TextIOWrapper name='/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_1_genera_25_genomes/2021.03.14_07.51.47_sample_0/contigs/anonymous_gsa_filtered.fasta.gz' mode='w' encoding='UTF-8'>\n",
      "<vamb.vambtools.Reader object at 0x7f0e2d159070>\n",
      "<_io.TextIOWrapper name='/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_5_genera_50_genomes/2021.03.14_07.56.49_sample_0/contigs/anonymous_gsa_filtered.fasta.gz' mode='w' encoding='UTF-8'>\n",
      "<vamb.vambtools.Reader object at 0x7f0e2d1594c0>\n",
      "<_io.TextIOWrapper name='/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/2021.03.14_08.01.38_sample_0/contigs/anonymous_gsa_filtered.fasta.gz' mode='w' encoding='UTF-8'>\n",
      "<vamb.vambtools.Reader object at 0x7f0e2d1598e0>\n",
      "<_io.TextIOWrapper name='/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_1_genera_50_genomes/2021.03.14_08.06.23_sample_0/contigs/anonymous_gsa_filtered.fasta.gz' mode='w' encoding='UTF-8'>\n",
      "<vamb.vambtools.Reader object at 0x7f0e2d159070>\n",
      "<_io.TextIOWrapper name='/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_5_genera_250_genomes/2021.03.14_08.11.27_sample_0/contigs/anonymous_gsa_filtered.fasta.gz' mode='w' encoding='UTF-8'>\n",
      "<vamb.vambtools.Reader object at 0x7f0e2d1594c0>\n",
      "<_io.TextIOWrapper name='/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_5_genera_25_genomes/2021.03.14_08.19.20_sample_0/contigs/anonymous_gsa_filtered.fasta.gz' mode='w' encoding='UTF-8'>\n",
      "<vamb.vambtools.Reader object at 0x7f0e2d1598e0>\n",
      "<_io.TextIOWrapper name='/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_50_genomes/2021.03.14_08.23.34_sample_0/contigs/anonymous_gsa_filtered.fasta.gz' mode='w' encoding='UTF-8'>\n"
     ]
    }
   ],
   "source": [
    "for SIM_FASTA_FILE in glob.glob('example_input_data/new_simulations/complexity_sim*/*sample_0*'):\n",
    "    input_contigs_fasta = os.path.join(BASE_DIR, f\"{SIM_FASTA_FILE}/contigs/anonymous_gsa.fasta.gz\")\n",
    "    output_filtered_contigs_fasta = os.path.join(BASE_DIR, f\"{SIM_FASTA_FILE}/contigs/anonymous_gsa_filtered.fasta.gz\")\n",
    "\n",
    "    with vamb.vambtools.Reader(input_contigs_fasta, 'rb') as inputfile:\n",
    "        with open(output_filtered_contigs_fasta, 'w') as outputfile:\n",
    "            print(inputfile)\n",
    "            print(outputfile)\n",
    "            vamb.vambtools.filtercontigs(inputfile, outputfile, minlength=2000)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: example_input_data/new_simulations/complexity_sim_10_genera_250_genomes/2021.03.14_07.34.03_sample_0\n",
      "[M::mm_idx_gen::0.189*1.21] collected minimizers\n",
      "[M::mm_idx_gen::0.252*1.64] sorted minimizers\n",
      "[M::main::0.326*1.50] loaded/built the index for 774 target sequence(s)\n",
      "[M::mm_idx_stat] kmer size: 15; skip: 10; is_hpc: 0; #seq: 774\n",
      "[M::mm_idx_stat::0.337*1.48] distinct minimizers: 1238312 (96.93% are singletons); average occurrences: 1.038; average spacing: 5.364\n",
      "[M::main] Version: 2.17-r941\n",
      "[M::main] CMD: /home/pathinformatics/miniconda3/envs/vamb_env/bin/minimap2 -d example_input_data/new_simulations/complexity_sim_10_genera_250_genomes/2021.03.14_07.34.03_sample_0/catalogue.mmi /home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_250_genomes/2021.03.14_07.34.03_sample_0/contigs/anonymous_gsa_filtered.fasta.gz\n",
      "[M::main] Real time: 0.342 sec; CPU: 0.503 sec; Peak RSS: 0.736 GB\n",
      "minimap index location example_input_data/new_simulations/complexity_sim_10_genera_250_genomes/2021.03.14_07.34.03_sample_0/catalogue.mmi\n",
      "[WARNING]\u001b[1;31m Indexing parameters (-k, -w or -H) overridden by parameters used in the prebuilt index.\u001b[0m\n",
      "[M::main::0.063*1.02] loaded/built the index for 774 target sequence(s)\n",
      "[M::mm_mapopt_update::0.063*1.02] mid_occ = 1000\n",
      "[M::mm_idx_stat] kmer size: 15; skip: 10; is_hpc: 0; #seq: 774\n",
      "[M::mm_idx_stat::0.073*1.02] distinct minimizers: 1238312 (96.93% are singletons); average occurrences: 1.038; average spacing: 5.364\n",
      "[M::worker_pipeline::1.432*4.20] mapped 333334 sequences\n",
      "[M::worker_pipeline::2.156*4.84] mapped 329174 sequences\n",
      "[M::main] Version: 2.17-r941\n",
      "[M::main] CMD: /home/pathinformatics/miniconda3/envs/vamb_env/bin/minimap2 -t 10 -N 50 -ax sr example_input_data/new_simulations/complexity_sim_10_genera_250_genomes/2021.03.14_07.34.03_sample_0/catalogue.mmi example_input_data/new_simulations/complexity_sim_10_genera_250_genomes/2021.03.14_07.34.03_sample_0/reads/anonymous_reads.fq.gz\n",
      "[M::main] Real time: 2.165 sec; CPU: 10.447 sec; Peak RSS: 0.325 GB\n",
      "Processing: example_input_data/new_simulations/complexity_sim_1_genera_250_genomes/2021.03.14_07.42.37_sample_0\n",
      "[M::mm_idx_gen::0.149*1.21] collected minimizers\n",
      "[M::mm_idx_gen::0.205*1.69] sorted minimizers\n",
      "[M::main::0.238*1.59] loaded/built the index for 586 target sequence(s)\n",
      "[M::mm_idx_stat] kmer size: 15; skip: 10; is_hpc: 0; #seq: 586\n",
      "[M::mm_idx_stat::0.249*1.57] distinct minimizers: 977379 (88.24% are singletons); average occurrences: 1.128; average spacing: 5.359\n",
      "[M::main] Version: 2.17-r941\n",
      "[M::main] CMD: /home/pathinformatics/miniconda3/envs/vamb_env/bin/minimap2 -d example_input_data/new_simulations/complexity_sim_1_genera_250_genomes/2021.03.14_07.42.37_sample_0/catalogue.mmi /home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_1_genera_250_genomes/2021.03.14_07.42.37_sample_0/contigs/anonymous_gsa_filtered.fasta.gz\n",
      "[M::main] Real time: 0.254 sec; CPU: 0.395 sec; Peak RSS: 0.736 GB\n",
      "minimap index location example_input_data/new_simulations/complexity_sim_1_genera_250_genomes/2021.03.14_07.42.37_sample_0/catalogue.mmi\n",
      "[WARNING]\u001b[1;31m Indexing parameters (-k, -w or -H) overridden by parameters used in the prebuilt index.\u001b[0m\n",
      "[M::main::0.067*1.02] loaded/built the index for 586 target sequence(s)\n",
      "[M::mm_mapopt_update::0.067*1.02] mid_occ = 1000\n",
      "[M::mm_idx_stat] kmer size: 15; skip: 10; is_hpc: 0; #seq: 586\n",
      "[M::mm_idx_stat::0.077*1.02] distinct minimizers: 977379 (88.24% are singletons); average occurrences: 1.128; average spacing: 5.359\n",
      "[M::worker_pipeline::1.801*6.52] mapped 333334 sequences\n",
      "[M::worker_pipeline::2.486*6.18] mapped 330750 sequences\n",
      "[M::main] Version: 2.17-r941\n",
      "[M::main] CMD: /home/pathinformatics/miniconda3/envs/vamb_env/bin/minimap2 -t 10 -N 50 -ax sr example_input_data/new_simulations/complexity_sim_1_genera_250_genomes/2021.03.14_07.42.37_sample_0/catalogue.mmi example_input_data/new_simulations/complexity_sim_1_genera_250_genomes/2021.03.14_07.42.37_sample_0/reads/anonymous_reads.fq.gz\n",
      "[M::main] Real time: 2.491 sec; CPU: 15.363 sec; Peak RSS: 0.342 GB\n",
      "Processing: example_input_data/new_simulations/complexity_sim_1_genera_25_genomes/2021.03.14_07.51.47_sample_0\n",
      "[M::mm_idx_gen::0.235*1.14] collected minimizers\n",
      "[M::mm_idx_gen::0.326*1.66] sorted minimizers\n",
      "[M::main::0.369*1.58] loaded/built the index for 2051 target sequence(s)\n",
      "[M::mm_idx_stat] kmer size: 15; skip: 10; is_hpc: 0; #seq: 2051\n",
      "[M::mm_idx_stat::0.382*1.56] distinct minimizers: 1264376 (62.07% are singletons); average occurrences: 1.488; average spacing: 5.368\n",
      "[M::main] Version: 2.17-r941\n",
      "[M::main] CMD: /home/pathinformatics/miniconda3/envs/vamb_env/bin/minimap2 -d example_input_data/new_simulations/complexity_sim_1_genera_25_genomes/2021.03.14_07.51.47_sample_0/catalogue.mmi /home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_1_genera_25_genomes/2021.03.14_07.51.47_sample_0/contigs/anonymous_gsa_filtered.fasta.gz\n",
      "[M::main] Real time: 0.387 sec; CPU: 0.600 sec; Peak RSS: 0.736 GB\n",
      "minimap index location example_input_data/new_simulations/complexity_sim_1_genera_25_genomes/2021.03.14_07.51.47_sample_0/catalogue.mmi\n",
      "[WARNING]\u001b[1;31m Indexing parameters (-k, -w or -H) overridden by parameters used in the prebuilt index.\u001b[0m\n",
      "[M::main::0.062*1.02] loaded/built the index for 2051 target sequence(s)\n",
      "[M::mm_mapopt_update::0.062*1.02] mid_occ = 1000\n",
      "[M::mm_idx_stat] kmer size: 15; skip: 10; is_hpc: 0; #seq: 2051\n",
      "[M::mm_idx_stat::0.075*1.02] distinct minimizers: 1264376 (62.07% are singletons); average occurrences: 1.488; average spacing: 5.368\n",
      "[M::worker_pipeline::2.085*7.48] mapped 333334 sequences\n",
      "[M::worker_pipeline::2.978*7.02] mapped 330096 sequences\n",
      "[M::main] Version: 2.17-r941\n",
      "[M::main] CMD: /home/pathinformatics/miniconda3/envs/vamb_env/bin/minimap2 -t 10 -N 50 -ax sr example_input_data/new_simulations/complexity_sim_1_genera_25_genomes/2021.03.14_07.51.47_sample_0/catalogue.mmi example_input_data/new_simulations/complexity_sim_1_genera_25_genomes/2021.03.14_07.51.47_sample_0/reads/anonymous_reads.fq.gz\n",
      "[M::main] Real time: 2.986 sec; CPU: 20.902 sec; Peak RSS: 0.379 GB\n",
      "Processing: example_input_data/new_simulations/complexity_sim_5_genera_50_genomes/2021.03.14_07.56.49_sample_0\n",
      "[M::mm_idx_gen::0.285*1.12] collected minimizers\n",
      "[M::mm_idx_gen::0.379*1.58] sorted minimizers\n",
      "[M::main::0.446*1.49] loaded/built the index for 2516 target sequence(s)\n",
      "[M::mm_idx_stat] kmer size: 15; skip: 10; is_hpc: 0; #seq: 2516\n",
      "[M::mm_idx_stat::0.467*1.47] distinct minimizers: 2162200 (95.69% are singletons); average occurrences: 1.051; average spacing: 5.368\n",
      "[M::main] Version: 2.17-r941\n",
      "[M::main] CMD: /home/pathinformatics/miniconda3/envs/vamb_env/bin/minimap2 -d example_input_data/new_simulations/complexity_sim_5_genera_50_genomes/2021.03.14_07.56.49_sample_0/catalogue.mmi /home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_5_genera_50_genomes/2021.03.14_07.56.49_sample_0/contigs/anonymous_gsa_filtered.fasta.gz\n",
      "[M::main] Real time: 0.473 sec; CPU: 0.693 sec; Peak RSS: 0.736 GB\n",
      "minimap index location example_input_data/new_simulations/complexity_sim_5_genera_50_genomes/2021.03.14_07.56.49_sample_0/catalogue.mmi\n",
      "[WARNING]\u001b[1;31m Indexing parameters (-k, -w or -H) overridden by parameters used in the prebuilt index.\u001b[0m\n",
      "[M::main::0.130*1.01] loaded/built the index for 2516 target sequence(s)\n",
      "[M::mm_mapopt_update::0.131*1.01] mid_occ = 1000\n",
      "[M::mm_idx_stat] kmer size: 15; skip: 10; is_hpc: 0; #seq: 2516\n",
      "[M::mm_idx_stat::0.151*1.01] distinct minimizers: 2162200 (95.69% are singletons); average occurrences: 1.051; average spacing: 5.368\n",
      "[M::worker_pipeline::1.599*4.66] mapped 333334 sequences\n",
      "[M::worker_pipeline::2.297*5.11] mapped 328830 sequences\n",
      "[M::main] Version: 2.17-r941\n",
      "[M::main] CMD: /home/pathinformatics/miniconda3/envs/vamb_env/bin/minimap2 -t 10 -N 50 -ax sr example_input_data/new_simulations/complexity_sim_5_genera_50_genomes/2021.03.14_07.56.49_sample_0/catalogue.mmi example_input_data/new_simulations/complexity_sim_5_genera_50_genomes/2021.03.14_07.56.49_sample_0/reads/anonymous_reads.fq.gz\n",
      "[M::main] Real time: 2.306 sec; CPU: 11.744 sec; Peak RSS: 0.359 GB\n",
      "Processing: example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/2021.03.14_08.01.38_sample_0\n",
      "[M::mm_idx_gen::0.239*1.14] collected minimizers\n",
      "[M::mm_idx_gen::0.333*1.66] sorted minimizers\n",
      "[M::main::0.388*1.56] loaded/built the index for 1629 target sequence(s)\n",
      "[M::mm_idx_stat] kmer size: 15; skip: 10; is_hpc: 0; #seq: 1629\n",
      "[M::mm_idx_stat::0.406*1.54] distinct minimizers: 1703701 (86.60% are singletons); average occurrences: 1.162; average spacing: 5.366\n",
      "[M::main] Version: 2.17-r941\n",
      "[M::main] CMD: /home/pathinformatics/miniconda3/envs/vamb_env/bin/minimap2 -d example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/2021.03.14_08.01.38_sample_0/catalogue.mmi /home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/2021.03.14_08.01.38_sample_0/contigs/anonymous_gsa_filtered.fasta.gz\n",
      "[M::main] Real time: 0.412 sec; CPU: 0.631 sec; Peak RSS: 0.736 GB\n",
      "minimap index location example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/2021.03.14_08.01.38_sample_0/catalogue.mmi\n",
      "[WARNING]\u001b[1;31m Indexing parameters (-k, -w or -H) overridden by parameters used in the prebuilt index.\u001b[0m\n",
      "[M::main::0.102*1.01] loaded/built the index for 1629 target sequence(s)\n",
      "[M::mm_mapopt_update::0.102*1.01] mid_occ = 1000\n",
      "[M::mm_idx_stat] kmer size: 15; skip: 10; is_hpc: 0; #seq: 1629\n",
      "[M::mm_idx_stat::0.120*1.01] distinct minimizers: 1703701 (86.60% are singletons); average occurrences: 1.162; average spacing: 5.366\n",
      "[M::worker_pipeline::1.694*5.61] mapped 333334 sequences\n",
      "[M::worker_pipeline::2.452*5.70] mapped 329350 sequences\n",
      "[M::main] Version: 2.17-r941\n",
      "[M::main] CMD: /home/pathinformatics/miniconda3/envs/vamb_env/bin/minimap2 -t 10 -N 50 -ax sr example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/2021.03.14_08.01.38_sample_0/catalogue.mmi example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/2021.03.14_08.01.38_sample_0/reads/anonymous_reads.fq.gz\n",
      "[M::main] Real time: 2.459 sec; CPU: 13.982 sec; Peak RSS: 0.362 GB\n",
      "Processing: example_input_data/new_simulations/complexity_sim_1_genera_50_genomes/2021.03.14_08.06.23_sample_0\n",
      "[M::mm_idx_gen::0.259*1.12] collected minimizers\n",
      "[M::mm_idx_gen::0.339*1.56] sorted minimizers\n",
      "[M::main::0.435*1.44] loaded/built the index for 2895 target sequence(s)\n",
      "[M::mm_idx_stat] kmer size: 15; skip: 10; is_hpc: 0; #seq: 2895\n",
      "[M::mm_idx_stat::0.454*1.42] distinct minimizers: 1263412 (56.77% are singletons); average occurrences: 1.611; average spacing: 5.375\n",
      "[M::main] Version: 2.17-r941\n",
      "[M::main] CMD: /home/pathinformatics/miniconda3/envs/vamb_env/bin/minimap2 -d example_input_data/new_simulations/complexity_sim_1_genera_50_genomes/2021.03.14_08.06.23_sample_0/catalogue.mmi /home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_1_genera_50_genomes/2021.03.14_08.06.23_sample_0/contigs/anonymous_gsa_filtered.fasta.gz\n",
      "[M::main] Real time: 0.460 sec; CPU: 0.652 sec; Peak RSS: 0.736 GB\n",
      "minimap index location example_input_data/new_simulations/complexity_sim_1_genera_50_genomes/2021.03.14_08.06.23_sample_0/catalogue.mmi\n",
      "[WARNING]\u001b[1;31m Indexing parameters (-k, -w or -H) overridden by parameters used in the prebuilt index.\u001b[0m\n",
      "[M::main::0.084*1.02] loaded/built the index for 2895 target sequence(s)\n",
      "[M::mm_mapopt_update::0.084*1.02] mid_occ = 1000\n",
      "[M::mm_idx_stat] kmer size: 15; skip: 10; is_hpc: 0; #seq: 2895\n",
      "[M::mm_idx_stat::0.098*1.01] distinct minimizers: 1263412 (56.77% are singletons); average occurrences: 1.611; average spacing: 5.375\n",
      "[M::worker_pipeline::2.206*7.52] mapped 333334 sequences\n",
      "[M::worker_pipeline::3.194*7.06] mapped 331980 sequences\n",
      "[M::main] Version: 2.17-r941\n",
      "[M::main] CMD: /home/pathinformatics/miniconda3/envs/vamb_env/bin/minimap2 -t 10 -N 50 -ax sr example_input_data/new_simulations/complexity_sim_1_genera_50_genomes/2021.03.14_08.06.23_sample_0/catalogue.mmi example_input_data/new_simulations/complexity_sim_1_genera_50_genomes/2021.03.14_08.06.23_sample_0/reads/anonymous_reads.fq.gz\n",
      "[M::main] Real time: 3.201 sec; CPU: 22.574 sec; Peak RSS: 0.389 GB\n",
      "Processing: example_input_data/new_simulations/complexity_sim_5_genera_250_genomes/2021.03.14_08.11.27_sample_0\n",
      "[M::mm_idx_gen::0.186*1.18] collected minimizers\n",
      "[M::mm_idx_gen::0.251*1.65] sorted minimizers\n",
      "[M::main::0.326*1.50] loaded/built the index for 568 target sequence(s)\n",
      "[M::mm_idx_stat] kmer size: 15; skip: 10; is_hpc: 0; #seq: 568\n",
      "[M::mm_idx_stat::0.336*1.48] distinct minimizers: 1280562 (96.72% are singletons); average occurrences: 1.040; average spacing: 5.356\n",
      "[M::main] Version: 2.17-r941\n",
      "[M::main] CMD: /home/pathinformatics/miniconda3/envs/vamb_env/bin/minimap2 -d example_input_data/new_simulations/complexity_sim_5_genera_250_genomes/2021.03.14_08.11.27_sample_0/catalogue.mmi /home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_5_genera_250_genomes/2021.03.14_08.11.27_sample_0/contigs/anonymous_gsa_filtered.fasta.gz\n",
      "[M::main] Real time: 0.341 sec; CPU: 0.503 sec; Peak RSS: 0.736 GB\n",
      "minimap index location example_input_data/new_simulations/complexity_sim_5_genera_250_genomes/2021.03.14_08.11.27_sample_0/catalogue.mmi\n",
      "[WARNING]\u001b[1;31m Indexing parameters (-k, -w or -H) overridden by parameters used in the prebuilt index.\u001b[0m\n",
      "[M::main::0.080*1.02] loaded/built the index for 568 target sequence(s)\n",
      "[M::mm_mapopt_update::0.080*1.02] mid_occ = 1000\n",
      "[M::mm_idx_stat] kmer size: 15; skip: 10; is_hpc: 0; #seq: 568\n",
      "[M::mm_idx_stat::0.090*1.01] distinct minimizers: 1280562 (96.72% are singletons); average occurrences: 1.040; average spacing: 5.356\n",
      "[M::worker_pipeline::1.475*3.94] mapped 333334 sequences\n",
      "[M::worker_pipeline::2.189*4.58] mapped 327978 sequences\n",
      "[M::main] Version: 2.17-r941\n",
      "[M::main] CMD: /home/pathinformatics/miniconda3/envs/vamb_env/bin/minimap2 -t 10 -N 50 -ax sr example_input_data/new_simulations/complexity_sim_5_genera_250_genomes/2021.03.14_08.11.27_sample_0/catalogue.mmi example_input_data/new_simulations/complexity_sim_5_genera_250_genomes/2021.03.14_08.11.27_sample_0/reads/anonymous_reads.fq.gz\n",
      "[M::main] Real time: 2.195 sec; CPU: 10.024 sec; Peak RSS: 0.328 GB\n",
      "Processing: example_input_data/new_simulations/complexity_sim_5_genera_25_genomes/2021.03.14_08.19.20_sample_0\n",
      "[M::mm_idx_gen::0.239*1.14] collected minimizers\n",
      "[M::mm_idx_gen::0.336*1.67] sorted minimizers\n",
      "[M::main::0.397*1.57] loaded/built the index for 1768 target sequence(s)\n",
      "[M::mm_idx_stat] kmer size: 15; skip: 10; is_hpc: 0; #seq: 1768\n",
      "[M::mm_idx_stat::0.416*1.54] distinct minimizers: 1848394 (97.59% are singletons); average occurrences: 1.026; average spacing: 5.366\n",
      "[M::main] Version: 2.17-r941\n",
      "[M::main] CMD: /home/pathinformatics/miniconda3/envs/vamb_env/bin/minimap2 -d example_input_data/new_simulations/complexity_sim_5_genera_25_genomes/2021.03.14_08.19.20_sample_0/catalogue.mmi /home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_5_genera_25_genomes/2021.03.14_08.19.20_sample_0/contigs/anonymous_gsa_filtered.fasta.gz\n",
      "[M::main] Real time: 0.422 sec; CPU: 0.648 sec; Peak RSS: 0.736 GB\n",
      "minimap index location example_input_data/new_simulations/complexity_sim_5_genera_25_genomes/2021.03.14_08.19.20_sample_0/catalogue.mmi\n",
      "[WARNING]\u001b[1;31m Indexing parameters (-k, -w or -H) overridden by parameters used in the prebuilt index.\u001b[0m\n",
      "[M::main::0.106*1.01] loaded/built the index for 1768 target sequence(s)\n",
      "[M::mm_mapopt_update::0.106*1.01] mid_occ = 1000\n",
      "[M::mm_idx_stat] kmer size: 15; skip: 10; is_hpc: 0; #seq: 1768\n",
      "[M::mm_idx_stat::0.124*1.01] distinct minimizers: 1848394 (97.59% are singletons); average occurrences: 1.026; average spacing: 5.366\n",
      "[M::worker_pipeline::1.485*4.10] mapped 333334 sequences\n",
      "[M::worker_pipeline::2.240*4.72] mapped 324374 sequences\n",
      "[M::main] Version: 2.17-r941\n",
      "[M::main] CMD: /home/pathinformatics/miniconda3/envs/vamb_env/bin/minimap2 -t 10 -N 50 -ax sr example_input_data/new_simulations/complexity_sim_5_genera_25_genomes/2021.03.14_08.19.20_sample_0/catalogue.mmi example_input_data/new_simulations/complexity_sim_5_genera_25_genomes/2021.03.14_08.19.20_sample_0/reads/anonymous_reads.fq.gz\n",
      "[M::main] Real time: 2.247 sec; CPU: 10.577 sec; Peak RSS: 0.361 GB\n",
      "Processing: example_input_data/new_simulations/complexity_sim_10_genera_50_genomes/2021.03.14_08.23.34_sample_0\n",
      "[M::mm_idx_gen::0.257*1.13] collected minimizers\n",
      "[M::mm_idx_gen::0.340*1.58] sorted minimizers\n",
      "[M::main::0.402*1.49] loaded/built the index for 2425 target sequence(s)\n",
      "[M::mm_idx_stat] kmer size: 15; skip: 10; is_hpc: 0; #seq: 2425\n",
      "[M::mm_idx_stat::0.422*1.47] distinct minimizers: 1973910 (97.77% are singletons); average occurrences: 1.030; average spacing: 5.370\n",
      "[M::main] Version: 2.17-r941\n",
      "[M::main] CMD: /home/pathinformatics/miniconda3/envs/vamb_env/bin/minimap2 -d example_input_data/new_simulations/complexity_sim_10_genera_50_genomes/2021.03.14_08.23.34_sample_0/catalogue.mmi /home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_50_genomes/2021.03.14_08.23.34_sample_0/contigs/anonymous_gsa_filtered.fasta.gz\n",
      "[M::main] Real time: 0.429 sec; CPU: 0.626 sec; Peak RSS: 0.736 GB\n",
      "minimap index location example_input_data/new_simulations/complexity_sim_10_genera_50_genomes/2021.03.14_08.23.34_sample_0/catalogue.mmi\n",
      "[WARNING]\u001b[1;31m Indexing parameters (-k, -w or -H) overridden by parameters used in the prebuilt index.\u001b[0m\n",
      "[M::main::0.116*1.01] loaded/built the index for 2425 target sequence(s)\n",
      "[M::mm_mapopt_update::0.116*1.01] mid_occ = 1000\n",
      "[M::mm_idx_stat] kmer size: 15; skip: 10; is_hpc: 0; #seq: 2425\n",
      "[M::mm_idx_stat::0.135*1.01] distinct minimizers: 1973910 (97.77% are singletons); average occurrences: 1.030; average spacing: 5.370\n",
      "[M::worker_pipeline::1.456*4.03] mapped 333334 sequences\n",
      "[M::worker_pipeline::2.156*4.60] mapped 310130 sequences\n",
      "[M::main] Version: 2.17-r941\n",
      "[M::main] CMD: /home/pathinformatics/miniconda3/envs/vamb_env/bin/minimap2 -t 10 -N 50 -ax sr example_input_data/new_simulations/complexity_sim_10_genera_50_genomes/2021.03.14_08.23.34_sample_0/catalogue.mmi example_input_data/new_simulations/complexity_sim_10_genera_50_genomes/2021.03.14_08.23.34_sample_0/reads/anonymous_reads.fq.gz\n",
      "[M::main] Real time: 2.163 sec; CPU: 9.920 sec; Peak RSS: 0.346 GB\n"
     ]
    }
   ],
   "source": [
    "for SIM_FASTA_FILE in glob.glob('example_input_data/new_simulations/complexity_sim*/*sample_0*'):\n",
    "    print('Processing:', SIM_FASTA_FILE)\n",
    "    minimap2_input = f\"{SIM_FASTA_FILE}/reads/anonymous_reads.fq.gz\"\n",
    "    minimap2_output = f\"{SIM_FASTA_FILE}/assemblies/re_mapped.bam\"\n",
    "\n",
    "    minimap_index_location = f\"{SIM_FASTA_FILE}/catalogue.mmi\"\n",
    "    \n",
    "    output_filtered_contigs_fasta = os.path.join(BASE_DIR, f\"{SIM_FASTA_FILE}/contigs/anonymous_gsa_filtered.fasta.gz\")\n",
    "\n",
    "    !~/miniconda3/envs/vamb_env/bin/minimap2 -d $minimap_index_location $output_filtered_contigs_fasta \n",
    "    \n",
    "    print('minimap index location', minimap_index_location)\n",
    "\n",
    "    if not os.path.exists(os.path.join(SIM_FASTA_FILE,'assemblies')):\n",
    "        os.mkdir(os.path.join(SIM_FASTA_FILE,'assemblies'))\n",
    "\n",
    "    !~/miniconda3/envs/vamb_env/bin/minimap2 \\\n",
    "        -t 10 \\\n",
    "        -N 50 \\\n",
    "        -ax sr $minimap_index_location \\\n",
    "        $minimap2_input | samtools view -F 3584 -b --threads 8 > $minimap2_output"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "And we calculate the TNF and RPKM matrices"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bamfile for rpkm: ['example_input_data/new_simulations/complexity_sim_10_genera_250_genomes/2021.03.14_07.34.03_sample_0/assemblies/re_mapped.bam']\n",
      "Type of rpkms: <class 'numpy.ndarray'> of dtype float32\n",
      "Shape of rpkms (774, 1)\n",
      "tnfs shape: (774, 103)\n",
      "bamfile for rpkm: ['example_input_data/new_simulations/complexity_sim_1_genera_250_genomes/2021.03.14_07.42.37_sample_0/assemblies/re_mapped.bam']\n",
      "Type of rpkms: <class 'numpy.ndarray'> of dtype float32\n",
      "Shape of rpkms (586, 1)\n",
      "tnfs shape: (586, 103)\n",
      "bamfile for rpkm: ['example_input_data/new_simulations/complexity_sim_1_genera_25_genomes/2021.03.14_07.51.47_sample_0/assemblies/re_mapped.bam']\n",
      "Type of rpkms: <class 'numpy.ndarray'> of dtype float32\n",
      "Shape of rpkms (2051, 1)\n",
      "tnfs shape: (2051, 103)\n",
      "bamfile for rpkm: ['example_input_data/new_simulations/complexity_sim_5_genera_50_genomes/2021.03.14_07.56.49_sample_0/assemblies/re_mapped.bam']\n",
      "Type of rpkms: <class 'numpy.ndarray'> of dtype float32\n",
      "Shape of rpkms (2516, 1)\n",
      "tnfs shape: (2516, 103)\n",
      "bamfile for rpkm: ['example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/2021.03.14_08.01.38_sample_0/assemblies/re_mapped.bam']\n",
      "Type of rpkms: <class 'numpy.ndarray'> of dtype float32\n",
      "Shape of rpkms (1629, 1)\n",
      "tnfs shape: (1629, 103)\n",
      "bamfile for rpkm: ['example_input_data/new_simulations/complexity_sim_1_genera_50_genomes/2021.03.14_08.06.23_sample_0/assemblies/re_mapped.bam']\n",
      "Type of rpkms: <class 'numpy.ndarray'> of dtype float32\n",
      "Shape of rpkms (2895, 1)\n",
      "tnfs shape: (2895, 103)\n",
      "bamfile for rpkm: ['example_input_data/new_simulations/complexity_sim_5_genera_250_genomes/2021.03.14_08.11.27_sample_0/assemblies/re_mapped.bam']\n",
      "Type of rpkms: <class 'numpy.ndarray'> of dtype float32\n",
      "Shape of rpkms (568, 1)\n",
      "tnfs shape: (568, 103)\n",
      "bamfile for rpkm: ['example_input_data/new_simulations/complexity_sim_5_genera_25_genomes/2021.03.14_08.19.20_sample_0/assemblies/re_mapped.bam']\n",
      "Type of rpkms: <class 'numpy.ndarray'> of dtype float32\n",
      "Shape of rpkms (1768, 1)\n",
      "tnfs shape: (1768, 103)\n",
      "bamfile for rpkm: ['example_input_data/new_simulations/complexity_sim_10_genera_50_genomes/2021.03.14_08.23.34_sample_0/assemblies/re_mapped.bam']\n",
      "Type of rpkms: <class 'numpy.ndarray'> of dtype float32\n",
      "Shape of rpkms (2425, 1)\n",
      "tnfs shape: (2425, 103)\n"
     ]
    }
   ],
   "source": [
    "for SIM_FASTA_FILE in glob.glob('example_input_data/new_simulations/complexity_sim*/*sample_0*'):\n",
    "    output_filtered_contigs_fasta = f\"{SIM_FASTA_FILE}/contigs/anonymous_gsa_filtered.fasta.gz\"\n",
    "    \n",
    "    # Use Reader to open plain or zipped files. File must be opened in binary mode\n",
    "    with vamb.vambtools.Reader(output_filtered_contigs_fasta, 'rb') as inputfile:\n",
    "        tnfs, contignames, lengths = vamb.parsecontigs.read_contigs(inputfile)\n",
    "        \n",
    "    sample_bamfile = glob.glob(f\"{SIM_FASTA_FILE}/assemblies/re_mapped.bam\")\n",
    "    print(\"bamfile for rpkm:\", sample_bamfile)\n",
    "    \n",
    "    rpkms = vamb.parsebam.read_bamfiles(sample_bamfile) \n",
    "    print('Type of rpkms:', type(rpkms), 'of dtype', rpkms.dtype)\n",
    "    print('Shape of rpkms', rpkms.shape)\n",
    "    \n",
    "    print(\"tnfs shape:\", tnfs.shape)\n",
    "    \n",
    "    \n",
    "    vamb_inputs_base = os.path.join(SIM_FASTA_FILE,'vamb_inputs')\n",
    "    \n",
    "    \n",
    "    if not os.path.exists(vamb_inputs_base):\n",
    "        os.mkdir(vamb_inputs_base)\n",
    "\n",
    "    with open(os.path.join(vamb_inputs_base, 'contignames.npz'), 'wb') as file:\n",
    "        vamb.vambtools.write_npz(file, np.array(contignames))\n",
    "\n",
    "    with open(os.path.join(vamb_inputs_base, 'lengths.npz'), 'wb') as file:\n",
    "        vamb.vambtools.write_npz(file, lengths)\n",
    "\n",
    "    with open(os.path.join(vamb_inputs_base, 'tnfs.npz'), 'wb') as file:\n",
    "        vamb.vambtools.write_npz(file, tnfs)\n",
    "\n",
    "    with open(os.path.join(vamb_inputs_base, 'rpkms.npz'), 'wb') as file:\n",
    "        vamb.vambtools.write_npz(file, rpkms)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "False\n",
      "Binary files example_input_data/new_simulations/complexity_sim_10_genera_250_genomes/2021.03.14_07.34.03_sample_0/assemblies/re_mapped.bam and example_input_data/new_simulations/complexity_sim_1_genera_25_genomes/2021.03.14_07.51.47_sample_0/assemblies/re_mapped.bam differ\n"
     ]
    }
   ],
   "source": [
    "f1, f2 = np.random.choice(glob.glob('example_input_data/new_simulations/complexity_sim*/*sample_0*/*assem*/*re_mapped*'), 2)\n",
    "\n",
    "print(f1 == f2)\n",
    "\n",
    "!diff $f1 $f2"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>num_genera</th>\n",
       "      <th>num_genomes</th>\n",
       "      <th>num_bins</th>\n",
       "      <th>num_mappable_bins</th>\n",
       "      <th>mean_completeness</th>\n",
       "      <th>mean_contamination</th>\n",
       "      <th>beta</th>\n",
       "      <th>dropout</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>10</td>\n",
       "      <td>250</td>\n",
       "      <td>29</td>\n",
       "      <td>7</td>\n",
       "      <td>35.367005</td>\n",
       "      <td>0.387696</td>\n",
       "      <td>1</td>\n",
       "      <td>0.2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1</td>\n",
       "      <td>250</td>\n",
       "      <td>133</td>\n",
       "      <td>9</td>\n",
       "      <td>11.004557</td>\n",
       "      <td>0.410424</td>\n",
       "      <td>1</td>\n",
       "      <td>0.2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>1</td>\n",
       "      <td>25</td>\n",
       "      <td>189</td>\n",
       "      <td>14</td>\n",
       "      <td>16.133731</td>\n",
       "      <td>1.513042</td>\n",
       "      <td>1</td>\n",
       "      <td>0.2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>5</td>\n",
       "      <td>50</td>\n",
       "      <td>75</td>\n",
       "      <td>8</td>\n",
       "      <td>49.573638</td>\n",
       "      <td>5.219702</td>\n",
       "      <td>1</td>\n",
       "      <td>0.2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>10</td>\n",
       "      <td>25</td>\n",
       "      <td>193</td>\n",
       "      <td>20</td>\n",
       "      <td>10.836362</td>\n",
       "      <td>0.224332</td>\n",
       "      <td>1</td>\n",
       "      <td>0.2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>1</td>\n",
       "      <td>50</td>\n",
       "      <td>77</td>\n",
       "      <td>5</td>\n",
       "      <td>35.000355</td>\n",
       "      <td>12.480161</td>\n",
       "      <td>1</td>\n",
       "      <td>0.2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>5</td>\n",
       "      <td>250</td>\n",
       "      <td>59</td>\n",
       "      <td>6</td>\n",
       "      <td>31.815141</td>\n",
       "      <td>0.512513</td>\n",
       "      <td>1</td>\n",
       "      <td>0.2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>5</td>\n",
       "      <td>25</td>\n",
       "      <td>158</td>\n",
       "      <td>9</td>\n",
       "      <td>29.103874</td>\n",
       "      <td>2.806890</td>\n",
       "      <td>1</td>\n",
       "      <td>0.2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>10</td>\n",
       "      <td>50</td>\n",
       "      <td>107</td>\n",
       "      <td>22</td>\n",
       "      <td>15.176583</td>\n",
       "      <td>4.291655</td>\n",
       "      <td>1</td>\n",
       "      <td>0.2</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   num_genera  num_genomes  num_bins  num_mappable_bins  mean_completeness  \\\n",
       "0          10          250        29                  7          35.367005   \n",
       "1           1          250       133                  9          11.004557   \n",
       "2           1           25       189                 14          16.133731   \n",
       "3           5           50        75                  8          49.573638   \n",
       "4          10           25       193                 20          10.836362   \n",
       "5           1           50        77                  5          35.000355   \n",
       "6           5          250        59                  6          31.815141   \n",
       "7           5           25       158                  9          29.103874   \n",
       "8          10           50       107                 22          15.176583   \n",
       "\n",
       "   mean_contamination  beta  dropout  \n",
       "0            0.387696     1      0.2  \n",
       "1            0.410424     1      0.2  \n",
       "2            1.513042     1      0.2  \n",
       "3            5.219702     1      0.2  \n",
       "4            0.224332     1      0.2  \n",
       "5           12.480161     1      0.2  \n",
       "6            0.512513     1      0.2  \n",
       "7            2.806890     1      0.2  \n",
       "8            4.291655     1      0.2  "
      ]
     },
     "execution_count": 35,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pd.read_csv('experiment_beta1/experiment_stats_beta1.tsv', sep='\\t')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "vamb_env",
   "language": "python",
   "name": "vamb_env"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.5"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
