{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "import numpy as np\n",
    "\n",
    "import dendropy\n",
    "\n",
    "import os\n",
    "\n",
    "import pysam\n",
    "import vamb\n",
    "\n",
    "import glob\n",
    "\n",
    "import pickle"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Purpose of this Notebook\n",
    "\n",
    "In order to test the relevant impact of tuning the beta parameter for out disentangled autoencoder, I want to set up some compelling experiments. These will consist of a small grid search methodology where we modify both the number of genomes in a simulation as well as the phylogenetic distance between those genomes. That strategy would look something like this:\n",
    "\n",
    "\n",
    "\n",
    "|     | 2   |  10 | 50  |\n",
    "|-----|-----|-----|-----|\n",
    "| 5   |     |     |     |\n",
    "| 50  |     |     |     |\n",
    "| 250 |     |     |     |\n",
    "\n",
    "\n",
    "In order do do this, I'm going to set up these simulations first by deciding how I'll collect input reference genomes of varying levels of phylogenetic similarity."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 1. Gathering Genomes and Adjusting Similarity\n",
    "\n",
    "We'll start by obtaining the complete list of bacterial genomes present in the NCBI Genome database:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--2021-03-14 00:32:06--  ftp://ftp.ncbi.nlm.nih.gov/genomes/genbank/bacteria/assembly_summary.txt\n",
      "           => ‘assembly_summary.txt’\n",
      "Resolving ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)... 130.14.250.13, 130.14.250.12, 2607:f220:41e:250::10, ...\n",
      "Connecting to ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)|130.14.250.13|:21... connected.\n",
      "Logging in as anonymous ... Logged in!\n",
      "==> SYST ... done.    ==> PWD ... done.\n",
      "==> TYPE I ... done.  ==> CWD (1) /genomes/genbank/bacteria ... done.\n",
      "==> SIZE assembly_summary.txt ... 276853487\n",
      "==> PASV ... done.    ==> RETR assembly_summary.txt ... done.\n",
      "Length: 276853487 (264M) (unauthoritative)\n",
      "\n",
      "assembly_summary.tx 100%[===================>] 264.03M  23.2MB/s    in 12s     \n",
      "\n",
      "2021-03-14 00:32:19 (21.4 MB/s) - ‘assembly_summary.txt’ saved [276853487]\n",
      "\n"
     ]
    }
   ],
   "source": [
    "!wget ftp://ftp.ncbi.nlm.nih.gov/genomes/genbank/bacteria/assembly_summary.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/pathinformatics/miniconda3/envs/vamb_env/lib/python3.8/site-packages/IPython/core/interactiveshell.py:3146: DtypeWarning: Columns (21) have mixed types.Specify dtype option on import or set low_memory=False.\n",
      "  has_raised = await self.run_ast_nodes(code_ast.body, cell_name,\n"
     ]
    }
   ],
   "source": [
    "all_ncbi_genomes = pd.read_csv('assembly_summary.txt', sep='\\t', skiprows=1)\n",
    "all_complete_genomes = all_ncbi_genomes\n",
    "#all_complete_genomes = all_ncbi_genomes[all_ncbi_genomes['assembly_level']=='Complete Genome']\n",
    "all_complete_genomes = all_complete_genomes[(~all_complete_genomes['organism_name'].str.contains('sp.')) & \n",
    "                                            (~all_complete_genomes['organism_name'].str.contains('endosymbiont')) & \n",
    "                                            (~all_complete_genomes['organism_name'].str.contains('candidate'))]\n",
    "all_complete_genomes = all_complete_genomes[pd.notna(all_complete_genomes['excluded_from_refseq'])]\n",
    "\n",
    "all_complete_genomes = all_complete_genomes[['# assembly_accession','wgs_master','taxid','organism_name',\n",
    "                                             'infraspecific_name','ftp_path']].drop_duplicates()\n",
    "\n",
    "all_complete_genomes = all_complete_genomes.groupby('organism_name').first().reset_index()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>organism_name</th>\n",
       "      <th># assembly_accession</th>\n",
       "      <th>wgs_master</th>\n",
       "      <th>taxid</th>\n",
       "      <th>infraspecific_name</th>\n",
       "      <th>ftp_path</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>'Brassica napus' phytoplasma</td>\n",
       "      <td>GCA_003181115.1</td>\n",
       "      <td>QGKT00000000.1</td>\n",
       "      <td>469009</td>\n",
       "      <td>None</td>\n",
       "      <td>ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/003...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>'Candidatus Kapabacteria' thiocyanatum</td>\n",
       "      <td>GCA_001899175.1</td>\n",
       "      <td>MKVH00000000.1</td>\n",
       "      <td>1895771</td>\n",
       "      <td>None</td>\n",
       "      <td>ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/001...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                            organism_name # assembly_accession  \\\n",
       "0            'Brassica napus' phytoplasma      GCA_003181115.1   \n",
       "1  'Candidatus Kapabacteria' thiocyanatum      GCA_001899175.1   \n",
       "\n",
       "       wgs_master    taxid infraspecific_name  \\\n",
       "0  QGKT00000000.1   469009               None   \n",
       "1  MKVH00000000.1  1895771               None   \n",
       "\n",
       "                                            ftp_path  \n",
       "0  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/003...  \n",
       "1  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/001...  "
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "all_complete_genomes.head(2)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Having this, I've also obtained a reference phylogenetic tree which we can use to group genome by differing levels of phylogenetic similarity. The particular tree I'll be using is one we obtained from the [Web Of Life (WOL)](https://biocore.github.io/wol/start) project. I read this in with dendropy and then convert this into a phylogenetic distance matrix `pdm`. This is calculated using the patristic distances between items in the tree and, thus, necessitates a pairwise calculation which can take quite a bit. Moreover, this creates a pretty big pickle file (several GB)."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "# with open('astral.cons.nid.e5p50.nwk', 'r') as treefile:\n",
    "#     tree = dendropy.Tree.get(\n",
    "#             data=treefile.read(),\n",
    "#             schema=\"newick\",\n",
    "#             )\n",
    "    \n",
    "#     pdm = tree.phylogenetic_distance_matrix()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "#with open('bacteria_pdm.pickle', 'wb') as f:\n",
    "#    pickle.dump(pdm, f, pickle.HIGHEST_PROTOCOL)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "I will then use this distance matrix to grab a collection of genomes relevant to each of our experiments. This will be a collection of genomes from 1 genus, from 5 genera, and from 10 genera. I'll first select which genera these are and then I'll create three specimens for each genera set: one with 25 genomes, one with 50 genomes, and one with 250 genomes genomes.\n",
    "\n",
    "So we start with our genera list which represent both common bacterial genera and things that are not totally unique giving us the ability to demonstrate similar vs un-similar genomes."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "genus_set_1 = 'Escherichia'\n",
    "genus_set_2, genus_set_3, genus_set_4, genus_set_5 = 'Vibrio', 'Bacteroides', 'Clostridium', 'Listeria'\n",
    "genus_set_6, genus_set_7, genus_set_8, genus_set_9, genus_set_10 = 'Lactobacillus', 'Staphylococcus', 'Shigella', 'Pseudomonas', 'Bifidobacterium'"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "So finally, I can use these genera labels to create the actual genome lists to use for sample simulation."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Escherichia 2.5 2\n",
      "--2021-03-14 01:23:23--  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000/397/385/GCA_000397385.1_Ec13_E2F4_TOP293-4_120714/GCA_000397385.1_Ec13_E2F4_TOP293-4_120714_genomic.fna.gz\n",
      "           => ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_000397385.1_Ec13_E2F4_TOP293-4_120714_genomic.fna.gz’\n",
      "Resolving ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)... 130.14.250.12, 130.14.250.13, 2607:f220:41e:250::10, ...\n",
      "Connecting to ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)|130.14.250.12|:21... connected.\n",
      "Logging in as anonymous ... Logged in!\n",
      "==> SYST ... done.    ==> PWD ... done.\n",
      "==> TYPE I ... done.  ==> CWD (1) /genomes/all/GCA/000/397/385/GCA_000397385.1_Ec13_E2F4_TOP293-4_120714 ... done.\n",
      "==> SIZE GCA_000397385.1_Ec13_E2F4_TOP293-4_120714_genomic.fna.gz ... 1626284\n",
      "==> PASV ... done.    ==> RETR GCA_000397385.1_Ec13_E2F4_TOP293-4_120714_genomic.fna.gz ... done.\n",
      "Length: 1626284 (1.6M) (unauthoritative)\n",
      "\n",
      "GCA_000397385.1_Ec1 100%[===================>]   1.55M  3.33MB/s    in 0.5s    \n",
      "\n",
      "2021-03-14 01:23:24 (3.33 MB/s) - ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_000397385.1_Ec13_E2F4_TOP293-4_120714_genomic.fna.gz’ saved [1626284]\n",
      "\n",
      "--2021-03-14 01:23:24--  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/011/855/365/GCA_011855365.1_PDT000259510.3/GCA_011855365.1_PDT000259510.3_genomic.fna.gz\n",
      "           => ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_011855365.1_PDT000259510.3_genomic.fna.gz’\n",
      "Resolving ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)... 130.14.250.12, 130.14.250.13, 2607:f220:41e:250::10, ...\n",
      "Connecting to ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)|130.14.250.12|:21... connected.\n",
      "Logging in as anonymous ... Logged in!\n",
      "==> SYST ... done.    ==> PWD ... done.\n",
      "==> TYPE I ... done.  ==> CWD (1) /genomes/all/GCA/011/855/365/GCA_011855365.1_PDT000259510.3 ... done.\n",
      "==> SIZE GCA_011855365.1_PDT000259510.3_genomic.fna.gz ... 1574940\n",
      "==> PASV ... done.    ==> RETR GCA_011855365.1_PDT000259510.3_genomic.fna.gz ... done.\n",
      "Length: 1574940 (1.5M) (unauthoritative)\n",
      "\n",
      "GCA_011855365.1_PDT 100%[===================>]   1.50M  2.71MB/s    in 0.6s    \n",
      "\n",
      "2021-03-14 01:23:25 (2.71 MB/s) - ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_011855365.1_PDT000259510.3_genomic.fna.gz’ saved [1574940]\n",
      "\n",
      "Vibrio 2.5 2\n",
      "--2021-03-14 01:23:26--  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/001/597/655/GCA_001597655.1_ASM159765v1/GCA_001597655.1_ASM159765v1_genomic.fna.gz\n",
      "           => ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_001597655.1_ASM159765v1_genomic.fna.gz’\n",
      "Resolving ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)... 130.14.250.12, 130.14.250.13, 2607:f220:41e:250::10, ...\n",
      "Connecting to ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)|130.14.250.12|:21... connected.\n",
      "Logging in as anonymous ... Logged in!\n",
      "==> SYST ... done.    ==> PWD ... done.\n",
      "==> TYPE I ... done.  ==> CWD (1) /genomes/all/GCA/001/597/655/GCA_001597655.1_ASM159765v1 ... done.\n",
      "==> SIZE GCA_001597655.1_ASM159765v1_genomic.fna.gz ... 1399787\n",
      "==> PASV ... done.    ==> RETR GCA_001597655.1_ASM159765v1_genomic.fna.gz ... done.\n",
      "Length: 1399787 (1.3M) (unauthoritative)\n",
      "\n",
      "GCA_001597655.1_ASM 100%[===================>]   1.33M  3.51MB/s    in 0.4s    \n",
      "\n",
      "2021-03-14 01:23:27 (3.51 MB/s) - ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_001597655.1_ASM159765v1_genomic.fna.gz’ saved [1399787]\n",
      "\n",
      "--2021-03-14 01:23:27--  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000/269/765/GCA_000269765.1_ASM26976v1/GCA_000269765.1_ASM26976v1_genomic.fna.gz\n",
      "           => ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_000269765.1_ASM26976v1_genomic.fna.gz’\n",
      "Resolving ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)... 130.14.250.12, 130.14.250.13, 2607:f220:41e:250::10, ...\n",
      "Connecting to ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)|130.14.250.12|:21... connected.\n",
      "Logging in as anonymous ... Logged in!\n",
      "==> SYST ... done.    ==> PWD ... done.\n",
      "==> TYPE I ... done.  ==> CWD (1) /genomes/all/GCA/000/269/765/GCA_000269765.1_ASM26976v1 ... done.\n",
      "==> SIZE GCA_000269765.1_ASM26976v1_genomic.fna.gz ... 1481534\n",
      "==> PASV ... done.    ==> RETR GCA_000269765.1_ASM26976v1_genomic.fna.gz ... done.\n",
      "Length: 1481534 (1.4M) (unauthoritative)\n",
      "\n",
      "GCA_000269765.1_ASM 100%[===================>]   1.41M  3.66MB/s    in 0.4s    \n",
      "\n",
      "2021-03-14 01:23:28 (3.66 MB/s) - ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_000269765.1_ASM26976v1_genomic.fna.gz’ saved [1481534]\n",
      "\n",
      "Bacteroides 2.5 2\n",
      "--2021-03-14 01:23:28--  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000/438/035/GCA_000438035.1_MGS6/GCA_000438035.1_MGS6_genomic.fna.gz\n",
      "           => ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_000438035.1_MGS6_genomic.fna.gz’\n",
      "Resolving ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)... 130.14.250.12, 130.14.250.13, 2607:f220:41e:250::10, ...\n",
      "Connecting to ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)|130.14.250.12|:21... connected.\n",
      "Logging in as anonymous ... Logged in!\n",
      "==> SYST ... done.    ==> PWD ... done.\n",
      "==> TYPE I ... done.  ==> CWD (1) /genomes/all/GCA/000/438/035/GCA_000438035.1_MGS6 ... done.\n",
      "==> SIZE GCA_000438035.1_MGS6_genomic.fna.gz ... 1151242\n",
      "==> PASV ... done.    ==> RETR GCA_000438035.1_MGS6_genomic.fna.gz ... done.\n",
      "Length: 1151242 (1.1M) (unauthoritative)\n",
      "\n",
      "GCA_000438035.1_MGS 100%[===================>]   1.10M  2.56MB/s    in 0.4s    \n",
      "\n",
      "2021-03-14 01:23:29 (2.56 MB/s) - ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_000438035.1_MGS6_genomic.fna.gz’ saved [1151242]\n",
      "\n",
      "--2021-03-14 01:23:29--  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000/433/375/GCA_000433375.1_MGS32/GCA_000433375.1_MGS32_genomic.fna.gz\n",
      "           => ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_000433375.1_MGS32_genomic.fna.gz’\n",
      "Resolving ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)... 130.14.250.12, 130.14.250.13, 2607:f220:41e:250::10, ...\n",
      "Connecting to ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)|130.14.250.12|:21... connected.\n",
      "Logging in as anonymous ... Logged in!\n",
      "==> SYST ... done.    ==> PWD ... done.\n",
      "==> TYPE I ... done.  ==> CWD (1) /genomes/all/GCA/000/433/375/GCA_000433375.1_MGS32 ... done.\n",
      "==> SIZE GCA_000433375.1_MGS32_genomic.fna.gz ... 1430411\n",
      "==> PASV ... done.    ==> RETR GCA_000433375.1_MGS32_genomic.fna.gz ... done.\n",
      "Length: 1430411 (1.4M) (unauthoritative)\n",
      "\n",
      "GCA_000433375.1_MGS 100%[===================>]   1.36M  3.23MB/s    in 0.4s    \n",
      "\n",
      "2021-03-14 01:23:30 (3.23 MB/s) - ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_000433375.1_MGS32_genomic.fna.gz’ saved [1430411]\n",
      "\n",
      "Clostridium 2.5 2\n",
      "--2021-03-14 01:23:30--  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000/436/455/GCA_000436455.1_MGS132/GCA_000436455.1_MGS132_genomic.fna.gz\n",
      "           => ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_000436455.1_MGS132_genomic.fna.gz’\n",
      "Resolving ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)... 130.14.250.12, 130.14.250.13, 2607:f220:41e:250::10, ...\n",
      "Connecting to ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)|130.14.250.12|:21... connected.\n",
      "Logging in as anonymous ... Logged in!\n",
      "==> SYST ... done.    ==> PWD ... done.\n",
      "==> TYPE I ... done.  ==> CWD (1) /genomes/all/GCA/000/436/455/GCA_000436455.1_MGS132 ... done.\n",
      "==> SIZE GCA_000436455.1_MGS132_genomic.fna.gz ... 1342123\n",
      "==> PASV ... done.    ==> RETR GCA_000436455.1_MGS132_genomic.fna.gz ... done.\n",
      "Length: 1342123 (1.3M) (unauthoritative)\n",
      "\n",
      "GCA_000436455.1_MGS 100%[===================>]   1.28M  3.01MB/s    in 0.4s    \n",
      "\n",
      "2021-03-14 01:23:31 (3.01 MB/s) - ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_000436455.1_MGS132_genomic.fna.gz’ saved [1342123]\n",
      "\n",
      "--2021-03-14 01:23:31--  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/016/902/235/GCA_016902235.1_ASM1690223v1/GCA_016902235.1_ASM1690223v1_genomic.fna.gz\n",
      "           => ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_016902235.1_ASM1690223v1_genomic.fna.gz’\n",
      "Resolving ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)... 130.14.250.12, 130.14.250.13, 2607:f220:41e:250::10, ...\n",
      "Connecting to ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)|130.14.250.12|:21... connected.\n",
      "Logging in as anonymous ... Logged in!\n",
      "==> SYST ... done.    ==> PWD ... done.\n",
      "==> TYPE I ... done.  ==> CWD (1) /genomes/all/GCA/016/902/235/GCA_016902235.1_ASM1690223v1 ... done.\n",
      "==> SIZE GCA_016902235.1_ASM1690223v1_genomic.fna.gz ... 678653\n",
      "==> PASV ... done.    ==> RETR GCA_016902235.1_ASM1690223v1_genomic.fna.gz ... done.\n",
      "Length: 678653 (663K) (unauthoritative)\n",
      "\n",
      "GCA_016902235.1_ASM 100%[===================>] 662.75K  2.04MB/s    in 0.3s    \n",
      "\n",
      "2021-03-14 01:23:32 (2.04 MB/s) - ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_016902235.1_ASM1690223v1_genomic.fna.gz’ saved [678653]\n",
      "\n",
      "Listeria 2.5 2\n",
      "--2021-03-14 01:23:33--  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/004/710/765/GCA_004710765.1_PDT000030589.4/GCA_004710765.1_PDT000030589.4_genomic.fna.gz\n",
      "           => ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_004710765.1_PDT000030589.4_genomic.fna.gz’\n",
      "Resolving ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)... 130.14.250.12, 130.14.250.13, 2607:f220:41e:250::10, ...\n",
      "Connecting to ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)|130.14.250.12|:21... connected.\n",
      "Logging in as anonymous ... Logged in!\n",
      "==> SYST ... done.    ==> PWD ... done.\n",
      "==> TYPE I ... done.  ==> CWD (1) /genomes/all/GCA/004/710/765/GCA_004710765.1_PDT000030589.4 ... done.\n",
      "==> SIZE GCA_004710765.1_PDT000030589.4_genomic.fna.gz ... 858215\n",
      "==> PASV ... done.    ==> RETR GCA_004710765.1_PDT000030589.4_genomic.fna.gz ... done.\n",
      "Length: 858215 (838K) (unauthoritative)\n",
      "\n",
      "GCA_004710765.1_PDT 100%[===================>] 838.10K  2.49MB/s    in 0.3s    \n",
      "\n",
      "2021-03-14 01:23:33 (2.49 MB/s) - ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_004710765.1_PDT000030589.4_genomic.fna.gz’ saved [858215]\n",
      "\n",
      "--2021-03-14 01:23:34--  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000/567/965/GCA_000567965.1_ASM56796v1/GCA_000567965.1_ASM56796v1_genomic.fna.gz\n",
      "           => ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_000567965.1_ASM56796v1_genomic.fna.gz’\n",
      "Resolving ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)... 130.14.250.12, 130.14.250.13, 2607:f220:41e:250::10, ...\n",
      "Connecting to ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)|130.14.250.12|:21... connected.\n",
      "Logging in as anonymous ... Logged in!\n",
      "==> SYST ... done.    ==> PWD ... done.\n",
      "==> TYPE I ... done.  ==> CWD (1) /genomes/all/GCA/000/567/965/GCA_000567965.1_ASM56796v1 ... done.\n",
      "==> SIZE GCA_000567965.1_ASM56796v1_genomic.fna.gz ... 879223\n",
      "==> PASV ... done.    ==> RETR GCA_000567965.1_ASM56796v1_genomic.fna.gz ... done.\n",
      "Length: 879223 (859K) (unauthoritative)\n",
      "\n",
      "GCA_000567965.1_ASM 100%[===================>] 858.62K  2.39MB/s    in 0.4s    \n",
      "\n",
      "2021-03-14 01:23:35 (2.39 MB/s) - ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_000567965.1_ASM56796v1_genomic.fna.gz’ saved [879223]\n",
      "\n",
      "Lactobacillus 2.5 2\n",
      "--2021-03-14 01:23:35--  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/001/311/525/GCA_001311525.1_ASM131152v1/GCA_001311525.1_ASM131152v1_genomic.fna.gz\n",
      "           => ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_001311525.1_ASM131152v1_genomic.fna.gz’\n",
      "Resolving ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)... 130.14.250.12, 130.14.250.13, 2607:f220:41e:250::10, ...\n",
      "Connecting to ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)|130.14.250.12|:21... connected.\n",
      "Logging in as anonymous ... Logged in!\n",
      "==> SYST ... done.    ==> PWD ... done.\n",
      "==> TYPE I ... done.  ==> CWD (1) /genomes/all/GCA/001/311/525/GCA_001311525.1_ASM131152v1 ... done.\n",
      "==> SIZE GCA_001311525.1_ASM131152v1_genomic.fna.gz ... 554526\n",
      "==> PASV ... done.    ==> RETR GCA_001311525.1_ASM131152v1_genomic.fna.gz ... done.\n",
      "Length: 554526 (542K) (unauthoritative)\n",
      "\n",
      "GCA_001311525.1_ASM 100%[===================>] 541.53K  1.68MB/s    in 0.3s    \n",
      "\n",
      "2021-03-14 01:23:36 (1.68 MB/s) - ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_001311525.1_ASM131152v1_genomic.fna.gz’ saved [554526]\n",
      "\n",
      "--2021-03-14 01:23:36--  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000/165/775/GCA_000165775.3_ASM16577v3/GCA_000165775.3_ASM16577v3_genomic.fna.gz\n",
      "           => ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_000165775.3_ASM16577v3_genomic.fna.gz’\n",
      "Resolving ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)... 130.14.250.12, 130.14.250.13, 2607:f220:41e:250::10, ...\n",
      "Connecting to ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)|130.14.250.12|:21... connected.\n",
      "Logging in as anonymous ... Logged in!\n",
      "==> SYST ... done.    ==> PWD ... done.\n",
      "==> TYPE I ... done.  ==> CWD (1) /genomes/all/GCA/000/165/775/GCA_000165775.3_ASM16577v3 ... done.\n",
      "==> SIZE GCA_000165775.3_ASM16577v3_genomic.fna.gz ... 623427\n",
      "==> PASV ... done.    ==> RETR GCA_000165775.3_ASM16577v3_genomic.fna.gz ... done.\n",
      "Length: 623427 (609K) (unauthoritative)\n",
      "\n",
      "GCA_000165775.3_ASM 100%[===================>] 608.82K  1.82MB/s    in 0.3s    \n",
      "\n",
      "2021-03-14 01:23:37 (1.82 MB/s) - ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_000165775.3_ASM16577v3_genomic.fna.gz’ saved [623427]\n",
      "\n",
      "Staphylococcus 2.5 2\n",
      "--2021-03-14 01:23:37--  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/002/902/575/GCA_002902575.1_ASM290257v1/GCA_002902575.1_ASM290257v1_genomic.fna.gz\n",
      "           => ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_002902575.1_ASM290257v1_genomic.fna.gz’\n",
      "Resolving ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)... 130.14.250.12, 130.14.250.13, 2607:f220:41e:250::10, ...\n",
      "Connecting to ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)|130.14.250.12|:21... connected.\n",
      "Logging in as anonymous ... Logged in!\n",
      "==> SYST ... done.    ==> PWD ... done.\n",
      "==> TYPE I ... done.  ==> CWD (1) /genomes/all/GCA/002/902/575/GCA_002902575.1_ASM290257v1 ... done.\n",
      "==> SIZE GCA_002902575.1_ASM290257v1_genomic.fna.gz ... 704198\n",
      "==> PASV ... done.    ==> RETR GCA_002902575.1_ASM290257v1_genomic.fna.gz ... done.\n",
      "Length: 704198 (688K) (unauthoritative)\n",
      "\n",
      "GCA_002902575.1_ASM 100%[===================>] 687.69K  1.93MB/s    in 0.3s    \n",
      "\n",
      "2021-03-14 01:23:38 (1.93 MB/s) - ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_002902575.1_ASM290257v1_genomic.fna.gz’ saved [704198]\n",
      "\n",
      "--2021-03-14 01:23:38--  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000/555/645/GCA_000555645.1_Stap_aure_M0406_V1/GCA_000555645.1_Stap_aure_M0406_V1_genomic.fna.gz\n",
      "           => ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_000555645.1_Stap_aure_M0406_V1_genomic.fna.gz’\n",
      "Resolving ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)... 130.14.250.12, 130.14.250.13, 2607:f220:41e:250::10, ...\n",
      "Connecting to ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)|130.14.250.12|:21... connected.\n",
      "Logging in as anonymous ... Logged in!\n",
      "==> SYST ... done.    ==> PWD ... done.\n",
      "==> TYPE I ... done.  ==> CWD (1) /genomes/all/GCA/000/555/645/GCA_000555645.1_Stap_aure_M0406_V1 ... done.\n",
      "==> SIZE GCA_000555645.1_Stap_aure_M0406_V1_genomic.fna.gz ... 919251\n",
      "==> PASV ... done.    ==> RETR GCA_000555645.1_Stap_aure_M0406_V1_genomic.fna.gz ... done.\n",
      "Length: 919251 (898K) (unauthoritative)\n",
      "\n",
      "GCA_000555645.1_Sta 100%[===================>] 897.71K  2.20MB/s    in 0.4s    \n",
      "\n",
      "2021-03-14 01:23:39 (2.20 MB/s) - ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_000555645.1_Stap_aure_M0406_V1_genomic.fna.gz’ saved [919251]\n",
      "\n",
      "Shigella 2.5 2\n",
      "--2021-03-14 01:23:39--  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000/268/085/GCA_000268085.1_ASM26808v1/GCA_000268085.1_ASM26808v1_genomic.fna.gz\n",
      "           => ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_000268085.1_ASM26808v1_genomic.fna.gz’\n",
      "Resolving ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)... 130.14.250.12, 130.14.250.13, 2607:f220:41e:250::10, ...\n",
      "Connecting to ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)|130.14.250.12|:21... connected.\n",
      "Logging in as anonymous ... Logged in!\n",
      "==> SYST ... done.    ==> PWD ... done.\n",
      "==> TYPE I ... done.  ==> CWD (1) /genomes/all/GCA/000/268/085/GCA_000268085.1_ASM26808v1 ... done.\n",
      "==> SIZE GCA_000268085.1_ASM26808v1_genomic.fna.gz ... 1415441\n",
      "==> PASV ... done.    ==> RETR GCA_000268085.1_ASM26808v1_genomic.fna.gz ... done.\n",
      "Length: 1415441 (1.3M) (unauthoritative)\n",
      "\n",
      "GCA_000268085.1_ASM 100%[===================>]   1.35M  3.04MB/s    in 0.4s    \n",
      "\n",
      "2021-03-14 01:23:40 (3.04 MB/s) - ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_000268085.1_ASM26808v1_genomic.fna.gz’ saved [1415441]\n",
      "\n",
      "--2021-03-14 01:23:41--  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000/268/165/GCA_000268165.1_ASM26816v1/GCA_000268165.1_ASM26816v1_genomic.fna.gz\n",
      "           => ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_000268165.1_ASM26816v1_genomic.fna.gz’\n",
      "Resolving ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)... 130.14.250.12, 130.14.250.13, 2607:f220:41e:250::10, ...\n",
      "Connecting to ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)|130.14.250.12|:21... connected.\n",
      "Logging in as anonymous ... Logged in!\n",
      "==> SYST ... done.    ==> PWD ... done.\n",
      "==> TYPE I ... done.  ==> CWD (1) /genomes/all/GCA/000/268/165/GCA_000268165.1_ASM26816v1 ... done.\n",
      "==> SIZE GCA_000268165.1_ASM26816v1_genomic.fna.gz ... 1325670\n",
      "==> PASV ... done.    ==> RETR GCA_000268165.1_ASM26816v1_genomic.fna.gz ... done.\n",
      "Length: 1325670 (1.3M) (unauthoritative)\n",
      "\n",
      "GCA_000268165.1_ASM 100%[===================>]   1.26M  2.67MB/s    in 0.5s    \n",
      "\n",
      "2021-03-14 01:23:42 (2.67 MB/s) - ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_000268165.1_ASM26816v1_genomic.fna.gz’ saved [1325670]\n",
      "\n",
      "Pseudomonas 2.5 2\n",
      "--2021-03-14 01:23:42--  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000/953/455/GCA_000953455.1_Ppseudo_Pac/GCA_000953455.1_Ppseudo_Pac_genomic.fna.gz\n",
      "           => ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_000953455.1_Ppseudo_Pac_genomic.fna.gz’\n",
      "Resolving ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)... 130.14.250.12, 130.14.250.13, 2607:f220:41e:250::10, ...\n",
      "Connecting to ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)|130.14.250.12|:21... connected.\n",
      "Logging in as anonymous ... Logged in!\n",
      "==> SYST ... done.    ==> PWD ... done.\n",
      "==> TYPE I ... done.  ==> CWD (1) /genomes/all/GCA/000/953/455/GCA_000953455.1_Ppseudo_Pac ... done.\n",
      "==> SIZE GCA_000953455.1_Ppseudo_Pac_genomic.fna.gz ... 1366143\n",
      "==> PASV ... done.    ==> RETR GCA_000953455.1_Ppseudo_Pac_genomic.fna.gz ... done.\n",
      "Length: 1366143 (1.3M) (unauthoritative)\n",
      "\n",
      "GCA_000953455.1_Pps 100%[===================>]   1.30M  2.99MB/s    in 0.4s    \n",
      "\n",
      "2021-03-14 01:23:43 (2.99 MB/s) - ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_000953455.1_Ppseudo_Pac_genomic.fna.gz’ saved [1366143]\n",
      "\n",
      "--2021-03-14 01:23:43--  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/009/911/885/GCA_009911885.1_ASM991188v1/GCA_009911885.1_ASM991188v1_genomic.fna.gz\n",
      "           => ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_009911885.1_ASM991188v1_genomic.fna.gz’\n",
      "Resolving ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)... 130.14.250.12, 130.14.250.13, 2607:f220:41e:250::10, ...\n",
      "Connecting to ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)|130.14.250.12|:21... connected.\n",
      "Logging in as anonymous ... Logged in!\n",
      "==> SYST ... done.    ==> PWD ... done.\n",
      "==> TYPE I ... done.  ==> CWD (1) /genomes/all/GCA/009/911/885/GCA_009911885.1_ASM991188v1 ... done.\n",
      "==> SIZE GCA_009911885.1_ASM991188v1_genomic.fna.gz ... 104893\n",
      "==> PASV ... done.    ==> RETR GCA_009911885.1_ASM991188v1_genomic.fna.gz ... done.\n",
      "Length: 104893 (102K) (unauthoritative)\n",
      "\n",
      "GCA_009911885.1_ASM 100%[===================>] 102.43K  --.-KB/s    in 0.1s    \n",
      "\n",
      "2021-03-14 01:23:44 (742 KB/s) - ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_009911885.1_ASM991188v1_genomic.fna.gz’ saved [104893]\n",
      "\n",
      "Bifidobacterium 2.5 2\n",
      "--2021-03-14 01:23:44--  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/002/311/215/GCA_002311215.1_ASM231121v1/GCA_002311215.1_ASM231121v1_genomic.fna.gz\n",
      "           => ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_002311215.1_ASM231121v1_genomic.fna.gz’\n",
      "Resolving ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)... 130.14.250.12, 130.14.250.13, 2607:f220:41e:250::10, ...\n",
      "Connecting to ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)|130.14.250.12|:21... connected.\n",
      "Logging in as anonymous ... Logged in!\n",
      "==> SYST ... done.    ==> PWD ... done.\n",
      "==> TYPE I ... done.  ==> CWD (1) /genomes/all/GCA/002/311/215/GCA_002311215.1_ASM231121v1 ... done.\n",
      "==> SIZE GCA_002311215.1_ASM231121v1_genomic.fna.gz ... 523383\n",
      "==> PASV ... done.    ==> RETR GCA_002311215.1_ASM231121v1_genomic.fna.gz ... done.\n",
      "Length: 523383 (511K) (unauthoritative)\n",
      "\n",
      "GCA_002311215.1_ASM 100%[===================>] 511.12K  1.57MB/s    in 0.3s    \n",
      "\n",
      "2021-03-14 01:23:45 (1.57 MB/s) - ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_002311215.1_ASM231121v1_genomic.fna.gz’ saved [523383]\n",
      "\n",
      "--2021-03-14 01:23:45--  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000/435/235/GCA_000435235.1_MGS119/GCA_000435235.1_MGS119_genomic.fna.gz\n",
      "           => ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_000435235.1_MGS119_genomic.fna.gz’\n",
      "Resolving ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)... 130.14.250.12, 130.14.250.13, 2607:f220:41e:250::10, ...\n",
      "Connecting to ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)|130.14.250.12|:21... connected.\n",
      "Logging in as anonymous ... Logged in!\n",
      "==> SYST ... done.    ==> PWD ... done.\n",
      "==> TYPE I ... done.  ==> CWD (1) /genomes/all/GCA/000/435/235/GCA_000435235.1_MGS119 ... done.\n",
      "==> SIZE GCA_000435235.1_MGS119_genomic.fna.gz ... 570236\n",
      "==> PASV ... done.    ==> RETR GCA_000435235.1_MGS119_genomic.fna.gz ... done.\n",
      "Length: 570236 (557K) (unauthoritative)\n",
      "\n",
      "GCA_000435235.1_MGS 100%[===================>] 556.87K  1.74MB/s    in 0.3s    \n",
      "\n",
      "2021-03-14 01:23:46 (1.74 MB/s) - ‘/home/pathinformatics/jupyter_projects/vamb/stanford_cs230_project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/genomes/GCA_000435235.1_MGS119_genomic.fna.gz’ saved [570236]\n",
      "\n"
     ]
    }
   ],
   "source": [
    "for idx, genus_set in enumerate([[genus_set_1], \n",
    "                  [genus_set_1, genus_set_2, genus_set_3, genus_set_4, genus_set_5], \n",
    "                  [genus_set_1, genus_set_2, genus_set_3, genus_set_4, genus_set_5, genus_set_6, genus_set_7, genus_set_8, genus_set_9, genus_set_10]]):\n",
    "    \n",
    "    for num_genomes in [25, 50, 250]:\n",
    "        if idx == 0:\n",
    "            num_genera = 1\n",
    "        else:\n",
    "            num_genera = idx*5\n",
    "        complexity_sim_folder = os.path.join(os.getcwd(), f\"example_input_data/new_simulations/complexity_sim_{num_genera}_genera_{num_genomes}_genomes\")\n",
    "        if not os.path.exists(complexity_sim_folder):\n",
    "            os.mkdir(complexity_sim_folder)\n",
    "            os.mkdir(os.path.join(complexity_sim_folder, 'genomes'))\n",
    "\n",
    "            for genus in genus_set:\n",
    "                genus_subset = all_complete_genomes[all_complete_genomes['organism_name'].str.startswith(genus)]\n",
    "\n",
    "                per_genus_sample_size = num_genomes/len(genus_set)\n",
    "                genomes = genus_subset.sample(int(per_genus_sample_size))\n",
    "                print(genus, per_genus_sample_size, genomes['ftp_path'].nunique())\n",
    "\n",
    "                for ftp_path in genomes['ftp_path'].values:\n",
    "                    genome_path = ftp_path + f\"/{os.path.basename(ftp_path)}_genomic.fna.gz\"\n",
    "                    genome_folder = os.path.join(complexity_sim_folder, 'genomes')\n",
    "                    !wget $genome_path -P $genome_folder"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "I'll also just verify that we got the right number of genomes downloaded and the right number of genera represented in each of these experiments."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1_genera_25_genomes\n",
      "25\n",
      "1_genera_50_genomes\n",
      "50\n",
      "1_genera_250_genomes\n",
      "250\n",
      "5_genera_25_genomes\n",
      "25\n",
      "5_genera_50_genomes\n",
      "50\n",
      "5_genera_250_genomes\n",
      "250\n",
      "10_genera_25_genomes\n",
      "25\n",
      "10_genera_50_genomes\n",
      "50\n",
      "10_genera_250_genomes\n",
      "250\n"
     ]
    }
   ],
   "source": [
    "for idx, genus_set in enumerate([[genus_set_1], \n",
    "                  [genus_set_1, genus_set_2, genus_set_3, genus_set_4, genus_set_5], \n",
    "                  [genus_set_1, genus_set_2, genus_set_3, genus_set_4, genus_set_5, genus_set_6, genus_set_7, genus_set_8, genus_set_9, genus_set_10]]):\n",
    "    \n",
    "    for num_genomes in [25, 50, 250]:\n",
    "        param_thing = f\"{len(genus_set)}_genera_{num_genomes}_genomes\"\n",
    "        print(param_thing)\n",
    "\n",
    "        print(len(glob.glob(f\"example_input_data/new_simulations/complexity_sim_{param_thing}/genomes/*\")) )\n",
    "\n",
    "        targets = ['_'.join(i.split('/')[-1].strip('.fna.gz').split('_')[:2]) for \n",
    "                   i in glob.glob(f\"example_input_data/new_simulations/complexity_sim_{param_thing}/genomes/*\")]\n",
    "        \n",
    "        assert len(targets) == num_genomes\n",
    "\n",
    "        all_complete_genomes[all_complete_genomes['# assembly_accession'].isin(targets)]['organism_name'].str.contains(genus_set_1).all()\n",
    "        \n",
    "        assert len(set(genus_set).intersection(\n",
    "            set([i.split(' ')[0] for \n",
    "                 i in all_complete_genomes[all_complete_genomes['# assembly_accession'].isin(targets)]['organism_name'].values]))) == len(genus_set)    "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "We also have to set up the genomes and metadata tsv files for simulation"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "for complexity_experiment in glob.glob('example_input_data/new_simulations/complexity*'):\n",
    "    experiment_name = os.path.basename(complexity_experiment)\n",
    "    \n",
    "    for fna_file in glob.glob(f\"example_input_data/new_simulations/{experiment_name}/genomes/*.gz\"):\n",
    "        !gzip -d $fna_file\n",
    "    \n",
    "    genomes = glob.glob(os.path.join(complexity_experiment,'genomes','*'))\n",
    "    genomes_mod = [os.path.join('/input/genomes',os.path.basename(i)) for i in genomes]\n",
    "    \n",
    "    df = pd.DataFrame(genomes_mod)\n",
    "    df['genome_ID'] = [ f\"Genome{float(i+1)}\" for i in range(len(df))]\n",
    "    df = df[['genome_ID',0]]\n",
    "    df.to_csv(os.path.join(complexity_experiment,'genome_to_id.tsv'), sep='\\t', header=False, index=False)\n",
    "    \n",
    "    metadata_df = df\n",
    "    metadata_df['novelty_category'] = 'known_strain'\n",
    "    metadata_df['NCBI_ID'] = metadata_df[0].str.split('/', expand=True)[3].str.split('_', expand=True)[[0,1]].agg('_'.join, axis=1)\n",
    "    \n",
    "    metadata_df = metadata_df.merge(all_complete_genomes, left_on='NCBI_ID', \n",
    "                      right_on='# assembly_accession', \n",
    "                      how='inner')\n",
    "    metadata_df['NCBI_ID'] = metadata_df['ftp_path'].str.split('/', expand=True)[9].str.strip('.fna.gz')\n",
    "    \n",
    "    metadata_df = metadata_df[['genome_ID','NCBI_ID','taxid','novelty_category']].rename(columns={'NCBI_ID':'OTU','taxid':'NCBI_ID',})\n",
    "    metadata_df['NCBI_ID'] = 2\n",
    "    \n",
    "    metadata_df.to_csv(os.path.join(complexity_experiment,'metadata.tsv'), sep='\\t', index=False)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "And now I can run this to create the simulated samples"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 301,
   "metadata": {
    "collapsed": true,
    "jupyter": {
     "outputs_hidden": true
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "simulating complexity_sim_1_genera_250_genomes\n",
      "Password:2021-03-13 09:38:14 INFO: [MetagenomeSimulationPipeline] Metagenome simulation starting\n",
      "2021-03-13 09:38:14 INFO: [MetagenomeSimulationPipeline] Validating Genomes\n",
      "2021-03-13 09:38:14 INFO: [MetadataReader] Reading file: '/input/genome_to_id.tsv'\n",
      "2021-03-13 09:43:56 INFO: [MetagenomeSimulationPipeline] Design Communities\n",
      "2021-03-13 09:43:56 INFO: [CommunityDesign] Drawing strains.\n",
      "2021-03-13 09:43:56 INFO: [MetadataReader 82380310288] Reading file: '/input/metadata.tsv'\n",
      "2021-03-13 09:43:56 INFO: [MetadataReader 28782960663] Reading file: '/input/genome_to_id.tsv'\n",
      "2021-03-13 09:43:56 INFO: [CommunityDesign] Validating raw sequence files!\n",
      "2021-03-13 09:44:34 INFO: [NcbiTaxonomy] Building taxonomy tree...\n",
      "2021-03-13 09:44:34 INFO: [NcbiTaxonomy] Reading 'nodes' file:\t'/tmp/tmpHCLv64/NCBI/nodes.dmp'\n",
      "2021-03-13 09:44:50 INFO: [NcbiTaxonomy] Reading 'names' file:\t'/tmp/tmpHCLv64/NCBI/names.dmp'\n",
      "2021-03-13 09:44:53 INFO: [NcbiTaxonomy] Reading 'merged' file:\t'/tmp/tmpHCLv64/NCBI/merged.dmp'\n",
      "2021-03-13 09:44:53 INFO: [NcbiTaxonomy] Done (19.0s)\n",
      "2021-03-13 09:44:53 INFO: [MetagenomeSimulationPipeline] Move Genomes\n",
      "2021-03-13 09:45:00 INFO: [MetagenomeSimulationPipeline] Read simulation\n",
      "2021-03-13 09:45:00 INFO: [GenomePreparation 82430254076] Using 'mbarc' error profile.\n",
      "2021-03-13 09:45:00 INFO: [GenomePreparation 82430254076] Reading distribution file\n",
      "2021-03-13 09:45:00 INFO: [GenomePreparation 82430254076] Reading genome location file\n",
      "2021-03-13 09:45:11 INFO: [GenomePreparation 82430254076] Genome 'Genome119.0' has sequences below minimum, creating filtered copy.\n",
      "2021-03-13 09:45:17 INFO: [GenomePreparation 82430254076] Genome 'Genome28.0' has sequences below minimum, creating filtered copy.\n",
      "2021-03-13 09:45:24 INFO: [GenomePreparation 82430254076] Genome 'Genome124.0' has sequences below minimum, creating filtered copy.\n",
      "2021-03-13 09:45:37 INFO: [GenomePreparation 82430254076] Simulating reads using ReadSimulationArtIllumina readsimulator...\n",
      "2021-03-13 09:45:37 INFO: [GenomePreparation 82430254076] Simulating reads from Genome33.0: '/output/source_genomes/GCA_002741215.1_ASM274121v1_genomic.fna'\n",
      "2021-03-13 09:45:37 INFO: [GenomePreparation 82430254076] Simulating reads from Genome201.0: '/output/source_genomes/GCA_013049715.1_PDT000290410.4_genomic.fna'\n",
      "2021-03-13 09:45:37 INFO: [GenomePreparation 82430254076] Simulating reads from Genome15.0: '/output/source_genomes/GCA_016954795.1_PDT000971232.1_genomic.fna'\n",
      "2021-03-13 09:45:37 INFO: [GenomePreparation 82430254076] Simulating reads from Genome188.0: '/output/source_genomes/GCA_011855365.1_PDT000259510.3_genomic.fna'\n",
      "2021-03-13 09:45:37 INFO: [GenomePreparation 82430254076] Simulating reads from Genome12.0: '/output/source_genomes/GCA_012246945.1_PDT000290529.2_genomic.fna'\n",
      "2021-03-13 09:45:37 INFO: [GenomePreparation 82430254076] Simulating reads from Genome216.0: '/output/source_genomes/GCA_000176655.2_ASM17665v2_genomic.fna'\n",
      "2021-03-13 09:45:37 INFO: [GenomePreparation 82430254076] Simulating reads from Genome77.0: '/output/source_genomes/GCA_013076425.1_PDT000186183.4_genomic.fna'\n",
      "2021-03-13 09:45:37 INFO: [GenomePreparation 82430254076] Simulating reads from Genome119.0: '/tmp/tmpubQJEs/tmpIIljmM'\n",
      "2021-03-13 09:45:37 INFO: [GenomePreparation 82430254076] Simulating reads from Genome189.0: '/output/source_genomes/GCA_902729235.2_easoj011_ecoli_genomic_umi_ont_r10_g324hac_genomic.fna'\n",
      "2021-03-13 09:45:37 INFO: [GenomePreparation 82430254076] Simulating reads from Genome28.0: '/tmp/tmpubQJEs/tmpIgf6wJ'\n",
      "2021-03-13 09:45:37 INFO: [GenomePreparation 82430254076] Simulating reads from Genome207.0: '/output/source_genomes/GCA_012246885.1_PDT000290429.2_genomic.fna'\n",
      "2021-03-13 09:45:37 INFO: [GenomePreparation 82430254076] Simulating reads from Genome219.0: '/output/source_genomes/GCA_009002655.1_ASM900265v1_genomic.fna'\n",
      "2021-03-13 09:45:37 INFO: [GenomePreparation 82430254076] Simulating reads from Genome104.0: '/output/source_genomes/GCA_000249835.2_ASM24983v2_genomic.fna'\n",
      "2021-03-13 09:45:37 INFO: [GenomePreparation 82430254076] Simulating reads from Genome74.0: '/output/source_genomes/GCA_012247285.1_PDT000290389.2_genomic.fna'\n",
      "2021-03-13 09:45:37 INFO: [GenomePreparation 82430254076] Simulating reads from Genome124.0: '/tmp/tmpubQJEs/tmpQCl6xZ'\n",
      "2021-03-13 09:45:37 INFO: [GenomePreparation 82430254076] Simulating reads from Genome231.0: '/output/source_genomes/GCA_016954935.1_PDT000971224.1_genomic.fna'\n",
      "2021-03-13 09:45:37 INFO: [GenomePreparation 82430254076] Simulating reads from Genome52.0: '/output/source_genomes/GCA_016956075.1_PDT000971141.1_genomic.fna'\n",
      "2021-03-13 09:45:37 INFO: [GenomePreparation 82430254076] Simulating reads from Genome205.0: '/output/source_genomes/GCA_016955215.1_PDT000971186.1_genomic.fna'\n",
      "2021-03-13 09:45:37 INFO: [GenomePreparation 82430254076] Simulating reads from Genome2.0: '/output/source_genomes/GCA_016956155.1_PDT000971137.1_genomic.fna'\n",
      "2021-03-13 09:45:37 INFO: [GenomePreparation 82430254076] Simulating reads from Genome93.0: '/output/source_genomes/GCA_016956115.1_PDT000971138.1_genomic.fna'\n",
      "2021-03-13 09:45:37 INFO: [GenomePreparation 82430254076] Simulating reads from Genome140.0: '/output/source_genomes/GCA_012247505.1_PDT000290375.2_genomic.fna'\n",
      "2021-03-13 09:45:37 INFO: [GenomePreparation 82430254076] Simulating reads from Genome23.0: '/output/source_genomes/GCA_016955775.1_PDT000971156.1_genomic.fna'\n",
      "2021-03-13 09:45:37 INFO: [GenomePreparation 82430254076] Simulating reads from Genome171.0: '/output/source_genomes/GCA_000601175.1_ASM60117v1_genomic.fna'\n",
      "2021-03-13 09:45:37 INFO: [GenomePreparation 82430254076] Simulating reads from Genome227.0: '/output/source_genomes/GCA_003382705.1_ASM338270v1_genomic.fna'\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.15\n",
      "\n",
      "The random seed for the run: 1594031057\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0510977X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome119.01.fq\n",
      "\t the 2nd reads: /tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome119.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome119.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.18\n",
      "\n",
      "The random seed for the run: 50971974\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0485703X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome77.01.fq\n",
      "\t the 2nd reads: /tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome77.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome77.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.15\n",
      "\n",
      "The random seed for the run: 1044513605\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0196252X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome216.01.fq\n",
      "\t the 2nd reads: /tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome216.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome216.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.26\n",
      "\n",
      "The random seed for the run: 507207848\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0740688X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome33.01.fq\n",
      "\t the 2nd reads: /tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome33.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome33.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.27\n",
      "\n",
      "The random seed for the run: 393287511\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.105018X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome12.01.fq\n",
      "\t the 2nd reads: /tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome12.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome12.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.14\n",
      "\n",
      "The random seed for the run: 1414977469\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0444804X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome207.01.fq\n",
      "\t the 2nd reads: /tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome207.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome207.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.14\n",
      "\n",
      "The random seed for the run: 1937464905\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0156039X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome104.01.fq\n",
      "\t the 2nd reads: /tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome104.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome104.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.13\n",
      "\n",
      "The random seed for the run: 411957455\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0247527X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome74.01.fq\n",
      "\t the 2nd reads: /tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome74.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome74.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.33\n",
      "\n",
      "The random seed for the run: 2130432890\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.167137X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome219.01.fq\n",
      "\t the 2nd reads: /tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome219.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome219.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.99\n",
      "\n",
      "The random seed for the run: 1093218188\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.639141X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome15.01.fq\n",
      "\t the 2nd reads: /tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome15.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome15.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.7\n",
      "\n",
      "The random seed for the run: 1027999508\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.459421X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome28.01.fq\n",
      "\t the 2nd reads: /tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome28.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome28.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.21\n",
      "\n",
      "The random seed for the run: 1669488642\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0755899X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome205.01.fq\n",
      "\t the 2nd reads: /tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome205.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome205.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.14\n",
      "\n",
      "The random seed for the run: 263232151\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0350771X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome93.01.fq\n",
      "\t the 2nd reads: /tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome93.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome93.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 1.81\n",
      "\n",
      "The random seed for the run: 963059643\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            1.23057X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome188.01.fq\n",
      "\t the 2nd reads: /tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome188.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome188.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.99\n",
      "\n",
      "The random seed for the run: 460212766\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.66195X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome231.01.fq\n",
      "\t the 2nd reads: /tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome231.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome231.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.7\n",
      "\n",
      "The random seed for the run: 1097061061\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.44101X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome52.01.fq\n",
      "\t the 2nd reads: /tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome52.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome52.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.12\n",
      "\n",
      "The random seed for the run: 215267604\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.010146X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome140.01.fq\n",
      "\t the 2nd reads: /tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome140.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome140.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 2.06\n",
      "\n",
      "The random seed for the run: 941171800\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.375408X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome189.01.fq\n",
      "\t the 2nd reads: /tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome189.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome189.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.47\n",
      "\n",
      "The random seed for the run: 370300383\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.304572X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome23.01.fq\n",
      "\t the 2nd reads: /tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome23.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome23.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.55\n",
      "\n",
      "The random seed for the run: 642284040\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.400652X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome227.01.fq\n",
      "\t the 2nd reads: /tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome227.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome227.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 2.46\n",
      "\n",
      "The random seed for the run: 1795828534\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            1.84548X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome201.01.fq\n",
      "\t the 2nd reads: /tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome201.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome201.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 2.2\n",
      "\n",
      "The random seed for the run: 670345468\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            1.70513X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome124.01.fq\n",
      "\t the 2nd reads: /tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome124.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome124.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 2.08\n",
      "\n",
      "The random seed for the run: 2029296162\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            1.79664X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome2.01.fq\n",
      "\t the 2nd reads: /tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome2.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome2.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 7.63\n",
      "\n",
      "The random seed for the run: 1940362427\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            7.50844X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome171.01.fq\n",
      "\t the 2nd reads: /tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome171.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpubQJEs/2021.03.13_09.38.14_sample_0/reads/Genome171.0.sam\n",
      "\n",
      "2021-03-13 09:45:48 INFO: [GenomePreparation 82430254076] Simulating reads finished\n",
      "2021-03-13 09:46:00 INFO: [MetagenomeSimulationPipeline] Generate gold standard assembly\n",
      "2021-03-13 09:46:00 INFO: [MetadataReader 87205974322] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-13 09:46:56 INFO: [MetagenomeSimulationPipeline] Generate pooled strains gold standard assembly\n",
      "2021-03-13 09:46:56 INFO: [MetadataReader 114087710] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-13 09:46:56 INFO: [MetagenomeSimulationPipeline] Samples used for pooled assembly: '2021.03.13_09.38.14_sample_0'\n",
      "2021-03-13 09:46:56 INFO: [Validator 10795378910] Creating pooled gold standard\n",
      "2021-03-13 09:46:56 INFO: [Validator 10795378910] Merging bam files.\n",
      "2021-03-13 09:46:56 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_09.38.14_sample_0/bam/Genome15.0.bam'\n",
      "2021-03-13 09:46:56 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_09.38.14_sample_0/bam/Genome119.0.bam'\n",
      "2021-03-13 09:46:56 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_09.38.14_sample_0/bam/Genome33.0.bam'\n",
      "2021-03-13 09:46:56 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_09.38.14_sample_0/bam/Genome188.0.bam'\n",
      "2021-03-13 09:46:56 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_09.38.14_sample_0/bam/Genome12.0.bam'\n",
      "2021-03-13 09:46:56 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_09.38.14_sample_0/bam/Genome216.0.bam'\n",
      "2021-03-13 09:46:56 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_09.38.14_sample_0/bam/Genome77.0.bam'\n",
      "2021-03-13 09:46:56 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_09.38.14_sample_0/bam/Genome201.0.bam'\n",
      "2021-03-13 09:46:56 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_09.38.14_sample_0/bam/Genome189.0.bam'\n",
      "2021-03-13 09:46:56 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_09.38.14_sample_0/bam/Genome28.0.bam'\n",
      "2021-03-13 09:46:56 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_09.38.14_sample_0/bam/Genome207.0.bam'\n",
      "2021-03-13 09:46:56 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_09.38.14_sample_0/bam/Genome219.0.bam'\n",
      "2021-03-13 09:46:56 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_09.38.14_sample_0/bam/Genome104.0.bam'\n",
      "2021-03-13 09:46:56 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_09.38.14_sample_0/bam/Genome74.0.bam'\n",
      "2021-03-13 09:46:56 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_09.38.14_sample_0/bam/Genome124.0.bam'\n",
      "2021-03-13 09:46:56 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_09.38.14_sample_0/bam/Genome231.0.bam'\n",
      "2021-03-13 09:46:56 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_09.38.14_sample_0/bam/Genome52.0.bam'\n",
      "2021-03-13 09:46:56 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_09.38.14_sample_0/bam/Genome205.0.bam'\n",
      "2021-03-13 09:46:56 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_09.38.14_sample_0/bam/Genome2.0.bam'\n",
      "2021-03-13 09:46:56 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_09.38.14_sample_0/bam/Genome93.0.bam'\n",
      "2021-03-13 09:46:56 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_09.38.14_sample_0/bam/Genome140.0.bam'\n",
      "2021-03-13 09:46:56 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_09.38.14_sample_0/bam/Genome23.0.bam'\n",
      "2021-03-13 09:46:56 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_09.38.14_sample_0/bam/Genome171.0.bam'\n",
      "2021-03-13 09:46:56 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_09.38.14_sample_0/bam/Genome227.0.bam'\n",
      "2021-03-13 09:47:51 INFO: [MetagenomeSimulationPipeline] Anonymize Data\n",
      "2021-03-13 09:47:51 INFO: [FastaAnonymizer] Interweave shuffle and anonymize\n",
      "2021-03-13 09:49:22 INFO: [MetadataReader 4631107874] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-13 09:49:24 INFO: [MetadataReader 14300761082] Reading file: '/output/internal/meta_data.tsv'\n",
      "2021-03-13 09:49:24 INFO: [MetadataReader 87652392539] Reading file: '/tmp/tmpubQJEs/tmp8uuFRR'\n",
      "2021-03-13 09:49:28 INFO: [FastaAnonymizer 66694448886] Shuffle and anonymize '/tmp/tmpubQJEs/tmpSVl6Tl'\n",
      "2021-03-13 09:49:39 INFO: [MetadataReader 4079751341] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-13 09:49:41 INFO: [MetadataReader 84063950941] Reading file: '/output/internal/meta_data.tsv'\n",
      "2021-03-13 09:49:41 INFO: [MetadataReader 48704713275] Reading file: '/tmp/tmpubQJEs/read_start_positionszJeg7h'\n",
      "2021-03-13 09:49:43 INFO: [MetadataReader 43119039112] Reading file: '/tmp/tmpubQJEs/tmpKfC1AT'\n",
      "2021-03-13 09:50:07 INFO: [FastaAnonymizer 64368966291] Shuffle and anonymize '/tmp/tmpubQJEs/tmppKsGmI'\n",
      "2021-03-13 09:50:17 INFO: [MetadataReader 78717221707] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-13 09:50:19 INFO: [MetadataReader 89765374142] Reading file: '/output/internal/meta_data.tsv'\n",
      "2021-03-13 09:50:19 INFO: [MetadataReader 98541404663] Reading file: '/tmp/tmpubQJEs/read_start_positions6b0aIB'\n",
      "2021-03-13 09:50:22 INFO: [MetadataReader 99062647065] Reading file: '/tmp/tmpubQJEs/tmpL0Aivy'\n",
      "2021-03-13 09:50:46 INFO: [MetagenomeSimulationPipeline] Compress Data\n",
      "2021-03-13 09:50:46 INFO: [Compress 27146481853] Compressing 'tmpGYjKKm'\n",
      "2021-03-13 09:50:46 INFO: [Compress 5059325991] Compressing 'anonymous_gsa_mapping_rCBXz'\n",
      "2021-03-13 09:50:46 INFO: [Compress 3713287912] Compressing 'tmpksrBcK'\n",
      "2021-03-13 09:50:46 INFO: [Compress 28761400014] Compressing 'anonymous_gs_mappingxrPQ0j'\n",
      "2021-03-13 09:50:46 INFO: [Compress 57512415149] Compressing 'tmpoXw22Q'\n",
      "2021-03-13 09:50:46 INFO: [Compress 74155927945] Compressing 'anonymous_gsa_pooled_mappingBXHB3i'\n",
      "2021-03-13 09:50:47 INFO: [Compress 5059325991] Done compressing 'gsa_mapping.tsv.gz' in 0:00:01s.\n",
      "2021-03-13 09:50:47 INFO: [Compress 74155927945] Done compressing 'gsa_pooled_mapping.tsv.gz' in 0:00:01s.\n",
      "2021-03-13 09:50:49 INFO: [Compress 28761400014] Done compressing 'reads_mapping.tsv.gz' in 0:00:03s.\n",
      "2021-03-13 09:50:50 INFO: [Compress 27146481853] Done compressing 'anonymous_gsa_pooled.fasta.gz' in 0:00:04s.\n",
      "2021-03-13 09:50:50 INFO: [Compress 57512415149] Done compressing 'anonymous_gsa.fasta.gz' in 0:00:04s.\n",
      "2021-03-13 09:51:02 INFO: [Compress 3713287912] Done compressing 'anonymous_reads.fq.gz' in 0:00:16s.\n",
      "2021-03-13 09:51:02 INFO: [MetagenomeSimulationPipeline] Metagenome simulation finished\n",
      "simulating complexity_sim_5_genera_25_genomes\n",
      "Password:2021-03-13 09:51:05 INFO: [MetagenomeSimulationPipeline] Metagenome simulation starting\n",
      "2021-03-13 09:51:05 INFO: [MetagenomeSimulationPipeline] Validating Genomes\n",
      "2021-03-13 09:51:05 INFO: [MetadataReader] Reading file: '/input/genome_to_id.tsv'\n",
      "2021-03-13 09:51:33 INFO: [MetagenomeSimulationPipeline] Design Communities\n",
      "2021-03-13 09:51:33 INFO: [CommunityDesign] Drawing strains.\n",
      "2021-03-13 09:51:33 INFO: [MetadataReader 82380310288] Reading file: '/input/metadata.tsv'\n",
      "2021-03-13 09:51:33 INFO: [MetadataReader 25595780401] Reading file: '/input/genome_to_id.tsv'\n",
      "2021-03-13 09:51:33 INFO: [CommunityDesign] Validating raw sequence files!\n",
      "2021-03-13 09:52:01 INFO: [NcbiTaxonomy] Building taxonomy tree...\n",
      "2021-03-13 09:52:01 INFO: [NcbiTaxonomy] Reading 'nodes' file:\t'/tmp/tmpxWHBq3/NCBI/nodes.dmp'\n",
      "2021-03-13 09:52:17 INFO: [NcbiTaxonomy] Reading 'names' file:\t'/tmp/tmpxWHBq3/NCBI/names.dmp'\n",
      "2021-03-13 09:52:20 INFO: [NcbiTaxonomy] Reading 'merged' file:\t'/tmp/tmpxWHBq3/NCBI/merged.dmp'\n",
      "2021-03-13 09:52:20 INFO: [NcbiTaxonomy] Done (19.0s)\n",
      "2021-03-13 09:52:20 INFO: [MetagenomeSimulationPipeline] Move Genomes\n",
      "2021-03-13 09:52:24 INFO: [MetagenomeSimulationPipeline] Read simulation\n",
      "2021-03-13 09:52:24 INFO: [GenomePreparation 70514612869] Using 'mbarc' error profile.\n",
      "2021-03-13 09:52:24 INFO: [GenomePreparation 70514612869] Reading distribution file\n",
      "2021-03-13 09:52:24 INFO: [GenomePreparation 70514612869] Reading genome location file\n",
      "2021-03-13 09:52:46 INFO: [GenomePreparation 70514612869] Genome 'Genome7.0' has sequences below minimum, creating filtered copy.\n",
      "2021-03-13 09:52:48 INFO: [GenomePreparation 70514612869] Genome 'Genome13.0' has sequences below minimum, creating filtered copy.\n",
      "2021-03-13 09:52:51 INFO: [GenomePreparation 70514612869] Genome 'Genome8.0' has sequences below minimum, creating filtered copy.\n",
      "2021-03-13 09:52:52 INFO: [GenomePreparation 70514612869] Simulating reads using ReadSimulationArtIllumina readsimulator...\n",
      "2021-03-13 09:52:52 INFO: [GenomePreparation 70514612869] Simulating reads from Genome15.0: '/output/source_genomes/GCA_000436655.1_MGS160_genomic.fna'\n",
      "2021-03-13 09:52:52 INFO: [GenomePreparation 70514612869] Simulating reads from Genome14.0: '/output/source_genomes/GCA_003318175.1_ASM331817v1_genomic.fna'\n",
      "2021-03-13 09:52:52 INFO: [GenomePreparation 70514612869] Simulating reads from Genome11.0: '/output/source_genomes/GCA_013014285.1_PDT000430776.1_genomic.fna'\n",
      "2021-03-13 09:52:52 INFO: [GenomePreparation 70514612869] Simulating reads from Genome24.0: '/output/source_genomes/GCA_000436135.1_MGS222_genomic.fna'\n",
      "2021-03-13 09:52:52 INFO: [GenomePreparation 70514612869] Simulating reads from Genome25.0: '/output/source_genomes/GCA_000705295.1_V.cholerae_O1_biovar_El_Tor_str._M-1293_genomic.fna'\n",
      "2021-03-13 09:52:52 INFO: [GenomePreparation 70514612869] Simulating reads from Genome18.0: '/output/source_genomes/GCA_004710365.1_PDT000000906.5_genomic.fna'\n",
      "2021-03-13 09:52:52 INFO: [GenomePreparation 70514612869] Simulating reads from Genome17.0: '/output/source_genomes/GCA_004710385.1_PDT000000905.5_genomic.fna'\n",
      "2021-03-13 09:52:52 INFO: [GenomePreparation 70514612869] Simulating reads from Genome16.0: '/output/source_genomes/GCA_013407315.1_ASM1340731v1_genomic.fna'\n",
      "2021-03-13 09:52:52 INFO: [GenomePreparation 70514612869] Simulating reads from Genome19.0: '/output/source_genomes/GCA_016955495.1_PDT000971171.1_genomic.fna'\n",
      "2021-03-13 09:52:52 INFO: [GenomePreparation 70514612869] Simulating reads from Genome2.0: '/output/source_genomes/GCA_000431175.1_MGS59_genomic.fna'\n",
      "2021-03-13 09:52:52 INFO: [GenomePreparation 70514612869] Simulating reads from Genome1.0: '/output/source_genomes/GCA_001915515.1_ASM191551v1_genomic.fna'\n",
      "2021-03-13 09:52:52 INFO: [GenomePreparation 70514612869] Simulating reads from Genome21.0: '/output/source_genomes/GCA_000382945.1_ASM38294v1_genomic.fna'\n",
      "2021-03-13 09:52:52 INFO: [GenomePreparation 70514612869] Simulating reads from Genome4.0: '/output/source_genomes/GCA_000307655.2_CFSAN0001628_2.0_genomic.fna'\n",
      "2021-03-13 09:52:52 INFO: [GenomePreparation 70514612869] Simulating reads from Genome5.0: '/output/source_genomes/GCA_004710325.1_PDT000000912.5_genomic.fna'\n",
      "2021-03-13 09:52:52 INFO: [GenomePreparation 70514612869] Simulating reads from Genome3.0: '/output/source_genomes/GCA_003678225.1_PDT000362406.1_genomic.fna'\n",
      "2021-03-13 09:52:52 INFO: [GenomePreparation 70514612869] Simulating reads from Genome20.0: '/output/source_genomes/GCA_016955055.1_PDT000971198.1_genomic.fna'\n",
      "2021-03-13 09:52:52 INFO: [GenomePreparation 70514612869] Simulating reads from Genome6.0: '/output/source_genomes/GCA_900461265.1_50256_C01_genomic.fna'\n",
      "2021-03-13 09:52:52 INFO: [GenomePreparation 70514612869] Simulating reads from Genome23.0: '/output/source_genomes/GCA_000237625.2_ASM23762v2_genomic.fna'\n",
      "2021-03-13 09:52:52 INFO: [GenomePreparation 70514612869] Simulating reads from Genome9.0: '/output/source_genomes/GCA_000618005.3_ASM61800v3_genomic.fna'\n",
      "2021-03-13 09:52:52 INFO: [GenomePreparation 70514612869] Simulating reads from Genome7.0: '/tmp/tmpOzQdhY/tmpMwkXq2'\n",
      "2021-03-13 09:52:52 INFO: [GenomePreparation 70514612869] Simulating reads from Genome13.0: '/tmp/tmpOzQdhY/tmpvQMWin'\n",
      "2021-03-13 09:52:52 INFO: [GenomePreparation 70514612869] Simulating reads from Genome10.0: '/output/source_genomes/GCA_003463775.1_ASM346377v1_genomic.fna'\n",
      "2021-03-13 09:52:52 INFO: [GenomePreparation 70514612869] Simulating reads from Genome8.0: '/tmp/tmpOzQdhY/tmpsiOD_9'\n",
      "2021-03-13 09:52:52 INFO: [GenomePreparation 70514612869] Simulating reads from Genome22.0: '/output/source_genomes/GCA_905211905.1_SRR413683-mag-bin.12_genomic.fna'\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.08\n",
      "\n",
      "The random seed for the run: 540968737\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.00235883X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome15.01.fq\n",
      "\t the 2nd reads: /tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome15.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome15.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.09\n",
      "\n",
      "The random seed for the run: 2129338277\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.018133X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome18.01.fq\n",
      "\t the 2nd reads: /tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome18.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome18.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.12\n",
      "\n",
      "The random seed for the run: 1847505585\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0109081X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome14.01.fq\n",
      "\t the 2nd reads: /tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome14.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome14.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.13\n",
      "\n",
      "The random seed for the run: 170568346\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0585527X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome17.01.fq\n",
      "\t the 2nd reads: /tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome17.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome17.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.21\n",
      "\n",
      "The random seed for the run: 65634987\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0532078X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome11.01.fq\n",
      "\t the 2nd reads: /tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome11.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome11.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.16\n",
      "\n",
      "The random seed for the run: 1703660943\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0246723X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome16.01.fq\n",
      "\t the 2nd reads: /tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome16.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome16.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.06\n",
      "\n",
      "The random seed for the run: 1329514041\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.00328132X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome5.01.fq\n",
      "\t the 2nd reads: /tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome5.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome5.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.2\n",
      "\n",
      "The random seed for the run: 1577639001\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0614933X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome2.01.fq\n",
      "\t the 2nd reads: /tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome2.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome2.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.57\n",
      "\n",
      "The random seed for the run: 573380147\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.407411X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome25.01.fq\n",
      "\t the 2nd reads: /tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome25.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome25.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.13\n",
      "\n",
      "The random seed for the run: 1168871082\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.103102X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome3.01.fq\n",
      "\t the 2nd reads: /tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome3.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome3.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.54\n",
      "\n",
      "The random seed for the run: 49814198\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.708991X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome21.01.fq\n",
      "\t the 2nd reads: /tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome21.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome21.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.11\n",
      "\n",
      "The random seed for the run: 1550755105\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0347743X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome23.01.fq\n",
      "\t the 2nd reads: /tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome23.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome23.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.46\n",
      "\n",
      "The random seed for the run: 1961570421\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.299957X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome20.01.fq\n",
      "\t the 2nd reads: /tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome20.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome20.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.06\n",
      "\n",
      "The random seed for the run: 1923521921\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.00278049X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome7.01.fq\n",
      "\t the 2nd reads: /tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome7.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome7.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.22\n",
      "\n",
      "The random seed for the run: 854832072\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.119171X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome13.01.fq\n",
      "\t the 2nd reads: /tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome13.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome13.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.83\n",
      "\n",
      "The random seed for the run: 923704085\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            1.31389X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome6.01.fq\n",
      "\t the 2nd reads: /tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome6.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome6.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.15\n",
      "\n",
      "The random seed for the run: 2102807139\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0478747X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome8.01.fq\n",
      "\t the 2nd reads: /tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome8.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome8.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.14\n",
      "\n",
      "The random seed for the run: 1940270192\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.107609X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome22.01.fq\n",
      "\t the 2nd reads: /tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome22.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome22.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 1.03\n",
      "\n",
      "The random seed for the run: 689939719\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.73288X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome10.01.fq\n",
      "\t the 2nd reads: /tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome10.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome10.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 2.43\n",
      "\n",
      "The random seed for the run: 1557933274\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            2.64328X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome4.01.fq\n",
      "\t the 2nd reads: /tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome4.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome4.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 1.87\n",
      "\n",
      "The random seed for the run: 1992431107\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            1.64442X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome9.01.fq\n",
      "\t the 2nd reads: /tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome9.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome9.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 2.95\n",
      "\n",
      "The random seed for the run: 1661055960\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            2.45526X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome19.01.fq\n",
      "\t the 2nd reads: /tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome19.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome19.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 5.2\n",
      "\n",
      "The random seed for the run: 1294107793\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            6.57772X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome24.01.fq\n",
      "\t the 2nd reads: /tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome24.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome24.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 5.83\n",
      "\n",
      "The random seed for the run: 1713385206\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            4.70755X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome1.01.fq\n",
      "\t the 2nd reads: /tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome1.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpOzQdhY/2021.03.13_09.51.05_sample_0/reads/Genome1.0.sam\n",
      "\n",
      "2021-03-13 09:52:59 INFO: [GenomePreparation 70514612869] Simulating reads finished\n",
      "2021-03-13 09:53:09 INFO: [MetagenomeSimulationPipeline] Generate gold standard assembly\n",
      "2021-03-13 09:53:09 INFO: [MetadataReader 64136302938] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-13 09:53:54 INFO: [MetagenomeSimulationPipeline] Generate pooled strains gold standard assembly\n",
      "2021-03-13 09:53:54 INFO: [MetadataReader 58498923446] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-13 09:53:54 INFO: [MetagenomeSimulationPipeline] Samples used for pooled assembly: '2021.03.13_09.51.05_sample_0'\n",
      "2021-03-13 09:53:54 INFO: [Validator 48443846240] Creating pooled gold standard\n",
      "2021-03-13 09:53:54 INFO: [Validator 48443846240] Merging bam files.\n",
      "2021-03-13 09:53:54 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.13_09.51.05_sample_0/bam/Genome15.0.bam'\n",
      "2021-03-13 09:53:54 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.13_09.51.05_sample_0/bam/Genome14.0.bam'\n",
      "2021-03-13 09:53:54 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.13_09.51.05_sample_0/bam/Genome11.0.bam'\n",
      "2021-03-13 09:53:54 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.13_09.51.05_sample_0/bam/Genome24.0.bam'\n",
      "2021-03-13 09:53:54 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.13_09.51.05_sample_0/bam/Genome25.0.bam'\n",
      "2021-03-13 09:53:54 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.13_09.51.05_sample_0/bam/Genome18.0.bam'\n",
      "2021-03-13 09:53:54 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.13_09.51.05_sample_0/bam/Genome17.0.bam'\n",
      "2021-03-13 09:53:54 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.13_09.51.05_sample_0/bam/Genome16.0.bam'\n",
      "2021-03-13 09:53:54 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.13_09.51.05_sample_0/bam/Genome19.0.bam'\n",
      "2021-03-13 09:53:54 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.13_09.51.05_sample_0/bam/Genome2.0.bam'\n",
      "2021-03-13 09:53:54 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.13_09.51.05_sample_0/bam/Genome1.0.bam'\n",
      "2021-03-13 09:53:54 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.13_09.51.05_sample_0/bam/Genome21.0.bam'\n",
      "2021-03-13 09:53:54 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.13_09.51.05_sample_0/bam/Genome4.0.bam'\n",
      "2021-03-13 09:53:54 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.13_09.51.05_sample_0/bam/Genome5.0.bam'\n",
      "2021-03-13 09:53:54 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.13_09.51.05_sample_0/bam/Genome3.0.bam'\n",
      "2021-03-13 09:53:54 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.13_09.51.05_sample_0/bam/Genome20.0.bam'\n",
      "2021-03-13 09:53:54 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.13_09.51.05_sample_0/bam/Genome6.0.bam'\n",
      "2021-03-13 09:53:54 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.13_09.51.05_sample_0/bam/Genome23.0.bam'\n",
      "2021-03-13 09:53:54 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.13_09.51.05_sample_0/bam/Genome9.0.bam'\n",
      "2021-03-13 09:53:54 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.13_09.51.05_sample_0/bam/Genome7.0.bam'\n",
      "2021-03-13 09:53:54 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.13_09.51.05_sample_0/bam/Genome13.0.bam'\n",
      "2021-03-13 09:53:54 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.13_09.51.05_sample_0/bam/Genome10.0.bam'\n",
      "2021-03-13 09:53:54 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.13_09.51.05_sample_0/bam/Genome8.0.bam'\n",
      "2021-03-13 09:53:54 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.13_09.51.05_sample_0/bam/Genome22.0.bam'\n",
      "2021-03-13 09:54:38 INFO: [MetagenomeSimulationPipeline] Anonymize Data\n",
      "2021-03-13 09:54:38 INFO: [FastaAnonymizer] Interweave shuffle and anonymize\n",
      "2021-03-13 09:56:09 INFO: [MetadataReader 72802808290] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-13 09:56:11 INFO: [MetadataReader 47315449936] Reading file: '/output/internal/meta_data.tsv'\n",
      "2021-03-13 09:56:11 INFO: [MetadataReader 65300095466] Reading file: '/tmp/tmpOzQdhY/tmpmYdqOY'\n",
      "2021-03-13 09:56:15 INFO: [FastaAnonymizer 43075827506] Shuffle and anonymize '/tmp/tmpOzQdhY/tmpWpmrHx'\n",
      "2021-03-13 09:56:22 INFO: [MetadataReader 71873526544] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-13 09:56:23 INFO: [MetadataReader 45105173502] Reading file: '/output/internal/meta_data.tsv'\n",
      "2021-03-13 09:56:23 INFO: [MetadataReader 84613406616] Reading file: '/tmp/tmpOzQdhY/read_start_positionssGNxN_'\n",
      "2021-03-13 09:56:26 INFO: [MetadataReader 10674074437] Reading file: '/tmp/tmpOzQdhY/tmp2tiWBF'\n",
      "2021-03-13 09:57:45 INFO: [FastaAnonymizer 86623674311] Shuffle and anonymize '/tmp/tmpOzQdhY/tmpZ9kqwH'\n",
      "2021-03-13 09:57:53 INFO: [MetadataReader 15345066727] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-13 09:57:54 INFO: [MetadataReader 77753400540] Reading file: '/output/internal/meta_data.tsv'\n",
      "2021-03-13 09:57:54 INFO: [MetadataReader 12266665971] Reading file: '/tmp/tmpOzQdhY/read_start_positionsIJCRo_'\n",
      "2021-03-13 09:57:57 INFO: [MetadataReader 42355547664] Reading file: '/tmp/tmpOzQdhY/tmp9hN6Xy'\n",
      "2021-03-13 09:59:16 INFO: [MetagenomeSimulationPipeline] Compress Data\n",
      "2021-03-13 09:59:16 INFO: [Compress 26797537663] Compressing 'anonymous_gsa_mapping0q17GA'\n",
      "2021-03-13 09:59:16 INFO: [Compress 89437880737] Compressing 'tmpAYtiT0'\n",
      "2021-03-13 09:59:16 INFO: [Compress 9763732719] Compressing 'tmpbX_KLr'\n",
      "2021-03-13 09:59:16 INFO: [Compress 61506368141] Compressing 'tmpMdKlQh'\n",
      "2021-03-13 09:59:16 INFO: [Compress 7831310072] Compressing 'anonymous_gsa_pooled_mappingB8ALiN'\n",
      "2021-03-13 09:59:16 INFO: [Compress 51464108618] Compressing 'anonymous_gs_mapping6YiM0w'\n",
      "2021-03-13 09:59:16 INFO: [Compress 7831310072] Done compressing 'gsa_pooled_mapping.tsv.gz' in 0:00:00s.\n",
      "2021-03-13 09:59:16 INFO: [Compress 26797537663] Done compressing 'gsa_mapping.tsv.gz' in 0:00:00s.\n",
      "2021-03-13 09:59:19 INFO: [Compress 51464108618] Done compressing 'reads_mapping.tsv.gz' in 0:00:03s.\n",
      "2021-03-13 09:59:19 INFO: [Compress 61506368141] Done compressing 'anonymous_gsa_pooled.fasta.gz' in 0:00:03s.\n",
      "2021-03-13 09:59:19 INFO: [Compress 89437880737] Done compressing 'anonymous_gsa.fasta.gz' in 0:00:03s.\n",
      "2021-03-13 09:59:31 INFO: [Compress 9763732719] Done compressing 'anonymous_reads.fq.gz' in 0:00:15s.\n",
      "2021-03-13 09:59:31 INFO: [MetagenomeSimulationPipeline] Metagenome simulation finished\n",
      "simulating complexity_sim_10_genera_25_genomes\n",
      "Password:2021-03-13 09:59:34 INFO: [MetagenomeSimulationPipeline] Metagenome simulation starting\n",
      "2021-03-13 09:59:34 INFO: [MetagenomeSimulationPipeline] Validating Genomes\n",
      "2021-03-13 09:59:34 INFO: [MetadataReader] Reading file: '/input/genome_to_id.tsv'\n",
      "2021-03-13 10:00:03 INFO: [MetagenomeSimulationPipeline] Design Communities\n",
      "2021-03-13 10:00:03 INFO: [CommunityDesign] Drawing strains.\n",
      "2021-03-13 10:00:03 INFO: [MetadataReader 82380310288] Reading file: '/input/metadata.tsv'\n",
      "2021-03-13 10:00:03 INFO: [MetadataReader 25595780401] Reading file: '/input/genome_to_id.tsv'\n",
      "2021-03-13 10:00:03 INFO: [CommunityDesign] Validating raw sequence files!\n",
      "2021-03-13 10:00:32 INFO: [NcbiTaxonomy] Building taxonomy tree...\n",
      "2021-03-13 10:00:32 INFO: [NcbiTaxonomy] Reading 'nodes' file:\t'/tmp/tmpNlmTZR/NCBI/nodes.dmp'\n",
      "2021-03-13 10:00:48 INFO: [NcbiTaxonomy] Reading 'names' file:\t'/tmp/tmpNlmTZR/NCBI/names.dmp'\n",
      "2021-03-13 10:00:51 INFO: [NcbiTaxonomy] Reading 'merged' file:\t'/tmp/tmpNlmTZR/NCBI/merged.dmp'\n",
      "2021-03-13 10:00:51 INFO: [NcbiTaxonomy] Done (19.0s)\n",
      "2021-03-13 10:00:51 INFO: [MetagenomeSimulationPipeline] Move Genomes\n",
      "2021-03-13 10:00:57 INFO: [MetagenomeSimulationPipeline] Read simulation\n",
      "2021-03-13 10:00:57 INFO: [GenomePreparation 70514612869] Using 'mbarc' error profile.\n",
      "2021-03-13 10:00:57 INFO: [GenomePreparation 70514612869] Reading distribution file\n",
      "2021-03-13 10:00:57 INFO: [GenomePreparation 70514612869] Reading genome location file\n",
      "2021-03-13 10:01:00 INFO: [GenomePreparation 70514612869] Genome 'Genome14.0' has sequences below minimum, creating filtered copy.\n",
      "2021-03-13 10:01:04 INFO: [GenomePreparation 70514612869] Genome 'Genome24.0' has sequences below minimum, creating filtered copy.\n",
      "2021-03-13 10:01:11 INFO: [GenomePreparation 70514612869] Genome 'Genome19.0' has sequences below minimum, creating filtered copy.\n",
      "2021-03-13 10:01:17 INFO: [GenomePreparation 70514612869] Genome 'Genome5.0' has sequences below minimum, creating filtered copy.\n",
      "2021-03-13 10:01:24 INFO: [GenomePreparation 70514612869] Genome 'Genome13.0' has sequences below minimum, creating filtered copy.\n",
      "2021-03-13 10:01:27 INFO: [GenomePreparation 70514612869] Genome 'Genome8.0' has sequences below minimum, creating filtered copy.\n",
      "2021-03-13 10:01:28 INFO: [GenomePreparation 70514612869] Simulating reads using ReadSimulationArtIllumina readsimulator...\n",
      "2021-03-13 10:01:28 INFO: [GenomePreparation 70514612869] Simulating reads from Genome15.0: '/output/source_genomes/GCA_900458615.1_35910_D02_genomic.fna'\n",
      "2021-03-13 10:01:28 INFO: [GenomePreparation 70514612869] Simulating reads from Genome14.0: '/tmp/tmpTb9_lu/tmpiXh16r'\n",
      "2021-03-13 10:01:28 INFO: [GenomePreparation 70514612869] Simulating reads from Genome11.0: '/output/source_genomes/GCA_000188755.2_ASM18875v2_genomic.fna'\n",
      "2021-03-13 10:01:28 INFO: [GenomePreparation 70514612869] Simulating reads from Genome24.0: '/tmp/tmpTb9_lu/tmpidtLj8'\n",
      "2021-03-13 10:01:28 INFO: [GenomePreparation 70514612869] Simulating reads from Genome12.0: '/output/source_genomes/GCA_000190975.1_ASM19097v1_genomic.fna'\n",
      "2021-03-13 10:01:28 INFO: [GenomePreparation 70514612869] Simulating reads from Genome25.0: '/output/source_genomes/GCA_003129525.1_ASM312952v1_genomic.fna'\n",
      "2021-03-13 10:01:28 INFO: [GenomePreparation 70514612869] Simulating reads from Genome18.0: '/output/source_genomes/GCA_003463715.1_ASM346371v1_genomic.fna'\n",
      "2021-03-13 10:01:28 INFO: [GenomePreparation 70514612869] Simulating reads from Genome17.0: '/output/source_genomes/GCA_902791925.1_Rumen_uncultured_genome_RUG13721_genomic.fna'\n",
      "2021-03-13 10:01:28 INFO: [GenomePreparation 70514612869] Simulating reads from Genome16.0: '/output/source_genomes/GCA_905207145.1_ERR260204-mag-bin.19_genomic.fna'\n",
      "2021-03-13 10:01:28 INFO: [GenomePreparation 70514612869] Simulating reads from Genome19.0: '/tmp/tmpTb9_lu/tmprlJhEe'\n",
      "2021-03-13 10:01:28 INFO: [GenomePreparation 70514612869] Simulating reads from Genome2.0: '/output/source_genomes/GCA_000249615.2_ASM24961v2_genomic.fna'\n",
      "2021-03-13 10:01:28 INFO: [GenomePreparation 70514612869] Simulating reads from Genome1.0: '/output/source_genomes/GCA_000614735.1_ASM61473v1_genomic.fna'\n",
      "2021-03-13 10:01:28 INFO: [GenomePreparation 70514612869] Simulating reads from Genome21.0: '/output/source_genomes/GCA_000249795.2_ASM24979v2_genomic.fna'\n",
      "2021-03-13 10:01:28 INFO: [GenomePreparation 70514612869] Simulating reads from Genome4.0: '/output/source_genomes/GCA_017189635.1_PDT000943498.1_genomic.fna'\n",
      "2021-03-13 10:01:28 INFO: [GenomePreparation 70514612869] Simulating reads from Genome5.0: '/tmp/tmpTb9_lu/tmpM28vPi'\n",
      "2021-03-13 10:01:28 INFO: [GenomePreparation 70514612869] Simulating reads from Genome3.0: '/output/source_genomes/GCA_003678225.1_PDT000362406.1_genomic.fna'\n",
      "2021-03-13 10:01:28 INFO: [GenomePreparation 70514612869] Simulating reads from Genome20.0: '/output/source_genomes/GCA_014871405.1_ASM1487140v1_genomic.fna'\n",
      "2021-03-13 10:01:28 INFO: [GenomePreparation 70514612869] Simulating reads from Genome6.0: '/output/source_genomes/GCA_012079875.1_PDT000274835.6_genomic.fna'\n",
      "2021-03-13 10:01:28 INFO: [GenomePreparation 70514612869] Simulating reads from Genome23.0: '/output/source_genomes/GCA_000249815.2_ASM24981v2_genomic.fna'\n",
      "2021-03-13 10:01:28 INFO: [GenomePreparation 70514612869] Simulating reads from Genome7.0: '/output/source_genomes/GCA_004710325.1_PDT000000912.5_genomic.fna'\n",
      "2021-03-13 10:01:28 INFO: [GenomePreparation 70514612869] Simulating reads from Genome13.0: '/tmp/tmpTb9_lu/tmp6btzOO'\n",
      "2021-03-13 10:01:28 INFO: [GenomePreparation 70514612869] Simulating reads from Genome10.0: '/output/source_genomes/GCA_000422165.1_ASM42216v1_genomic.fna'\n",
      "2021-03-13 10:01:28 INFO: [GenomePreparation 70514612869] Simulating reads from Genome8.0: '/tmp/tmpTb9_lu/tmpnRQ0WC'\n",
      "2021-03-13 10:01:28 INFO: [GenomePreparation 70514612869] Simulating reads from Genome22.0: '/output/source_genomes/GCA_000508465.1_CloBut_DORA_1_velvet_1.1_genomic.fna'\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.09\n",
      "\n",
      "The random seed for the run: 540968737\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.00257988X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome15.01.fq\n",
      "\t the 2nd reads: /tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome15.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome15.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.18\n",
      "\n",
      "The random seed for the run: 1294107793\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0228923X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome24.01.fq\n",
      "\t the 2nd reads: /tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome24.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome24.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.09\n",
      "\n",
      "The random seed for the run: 1703660943\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0570566X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome17.01.fq\n",
      "\t the 2nd reads: /tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome17.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome17.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.16\n",
      "\n",
      "The random seed for the run: 2129338277\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0444206X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome25.01.fq\n",
      "\t the 2nd reads: /tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome25.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome25.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.08\n",
      "\n",
      "The random seed for the run: 1661055960\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0322654X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome16.01.fq\n",
      "\t the 2nd reads: /tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome16.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome16.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.27\n",
      "\n",
      "The random seed for the run: 170568346\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0543282X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome18.01.fq\n",
      "\t the 2nd reads: /tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome18.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome18.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.05\n",
      "\n",
      "The random seed for the run: 49814198\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.00304458X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome1.01.fq\n",
      "\t the 2nd reads: /tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome1.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome1.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.3\n",
      "\n",
      "The random seed for the run: 1713385206\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.110573X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome2.01.fq\n",
      "\t the 2nd reads: /tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome2.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome2.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.14\n",
      "\n",
      "The random seed for the run: 1961570421\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0956632X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome3.01.fq\n",
      "\t the 2nd reads: /tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome3.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome3.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.51\n",
      "\n",
      "The random seed for the run: 1329514041\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.378017X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome4.01.fq\n",
      "\t the 2nd reads: /tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome4.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome4.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.15\n",
      "\n",
      "The random seed for the run: 923704085\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.049369X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome20.01.fq\n",
      "\t the 2nd reads: /tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome20.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome20.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.92\n",
      "\n",
      "The random seed for the run: 1168871082\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.657838X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome5.01.fq\n",
      "\t the 2nd reads: /tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome5.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome5.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.1\n",
      "\n",
      "The random seed for the run: 1992431107\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.00218864X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome23.01.fq\n",
      "\t the 2nd reads: /tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome23.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome23.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.07\n",
      "\n",
      "The random seed for the run: 1923521921\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0101211X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome7.01.fq\n",
      "\t the 2nd reads: /tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome7.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome7.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 1.09\n",
      "\n",
      "The random seed for the run: 1557933274\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.680004X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome21.01.fq\n",
      "\t the 2nd reads: /tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome21.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome21.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 2\n",
      "\n",
      "The random seed for the run: 1847505585\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            1.52577X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome14.01.fq\n",
      "\t the 2nd reads: /tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome14.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome14.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.05\n",
      "\n",
      "The random seed for the run: 689939719\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0168247X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome10.01.fq\n",
      "\t the 2nd reads: /tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome10.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome10.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.49\n",
      "\n",
      "The random seed for the run: 854832072\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.278315X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome13.01.fq\n",
      "\t the 2nd reads: /tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome13.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome13.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.22\n",
      "\n",
      "The random seed for the run: 2102807139\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0998456X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome8.01.fq\n",
      "\t the 2nd reads: /tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome8.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome8.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 1.78\n",
      "\n",
      "The random seed for the run: 1577639001\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            1.2191X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome19.01.fq\n",
      "\t the 2nd reads: /tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome19.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome19.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 3.18\n",
      "\n",
      "The random seed for the run: 65634987\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            2.45257X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome11.01.fq\n",
      "\t the 2nd reads: /tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome11.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome11.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 2.61\n",
      "\n",
      "The random seed for the run: 1550755105\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            2.27812X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome6.01.fq\n",
      "\t the 2nd reads: /tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome6.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome6.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 5.06\n",
      "\n",
      "The random seed for the run: 573380147\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            4.36791X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome12.01.fq\n",
      "\t the 2nd reads: /tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome12.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome12.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 4.69\n",
      "\n",
      "The random seed for the run: 1940270192\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            6.10315X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome22.01.fq\n",
      "\t the 2nd reads: /tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome22.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpTb9_lu/2021.03.13_09.59.34_sample_0/reads/Genome22.0.sam\n",
      "\n",
      "2021-03-13 10:01:36 INFO: [GenomePreparation 70514612869] Simulating reads finished\n",
      "2021-03-13 10:01:45 INFO: [MetagenomeSimulationPipeline] Generate gold standard assembly\n",
      "2021-03-13 10:01:45 INFO: [MetadataReader 64136302938] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-13 10:02:33 INFO: [MetagenomeSimulationPipeline] Generate pooled strains gold standard assembly\n",
      "2021-03-13 10:02:33 INFO: [MetadataReader 58498923446] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-13 10:02:33 INFO: [MetagenomeSimulationPipeline] Samples used for pooled assembly: '2021.03.13_09.59.34_sample_0'\n",
      "2021-03-13 10:02:33 INFO: [Validator 48443846240] Creating pooled gold standard\n",
      "2021-03-13 10:02:33 INFO: [Validator 48443846240] Merging bam files.\n",
      "2021-03-13 10:02:33 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.13_09.59.34_sample_0/bam/Genome15.0.bam'\n",
      "2021-03-13 10:02:33 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.13_09.59.34_sample_0/bam/Genome14.0.bam'\n",
      "2021-03-13 10:02:33 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.13_09.59.34_sample_0/bam/Genome11.0.bam'\n",
      "2021-03-13 10:02:33 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.13_09.59.34_sample_0/bam/Genome24.0.bam'\n",
      "2021-03-13 10:02:33 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.13_09.59.34_sample_0/bam/Genome12.0.bam'\n",
      "2021-03-13 10:02:33 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.13_09.59.34_sample_0/bam/Genome25.0.bam'\n",
      "2021-03-13 10:02:33 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.13_09.59.34_sample_0/bam/Genome18.0.bam'\n",
      "2021-03-13 10:02:33 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.13_09.59.34_sample_0/bam/Genome17.0.bam'\n",
      "2021-03-13 10:02:33 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.13_09.59.34_sample_0/bam/Genome16.0.bam'\n",
      "2021-03-13 10:02:33 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.13_09.59.34_sample_0/bam/Genome19.0.bam'\n",
      "2021-03-13 10:02:33 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.13_09.59.34_sample_0/bam/Genome2.0.bam'\n",
      "2021-03-13 10:02:33 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.13_09.59.34_sample_0/bam/Genome1.0.bam'\n",
      "2021-03-13 10:02:33 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.13_09.59.34_sample_0/bam/Genome21.0.bam'\n",
      "2021-03-13 10:02:33 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.13_09.59.34_sample_0/bam/Genome4.0.bam'\n",
      "2021-03-13 10:02:33 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.13_09.59.34_sample_0/bam/Genome5.0.bam'\n",
      "2021-03-13 10:02:33 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.13_09.59.34_sample_0/bam/Genome3.0.bam'\n",
      "2021-03-13 10:02:33 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.13_09.59.34_sample_0/bam/Genome20.0.bam'\n",
      "2021-03-13 10:02:33 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.13_09.59.34_sample_0/bam/Genome6.0.bam'\n",
      "2021-03-13 10:02:33 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.13_09.59.34_sample_0/bam/Genome23.0.bam'\n",
      "2021-03-13 10:02:33 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.13_09.59.34_sample_0/bam/Genome7.0.bam'\n",
      "2021-03-13 10:02:33 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.13_09.59.34_sample_0/bam/Genome13.0.bam'\n",
      "2021-03-13 10:02:33 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.13_09.59.34_sample_0/bam/Genome10.0.bam'\n",
      "2021-03-13 10:02:33 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.13_09.59.34_sample_0/bam/Genome8.0.bam'\n",
      "2021-03-13 10:02:33 WARNING: [Validator 48443846240] List contains only one file: '/output/2021.03.13_09.59.34_sample_0/bam/Genome22.0.bam'\n",
      "2021-03-13 10:03:21 INFO: [MetagenomeSimulationPipeline] Anonymize Data\n",
      "2021-03-13 10:03:21 INFO: [FastaAnonymizer] Interweave shuffle and anonymize\n",
      "2021-03-13 10:04:51 INFO: [MetadataReader 72802808290] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-13 10:04:53 INFO: [MetadataReader 47315449936] Reading file: '/output/internal/meta_data.tsv'\n",
      "2021-03-13 10:04:53 INFO: [MetadataReader 65300095466] Reading file: '/tmp/tmpTb9_lu/tmpIoxCfa'\n",
      "2021-03-13 10:04:57 INFO: [FastaAnonymizer 43075827506] Shuffle and anonymize '/tmp/tmpTb9_lu/tmp_GWPTB'\n",
      "2021-03-13 10:05:05 INFO: [MetadataReader 71873526544] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-13 10:05:07 INFO: [MetadataReader 45105173502] Reading file: '/output/internal/meta_data.tsv'\n",
      "2021-03-13 10:05:07 INFO: [MetadataReader 84613406616] Reading file: '/tmp/tmpTb9_lu/read_start_positionsfdrthf'\n",
      "2021-03-13 10:05:09 INFO: [MetadataReader 10674074437] Reading file: '/tmp/tmpTb9_lu/tmp8XAVsU'\n",
      "2021-03-13 10:05:23 INFO: [FastaAnonymizer 86623674311] Shuffle and anonymize '/tmp/tmpTb9_lu/tmpviqNva'\n",
      "2021-03-13 10:05:31 INFO: [MetadataReader 15345066727] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-13 10:05:32 INFO: [MetadataReader 77753400540] Reading file: '/output/internal/meta_data.tsv'\n",
      "2021-03-13 10:05:32 INFO: [MetadataReader 12266665971] Reading file: '/tmp/tmpTb9_lu/read_start_positionsa_3jZr'\n",
      "2021-03-13 10:05:35 INFO: [MetadataReader 42355547664] Reading file: '/tmp/tmpTb9_lu/tmpTblWGV'\n",
      "2021-03-13 10:05:48 INFO: [MetagenomeSimulationPipeline] Compress Data\n",
      "2021-03-13 10:05:49 INFO: [Compress 87480216819] Compressing 'anonymous_gsa_mappingasaSgm'\n",
      "2021-03-13 10:05:49 INFO: [Compress 92586523794] Compressing 'anonymous_gs_mappingbsiqqM'\n",
      "2021-03-13 10:05:49 INFO: [Compress 46598131196] Compressing 'anonymous_gsa_pooled_mapping961tiQ'\n",
      "2021-03-13 10:05:49 INFO: [Compress 34674633099] Compressing 'tmpnHVblF'\n",
      "2021-03-13 10:05:49 INFO: [Compress 15423503801] Compressing 'tmp3wbRps'\n",
      "2021-03-13 10:05:49 INFO: [Compress 98309081475] Compressing 'tmpHYXDAE'\n",
      "2021-03-13 10:05:49 INFO: [Compress 87480216819] Done compressing 'gsa_mapping.tsv.gz' in 0:00:00s.\n",
      "2021-03-13 10:05:49 INFO: [Compress 46598131196] Done compressing 'gsa_pooled_mapping.tsv.gz' in 0:00:00s.\n",
      "2021-03-13 10:05:51 INFO: [Compress 92586523794] Done compressing 'reads_mapping.tsv.gz' in 0:00:03s.\n",
      "2021-03-13 10:05:52 INFO: [Compress 15423503801] Done compressing 'anonymous_gsa.fasta.gz' in 0:00:04s.\n",
      "2021-03-13 10:05:52 INFO: [Compress 34674633099] Done compressing 'anonymous_gsa_pooled.fasta.gz' in 0:00:04s.\n",
      "2021-03-13 10:06:04 INFO: [Compress 98309081475] Done compressing 'anonymous_reads.fq.gz' in 0:00:16s.\n",
      "2021-03-13 10:06:04 INFO: [MetagenomeSimulationPipeline] Metagenome simulation finished\n",
      "simulating complexity_sim_1_genera_50_genomes\n",
      "Password:2021-03-13 10:06:07 INFO: [MetagenomeSimulationPipeline] Metagenome simulation starting\n",
      "2021-03-13 10:06:07 INFO: [MetagenomeSimulationPipeline] Validating Genomes\n",
      "2021-03-13 10:06:07 INFO: [MetadataReader] Reading file: '/input/genome_to_id.tsv'\n",
      "2021-03-13 10:07:16 INFO: [MetagenomeSimulationPipeline] Design Communities\n",
      "2021-03-13 10:07:16 INFO: [CommunityDesign] Drawing strains.\n",
      "2021-03-13 10:07:16 INFO: [MetadataReader 82380310288] Reading file: '/input/metadata.tsv'\n",
      "2021-03-13 10:07:16 INFO: [MetadataReader 89885674421] Reading file: '/input/genome_to_id.tsv'\n",
      "2021-03-13 10:07:16 INFO: [CommunityDesign] Validating raw sequence files!\n",
      "2021-03-13 10:07:51 INFO: [NcbiTaxonomy] Building taxonomy tree...\n",
      "2021-03-13 10:07:51 INFO: [NcbiTaxonomy] Reading 'nodes' file:\t'/tmp/tmpKuLffd/NCBI/nodes.dmp'\n",
      "2021-03-13 10:08:07 INFO: [NcbiTaxonomy] Reading 'names' file:\t'/tmp/tmpKuLffd/NCBI/names.dmp'\n",
      "2021-03-13 10:08:10 INFO: [NcbiTaxonomy] Reading 'merged' file:\t'/tmp/tmpKuLffd/NCBI/merged.dmp'\n",
      "2021-03-13 10:08:10 INFO: [NcbiTaxonomy] Done (19.0s)\n",
      "2021-03-13 10:08:10 INFO: [MetagenomeSimulationPipeline] Move Genomes\n",
      "2021-03-13 10:08:15 INFO: [MetagenomeSimulationPipeline] Read simulation\n",
      "2021-03-13 10:08:15 INFO: [GenomePreparation 64152656348] Using 'mbarc' error profile.\n",
      "2021-03-13 10:08:15 INFO: [GenomePreparation 64152656348] Reading distribution file\n",
      "2021-03-13 10:08:15 INFO: [GenomePreparation 64152656348] Reading genome location file\n",
      "2021-03-13 10:08:21 INFO: [GenomePreparation 64152656348] Genome 'Genome29.0' has sequences below minimum, creating filtered copy.\n",
      "2021-03-13 10:08:49 INFO: [GenomePreparation 64152656348] Simulating reads using ReadSimulationArtIllumina readsimulator...\n",
      "2021-03-13 10:08:49 INFO: [GenomePreparation 64152656348] Simulating reads from Genome15.0: '/output/source_genomes/GCA_012247485.1_PDT000290379.2_genomic.fna'\n",
      "2021-03-13 10:08:49 INFO: [GenomePreparation 64152656348] Simulating reads from Genome32.0: '/output/source_genomes/GCA_016955275.1_PDT000971180.1_genomic.fna'\n",
      "2021-03-13 10:08:49 INFO: [GenomePreparation 64152656348] Simulating reads from Genome33.0: '/output/source_genomes/GCA_000529855.1_ISC7v1_genomic.fna'\n",
      "2021-03-13 10:08:49 INFO: [GenomePreparation 64152656348] Simulating reads from Genome29.0: '/tmp/tmpgr58uS/tmplNhtkj'\n",
      "2021-03-13 10:08:49 INFO: [GenomePreparation 64152656348] Simulating reads from Genome24.0: '/output/source_genomes/GCA_000155005.1_ASM15500v1_genomic.fna'\n",
      "2021-03-13 10:08:49 INFO: [GenomePreparation 64152656348] Simulating reads from Genome12.0: '/output/source_genomes/GCA_013037205.1_PDT000421869.1_genomic.fna'\n",
      "2021-03-13 10:08:49 INFO: [GenomePreparation 64152656348] Simulating reads from Genome25.0: '/output/source_genomes/GCA_013039825.1_PDT000415768.1_genomic.fna'\n",
      "2021-03-13 10:08:49 INFO: [GenomePreparation 64152656348] Simulating reads from Genome34.0: '/output/source_genomes/GCA_012212365.1_PDT000307416.2_genomic.fna'\n",
      "2021-03-13 10:08:49 INFO: [GenomePreparation 64152656348] Simulating reads from Genome21.0: '/output/source_genomes/GCA_013012665.1_PDT000430809.1_genomic.fna'\n",
      "2021-03-13 10:08:49 INFO: [GenomePreparation 64152656348] Simulating reads from Genome13.0: '/output/source_genomes/GCA_003113755.1_ASM311375v1_genomic.fna'\n",
      "2021-03-13 10:08:49 INFO: [GenomePreparation 64152656348] Simulating reads from Genome50.0: '/output/source_genomes/GCA_012246645.1_PDT000290443.2_genomic.fna'\n",
      "2021-03-13 10:08:49 INFO: [GenomePreparation 64152656348] Simulating reads from Genome10.0: '/output/source_genomes/GCA_000601155.1_ASM60115v1_genomic.fna'\n",
      "2021-03-13 10:08:49 INFO: [GenomePreparation 64152656348] Simulating reads from Genome2.0: '/output/source_genomes/GCA_012226665.1_PDT000312646.2_genomic.fna'\n",
      "2021-03-13 10:08:49 INFO: [GenomePreparation 64152656348] Simulating reads from Genome47.0: '/output/source_genomes/GCA_000250055.2_ASM25005v2_genomic.fna'\n",
      "2021-03-13 10:08:49 INFO: [GenomePreparation 64152656348] Simulating reads from Genome20.0: '/output/source_genomes/GCA_012246545.1_PDT000290364.2_genomic.fna'\n",
      "2021-03-13 10:08:49 INFO: [GenomePreparation 64152656348] Simulating reads from Genome38.0: '/output/source_genomes/GCA_000618905.3_ASM61890v3_genomic.fna'\n",
      "2021-03-13 10:08:49 INFO: [GenomePreparation 64152656348] Simulating reads from Genome6.0: '/output/source_genomes/GCA_016955375.1_PDT000971177.1_genomic.fna'\n",
      "2021-03-13 10:08:49 INFO: [GenomePreparation 64152656348] Simulating reads from Genome9.0: '/output/source_genomes/GCA_013014505.1_PDT000430688.1_genomic.fna'\n",
      "2021-03-13 10:08:49 INFO: [GenomePreparation 64152656348] Simulating reads from Genome27.0: '/output/source_genomes/GCA_012226285.1_PDT000312629.2_genomic.fna'\n",
      "2021-03-13 10:08:49 INFO: [GenomePreparation 64152656348] Simulating reads from Genome35.0: '/output/source_genomes/GCA_002741275.1_ASM274127v1_genomic.fna'\n",
      "2021-03-13 10:08:49 INFO: [GenomePreparation 64152656348] Simulating reads from Genome36.0: '/output/source_genomes/GCA_012166075.1_PDT000336566.2_genomic.fna'\n",
      "2021-03-13 10:08:49 INFO: [GenomePreparation 64152656348] Simulating reads from Genome26.0: '/output/source_genomes/GCA_013040665.1_PDT000415659.1_genomic.fna'\n",
      "2021-03-13 10:08:49 INFO: [GenomePreparation 64152656348] Simulating reads from Genome41.0: '/output/source_genomes/GCA_017137395.1_PDT000948260.1_genomic.fna'\n",
      "2021-03-13 10:08:49 INFO: [GenomePreparation 64152656348] Simulating reads from Genome44.0: '/output/source_genomes/GCA_016955055.1_PDT000971198.1_genomic.fna'\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.2\n",
      "\n",
      "The random seed for the run: 689939719\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0374493X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome29.01.fq\n",
      "\t the 2nd reads: /tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome29.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome29.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.16\n",
      "\n",
      "The random seed for the run: 854832072\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0161777X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome33.01.fq\n",
      "\t the 2nd reads: /tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome33.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome33.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.31\n",
      "\n",
      "The random seed for the run: 783731559\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.128494X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome25.01.fq\n",
      "\t the 2nd reads: /tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome25.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome25.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.36\n",
      "\n",
      "The random seed for the run: 1992431107\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.155813X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome15.01.fq\n",
      "\t the 2nd reads: /tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome15.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome15.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.34\n",
      "\n",
      "The random seed for the run: 199492375\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.157825X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome34.01.fq\n",
      "\t the 2nd reads: /tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome34.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome34.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.31\n",
      "\n",
      "The random seed for the run: 1923521921\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.126583X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome32.01.fq\n",
      "\t the 2nd reads: /tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome32.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome32.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.46\n",
      "\n",
      "The random seed for the run: 2102807139\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0595261X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome24.01.fq\n",
      "\t the 2nd reads: /tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome24.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome24.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.2\n",
      "\n",
      "The random seed for the run: 1208022829\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0527827X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome10.01.fq\n",
      "\t the 2nd reads: /tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome10.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome10.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.22\n",
      "\n",
      "The random seed for the run: 840935803\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0726674X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome50.01.fq\n",
      "\t the 2nd reads: /tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome50.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome50.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.13\n",
      "\n",
      "The random seed for the run: 1937252460\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.00358039X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome2.01.fq\n",
      "\t the 2nd reads: /tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome2.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome2.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.21\n",
      "\n",
      "The random seed for the run: 918751087\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0924008X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome9.01.fq\n",
      "\t the 2nd reads: /tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome9.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome9.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.36\n",
      "\n",
      "The random seed for the run: 26380989\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.180886X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome6.01.fq\n",
      "\t the 2nd reads: /tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome6.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome6.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.49\n",
      "\n",
      "The random seed for the run: 387153331\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.26037X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome20.01.fq\n",
      "\t the 2nd reads: /tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome20.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome20.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.24\n",
      "\n",
      "The random seed for the run: 597928115\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.114647X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome27.01.fq\n",
      "\t the 2nd reads: /tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome27.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome27.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.71\n",
      "\n",
      "The random seed for the run: 458448008\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.381065X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome47.01.fq\n",
      "\t the 2nd reads: /tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome47.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome47.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.11\n",
      "\n",
      "The random seed for the run: 2139276349\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.00241212X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome26.01.fq\n",
      "\t the 2nd reads: /tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome26.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome26.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.29\n",
      "\n",
      "The random seed for the run: 2090645473\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.157864X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome36.01.fq\n",
      "\t the 2nd reads: /tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome36.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome36.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.95\n",
      "\n",
      "The random seed for the run: 526849290\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.623107X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome38.01.fq\n",
      "\t the 2nd reads: /tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome38.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome38.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 2.04\n",
      "\n",
      "The random seed for the run: 1168110496\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            1.516X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome21.01.fq\n",
      "\t the 2nd reads: /tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome21.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome21.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 2.31\n",
      "\n",
      "The random seed for the run: 1203576411\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            1.99431X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome41.01.fq\n",
      "\t the 2nd reads: /tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome41.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome41.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 1.98\n",
      "\n",
      "The random seed for the run: 2029410300\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            1.73949X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome44.01.fq\n",
      "\t the 2nd reads: /tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome44.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome44.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 4.51\n",
      "\n",
      "The random seed for the run: 1940270192\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            3.72676X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome12.01.fq\n",
      "\t the 2nd reads: /tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome12.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome12.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 4.34\n",
      "\n",
      "The random seed for the run: 1268568432\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            2.77535X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome13.01.fq\n",
      "\t the 2nd reads: /tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome13.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome13.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 4.24\n",
      "\n",
      "The random seed for the run: 565794829\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            3.87903X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome35.01.fq\n",
      "\t the 2nd reads: /tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome35.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpgr58uS/2021.03.13_10.06.07_sample_0/reads/Genome35.0.sam\n",
      "\n",
      "2021-03-13 10:08:56 INFO: [GenomePreparation 64152656348] Simulating reads finished\n",
      "2021-03-13 10:09:05 INFO: [MetagenomeSimulationPipeline] Generate gold standard assembly\n",
      "2021-03-13 10:09:05 INFO: [MetadataReader 38994325909] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-13 10:10:00 INFO: [MetagenomeSimulationPipeline] Generate pooled strains gold standard assembly\n",
      "2021-03-13 10:10:00 INFO: [MetadataReader 34289848075] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-13 10:10:00 INFO: [MetagenomeSimulationPipeline] Samples used for pooled assembly: '2021.03.13_08.45.06_sample_0', '2021.03.13_10.06.07_sample_0'\n",
      "2021-03-13 10:10:00 INFO: [Validator 5358345304] Creating pooled gold standard\n",
      "2021-03-13 10:10:00 INFO: [Validator 5358345304] Merging bam files.\n",
      "2021-03-13 10:10:00 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.06.07_sample_0/bam/Genome15.0.bam'\n",
      "2021-03-13 10:10:00 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.06.07_sample_0/bam/Genome32.0.bam'\n",
      "2021-03-13 10:10:00 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.06.07_sample_0/bam/Genome33.0.bam'\n",
      "2021-03-13 10:10:00 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.06.07_sample_0/bam/Genome29.0.bam'\n",
      "2021-03-13 10:10:00 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.06.07_sample_0/bam/Genome24.0.bam'\n",
      "2021-03-13 10:10:00 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.06.07_sample_0/bam/Genome12.0.bam'\n",
      "2021-03-13 10:10:00 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.06.07_sample_0/bam/Genome25.0.bam'\n",
      "2021-03-13 10:10:00 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.06.07_sample_0/bam/Genome34.0.bam'\n",
      "2021-03-13 10:10:00 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.06.07_sample_0/bam/Genome47.0.bam'\n",
      "2021-03-13 10:10:00 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.06.07_sample_0/bam/Genome13.0.bam'\n",
      "2021-03-13 10:10:00 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.06.07_sample_0/bam/Genome50.0.bam'\n",
      "2021-03-13 10:10:00 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.06.07_sample_0/bam/Genome10.0.bam'\n",
      "2021-03-13 10:10:00 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.06.07_sample_0/bam/Genome2.0.bam'\n",
      "2021-03-13 10:10:00 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.06.07_sample_0/bam/Genome21.0.bam'\n",
      "2021-03-13 10:10:00 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.06.07_sample_0/bam/Genome20.0.bam'\n",
      "2021-03-13 10:10:00 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.06.07_sample_0/bam/Genome38.0.bam'\n",
      "2021-03-13 10:10:00 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.06.07_sample_0/bam/Genome6.0.bam'\n",
      "2021-03-13 10:10:00 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.06.07_sample_0/bam/Genome9.0.bam'\n",
      "2021-03-13 10:10:00 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.06.07_sample_0/bam/Genome27.0.bam'\n",
      "2021-03-13 10:10:00 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.06.07_sample_0/bam/Genome35.0.bam'\n",
      "2021-03-13 10:10:00 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.06.07_sample_0/bam/Genome36.0.bam'\n",
      "2021-03-13 10:10:00 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.06.07_sample_0/bam/Genome26.0.bam'\n",
      "2021-03-13 10:10:00 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.06.07_sample_0/bam/Genome41.0.bam'\n",
      "2021-03-13 10:10:00 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.06.07_sample_0/bam/Genome44.0.bam'\n",
      "2021-03-13 10:10:54 INFO: [MetagenomeSimulationPipeline] Anonymize Data\n",
      "2021-03-13 10:10:54 INFO: [FastaAnonymizer] Interweave shuffle and anonymize\n",
      "2021-03-13 10:12:25 INFO: [MetadataReader 55801910510] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-13 10:12:28 INFO: [MetadataReader 35298217545] Reading file: '/output/internal/meta_data.tsv'\n",
      "2021-03-13 10:12:28 INFO: [MetadataReader 50273713801] Reading file: '/tmp/tmpgr58uS/tmpatSmBv'\n",
      "2021-03-13 10:12:32 INFO: [FastaAnonymizer 20754120427] Shuffle and anonymize '/tmp/tmpgr58uS/tmpUBSOBk'\n",
      "2021-03-13 10:12:41 INFO: [MetadataReader 64547429207] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-13 10:12:43 INFO: [MetadataReader 10989772577] Reading file: '/output/internal/meta_data.tsv'\n",
      "2021-03-13 10:12:43 INFO: [MetadataReader 53809671431] Reading file: '/tmp/tmpgr58uS/read_start_positionsEsIMYw'\n",
      "2021-03-13 10:12:46 INFO: [MetadataReader 18168782768] Reading file: '/tmp/tmpgr58uS/tmpKMDY_Y'\n",
      "2021-03-13 10:14:09 INFO: [FastaAnonymizer 92324153505] Shuffle and anonymize '/tmp/tmpgr58uS/tmpPzoHPc'\n",
      "2021-03-13 10:14:19 INFO: [MetadataReader 83768748887] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-13 10:14:20 INFO: [MetadataReader 11249978774] Reading file: '/output/internal/meta_data.tsv'\n",
      "2021-03-13 10:14:20 INFO: [MetadataReader 66861377693] Reading file: '/tmp/tmpgr58uS/read_start_positionspM_GmK'\n",
      "2021-03-13 10:14:23 INFO: [MetadataReader 8017839927] Reading file: '/tmp/tmpgr58uS/tmpfCwhqe'\n",
      "2021-03-13 10:15:47 INFO: [MetagenomeSimulationPipeline] Compress Data\n",
      "2021-03-13 10:15:47 INFO: [Compress 66863001710] Compressing 'tmp0d3aer'\n",
      "2021-03-13 10:15:47 INFO: [Compress 34356485948] Compressing 'anonymous_gs_mappingrSBqke'\n",
      "2021-03-13 10:15:47 INFO: [Compress 29140370217] Compressing 'tmptIj01v'\n",
      "2021-03-13 10:15:47 INFO: [Compress 71975260973] Compressing 'anonymous_gsa_mapping_7PMgT'\n",
      "2021-03-13 10:15:47 INFO: [Compress 94252567579] Compressing 'tmpO0kpEu'\n",
      "2021-03-13 10:15:47 INFO: [Compress 35567253805] Compressing 'anonymous_gsa_pooled_mappingxObcsC'\n",
      "2021-03-13 10:15:47 INFO: [Compress 35567253805] Done compressing 'gsa_pooled_mapping.tsv.gz' in 0:00:00s.\n",
      "2021-03-13 10:15:47 INFO: [Compress 71975260973] Done compressing 'gsa_mapping.tsv.gz' in 0:00:00s.\n",
      "2021-03-13 10:15:49 INFO: [Compress 34356485948] Done compressing 'reads_mapping.tsv.gz' in 0:00:03s.\n",
      "2021-03-13 10:15:51 INFO: [Compress 94252567579] Done compressing 'anonymous_gsa_pooled.fasta.gz' in 0:00:04s.\n",
      "2021-03-13 10:15:51 INFO: [Compress 29140370217] Done compressing 'anonymous_gsa.fasta.gz' in 0:00:04s.\n",
      "2021-03-13 10:16:02 INFO: [Compress 66863001710] Done compressing 'anonymous_reads.fq.gz' in 0:00:15s.\n",
      "2021-03-13 10:16:02 INFO: [MetagenomeSimulationPipeline] Metagenome simulation finished\n",
      "simulating complexity_sim_10_genera_50_genomes\n",
      "Password:2021-03-13 10:16:05 INFO: [MetagenomeSimulationPipeline] Metagenome simulation starting\n",
      "2021-03-13 10:16:05 INFO: [MetagenomeSimulationPipeline] Validating Genomes\n",
      "2021-03-13 10:16:05 INFO: [MetadataReader] Reading file: '/input/genome_to_id.tsv'\n",
      "2021-03-13 10:16:57 INFO: [MetagenomeSimulationPipeline] Design Communities\n",
      "2021-03-13 10:16:57 INFO: [CommunityDesign] Drawing strains.\n",
      "2021-03-13 10:16:57 INFO: [MetadataReader 82380310288] Reading file: '/input/metadata.tsv'\n",
      "2021-03-13 10:16:57 INFO: [MetadataReader 89885674421] Reading file: '/input/genome_to_id.tsv'\n",
      "2021-03-13 10:16:57 INFO: [CommunityDesign] Validating raw sequence files!\n",
      "2021-03-13 10:17:22 INFO: [NcbiTaxonomy] Building taxonomy tree...\n",
      "2021-03-13 10:17:22 INFO: [NcbiTaxonomy] Reading 'nodes' file:\t'/tmp/tmp2xgR19/NCBI/nodes.dmp'\n",
      "2021-03-13 10:17:39 INFO: [NcbiTaxonomy] Reading 'names' file:\t'/tmp/tmp2xgR19/NCBI/names.dmp'\n",
      "2021-03-13 10:17:41 INFO: [NcbiTaxonomy] Reading 'merged' file:\t'/tmp/tmp2xgR19/NCBI/merged.dmp'\n",
      "2021-03-13 10:17:41 INFO: [NcbiTaxonomy] Done (19.0s)\n",
      "2021-03-13 10:17:41 INFO: [MetagenomeSimulationPipeline] Move Genomes\n",
      "2021-03-13 10:17:49 INFO: [MetagenomeSimulationPipeline] Read simulation\n",
      "2021-03-13 10:17:49 INFO: [GenomePreparation 64152656348] Using 'mbarc' error profile.\n",
      "2021-03-13 10:17:49 INFO: [GenomePreparation 64152656348] Reading distribution file\n",
      "2021-03-13 10:17:49 INFO: [GenomePreparation 64152656348] Reading genome location file\n",
      "2021-03-13 10:17:54 INFO: [GenomePreparation 64152656348] Genome 'Genome8.0' has sequences below minimum, creating filtered copy.\n",
      "2021-03-13 10:18:03 INFO: [GenomePreparation 64152656348] Genome 'Genome49.0' has sequences below minimum, creating filtered copy.\n",
      "2021-03-13 10:18:14 INFO: [GenomePreparation 64152656348] Simulating reads using ReadSimulationArtIllumina readsimulator...\n",
      "2021-03-13 10:18:14 INFO: [GenomePreparation 64152656348] Simulating reads from Genome14.0: '/output/source_genomes/GCA_001597655.1_ASM159765v1_genomic.fna'\n",
      "2021-03-13 10:18:14 INFO: [GenomePreparation 64152656348] Simulating reads from Genome25.0: '/output/source_genomes/GCA_000219255.3_CSt2.0_genomic.fna'\n",
      "2021-03-13 10:18:14 INFO: [GenomePreparation 64152656348] Simulating reads from Genome20.0: '/output/source_genomes/GCA_002949575.1_ASM294957v1_genomic.fna'\n",
      "2021-03-13 10:18:14 INFO: [GenomePreparation 64152656348] Simulating reads from Genome8.0: '/tmp/tmpDcYaN9/tmplpEkhA'\n",
      "2021-03-13 10:18:14 INFO: [GenomePreparation 64152656348] Simulating reads from Genome17.0: '/output/source_genomes/GCA_015059615.1_ASM1505961v1_genomic.fna'\n",
      "2021-03-13 10:18:14 INFO: [GenomePreparation 64152656348] Simulating reads from Genome28.0: '/output/source_genomes/GCA_015058325.1_ASM1505832v1_genomic.fna'\n",
      "2021-03-13 10:18:14 INFO: [GenomePreparation 64152656348] Simulating reads from Genome16.0: '/output/source_genomes/GCA_000741445.1_Bifthe_sub.por_genomic.fna'\n",
      "2021-03-13 10:18:14 INFO: [GenomePreparation 64152656348] Simulating reads from Genome32.0: '/output/source_genomes/GCA_000361525.1_Stap_aure_M0562_V1_genomic.fna'\n",
      "2021-03-13 10:18:14 INFO: [GenomePreparation 64152656348] Simulating reads from Genome43.0: '/output/source_genomes/GCA_000937835.1_3a5B_assembly_genomic.fna'\n",
      "2021-03-13 10:18:14 INFO: [GenomePreparation 64152656348] Simulating reads from Genome50.0: '/output/source_genomes/GCA_012247705.1_PDT000289859.2_genomic.fna'\n",
      "2021-03-13 10:18:14 INFO: [GenomePreparation 64152656348] Simulating reads from Genome4.0: '/output/source_genomes/GCA_004728485.1_PDT000353211.2_genomic.fna'\n",
      "2021-03-13 10:18:14 INFO: [GenomePreparation 64152656348] Simulating reads from Genome48.0: '/output/source_genomes/GCA_002353005.1_ASM235300v1_genomic.fna'\n",
      "2021-03-13 10:18:14 INFO: [GenomePreparation 64152656348] Simulating reads from Genome49.0: '/tmp/tmpDcYaN9/tmp7Aut8k'\n",
      "2021-03-13 10:18:14 INFO: [GenomePreparation 64152656348] Simulating reads from Genome39.0: '/output/source_genomes/GCA_902791925.1_Rumen_uncultured_genome_RUG13721_genomic.fna'\n",
      "2021-03-13 10:18:14 INFO: [GenomePreparation 64152656348] Simulating reads from Genome46.0: '/output/source_genomes/GCA_001311725.1_ASM131172v1_genomic.fna'\n",
      "2021-03-13 10:18:14 INFO: [GenomePreparation 64152656348] Simulating reads from Genome38.0: '/output/source_genomes/GCA_000168675.1_ASM16867v1_genomic.fna'\n",
      "2021-03-13 10:18:14 INFO: [GenomePreparation 64152656348] Simulating reads from Genome6.0: '/output/source_genomes/GCA_000431175.1_MGS59_genomic.fna'\n",
      "2021-03-13 10:18:14 INFO: [GenomePreparation 64152656348] Simulating reads from Genome45.0: '/output/source_genomes/GCA_000268085.1_ASM26808v1_genomic.fna'\n",
      "2021-03-13 10:18:14 INFO: [GenomePreparation 64152656348] Simulating reads from Genome9.0: '/output/source_genomes/GCA_000695725.1_Vshiloi203_genomic.fna'\n",
      "2021-03-13 10:18:14 INFO: [GenomePreparation 64152656348] Simulating reads from Genome7.0: '/output/source_genomes/GCA_000435315.1_MGS162_genomic.fna'\n",
      "2021-03-13 10:18:14 INFO: [GenomePreparation 64152656348] Simulating reads from Genome35.0: '/output/source_genomes/GCA_011845705.1_PDT000606249.2_genomic.fna'\n",
      "2021-03-13 10:18:14 INFO: [GenomePreparation 64152656348] Simulating reads from Genome40.0: '/output/source_genomes/GCA_013049715.1_PDT000290410.4_genomic.fna'\n",
      "2021-03-13 10:18:14 INFO: [GenomePreparation 64152656348] Simulating reads from Genome41.0: '/output/source_genomes/GCA_000567965.1_ASM56796v1_genomic.fna'\n",
      "2021-03-13 10:18:14 INFO: [GenomePreparation 64152656348] Simulating reads from Genome22.0: '/output/source_genomes/GCA_011050525.1_ASM1105052v1_genomic.fna'\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.12\n",
      "\n",
      "The random seed for the run: 1923521921\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0506813X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome25.01.fq\n",
      "\t the 2nd reads: /tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome25.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome25.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.07\n",
      "\n",
      "The random seed for the run: 2102807139\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.00484545X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome17.01.fq\n",
      "\t the 2nd reads: /tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome17.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome17.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.23\n",
      "\n",
      "The random seed for the run: 1940270192\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.213643X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome28.01.fq\n",
      "\t the 2nd reads: /tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome28.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome28.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.2\n",
      "\n",
      "The random seed for the run: 783731559\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.210867X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome16.01.fq\n",
      "\t the 2nd reads: /tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome16.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome16.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.22\n",
      "\n",
      "The random seed for the run: 1268568432\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0805585X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome50.01.fq\n",
      "\t the 2nd reads: /tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome50.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome50.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.36\n",
      "\n",
      "The random seed for the run: 854832072\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.173895X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome20.01.fq\n",
      "\t the 2nd reads: /tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome20.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome20.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.54\n",
      "\n",
      "The random seed for the run: 689939719\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.244799X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome8.01.fq\n",
      "\t the 2nd reads: /tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome8.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome8.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.17\n",
      "\n",
      "The random seed for the run: 840935803\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.155156X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome4.01.fq\n",
      "\t the 2nd reads: /tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome4.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome4.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.08\n",
      "\n",
      "The random seed for the run: 1937252460\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.00326439X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome49.01.fq\n",
      "\t the 2nd reads: /tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome49.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome49.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.13\n",
      "\n",
      "The random seed for the run: 458448008\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.171309X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome39.01.fq\n",
      "\t the 2nd reads: /tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome39.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome39.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.35\n",
      "\n",
      "The random seed for the run: 526849290\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.515707X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome38.01.fq\n",
      "\t the 2nd reads: /tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome38.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome38.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.2\n",
      "\n",
      "The random seed for the run: 918751087\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.098343X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome45.01.fq\n",
      "\t the 2nd reads: /tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome45.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome45.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.39\n",
      "\n",
      "The random seed for the run: 26380989\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.213589X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome6.01.fq\n",
      "\t the 2nd reads: /tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome6.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome6.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.93\n",
      "\n",
      "The random seed for the run: 387153331\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            2.35411X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome46.01.fq\n",
      "\t the 2nd reads: /tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome46.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome46.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.26\n",
      "\n",
      "The random seed for the run: 2090645473\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.125049X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome35.01.fq\n",
      "\t the 2nd reads: /tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome35.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome35.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.07\n",
      "\n",
      "The random seed for the run: 1203576411\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0218938X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome41.01.fq\n",
      "\t the 2nd reads: /tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome41.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome41.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.18\n",
      "\n",
      "The random seed for the run: 2139276349\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0714324X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome40.01.fq\n",
      "\t the 2nd reads: /tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome40.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome40.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 1.06\n",
      "\n",
      "The random seed for the run: 597928115\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.84327X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome9.01.fq\n",
      "\t the 2nd reads: /tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome9.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome9.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.37\n",
      "\n",
      "The random seed for the run: 2029410300\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.352367X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome22.01.fq\n",
      "\t the 2nd reads: /tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome22.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome22.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 1.48\n",
      "\n",
      "The random seed for the run: 565794829\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            2.05165X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome7.01.fq\n",
      "\t the 2nd reads: /tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome7.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome7.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 3.11\n",
      "\n",
      "The random seed for the run: 1992431107\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            2.69897X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome14.01.fq\n",
      "\t the 2nd reads: /tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome14.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome14.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 3.48\n",
      "\n",
      "The random seed for the run: 199492375\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            5.24961X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome32.01.fq\n",
      "\t the 2nd reads: /tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome32.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome32.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 4.1\n",
      "\n",
      "The random seed for the run: 1208022829\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            3.75597X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome48.01.fq\n",
      "\t the 2nd reads: /tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome48.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome48.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 5.72\n",
      "\n",
      "The random seed for the run: 1168110496\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            5.04354X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome43.01.fq\n",
      "\t the 2nd reads: /tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome43.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpDcYaN9/2021.03.13_10.16.05_sample_0/reads/Genome43.0.sam\n",
      "\n",
      "2021-03-13 10:18:22 INFO: [GenomePreparation 64152656348] Simulating reads finished\n",
      "2021-03-13 10:18:31 INFO: [MetagenomeSimulationPipeline] Generate gold standard assembly\n",
      "2021-03-13 10:18:31 INFO: [MetadataReader 38994325909] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-13 10:19:15 INFO: [MetagenomeSimulationPipeline] Generate pooled strains gold standard assembly\n",
      "2021-03-13 10:19:15 INFO: [MetadataReader 34289848075] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-13 10:19:15 INFO: [MetagenomeSimulationPipeline] Samples used for pooled assembly: '2021.03.13_10.16.05_sample_0'\n",
      "2021-03-13 10:19:15 INFO: [Validator 5358345304] Creating pooled gold standard\n",
      "2021-03-13 10:19:15 INFO: [Validator 5358345304] Merging bam files.\n",
      "2021-03-13 10:19:15 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.16.05_sample_0/bam/Genome14.0.bam'\n",
      "2021-03-13 10:19:15 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.16.05_sample_0/bam/Genome43.0.bam'\n",
      "2021-03-13 10:19:16 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.16.05_sample_0/bam/Genome46.0.bam'\n",
      "2021-03-13 10:19:16 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.16.05_sample_0/bam/Genome8.0.bam'\n",
      "2021-03-13 10:19:16 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.16.05_sample_0/bam/Genome17.0.bam'\n",
      "2021-03-13 10:19:16 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.16.05_sample_0/bam/Genome28.0.bam'\n",
      "2021-03-13 10:19:16 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.16.05_sample_0/bam/Genome16.0.bam'\n",
      "2021-03-13 10:19:16 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.16.05_sample_0/bam/Genome32.0.bam'\n",
      "2021-03-13 10:19:16 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.16.05_sample_0/bam/Genome25.0.bam'\n",
      "2021-03-13 10:19:16 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.16.05_sample_0/bam/Genome50.0.bam'\n",
      "2021-03-13 10:19:16 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.16.05_sample_0/bam/Genome4.0.bam'\n",
      "2021-03-13 10:19:16 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.16.05_sample_0/bam/Genome48.0.bam'\n",
      "2021-03-13 10:19:16 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.16.05_sample_0/bam/Genome49.0.bam'\n",
      "2021-03-13 10:19:16 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.16.05_sample_0/bam/Genome39.0.bam'\n",
      "2021-03-13 10:19:16 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.16.05_sample_0/bam/Genome20.0.bam'\n",
      "2021-03-13 10:19:16 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.16.05_sample_0/bam/Genome38.0.bam'\n",
      "2021-03-13 10:19:16 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.16.05_sample_0/bam/Genome6.0.bam'\n",
      "2021-03-13 10:19:16 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.16.05_sample_0/bam/Genome45.0.bam'\n",
      "2021-03-13 10:19:16 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.16.05_sample_0/bam/Genome9.0.bam'\n",
      "2021-03-13 10:19:16 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.16.05_sample_0/bam/Genome7.0.bam'\n",
      "2021-03-13 10:19:16 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.16.05_sample_0/bam/Genome35.0.bam'\n",
      "2021-03-13 10:19:16 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.16.05_sample_0/bam/Genome40.0.bam'\n",
      "2021-03-13 10:19:16 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.16.05_sample_0/bam/Genome41.0.bam'\n",
      "2021-03-13 10:19:16 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.16.05_sample_0/bam/Genome22.0.bam'\n",
      "2021-03-13 10:20:00 INFO: [MetagenomeSimulationPipeline] Anonymize Data\n",
      "2021-03-13 10:20:00 INFO: [FastaAnonymizer] Interweave shuffle and anonymize\n",
      "2021-03-13 10:21:30 INFO: [MetadataReader 55801910510] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-13 10:21:32 INFO: [MetadataReader 35298217545] Reading file: '/output/internal/meta_data.tsv'\n",
      "2021-03-13 10:21:32 INFO: [MetadataReader 50273713801] Reading file: '/tmp/tmpDcYaN9/tmpyFlmgR'\n",
      "2021-03-13 10:21:35 INFO: [FastaAnonymizer 20754120427] Shuffle and anonymize '/tmp/tmpDcYaN9/tmpX_kDjW'\n",
      "2021-03-13 10:21:43 INFO: [MetadataReader 64547429207] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-13 10:21:44 INFO: [MetadataReader 10989772577] Reading file: '/output/internal/meta_data.tsv'\n",
      "2021-03-13 10:21:44 INFO: [MetadataReader 53809671431] Reading file: '/tmp/tmpDcYaN9/read_start_positionsTcI3WI'\n",
      "2021-03-13 10:21:47 INFO: [MetadataReader 18168782768] Reading file: '/tmp/tmpDcYaN9/tmpx9maKA'\n",
      "2021-03-13 10:22:34 INFO: [FastaAnonymizer 92324153505] Shuffle and anonymize '/tmp/tmpDcYaN9/tmp9fNu67'\n",
      "2021-03-13 10:22:41 INFO: [MetadataReader 83768748887] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-13 10:22:42 INFO: [MetadataReader 11249978774] Reading file: '/output/internal/meta_data.tsv'\n",
      "2021-03-13 10:22:42 INFO: [MetadataReader 66861377693] Reading file: '/tmp/tmpDcYaN9/read_start_positionsurBBf8'\n",
      "2021-03-13 10:22:45 INFO: [MetadataReader 8017839927] Reading file: '/tmp/tmpDcYaN9/tmpZQn8K7'\n",
      "2021-03-13 10:23:32 INFO: [MetagenomeSimulationPipeline] Compress Data\n",
      "2021-03-13 10:23:32 INFO: [Compress 69562257547] Compressing 'tmp2IBNeW'\n",
      "2021-03-13 10:23:32 INFO: [Compress 34171117072] Compressing 'tmpMBtzEC'\n",
      "2021-03-13 10:23:32 INFO: [Compress 98189093990] Compressing 'anonymous_gsa_mappingGnyJQ2'\n",
      "2021-03-13 10:23:32 INFO: [Compress 57987230012] Compressing 'tmpbFjboC'\n",
      "2021-03-13 10:23:32 INFO: [Compress 10535190427] Compressing 'anonymous_gs_mappingkQB25m'\n",
      "2021-03-13 10:23:32 INFO: [Compress 21134328741] Compressing 'anonymous_gsa_pooled_mappingK_nxUr'\n",
      "2021-03-13 10:23:32 INFO: [Compress 21134328741] Done compressing 'gsa_pooled_mapping.tsv.gz' in 0:00:00s.\n",
      "2021-03-13 10:23:32 INFO: [Compress 98189093990] Done compressing 'gsa_mapping.tsv.gz' in 0:00:00s.\n",
      "2021-03-13 10:23:34 INFO: [Compress 10535190427] Done compressing 'reads_mapping.tsv.gz' in 0:00:03s.\n",
      "2021-03-13 10:23:35 INFO: [Compress 34171117072] Done compressing 'anonymous_gsa.fasta.gz' in 0:00:03s.\n",
      "2021-03-13 10:23:35 INFO: [Compress 57987230012] Done compressing 'anonymous_gsa_pooled.fasta.gz' in 0:00:03s.\n",
      "2021-03-13 10:23:47 INFO: [Compress 69562257547] Done compressing 'anonymous_reads.fq.gz' in 0:00:15s.\n",
      "2021-03-13 10:23:47 INFO: [MetagenomeSimulationPipeline] Metagenome simulation finished\n",
      "simulating complexity_sim_5_genera_50_genomes\n",
      "Password:2021-03-13 10:23:50 INFO: [MetagenomeSimulationPipeline] Metagenome simulation starting\n",
      "2021-03-13 10:23:50 INFO: [MetagenomeSimulationPipeline] Validating Genomes\n",
      "2021-03-13 10:23:50 INFO: [MetadataReader] Reading file: '/input/genome_to_id.tsv'\n",
      "2021-03-13 10:24:48 INFO: [MetagenomeSimulationPipeline] Design Communities\n",
      "2021-03-13 10:24:48 INFO: [CommunityDesign] Drawing strains.\n",
      "2021-03-13 10:24:48 INFO: [MetadataReader 82380310288] Reading file: '/input/metadata.tsv'\n",
      "2021-03-13 10:24:48 INFO: [MetadataReader 89885674421] Reading file: '/input/genome_to_id.tsv'\n",
      "2021-03-13 10:24:48 INFO: [CommunityDesign] Validating raw sequence files!\n",
      "2021-03-13 10:25:17 INFO: [NcbiTaxonomy] Building taxonomy tree...\n",
      "2021-03-13 10:25:17 INFO: [NcbiTaxonomy] Reading 'nodes' file:\t'/tmp/tmpLXKAuK/NCBI/nodes.dmp'\n",
      "2021-03-13 10:25:33 INFO: [NcbiTaxonomy] Reading 'names' file:\t'/tmp/tmpLXKAuK/NCBI/names.dmp'\n",
      "2021-03-13 10:25:36 INFO: [NcbiTaxonomy] Reading 'merged' file:\t'/tmp/tmpLXKAuK/NCBI/merged.dmp'\n",
      "2021-03-13 10:25:36 INFO: [NcbiTaxonomy] Done (19.0s)\n",
      "2021-03-13 10:25:36 INFO: [MetagenomeSimulationPipeline] Move Genomes\n",
      "2021-03-13 10:25:42 INFO: [MetagenomeSimulationPipeline] Read simulation\n",
      "2021-03-13 10:25:42 INFO: [GenomePreparation 64152656348] Using 'mbarc' error profile.\n",
      "2021-03-13 10:25:42 INFO: [GenomePreparation 64152656348] Reading distribution file\n",
      "2021-03-13 10:25:42 INFO: [GenomePreparation 64152656348] Reading genome location file\n",
      "2021-03-13 10:25:50 INFO: [GenomePreparation 64152656348] Genome 'Genome46.0' has sequences below minimum, creating filtered copy.\n",
      "2021-03-13 10:25:54 INFO: [GenomePreparation 64152656348] Genome 'Genome34.0' has sequences below minimum, creating filtered copy.\n",
      "2021-03-13 10:25:59 INFO: [GenomePreparation 64152656348] Genome 'Genome1.0' has sequences below minimum, creating filtered copy.\n",
      "2021-03-13 10:26:11 INFO: [GenomePreparation 64152656348] Simulating reads using ReadSimulationArtIllumina readsimulator...\n",
      "2021-03-13 10:26:11 INFO: [GenomePreparation 64152656348] Simulating reads from Genome14.0: '/output/source_genomes/GCA_000760625.1_VF539_1.0_genomic.fna'\n",
      "2021-03-13 10:26:11 INFO: [GenomePreparation 64152656348] Simulating reads from Genome29.0: '/output/source_genomes/GCA_000755405.1_ASM75540v1_genomic.fna'\n",
      "2021-03-13 10:26:11 INFO: [GenomePreparation 64152656348] Simulating reads from Genome42.0: '/output/source_genomes/GCA_000614165.1_ASM61416v1_genomic.fna'\n",
      "2021-03-13 10:26:11 INFO: [GenomePreparation 64152656348] Simulating reads from Genome12.0: '/output/source_genomes/GCA_905203125.1_ERR1430548-mag-bin.13_genomic.fna'\n",
      "2021-03-13 10:26:11 INFO: [GenomePreparation 64152656348] Simulating reads from Genome25.0: '/output/source_genomes/GCA_000225105.2_ASM22510v2_genomic.fna'\n",
      "2021-03-13 10:26:11 INFO: [GenomePreparation 64152656348] Simulating reads from Genome46.0: '/tmp/tmpATPYoM/tmpH5EKzq'\n",
      "2021-03-13 10:26:11 INFO: [GenomePreparation 64152656348] Simulating reads from Genome31.0: '/output/source_genomes/GCA_002379765.1_ASM237976v1_genomic.fna'\n",
      "2021-03-13 10:26:11 INFO: [GenomePreparation 64152656348] Simulating reads from Genome28.0: '/output/source_genomes/GCA_004763725.1_PDT000456809.2_genomic.fna'\n",
      "2021-03-13 10:26:11 INFO: [GenomePreparation 64152656348] Simulating reads from Genome16.0: '/output/source_genomes/GCA_011724275.1_PDT000711349.1_genomic.fna'\n",
      "2021-03-13 10:26:11 INFO: [GenomePreparation 64152656348] Simulating reads from Genome19.0: '/output/source_genomes/GCA_000219255.3_CSt2.0_genomic.fna'\n",
      "2021-03-13 10:26:11 INFO: [GenomePreparation 64152656348] Simulating reads from Genome34.0: '/tmp/tmpATPYoM/tmpNe0CRH'\n",
      "2021-03-13 10:26:11 INFO: [GenomePreparation 64152656348] Simulating reads from Genome43.0: '/output/source_genomes/GCA_000176055.1_ASM17605v1_genomic.fna'\n",
      "2021-03-13 10:26:11 INFO: [GenomePreparation 64152656348] Simulating reads from Genome13.0: '/output/source_genomes/GCA_000461895.1_OCN008_genomic.fna'\n",
      "2021-03-13 10:26:11 INFO: [GenomePreparation 64152656348] Simulating reads from Genome10.0: '/output/source_genomes/GCA_004709795.1_PDT000000907.4_genomic.fna'\n",
      "2021-03-13 10:26:11 INFO: [GenomePreparation 64152656348] Simulating reads from Genome1.0: '/tmp/tmpATPYoM/tmpuer8vq'\n",
      "2021-03-13 10:26:11 INFO: [GenomePreparation 64152656348] Simulating reads from Genome47.0: '/output/source_genomes/GCA_000249815.2_ASM24981v2_genomic.fna'\n",
      "2021-03-13 10:26:11 INFO: [GenomePreparation 64152656348] Simulating reads from Genome49.0: '/output/source_genomes/GCA_012247705.1_PDT000289859.2_genomic.fna'\n",
      "2021-03-13 10:26:11 INFO: [GenomePreparation 64152656348] Simulating reads from Genome3.0: '/output/source_genomes/GCA_001373115.1_4d10B_assembly_genomic.fna'\n",
      "2021-03-13 10:26:11 INFO: [GenomePreparation 64152656348] Simulating reads from Genome39.0: '/output/source_genomes/GCA_012246885.1_PDT000290429.2_genomic.fna'\n",
      "2021-03-13 10:26:11 INFO: [GenomePreparation 64152656348] Simulating reads from Genome20.0: '/output/source_genomes/GCA_013040005.1_PDT000415773.1_genomic.fna'\n",
      "2021-03-13 10:26:11 INFO: [GenomePreparation 64152656348] Simulating reads from Genome7.0: '/output/source_genomes/GCA_013040205.1_PDT000415762.1_genomic.fna'\n",
      "2021-03-13 10:26:11 INFO: [GenomePreparation 64152656348] Simulating reads from Genome35.0: '/output/source_genomes/GCA_000154025.1_ASM15402v1_genomic.fna'\n",
      "2021-03-13 10:26:11 INFO: [GenomePreparation 64152656348] Simulating reads from Genome36.0: '/output/source_genomes/GCA_000436455.1_MGS132_genomic.fna'\n",
      "2021-03-13 10:26:11 INFO: [GenomePreparation 64152656348] Simulating reads from Genome44.0: '/output/source_genomes/GCA_004711505.1_PDT000030590.4_genomic.fna'\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.11\n",
      "\n",
      "The random seed for the run: 1940270192\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.00509514X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome46.01.fq\n",
      "\t the 2nd reads: /tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome46.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome46.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.24\n",
      "\n",
      "The random seed for the run: 1923521921\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0751134X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome29.01.fq\n",
      "\t the 2nd reads: /tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome29.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome29.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.4\n",
      "\n",
      "The random seed for the run: 854832072\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.163151X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome42.01.fq\n",
      "\t the 2nd reads: /tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome42.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome42.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.21\n",
      "\n",
      "The random seed for the run: 1168110496\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.224652X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome16.01.fq\n",
      "\t the 2nd reads: /tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome16.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome16.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.08\n",
      "\n",
      "The random seed for the run: 1268568432\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0532929X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome19.01.fq\n",
      "\t the 2nd reads: /tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome19.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome19.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.06\n",
      "\n",
      "The random seed for the run: 840935803\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.370525X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome34.01.fq\n",
      "\t the 2nd reads: /tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome34.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome34.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.41\n",
      "\n",
      "The random seed for the run: 2102807139\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.221733X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome25.01.fq\n",
      "\t the 2nd reads: /tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome25.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome25.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.12\n",
      "\n",
      "The random seed for the run: 1208022829\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.00343261X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome43.01.fq\n",
      "\t the 2nd reads: /tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome43.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome43.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.74\n",
      "\n",
      "The random seed for the run: 689939719\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.886724X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome12.01.fq\n",
      "\t the 2nd reads: /tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome12.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome12.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.2\n",
      "\n",
      "The random seed for the run: 26380989\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.103411X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome49.01.fq\n",
      "\t the 2nd reads: /tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome49.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome49.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.2\n",
      "\n",
      "The random seed for the run: 526849290\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0847097X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome47.01.fq\n",
      "\t the 2nd reads: /tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome47.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome47.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.34\n",
      "\n",
      "The random seed for the run: 1937252460\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.180137X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome13.01.fq\n",
      "\t the 2nd reads: /tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome13.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome13.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.22\n",
      "\n",
      "The random seed for the run: 597928115\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.131493X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome39.01.fq\n",
      "\t the 2nd reads: /tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome39.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome39.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.35\n",
      "\n",
      "The random seed for the run: 918751087\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.182856X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome3.01.fq\n",
      "\t the 2nd reads: /tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome3.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome3.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.11\n",
      "\n",
      "The random seed for the run: 2090645473\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.023022X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome7.01.fq\n",
      "\t the 2nd reads: /tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome7.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome7.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.38\n",
      "\n",
      "The random seed for the run: 2139276349\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.224595X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome35.01.fq\n",
      "\t the 2nd reads: /tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome35.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome35.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.69\n",
      "\n",
      "The random seed for the run: 565794829\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.542281X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome20.01.fq\n",
      "\t the 2nd reads: /tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome20.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome20.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.34\n",
      "\n",
      "The random seed for the run: 1203576411\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.257413X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome36.01.fq\n",
      "\t the 2nd reads: /tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome36.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome36.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 2.06\n",
      "\n",
      "The random seed for the run: 458448008\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            2.83805X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome10.01.fq\n",
      "\t the 2nd reads: /tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome10.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome10.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 2.69\n",
      "\n",
      "The random seed for the run: 1992431107\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            2.15737X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome14.01.fq\n",
      "\t the 2nd reads: /tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome14.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome14.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 2.72\n",
      "\n",
      "The random seed for the run: 199492375\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            3.94952X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome28.01.fq\n",
      "\t the 2nd reads: /tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome28.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome28.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 3.62\n",
      "\n",
      "The random seed for the run: 783731559\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            5.30343X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome31.01.fq\n",
      "\t the 2nd reads: /tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome31.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome31.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 1.46\n",
      "\n",
      "The random seed for the run: 2029410300\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            2.47541X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome44.01.fq\n",
      "\t the 2nd reads: /tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome44.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome44.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 6.07\n",
      "\n",
      "The random seed for the run: 387153331\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            5.52013X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome1.01.fq\n",
      "\t the 2nd reads: /tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome1.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpATPYoM/2021.03.13_10.23.50_sample_0/reads/Genome1.0.sam\n",
      "\n",
      "2021-03-13 10:26:19 INFO: [GenomePreparation 64152656348] Simulating reads finished\n",
      "2021-03-13 10:26:29 INFO: [MetagenomeSimulationPipeline] Generate gold standard assembly\n",
      "2021-03-13 10:26:29 INFO: [MetadataReader 38994325909] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-13 10:27:15 INFO: [MetagenomeSimulationPipeline] Generate pooled strains gold standard assembly\n",
      "2021-03-13 10:27:15 INFO: [MetadataReader 34289848075] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-13 10:27:15 INFO: [MetagenomeSimulationPipeline] Samples used for pooled assembly: '2021.03.13_10.23.50_sample_0'\n",
      "2021-03-13 10:27:15 INFO: [Validator 5358345304] Creating pooled gold standard\n",
      "2021-03-13 10:27:15 INFO: [Validator 5358345304] Merging bam files.\n",
      "2021-03-13 10:27:15 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.23.50_sample_0/bam/Genome14.0.bam'\n",
      "2021-03-13 10:27:15 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.23.50_sample_0/bam/Genome29.0.bam'\n",
      "2021-03-13 10:27:15 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.23.50_sample_0/bam/Genome42.0.bam'\n",
      "2021-03-13 10:27:15 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.23.50_sample_0/bam/Genome12.0.bam'\n",
      "2021-03-13 10:27:15 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.23.50_sample_0/bam/Genome25.0.bam'\n",
      "2021-03-13 10:27:15 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.23.50_sample_0/bam/Genome46.0.bam'\n",
      "2021-03-13 10:27:15 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.23.50_sample_0/bam/Genome31.0.bam'\n",
      "2021-03-13 10:27:15 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.23.50_sample_0/bam/Genome28.0.bam'\n",
      "2021-03-13 10:27:15 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.23.50_sample_0/bam/Genome16.0.bam'\n",
      "2021-03-13 10:27:15 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.23.50_sample_0/bam/Genome19.0.bam'\n",
      "2021-03-13 10:27:15 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.23.50_sample_0/bam/Genome34.0.bam'\n",
      "2021-03-13 10:27:15 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.23.50_sample_0/bam/Genome43.0.bam'\n",
      "2021-03-13 10:27:15 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.23.50_sample_0/bam/Genome13.0.bam'\n",
      "2021-03-13 10:27:15 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.23.50_sample_0/bam/Genome10.0.bam'\n",
      "2021-03-13 10:27:15 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.23.50_sample_0/bam/Genome1.0.bam'\n",
      "2021-03-13 10:27:15 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.23.50_sample_0/bam/Genome47.0.bam'\n",
      "2021-03-13 10:27:15 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.23.50_sample_0/bam/Genome49.0.bam'\n",
      "2021-03-13 10:27:15 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.23.50_sample_0/bam/Genome3.0.bam'\n",
      "2021-03-13 10:27:15 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.23.50_sample_0/bam/Genome39.0.bam'\n",
      "2021-03-13 10:27:15 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.23.50_sample_0/bam/Genome20.0.bam'\n",
      "2021-03-13 10:27:15 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.23.50_sample_0/bam/Genome7.0.bam'\n",
      "2021-03-13 10:27:15 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.23.50_sample_0/bam/Genome35.0.bam'\n",
      "2021-03-13 10:27:15 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.23.50_sample_0/bam/Genome36.0.bam'\n",
      "2021-03-13 10:27:15 WARNING: [Validator 5358345304] List contains only one file: '/output/2021.03.13_10.23.50_sample_0/bam/Genome44.0.bam'\n",
      "2021-03-13 10:28:00 INFO: [MetagenomeSimulationPipeline] Anonymize Data\n",
      "2021-03-13 10:28:00 INFO: [FastaAnonymizer] Interweave shuffle and anonymize\n",
      "2021-03-13 10:29:31 INFO: [MetadataReader 55801910510] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-13 10:29:32 INFO: [MetadataReader 35298217545] Reading file: '/output/internal/meta_data.tsv'\n",
      "2021-03-13 10:29:32 INFO: [MetadataReader 50273713801] Reading file: '/tmp/tmpATPYoM/tmpcmKVLR'\n",
      "2021-03-13 10:29:36 INFO: [FastaAnonymizer 20754120427] Shuffle and anonymize '/tmp/tmpATPYoM/tmpJksgwi'\n",
      "2021-03-13 10:29:44 INFO: [MetadataReader 64547429207] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-13 10:29:45 INFO: [MetadataReader 10989772577] Reading file: '/output/internal/meta_data.tsv'\n",
      "2021-03-13 10:29:45 INFO: [MetadataReader 53809671431] Reading file: '/tmp/tmpATPYoM/read_start_positions_1g0HY'\n",
      "2021-03-13 10:29:48 INFO: [MetadataReader 18168782768] Reading file: '/tmp/tmpATPYoM/tmpLjltRP'\n",
      "2021-03-13 10:30:07 INFO: [FastaAnonymizer 92324153505] Shuffle and anonymize '/tmp/tmpATPYoM/tmpYt0heW'\n",
      "2021-03-13 10:30:15 INFO: [MetadataReader 83768748887] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-13 10:30:16 INFO: [MetadataReader 11249978774] Reading file: '/output/internal/meta_data.tsv'\n",
      "2021-03-13 10:30:16 INFO: [MetadataReader 66861377693] Reading file: '/tmp/tmpATPYoM/read_start_positionshlgi0S'\n",
      "2021-03-13 10:30:19 INFO: [MetadataReader 8017839927] Reading file: '/tmp/tmpATPYoM/tmpvaNcuC'\n",
      "2021-03-13 10:30:38 INFO: [MetagenomeSimulationPipeline] Compress Data\n",
      "2021-03-13 10:30:38 INFO: [Compress 33402612185] Compressing 'anonymous_gsa_mapping4vD5TV'\n",
      "2021-03-13 10:30:38 INFO: [Compress 98556138476] Compressing 'anonymous_gs_mappingle_AFx'\n",
      "2021-03-13 10:30:38 INFO: [Compress 55161328301] Compressing 'tmpzX8v9M'\n",
      "2021-03-13 10:30:38 INFO: [Compress 38490510641] Compressing 'tmpmbRJO2'\n",
      "2021-03-13 10:30:38 INFO: [Compress 6766207330] Compressing 'tmp7TW3OI'\n",
      "2021-03-13 10:30:38 INFO: [Compress 815671116] Compressing 'anonymous_gsa_pooled_mappingxU_nNF'\n",
      "2021-03-13 10:30:39 INFO: [Compress 33402612185] Done compressing 'gsa_mapping.tsv.gz' in 0:00:00s.\n",
      "2021-03-13 10:30:39 INFO: [Compress 815671116] Done compressing 'gsa_pooled_mapping.tsv.gz' in 0:00:00s.\n",
      "2021-03-13 10:30:41 INFO: [Compress 98556138476] Done compressing 'reads_mapping.tsv.gz' in 0:00:03s.\n",
      "2021-03-13 10:30:42 INFO: [Compress 55161328301] Done compressing 'anonymous_gsa.fasta.gz' in 0:00:03s.\n",
      "2021-03-13 10:30:42 INFO: [Compress 6766207330] Done compressing 'anonymous_gsa_pooled.fasta.gz' in 0:00:03s.\n",
      "2021-03-13 10:30:54 INFO: [Compress 38490510641] Done compressing 'anonymous_reads.fq.gz' in 0:00:15s.\n",
      "2021-03-13 10:30:54 INFO: [MetagenomeSimulationPipeline] Metagenome simulation finished\n",
      "simulating complexity_sim_5_genera_250_genomes\n",
      "Password:2021-03-13 10:30:57 INFO: [MetagenomeSimulationPipeline] Metagenome simulation starting\n",
      "2021-03-13 10:30:57 INFO: [MetagenomeSimulationPipeline] Validating Genomes\n",
      "2021-03-13 10:30:57 INFO: [MetadataReader] Reading file: '/input/genome_to_id.tsv'\n",
      "2021-03-13 10:35:47 INFO: [MetagenomeSimulationPipeline] Design Communities\n",
      "2021-03-13 10:35:47 INFO: [CommunityDesign] Drawing strains.\n",
      "2021-03-13 10:35:47 INFO: [MetadataReader 82380310288] Reading file: '/input/metadata.tsv'\n",
      "2021-03-13 10:35:47 INFO: [MetadataReader 28782960663] Reading file: '/input/genome_to_id.tsv'\n",
      "2021-03-13 10:35:47 INFO: [CommunityDesign] Validating raw sequence files!\n",
      "2021-03-13 10:36:13 INFO: [NcbiTaxonomy] Building taxonomy tree...\n",
      "2021-03-13 10:36:13 INFO: [NcbiTaxonomy] Reading 'nodes' file:\t'/tmp/tmp2azXja/NCBI/nodes.dmp'\n",
      "2021-03-13 10:36:29 INFO: [NcbiTaxonomy] Reading 'names' file:\t'/tmp/tmp2azXja/NCBI/names.dmp'\n",
      "2021-03-13 10:36:32 INFO: [NcbiTaxonomy] Reading 'merged' file:\t'/tmp/tmp2azXja/NCBI/merged.dmp'\n",
      "2021-03-13 10:36:32 INFO: [NcbiTaxonomy] Done (19.0s)\n",
      "2021-03-13 10:36:32 INFO: [MetagenomeSimulationPipeline] Move Genomes\n",
      "2021-03-13 10:36:37 INFO: [MetagenomeSimulationPipeline] Read simulation\n",
      "2021-03-13 10:36:37 INFO: [GenomePreparation 82430254076] Using 'mbarc' error profile.\n",
      "2021-03-13 10:36:37 INFO: [GenomePreparation 82430254076] Reading distribution file\n",
      "2021-03-13 10:36:37 INFO: [GenomePreparation 82430254076] Reading genome location file\n",
      "2021-03-13 10:36:45 INFO: [GenomePreparation 82430254076] Genome 'Genome59.0' has sequences below minimum, creating filtered copy.\n",
      "2021-03-13 10:36:47 INFO: [GenomePreparation 82430254076] Genome 'Genome103.0' has sequences below minimum, creating filtered copy.\n",
      "2021-03-13 10:37:03 INFO: [GenomePreparation 82430254076] Simulating reads using ReadSimulationArtIllumina readsimulator...\n",
      "2021-03-13 10:37:03 INFO: [GenomePreparation 82430254076] Simulating reads from Genome131.0: '/output/source_genomes/GCA_000183965.1_ASM18396v1_genomic.fna'\n",
      "2021-03-13 10:37:03 INFO: [GenomePreparation 82430254076] Simulating reads from Genome201.0: '/output/source_genomes/GCA_003463715.1_ASM346371v1_genomic.fna'\n",
      "2021-03-13 10:37:03 INFO: [GenomePreparation 82430254076] Simulating reads from Genome173.0: '/output/source_genomes/GCA_013407315.1_ASM1340731v1_genomic.fna'\n",
      "2021-03-13 10:37:03 INFO: [GenomePreparation 82430254076] Simulating reads from Genome68.0: '/output/source_genomes/GCA_000436435.1_MGS22_genomic.fna'\n",
      "2021-03-13 10:37:03 INFO: [GenomePreparation 82430254076] Simulating reads from Genome59.0: '/tmp/tmpycTTFa/tmpd6JZwr'\n",
      "2021-03-13 10:37:03 INFO: [GenomePreparation 82430254076] Simulating reads from Genome64.0: '/output/source_genomes/GCA_014845585.1_ASM1484558v1_genomic.fna'\n",
      "2021-03-13 10:37:03 INFO: [GenomePreparation 82430254076] Simulating reads from Genome103.0: '/tmp/tmpycTTFa/tmpuV8HBz'\n",
      "2021-03-13 10:37:03 INFO: [GenomePreparation 82430254076] Simulating reads from Genome218.0: '/output/source_genomes/GCA_002410665.1_ASM241066v1_genomic.fna'\n",
      "2021-03-13 10:37:03 INFO: [GenomePreparation 82430254076] Simulating reads from Genome84.0: '/output/source_genomes/GCA_000195065.1_ASM19506v1_genomic.fna'\n",
      "2021-03-13 10:37:03 INFO: [GenomePreparation 82430254076] Simulating reads from Genome193.0: '/output/source_genomes/GCA_000724385.2_It1_0.1_genomic.fna'\n",
      "2021-03-13 10:37:03 INFO: [GenomePreparation 82430254076] Simulating reads from Genome178.0: '/output/source_genomes/GCA_900576925.1_CLIP201500305T_genomic.fna'\n",
      "2021-03-13 10:37:03 INFO: [GenomePreparation 82430254076] Simulating reads from Genome139.0: '/output/source_genomes/GCA_000946735.1_Vibrio_cholerae_G_298_Guinea_genomic.fna'\n",
      "2021-03-13 10:37:03 INFO: [GenomePreparation 82430254076] Simulating reads from Genome55.0: '/output/source_genomes/GCA_000613545.1_ASM61354v1_genomic.fna'\n",
      "2021-03-13 10:37:03 INFO: [GenomePreparation 82430254076] Simulating reads from Genome57.0: '/output/source_genomes/GCA_000163855.1_ASM16385v1_genomic.fna'\n",
      "2021-03-13 10:37:03 INFO: [GenomePreparation 82430254076] Simulating reads from Genome90.0: '/output/source_genomes/GCA_000434095.1_MGS47_genomic.fna'\n",
      "2021-03-13 10:37:03 INFO: [GenomePreparation 82430254076] Simulating reads from Genome212.0: '/output/source_genomes/GCA_000382945.1_ASM38294v1_genomic.fna'\n",
      "2021-03-13 10:37:03 INFO: [GenomePreparation 82430254076] Simulating reads from Genome66.0: '/output/source_genomes/GCA_004709815.1_PDT000030593.4_genomic.fna'\n",
      "2021-03-13 10:37:03 INFO: [GenomePreparation 82430254076] Simulating reads from Genome213.0: '/output/source_genomes/GCA_905206855.1_ERR1600702-mag-bin.11_genomic.fna'\n",
      "2021-03-13 10:37:03 INFO: [GenomePreparation 82430254076] Simulating reads from Genome244.0: '/output/source_genomes/GCA_000436515.1_MGS109_genomic.fna'\n",
      "2021-03-13 10:37:03 INFO: [GenomePreparation 82430254076] Simulating reads from Genome221.0: '/output/source_genomes/GCA_000176055.1_ASM17605v1_genomic.fna'\n",
      "2021-03-13 10:37:03 INFO: [GenomePreparation 82430254076] Simulating reads from Genome4.0: '/output/source_genomes/GCA_015741235.1_PDT000893871.1_genomic.fna'\n",
      "2021-03-13 10:37:03 INFO: [GenomePreparation 82430254076] Simulating reads from Genome209.0: '/output/source_genomes/GCA_003433595.1_ASM343359v1_genomic.fna'\n",
      "2021-03-13 10:37:03 INFO: [GenomePreparation 82430254076] Simulating reads from Genome170.0: '/output/source_genomes/GCA_004711105.1_PDT000030596.4_genomic.fna'\n",
      "2021-03-13 10:37:03 INFO: [GenomePreparation 82430254076] Simulating reads from Genome28.0: '/output/source_genomes/GCA_004512525.1_PDT000030591.3_genomic.fna'\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.17\n",
      "\n",
      "The random seed for the run: 50971974\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.029767X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome103.01.fq\n",
      "\t the 2nd reads: /tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome103.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome103.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.19\n",
      "\n",
      "The random seed for the run: 963059643\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0375443X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome68.01.fq\n",
      "\t the 2nd reads: /tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome68.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome68.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.15\n",
      "\n",
      "The random seed for the run: 1594031057\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.114653X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome218.01.fq\n",
      "\t the 2nd reads: /tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome218.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome218.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.23\n",
      "\n",
      "The random seed for the run: 507207848\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0674667X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome131.01.fq\n",
      "\t the 2nd reads: /tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome131.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome131.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.25\n",
      "\n",
      "The random seed for the run: 1093218188\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0736701X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome173.01.fq\n",
      "\t the 2nd reads: /tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome173.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome173.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.13\n",
      "\n",
      "The random seed for the run: 941171800\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0153892X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome84.01.fq\n",
      "\t the 2nd reads: /tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome84.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome84.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.36\n",
      "\n",
      "The random seed for the run: 1027999508\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.461968X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome193.01.fq\n",
      "\t the 2nd reads: /tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome193.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome193.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.69\n",
      "\n",
      "The random seed for the run: 1044513605\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.696838X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome64.01.fq\n",
      "\t the 2nd reads: /tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome64.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome64.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.15\n",
      "\n",
      "The random seed for the run: 670345468\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.053204X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome90.01.fq\n",
      "\t the 2nd reads: /tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome90.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome90.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.11\n",
      "\n",
      "The random seed for the run: 460212766\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0775037X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome212.01.fq\n",
      "\t the 2nd reads: /tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome212.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome212.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.82\n",
      "\n",
      "The random seed for the run: 411957455\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.668913X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome57.01.fq\n",
      "\t the 2nd reads: /tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome57.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome57.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.17\n",
      "\n",
      "The random seed for the run: 1097061061\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.159288X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome66.01.fq\n",
      "\t the 2nd reads: /tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome66.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome66.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 1.29\n",
      "\n",
      "The random seed for the run: 1795828534\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.607698X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome201.01.fq\n",
      "\t the 2nd reads: /tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome201.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome201.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.21\n",
      "\n",
      "The random seed for the run: 1669488642\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.112346X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome213.01.fq\n",
      "\t the 2nd reads: /tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome213.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome213.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 1.21\n",
      "\n",
      "The random seed for the run: 1937464905\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            1.00403X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome55.01.fq\n",
      "\t the 2nd reads: /tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome55.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome55.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 1.11\n",
      "\n",
      "The random seed for the run: 2130432890\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.969433X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome139.01.fq\n",
      "\t the 2nd reads: /tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome139.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome139.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.07\n",
      "\n",
      "The random seed for the run: 1940362427\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0236676X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome170.01.fq\n",
      "\t the 2nd reads: /tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome170.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome170.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.49\n",
      "\n",
      "The random seed for the run: 370300383\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.569409X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome209.01.fq\n",
      "\t the 2nd reads: /tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome209.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome209.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.21\n",
      "\n",
      "The random seed for the run: 642284040\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.25351X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome28.01.fq\n",
      "\t the 2nd reads: /tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome28.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome28.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 3.25\n",
      "\n",
      "The random seed for the run: 393287511\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            2.58629X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome59.01.fq\n",
      "\t the 2nd reads: /tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome59.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome59.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 1.51\n",
      "\n",
      "The random seed for the run: 2029296162\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            1.8665X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome244.01.fq\n",
      "\t the 2nd reads: /tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome244.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome244.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 2.41\n",
      "\n",
      "The random seed for the run: 215267604\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            2.72511X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome4.01.fq\n",
      "\t the 2nd reads: /tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome4.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome4.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 3.02\n",
      "\n",
      "The random seed for the run: 263232151\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            2.79918X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome221.01.fq\n",
      "\t the 2nd reads: /tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome221.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome221.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 6.26\n",
      "\n",
      "The random seed for the run: 1414977469\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            11.3886X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome178.01.fq\n",
      "\t the 2nd reads: /tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome178.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpycTTFa/2021.03.13_10.30.57_sample_0/reads/Genome178.0.sam\n",
      "\n",
      "2021-03-13 10:37:11 INFO: [GenomePreparation 82430254076] Simulating reads finished\n",
      "2021-03-13 10:37:22 INFO: [MetagenomeSimulationPipeline] Generate gold standard assembly\n",
      "2021-03-13 10:37:22 INFO: [MetadataReader 87205974322] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-13 10:38:10 INFO: [MetagenomeSimulationPipeline] Generate pooled strains gold standard assembly\n",
      "2021-03-13 10:38:10 INFO: [MetadataReader 114087710] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-13 10:38:10 INFO: [MetagenomeSimulationPipeline] Samples used for pooled assembly: '2021.03.13_10.30.57_sample_0'\n",
      "2021-03-13 10:38:10 INFO: [Validator 10795378910] Creating pooled gold standard\n",
      "2021-03-13 10:38:10 INFO: [Validator 10795378910] Merging bam files.\n",
      "2021-03-13 10:38:10 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_10.30.57_sample_0/bam/Genome131.0.bam'\n",
      "2021-03-13 10:38:10 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_10.30.57_sample_0/bam/Genome201.0.bam'\n",
      "2021-03-13 10:38:10 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_10.30.57_sample_0/bam/Genome173.0.bam'\n",
      "2021-03-13 10:38:10 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_10.30.57_sample_0/bam/Genome68.0.bam'\n",
      "2021-03-13 10:38:10 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_10.30.57_sample_0/bam/Genome59.0.bam'\n",
      "2021-03-13 10:38:10 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_10.30.57_sample_0/bam/Genome64.0.bam'\n",
      "2021-03-13 10:38:10 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_10.30.57_sample_0/bam/Genome103.0.bam'\n",
      "2021-03-13 10:38:10 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_10.30.57_sample_0/bam/Genome218.0.bam'\n",
      "2021-03-13 10:38:10 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_10.30.57_sample_0/bam/Genome84.0.bam'\n",
      "2021-03-13 10:38:10 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_10.30.57_sample_0/bam/Genome193.0.bam'\n",
      "2021-03-13 10:38:10 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_10.30.57_sample_0/bam/Genome178.0.bam'\n",
      "2021-03-13 10:38:10 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_10.30.57_sample_0/bam/Genome139.0.bam'\n",
      "2021-03-13 10:38:10 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_10.30.57_sample_0/bam/Genome55.0.bam'\n",
      "2021-03-13 10:38:10 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_10.30.57_sample_0/bam/Genome57.0.bam'\n",
      "2021-03-13 10:38:10 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_10.30.57_sample_0/bam/Genome90.0.bam'\n",
      "2021-03-13 10:38:10 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_10.30.57_sample_0/bam/Genome212.0.bam'\n",
      "2021-03-13 10:38:10 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_10.30.57_sample_0/bam/Genome66.0.bam'\n",
      "2021-03-13 10:38:10 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_10.30.57_sample_0/bam/Genome213.0.bam'\n",
      "2021-03-13 10:38:10 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_10.30.57_sample_0/bam/Genome244.0.bam'\n",
      "2021-03-13 10:38:10 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_10.30.57_sample_0/bam/Genome221.0.bam'\n",
      "2021-03-13 10:38:10 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_10.30.57_sample_0/bam/Genome4.0.bam'\n",
      "2021-03-13 10:38:10 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_10.30.57_sample_0/bam/Genome209.0.bam'\n",
      "2021-03-13 10:38:10 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_10.30.57_sample_0/bam/Genome170.0.bam'\n",
      "2021-03-13 10:38:10 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_10.30.57_sample_0/bam/Genome28.0.bam'\n",
      "2021-03-13 10:38:58 INFO: [MetagenomeSimulationPipeline] Anonymize Data\n",
      "2021-03-13 10:38:58 INFO: [FastaAnonymizer] Interweave shuffle and anonymize\n",
      "2021-03-13 10:40:30 INFO: [MetadataReader 4631107874] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-13 10:40:31 INFO: [MetadataReader 14300761082] Reading file: '/output/internal/meta_data.tsv'\n",
      "2021-03-13 10:40:31 INFO: [MetadataReader 87652392539] Reading file: '/tmp/tmpycTTFa/tmp8l8tAz'\n",
      "2021-03-13 10:40:35 INFO: [FastaAnonymizer 66694448886] Shuffle and anonymize '/tmp/tmpycTTFa/tmpJxIEZD'\n",
      "2021-03-13 10:40:44 INFO: [MetadataReader 4079751341] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-13 10:40:45 INFO: [MetadataReader 84063950941] Reading file: '/output/internal/meta_data.tsv'\n",
      "2021-03-13 10:40:45 INFO: [MetadataReader 48704713275] Reading file: '/tmp/tmpycTTFa/read_start_positions9wyJnG'\n",
      "2021-03-13 10:40:48 INFO: [MetadataReader 43119039112] Reading file: '/tmp/tmpycTTFa/tmpaLm6CX'\n",
      "2021-03-13 10:41:09 INFO: [FastaAnonymizer 64368966291] Shuffle and anonymize '/tmp/tmpycTTFa/tmpDmy7wV'\n",
      "2021-03-13 10:41:18 INFO: [MetadataReader 78717221707] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-13 10:41:19 INFO: [MetadataReader 89765374142] Reading file: '/output/internal/meta_data.tsv'\n",
      "2021-03-13 10:41:19 INFO: [MetadataReader 98541404663] Reading file: '/tmp/tmpycTTFa/read_start_positionsVHqkxo'\n",
      "2021-03-13 10:41:22 INFO: [MetadataReader 99062647065] Reading file: '/tmp/tmpycTTFa/tmp7NUMnL'\n",
      "2021-03-13 10:41:43 INFO: [MetagenomeSimulationPipeline] Compress Data\n",
      "2021-03-13 10:41:43 INFO: [Compress 91620105290] Compressing 'tmp1STEnn'\n",
      "2021-03-13 10:41:43 INFO: [Compress 98018931343] Compressing 'anonymous_gs_mappingtHMDhG'\n",
      "2021-03-13 10:41:43 INFO: [Compress 73838516429] Compressing 'tmpE8763j'\n",
      "2021-03-13 10:41:43 INFO: [Compress 45892943365] Compressing 'anonymous_gsa_mapping08c1W4'\n",
      "2021-03-13 10:41:43 INFO: [Compress 68653803610] Compressing 'anonymous_gsa_pooled_mappinggiJPGE'\n",
      "2021-03-13 10:41:43 INFO: [Compress 22635882602] Compressing 'tmpdWiYvA'\n",
      "2021-03-13 10:41:44 INFO: [Compress 68653803610] Done compressing 'gsa_pooled_mapping.tsv.gz' in 0:00:00s.\n",
      "2021-03-13 10:41:44 INFO: [Compress 45892943365] Done compressing 'gsa_mapping.tsv.gz' in 0:00:00s.\n",
      "2021-03-13 10:41:46 INFO: [Compress 98018931343] Done compressing 'reads_mapping.tsv.gz' in 0:00:03s.\n",
      "2021-03-13 10:41:47 INFO: [Compress 22635882602] Done compressing 'anonymous_gsa_pooled.fasta.gz' in 0:00:04s.\n",
      "2021-03-13 10:41:47 INFO: [Compress 73838516429] Done compressing 'anonymous_gsa.fasta.gz' in 0:00:04s.\n",
      "2021-03-13 10:41:59 INFO: [Compress 91620105290] Done compressing 'anonymous_reads.fq.gz' in 0:00:16s.\n",
      "2021-03-13 10:41:59 INFO: [MetagenomeSimulationPipeline] Metagenome simulation finished\n",
      "simulating complexity_sim_10_genera_250_genomes\n",
      "Password:2021-03-13 10:42:02 INFO: [MetagenomeSimulationPipeline] Metagenome simulation starting\n",
      "2021-03-13 10:42:02 INFO: [MetagenomeSimulationPipeline] Validating Genomes\n",
      "2021-03-13 10:42:02 INFO: [MetadataReader] Reading file: '/input/genome_to_id.tsv'\n",
      "2021-03-13 10:46:26 INFO: [MetagenomeSimulationPipeline] Design Communities\n",
      "2021-03-13 10:46:26 INFO: [CommunityDesign] Drawing strains.\n",
      "2021-03-13 10:46:26 INFO: [MetadataReader 82380310288] Reading file: '/input/metadata.tsv'\n",
      "2021-03-13 10:46:26 INFO: [MetadataReader 28782960663] Reading file: '/input/genome_to_id.tsv'\n",
      "2021-03-13 10:46:26 INFO: [CommunityDesign] Validating raw sequence files!\n",
      "2021-03-13 10:46:52 INFO: [NcbiTaxonomy] Building taxonomy tree...\n",
      "2021-03-13 10:46:52 INFO: [NcbiTaxonomy] Reading 'nodes' file:\t'/tmp/tmp0etdiN/NCBI/nodes.dmp'\n",
      "2021-03-13 10:47:08 INFO: [NcbiTaxonomy] Reading 'names' file:\t'/tmp/tmp0etdiN/NCBI/names.dmp'\n",
      "2021-03-13 10:47:11 INFO: [NcbiTaxonomy] Reading 'merged' file:\t'/tmp/tmp0etdiN/NCBI/merged.dmp'\n",
      "2021-03-13 10:47:11 INFO: [NcbiTaxonomy] Done (19.0s)\n",
      "2021-03-13 10:47:11 INFO: [MetagenomeSimulationPipeline] Move Genomes\n",
      "2021-03-13 10:47:18 INFO: [MetagenomeSimulationPipeline] Read simulation\n",
      "2021-03-13 10:47:18 INFO: [GenomePreparation 82430254076] Using 'mbarc' error profile.\n",
      "2021-03-13 10:47:18 INFO: [GenomePreparation 82430254076] Reading distribution file\n",
      "2021-03-13 10:47:18 INFO: [GenomePreparation 82430254076] Reading genome location file\n",
      "2021-03-13 10:47:19 INFO: [GenomePreparation 82430254076] Genome 'Genome33.0' has sequences below minimum, creating filtered copy.\n",
      "2021-03-13 10:47:24 INFO: [GenomePreparation 82430254076] Genome 'Genome137.0' has sequences below minimum, creating filtered copy.\n",
      "2021-03-13 10:47:37 INFO: [GenomePreparation 82430254076] Genome 'Genome5.0' has sequences below minimum, creating filtered copy.\n",
      "2021-03-13 10:47:41 INFO: [GenomePreparation 82430254076] Genome 'Genome99.0' has sequences below minimum, creating filtered copy.\n",
      "2021-03-13 10:47:43 INFO: [GenomePreparation 82430254076] Genome 'Genome187.0' has sequences below minimum, creating filtered copy.\n",
      "2021-03-13 10:47:44 INFO: [GenomePreparation 82430254076] Simulating reads using ReadSimulationArtIllumina readsimulator...\n",
      "2021-03-13 10:47:44 INFO: [GenomePreparation 82430254076] Simulating reads from Genome33.0: '/tmp/tmpnEsCS5/tmp5_lLDP'\n",
      "2021-03-13 10:47:44 INFO: [GenomePreparation 82430254076] Simulating reads from Genome128.0: '/output/source_genomes/GCA_000435915.1_MGS234_genomic.fna'\n",
      "2021-03-13 10:47:44 INFO: [GenomePreparation 82430254076] Simulating reads from Genome59.0: '/output/source_genomes/GCA_001576945.1_ASM157694v1_genomic.fna'\n",
      "2021-03-13 10:47:44 INFO: [GenomePreparation 82430254076] Simulating reads from Genome103.0: '/output/source_genomes/GCA_000193915.2_ASM19391v2_genomic.fna'\n",
      "2021-03-13 10:47:44 INFO: [GenomePreparation 82430254076] Simulating reads from Genome150.0: '/output/source_genomes/GCA_000436815.1_MGS263_genomic.fna'\n",
      "2021-03-13 10:47:44 INFO: [GenomePreparation 82430254076] Simulating reads from Genome137.0: '/tmp/tmpnEsCS5/tmpj6UocG'\n",
      "2021-03-13 10:47:44 INFO: [GenomePreparation 82430254076] Simulating reads from Genome75.0: '/output/source_genomes/GCA_000431955.1_MGS27_genomic.fna'\n",
      "2021-03-13 10:47:44 INFO: [GenomePreparation 82430254076] Simulating reads from Genome74.0: '/output/source_genomes/GCA_000936065.1_1a3B_assembly_genomic.fna'\n",
      "2021-03-13 10:47:44 INFO: [GenomePreparation 82430254076] Simulating reads from Genome149.0: '/output/source_genomes/GCA_000437135.1_MGS315_genomic.fna'\n",
      "2021-03-13 10:47:44 INFO: [GenomePreparation 82430254076] Simulating reads from Genome196.0: '/output/source_genomes/GCA_016955215.1_PDT000971186.1_genomic.fna'\n",
      "2021-03-13 10:47:44 INFO: [GenomePreparation 82430254076] Simulating reads from Genome191.0: '/output/source_genomes/GCA_000436455.1_MGS132_genomic.fna'\n",
      "2021-03-13 10:47:44 INFO: [GenomePreparation 82430254076] Simulating reads from Genome234.0: '/output/source_genomes/GCA_001576885.1_ASM157688v1_genomic.fna'\n",
      "2021-03-13 10:47:44 INFO: [GenomePreparation 82430254076] Simulating reads from Genome72.0: '/output/source_genomes/GCA_004709815.1_PDT000030593.4_genomic.fna'\n",
      "2021-03-13 10:47:44 INFO: [GenomePreparation 82430254076] Simulating reads from Genome244.0: '/output/source_genomes/GCA_000705295.1_V.cholerae_O1_biovar_El_Tor_str._M-1293_genomic.fna'\n",
      "2021-03-13 10:47:44 INFO: [GenomePreparation 82430254076] Simulating reads from Genome169.0: '/output/source_genomes/GCA_009773955.1_ASM977395v1_genomic.fna'\n",
      "2021-03-13 10:47:44 INFO: [GenomePreparation 82430254076] Simulating reads from Genome142.0: '/output/source_genomes/GCA_000783795.2_ASM78379v2_genomic.fna'\n",
      "2021-03-13 10:47:44 INFO: [GenomePreparation 82430254076] Simulating reads from Genome246.0: '/output/source_genomes/GCA_009738435.1_ASM973843v1_genomic.fna'\n",
      "2021-03-13 10:47:44 INFO: [GenomePreparation 82430254076] Simulating reads from Genome5.0: '/tmp/tmpnEsCS5/tmpPNgOOk'\n",
      "2021-03-13 10:47:44 INFO: [GenomePreparation 82430254076] Simulating reads from Genome223.0: '/output/source_genomes/GCA_000092525.1_ASM9252v1_genomic.fna'\n",
      "2021-03-13 10:47:44 INFO: [GenomePreparation 82430254076] Simulating reads from Genome170.0: '/output/source_genomes/GCA_905207145.1_ERR260204-mag-bin.19_genomic.fna'\n",
      "2021-03-13 10:47:44 INFO: [GenomePreparation 82430254076] Simulating reads from Genome240.0: '/output/source_genomes/GCA_001311705.1_ASM131170v1_genomic.fna'\n",
      "2021-03-13 10:47:44 INFO: [GenomePreparation 82430254076] Simulating reads from Genome99.0: '/tmp/tmpnEsCS5/tmp7BuK9n'\n",
      "2021-03-13 10:47:44 INFO: [GenomePreparation 82430254076] Simulating reads from Genome10.0: '/output/source_genomes/GCA_004728485.1_PDT000353211.2_genomic.fna'\n",
      "2021-03-13 10:47:44 INFO: [GenomePreparation 82430254076] Simulating reads from Genome187.0: '/tmp/tmpnEsCS5/tmp8FFt0m'\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.07\n",
      "\n",
      "The random seed for the run: 393287511\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0471417X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome150.01.fq\n",
      "\t the 2nd reads: /tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome150.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome150.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.14\n",
      "\n",
      "The random seed for the run: 963059643\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0193231X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome103.01.fq\n",
      "\t the 2nd reads: /tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome103.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome103.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.27\n",
      "\n",
      "The random seed for the run: 1044513605\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0847133X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome137.01.fq\n",
      "\t the 2nd reads: /tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome137.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome137.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.13\n",
      "\n",
      "The random seed for the run: 1027999508\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0297177X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome196.01.fq\n",
      "\t the 2nd reads: /tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome196.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome196.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.05\n",
      "\n",
      "The random seed for the run: 2130432890\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.097316X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome234.01.fq\n",
      "\t the 2nd reads: /tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome234.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome234.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.1\n",
      "\n",
      "The random seed for the run: 1937464905\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0668046X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome72.01.fq\n",
      "\t the 2nd reads: /tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome72.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome72.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 1.04\n",
      "\n",
      "The random seed for the run: 507207848\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.763044X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome33.01.fq\n",
      "\t the 2nd reads: /tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome33.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome33.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 1.03\n",
      "\n",
      "The random seed for the run: 941171800\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.874972X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome149.01.fq\n",
      "\t the 2nd reads: /tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome149.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome149.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.11\n",
      "\n",
      "The random seed for the run: 411957455\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0373764X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome244.01.fq\n",
      "\t the 2nd reads: /tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome244.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome244.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 1.74\n",
      "\n",
      "The random seed for the run: 1795828534\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            3.24743X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome128.01.fq\n",
      "\t the 2nd reads: /tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome128.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome128.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.28\n",
      "\n",
      "The random seed for the run: 670345468\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.580061X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome169.01.fq\n",
      "\t the 2nd reads: /tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome169.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome169.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.19\n",
      "\n",
      "The random seed for the run: 1097061061\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.141065X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome246.01.fq\n",
      "\t the 2nd reads: /tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome246.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome246.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.33\n",
      "\n",
      "The random seed for the run: 1669488642\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.200007X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome5.01.fq\n",
      "\t the 2nd reads: /tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome5.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome5.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.21\n",
      "\n",
      "The random seed for the run: 2029296162\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.0925025X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome223.01.fq\n",
      "\t the 2nd reads: /tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome223.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome223.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.18\n",
      "\n",
      "The random seed for the run: 263232151\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.318315X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome170.01.fq\n",
      "\t the 2nd reads: /tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome170.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome170.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.89\n",
      "\n",
      "The random seed for the run: 460212766\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.714967X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome142.01.fq\n",
      "\t the 2nd reads: /tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome142.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome142.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.14\n",
      "\n",
      "The random seed for the run: 1940362427\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.143962X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome10.01.fq\n",
      "\t the 2nd reads: /tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome10.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome10.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 2.28\n",
      "\n",
      "The random seed for the run: 50971974\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            3.42173X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome75.01.fq\n",
      "\t the 2nd reads: /tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome75.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome75.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 0.63\n",
      "\n",
      "The random seed for the run: 215267604\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            1.26069X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome240.01.fq\n",
      "\t the 2nd reads: /tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome240.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome240.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 3.58\n",
      "\n",
      "The random seed for the run: 1594031057\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            2.34364X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome74.01.fq\n",
      "\t the 2nd reads: /tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome74.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome74.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 1.13\n",
      "\n",
      "The random seed for the run: 370300383\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0.839908X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome99.01.fq\n",
      "\t the 2nd reads: /tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome99.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome99.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 3.41\n",
      "\n",
      "The random seed for the run: 1414977469\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            3.51474X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome191.01.fq\n",
      "\t the 2nd reads: /tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome191.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome191.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 4.43\n",
      "\n",
      "The random seed for the run: 1093218188\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            14.2999X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome59.01.fq\n",
      "\t the 2nd reads: /tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome59.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome59.0.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.6 (Mar 9, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 1.51\n",
      "\n",
      "The random seed for the run: 642284040\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            1.21725X\n",
      "\tMean Fragment Length:     270\n",
      "\tStandard Deviation:       27\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R1.txt (user's profile)\n",
      "\tSecond Read:  /usr/local/bin/tools/art_illumina-2.3.6/profiles/ART_MBARC-26_HiSeq_R2.txt (user's profile)\n",
      "\n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: /tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome187.01.fq\n",
      "\t the 2nd reads: /tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome187.02.fq\n",
      "\n",
      "  SAM Alignment File:\n",
      "\t/tmp/tmpnEsCS5/2021.03.13_10.42.02_sample_0/reads/Genome187.0.sam\n",
      "\n",
      "2021-03-13 10:47:49 INFO: [GenomePreparation 82430254076] Simulating reads finished\n",
      "2021-03-13 10:47:58 INFO: [MetagenomeSimulationPipeline] Generate gold standard assembly\n",
      "2021-03-13 10:47:58 INFO: [MetadataReader 87205974322] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-13 10:48:47 INFO: [MetagenomeSimulationPipeline] Generate pooled strains gold standard assembly\n",
      "2021-03-13 10:48:47 INFO: [MetadataReader 114087710] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-13 10:48:47 INFO: [MetagenomeSimulationPipeline] Samples used for pooled assembly: '2021.03.13_10.42.02_sample_0'\n",
      "2021-03-13 10:48:47 INFO: [Validator 10795378910] Creating pooled gold standard\n",
      "2021-03-13 10:48:47 INFO: [Validator 10795378910] Merging bam files.\n",
      "2021-03-13 10:48:47 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_10.42.02_sample_0/bam/Genome33.0.bam'\n",
      "2021-03-13 10:48:47 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_10.42.02_sample_0/bam/Genome128.0.bam'\n",
      "2021-03-13 10:48:48 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_10.42.02_sample_0/bam/Genome59.0.bam'\n",
      "2021-03-13 10:48:48 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_10.42.02_sample_0/bam/Genome103.0.bam'\n",
      "2021-03-13 10:48:48 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_10.42.02_sample_0/bam/Genome150.0.bam'\n",
      "2021-03-13 10:48:48 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_10.42.02_sample_0/bam/Genome137.0.bam'\n",
      "2021-03-13 10:48:48 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_10.42.02_sample_0/bam/Genome75.0.bam'\n",
      "2021-03-13 10:48:48 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_10.42.02_sample_0/bam/Genome74.0.bam'\n",
      "2021-03-13 10:48:48 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_10.42.02_sample_0/bam/Genome149.0.bam'\n",
      "2021-03-13 10:48:48 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_10.42.02_sample_0/bam/Genome72.0.bam'\n",
      "2021-03-13 10:48:48 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_10.42.02_sample_0/bam/Genome191.0.bam'\n",
      "2021-03-13 10:48:48 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_10.42.02_sample_0/bam/Genome234.0.bam'\n",
      "2021-03-13 10:48:48 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_10.42.02_sample_0/bam/Genome196.0.bam'\n",
      "2021-03-13 10:48:48 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_10.42.02_sample_0/bam/Genome244.0.bam'\n",
      "2021-03-13 10:48:48 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_10.42.02_sample_0/bam/Genome169.0.bam'\n",
      "2021-03-13 10:48:48 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_10.42.02_sample_0/bam/Genome142.0.bam'\n",
      "2021-03-13 10:48:48 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_10.42.02_sample_0/bam/Genome246.0.bam'\n",
      "2021-03-13 10:48:48 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_10.42.02_sample_0/bam/Genome5.0.bam'\n",
      "2021-03-13 10:48:48 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_10.42.02_sample_0/bam/Genome223.0.bam'\n",
      "2021-03-13 10:48:48 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_10.42.02_sample_0/bam/Genome170.0.bam'\n",
      "2021-03-13 10:48:48 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_10.42.02_sample_0/bam/Genome240.0.bam'\n",
      "2021-03-13 10:48:48 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_10.42.02_sample_0/bam/Genome99.0.bam'\n",
      "2021-03-13 10:48:48 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_10.42.02_sample_0/bam/Genome10.0.bam'\n",
      "2021-03-13 10:48:48 WARNING: [Validator 10795378910] List contains only one file: '/output/2021.03.13_10.42.02_sample_0/bam/Genome187.0.bam'\n",
      "2021-03-13 10:49:37 INFO: [MetagenomeSimulationPipeline] Anonymize Data\n",
      "2021-03-13 10:49:37 INFO: [FastaAnonymizer] Interweave shuffle and anonymize\n",
      "2021-03-13 10:51:06 INFO: [MetadataReader 4631107874] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-13 10:51:07 INFO: [MetadataReader 14300761082] Reading file: '/output/internal/meta_data.tsv'\n",
      "2021-03-13 10:51:07 INFO: [MetadataReader 87652392539] Reading file: '/tmp/tmpnEsCS5/tmp_fHln8'\n",
      "2021-03-13 10:51:11 INFO: [FastaAnonymizer 66694448886] Shuffle and anonymize '/tmp/tmpnEsCS5/tmpnULO5e'\n",
      "2021-03-13 10:51:20 INFO: [MetadataReader 4079751341] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-13 10:51:22 INFO: [MetadataReader 84063950941] Reading file: '/output/internal/meta_data.tsv'\n",
      "2021-03-13 10:51:22 INFO: [MetadataReader 48704713275] Reading file: '/tmp/tmpnEsCS5/read_start_positionsaNHNuy'\n",
      "2021-03-13 10:51:24 INFO: [MetadataReader 43119039112] Reading file: '/tmp/tmpnEsCS5/tmpyjWOe8'\n",
      "2021-03-13 10:51:52 INFO: [FastaAnonymizer 64368966291] Shuffle and anonymize '/tmp/tmpnEsCS5/tmplRvn_v'\n",
      "2021-03-13 10:52:01 INFO: [MetadataReader 78717221707] Reading file: '/output/internal/genome_locations.tsv'\n",
      "2021-03-13 10:52:02 INFO: [MetadataReader 89765374142] Reading file: '/output/internal/meta_data.tsv'\n",
      "2021-03-13 10:52:02 INFO: [MetadataReader 98541404663] Reading file: '/tmp/tmpnEsCS5/read_start_positions2fZ_O5'\n",
      "2021-03-13 10:52:05 INFO: [MetadataReader 99062647065] Reading file: '/tmp/tmpnEsCS5/tmpGC2h5z'\n",
      "2021-03-13 10:52:33 INFO: [MetagenomeSimulationPipeline] Compress Data\n",
      "2021-03-13 10:52:33 INFO: [Compress 3477227465] Compressing 'tmpzLJDxr'\n",
      "2021-03-13 10:52:33 INFO: [Compress 75517615449] Compressing 'anonymous_gs_mappingJhYfVd'\n",
      "2021-03-13 10:52:33 INFO: [Compress 84564194976] Compressing 'anonymous_gsa_mappingqaevyV'\n",
      "2021-03-13 10:52:33 INFO: [Compress 63224641871] Compressing 'tmpCQJQSr'\n",
      "2021-03-13 10:52:33 INFO: [Compress 11567118738] Compressing 'tmpcUE3an'\n",
      "2021-03-13 10:52:33 INFO: [Compress 9085777069] Compressing 'anonymous_gsa_pooled_mappingx1bPKa'\n",
      "2021-03-13 10:52:33 INFO: [Compress 9085777069] Done compressing 'gsa_pooled_mapping.tsv.gz' in 0:00:00s.\n",
      "2021-03-13 10:52:33 INFO: [Compress 84564194976] Done compressing 'gsa_mapping.tsv.gz' in 0:00:00s.\n",
      "2021-03-13 10:52:36 INFO: [Compress 75517615449] Done compressing 'reads_mapping.tsv.gz' in 0:00:03s.\n",
      "2021-03-13 10:52:36 INFO: [Compress 63224641871] Done compressing 'anonymous_gsa_pooled.fasta.gz' in 0:00:04s.\n",
      "2021-03-13 10:52:36 INFO: [Compress 3477227465] Done compressing 'anonymous_gsa.fasta.gz' in 0:00:04s.\n",
      "2021-03-13 10:52:49 INFO: [Compress 11567118738] Done compressing 'anonymous_reads.fq.gz' in 0:00:16s.\n",
      "2021-03-13 10:52:49 INFO: [MetagenomeSimulationPipeline] Metagenome simulation finished\n"
     ]
    }
   ],
   "source": [
    "experiments = [os.path.basename(i) for i in glob.glob('example_input_data/new_simulations/complexity*')]\n",
    "\n",
    "for experiment in experiments:\n",
    "    if len(glob.glob(os.path.join('example_input_data/new_simulations',experiment,'*','reads','anonymous*'))) == 0:\n",
    "        print('simulating',experiment)\n",
    "        \n",
    "        input_folder = os.path.join(os.getcwd(), f\"example_input_data/new_simulations/{experiment}\")\n",
    "        output_folder = os.path.join(os.getcwd(), f\"example_input_data/new_simulations/{experiment}\")\n",
    "\n",
    "        input_directory = f\"{input_folder}:/input:rw\"\n",
    "        output_directory = f\"{output_folder}:/output:rw\"\n",
    "\n",
    "        !echo awhf98aq | sudo -S docker run --rm -v $input_directory -v $output_directory  \\\n",
    "            cami/camisim:latest  metagenomesimulation.py /input/mini_config.ini"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 311,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['example_input_data/new_simulations/complexity_sim_1_genera_250_genomes/2021.03.13_09.38.14_sample_0/reads/anonymous_reads.fq.gz',\n",
       " 'example_input_data/new_simulations/complexity_sim_1_genera_25_genomes/2021.03.13_09.21.48_sample_0/reads/anonymous_reads.fq.gz',\n",
       " 'example_input_data/new_simulations/complexity_sim_5_genera_25_genomes/2021.03.13_09.51.05_sample_0/reads/anonymous_reads.fq.gz',\n",
       " 'example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/2021.03.13_09.59.34_sample_0/reads/anonymous_reads.fq.gz',\n",
       " 'example_input_data/new_simulations/complexity_sim_1_genera_50_genomes/2021.03.13_10.06.07_sample_0/reads/anonymous_reads.fq.gz',\n",
       " 'example_input_data/new_simulations/complexity_sim_10_genera_50_genomes/2021.03.13_10.16.05_sample_0/reads/anonymous_reads.fq.gz',\n",
       " 'example_input_data/new_simulations/complexity_sim_5_genera_50_genomes/2021.03.13_10.23.50_sample_0/reads/anonymous_reads.fq.gz',\n",
       " 'example_input_data/new_simulations/complexity_sim_5_genera_250_genomes/2021.03.13_10.30.57_sample_0/reads/anonymous_reads.fq.gz',\n",
       " 'example_input_data/new_simulations/complexity_sim_10_genera_250_genomes/2021.03.13_10.42.02_sample_0/reads/anonymous_reads.fq.gz']"
      ]
     },
     "execution_count": 311,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "glob.glob('example_input_data/new_simulations/com*/*sample_0*/reads/*anonymous_reads*')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Now we just need to run the assemblies\n",
    "\n",
    "Note that many of these steps are described in greater detail in the `preparing_data` notebook."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 365,
   "metadata": {},
   "outputs": [],
   "source": [
    "BASE_DIR = os.getcwd()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 417,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<vamb.vambtools.Reader object at 0x7feb547b9cd0>\n",
      "<_io.TextIOWrapper name='/Users/pmccaffrey/jupyter_projects/CS230_Stanford_Project/example_input_data/new_simulations/complexity_sim_1_genera_250_genomes/2021.03.13_09.38.14_sample_0/contigs/anonymous_gsa_filtered.fasta.gz' mode='w' encoding='UTF-8'>\n",
      "<vamb.vambtools.Reader object at 0x7feb58957110>\n",
      "<_io.TextIOWrapper name='/Users/pmccaffrey/jupyter_projects/CS230_Stanford_Project/example_input_data/new_simulations/complexity_sim_1_genera_25_genomes/2021.03.13_09.21.48_sample_0/contigs/anonymous_gsa_filtered.fasta.gz' mode='w' encoding='UTF-8'>\n",
      "<vamb.vambtools.Reader object at 0x7feb547b9190>\n",
      "<_io.TextIOWrapper name='/Users/pmccaffrey/jupyter_projects/CS230_Stanford_Project/example_input_data/new_simulations/complexity_sim_5_genera_25_genomes/2021.03.13_09.51.05_sample_0/contigs/anonymous_gsa_filtered.fasta.gz' mode='w' encoding='UTF-8'>\n",
      "<vamb.vambtools.Reader object at 0x7feb547b9cd0>\n",
      "<_io.TextIOWrapper name='/Users/pmccaffrey/jupyter_projects/CS230_Stanford_Project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/2021.03.13_09.59.34_sample_0/contigs/anonymous_gsa_filtered.fasta.gz' mode='w' encoding='UTF-8'>\n",
      "<vamb.vambtools.Reader object at 0x7feb58957110>\n",
      "<_io.TextIOWrapper name='/Users/pmccaffrey/jupyter_projects/CS230_Stanford_Project/example_input_data/new_simulations/complexity_sim_1_genera_50_genomes/2021.03.13_10.06.07_sample_0/contigs/anonymous_gsa_filtered.fasta.gz' mode='w' encoding='UTF-8'>\n",
      "<vamb.vambtools.Reader object at 0x7feb547b9190>\n",
      "<_io.TextIOWrapper name='/Users/pmccaffrey/jupyter_projects/CS230_Stanford_Project/example_input_data/new_simulations/complexity_sim_10_genera_50_genomes/2021.03.13_10.16.05_sample_0/contigs/anonymous_gsa_filtered.fasta.gz' mode='w' encoding='UTF-8'>\n",
      "<vamb.vambtools.Reader object at 0x7feb54660dd0>\n",
      "<_io.TextIOWrapper name='/Users/pmccaffrey/jupyter_projects/CS230_Stanford_Project/example_input_data/new_simulations/complexity_sim_5_genera_50_genomes/2021.03.13_10.23.50_sample_0/contigs/anonymous_gsa_filtered.fasta.gz' mode='w' encoding='UTF-8'>\n",
      "<vamb.vambtools.Reader object at 0x7feb58957110>\n",
      "<_io.TextIOWrapper name='/Users/pmccaffrey/jupyter_projects/CS230_Stanford_Project/example_input_data/new_simulations/complexity_sim_5_genera_250_genomes/2021.03.13_10.30.57_sample_0/contigs/anonymous_gsa_filtered.fasta.gz' mode='w' encoding='UTF-8'>\n",
      "<vamb.vambtools.Reader object at 0x7feb547b9190>\n",
      "<_io.TextIOWrapper name='/Users/pmccaffrey/jupyter_projects/CS230_Stanford_Project/example_input_data/new_simulations/complexity_sim_10_genera_250_genomes/2021.03.13_10.42.02_sample_0/contigs/anonymous_gsa_filtered.fasta.gz' mode='w' encoding='UTF-8'>\n"
     ]
    }
   ],
   "source": [
    "for SIM_FASTA_FILE in glob.glob('example_input_data/new_simulations/complexity_sim*/*sample_0*'):\n",
    "    input_contigs_fasta = os.path.join(BASE_DIR, f\"{SIM_FASTA_FILE}/contigs/anonymous_gsa.fasta.gz\")\n",
    "    output_filtered_contigs_fasta = os.path.join(BASE_DIR, f\"{SIM_FASTA_FILE}/contigs/anonymous_gsa_filtered.fasta.gz\")\n",
    "\n",
    "    with vamb.vambtools.Reader(input_contigs_fasta, 'rb') as inputfile:\n",
    "        with open(output_filtered_contigs_fasta, 'w') as outputfile:\n",
    "            print(inputfile)\n",
    "            print(outputfile)\n",
    "            vamb.vambtools.filtercontigs(inputfile, outputfile, minlength=2000)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 454,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: example_input_data/new_simulations/complexity_sim_1_genera_250_genomes/2021.03.13_09.38.14_sample_0\n",
      "[M::mm_idx_gen::0.186*1.08] collected minimizers\n",
      "[M::mm_idx_gen::0.236*1.48] sorted minimizers\n",
      "[M::main::0.323*1.35] loaded/built the index for 583 target sequence(s)\n",
      "[M::mm_idx_stat] kmer size: 15; skip: 10; is_hpc: 0; #seq: 583\n",
      "[M::mm_idx_stat::0.338*1.33] distinct minimizers: 1000598 (89.04% are singletons); average occurrences: 1.137; average spacing: 5.361\n",
      "[M::main] Version: 2.17-r941\n",
      "[M::main] CMD: /Users/pmccaffrey/anaconda3/envs/vamb_env/bin/minimap2 -d example_input_data/new_simulations/complexity_sim_1_genera_250_genomes/2021.03.13_09.38.14_sample_0/catalogue.mmi /Users/pmccaffrey/jupyter_projects/CS230_Stanford_Project/example_input_data/new_simulations/complexity_sim_1_genera_250_genomes/2021.03.13_09.38.14_sample_0/contigs/anonymous_gsa_filtered.fasta.gz\n",
      "[M::main] Real time: 0.351 sec; CPU: 0.462 sec; Peak RSS: 0.081 GB\n",
      "minimap index location example_input_data/new_simulations/complexity_sim_1_genera_250_genomes/2021.03.13_09.38.14_sample_0/catalogue.mmi\n",
      "[WARNING]\u001b[1;31m Indexing parameters (-k, -w or -H) overridden by parameters used in the prebuilt index.\u001b[0m\n",
      "[M::main::0.101*1.02] loaded/built the index for 583 target sequence(s)\n",
      "[M::mm_mapopt_update::0.101*1.02] mid_occ = 1000\n",
      "[M::mm_idx_stat] kmer size: 15; skip: 10; is_hpc: 0; #seq: 583\n",
      "[M::mm_idx_stat::0.114*1.02] distinct minimizers: 1000598 (89.04% are singletons); average occurrences: 1.137; average spacing: 5.361\n",
      "[M::worker_pipeline::3.013*6.48] mapped 333334 sequences\n",
      "[M::worker_pipeline::3.903*6.12] mapped 330738 sequences\n",
      "[M::main] Version: 2.17-r941\n",
      "[M::main] CMD: /Users/pmccaffrey/anaconda3/envs/vamb_env/bin/minimap2 -t 10 -N 50 -ax sr example_input_data/new_simulations/complexity_sim_1_genera_250_genomes/2021.03.13_09.38.14_sample_0/catalogue.mmi example_input_data/new_simulations/complexity_sim_1_genera_250_genomes/2021.03.13_09.38.14_sample_0/reads/anonymous_reads.fq.gz\n",
      "[M::main] Real time: 3.912 sec; CPU: 23.883 sec; Peak RSS: 0.336 GB\n",
      "Processing: example_input_data/new_simulations/complexity_sim_1_genera_25_genomes/2021.03.13_09.21.48_sample_0\n",
      "[M::mm_idx_gen::0.271*1.04] collected minimizers\n",
      "[M::mm_idx_gen::0.334*1.41] sorted minimizers\n",
      "[M::main::0.450*1.30] loaded/built the index for 2095 target sequence(s)\n",
      "[M::mm_idx_stat] kmer size: 15; skip: 10; is_hpc: 0; #seq: 2095\n",
      "[M::mm_idx_stat::0.471*1.29] distinct minimizers: 1270132 (58.47% are singletons); average occurrences: 1.528; average spacing: 5.373\n",
      "[M::main] Version: 2.17-r941\n",
      "[M::main] CMD: /Users/pmccaffrey/anaconda3/envs/vamb_env/bin/minimap2 -d example_input_data/new_simulations/complexity_sim_1_genera_25_genomes/2021.03.13_09.21.48_sample_0/catalogue.mmi /Users/pmccaffrey/jupyter_projects/CS230_Stanford_Project/example_input_data/new_simulations/complexity_sim_1_genera_25_genomes/2021.03.13_09.21.48_sample_0/contigs/anonymous_gsa_filtered.fasta.gz\n",
      "[M::main] Real time: 0.488 sec; CPU: 0.623 sec; Peak RSS: 0.101 GB\n",
      "minimap index location example_input_data/new_simulations/complexity_sim_1_genera_25_genomes/2021.03.13_09.21.48_sample_0/catalogue.mmi\n",
      "[WARNING]\u001b[1;31m Indexing parameters (-k, -w or -H) overridden by parameters used in the prebuilt index.\u001b[0m\n",
      "[M::main::0.113*1.02] loaded/built the index for 2095 target sequence(s)\n",
      "[M::mm_mapopt_update::0.113*1.02] mid_occ = 1000\n",
      "[M::mm_idx_stat] kmer size: 15; skip: 10; is_hpc: 0; #seq: 2095\n",
      "[M::mm_idx_stat::0.130*1.02] distinct minimizers: 1270132 (58.47% are singletons); average occurrences: 1.528; average spacing: 5.373\n",
      "[M::worker_pipeline::3.472*7.43] mapped 333334 sequences\n",
      "[M::worker_pipeline::4.585*6.96] mapped 330624 sequences\n",
      "[M::main] Version: 2.17-r941\n",
      "[M::main] CMD: /Users/pmccaffrey/anaconda3/envs/vamb_env/bin/minimap2 -t 10 -N 50 -ax sr example_input_data/new_simulations/complexity_sim_1_genera_25_genomes/2021.03.13_09.21.48_sample_0/catalogue.mmi example_input_data/new_simulations/complexity_sim_1_genera_25_genomes/2021.03.13_09.21.48_sample_0/reads/anonymous_reads.fq.gz\n",
      "[M::main] Real time: 4.597 sec; CPU: 31.908 sec; Peak RSS: 0.385 GB\n",
      "Processing: example_input_data/new_simulations/complexity_sim_5_genera_25_genomes/2021.03.13_09.51.05_sample_0\n",
      "[M::mm_idx_gen::0.289*1.04] collected minimizers\n",
      "[M::mm_idx_gen::0.356*1.41] sorted minimizers\n",
      "[M::main::0.513*1.26] loaded/built the index for 2071 target sequence(s)\n",
      "[M::mm_idx_stat] kmer size: 15; skip: 10; is_hpc: 0; #seq: 2071\n",
      "[M::mm_idx_stat::0.540*1.25] distinct minimizers: 2003541 (97.40% are singletons); average occurrences: 1.029; average spacing: 5.367\n",
      "[M::main] Version: 2.17-r941\n",
      "[M::main] CMD: /Users/pmccaffrey/anaconda3/envs/vamb_env/bin/minimap2 -d example_input_data/new_simulations/complexity_sim_5_genera_25_genomes/2021.03.13_09.51.05_sample_0/catalogue.mmi /Users/pmccaffrey/jupyter_projects/CS230_Stanford_Project/example_input_data/new_simulations/complexity_sim_5_genera_25_genomes/2021.03.13_09.51.05_sample_0/contigs/anonymous_gsa_filtered.fasta.gz\n",
      "[M::main] Real time: 0.565 sec; CPU: 0.697 sec; Peak RSS: 0.128 GB\n",
      "minimap index location example_input_data/new_simulations/complexity_sim_5_genera_25_genomes/2021.03.13_09.51.05_sample_0/catalogue.mmi\n",
      "[WARNING]\u001b[1;31m Indexing parameters (-k, -w or -H) overridden by parameters used in the prebuilt index.\u001b[0m\n",
      "[M::main::0.177*1.01] loaded/built the index for 2071 target sequence(s)\n",
      "[M::mm_mapopt_update::0.177*1.01] mid_occ = 1000\n",
      "[M::mm_idx_stat] kmer size: 15; skip: 10; is_hpc: 0; #seq: 2071\n",
      "[M::mm_idx_stat::0.206*1.01] distinct minimizers: 2003541 (97.40% are singletons); average occurrences: 1.029; average spacing: 5.367\n",
      "[M::worker_pipeline::2.117*4.09] mapped 333334 sequences\n",
      "[M::worker_pipeline::3.107*4.73] mapped 330644 sequences\n",
      "[M::main] Version: 2.17-r941\n",
      "[M::main] CMD: /Users/pmccaffrey/anaconda3/envs/vamb_env/bin/minimap2 -t 10 -N 50 -ax sr example_input_data/new_simulations/complexity_sim_5_genera_25_genomes/2021.03.13_09.51.05_sample_0/catalogue.mmi example_input_data/new_simulations/complexity_sim_5_genera_25_genomes/2021.03.13_09.51.05_sample_0/reads/anonymous_reads.fq.gz\n",
      "[M::main] Real time: 3.118 sec; CPU: 14.708 sec; Peak RSS: 0.295 GB\n",
      "Processing: example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/2021.03.13_09.59.34_sample_0\n",
      "[M::mm_idx_gen::0.252*1.05] collected minimizers\n",
      "[M::mm_idx_gen::0.318*1.45] sorted minimizers\n",
      "[M::main::0.456*1.29] loaded/built the index for 1977 target sequence(s)\n",
      "[M::mm_idx_stat] kmer size: 15; skip: 10; is_hpc: 0; #seq: 1977\n",
      "[M::mm_idx_stat::0.477*1.28] distinct minimizers: 1558062 (85.04% are singletons); average occurrences: 1.173; average spacing: 5.368\n",
      "[M::main] Version: 2.17-r941\n",
      "[M::main] CMD: /Users/pmccaffrey/anaconda3/envs/vamb_env/bin/minimap2 -d example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/2021.03.13_09.59.34_sample_0/catalogue.mmi /Users/pmccaffrey/jupyter_projects/CS230_Stanford_Project/example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/2021.03.13_09.59.34_sample_0/contigs/anonymous_gsa_filtered.fasta.gz\n",
      "[M::main] Real time: 0.491 sec; CPU: 0.625 sec; Peak RSS: 0.090 GB\n",
      "minimap index location example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/2021.03.13_09.59.34_sample_0/catalogue.mmi\n",
      "[WARNING]\u001b[1;31m Indexing parameters (-k, -w or -H) overridden by parameters used in the prebuilt index.\u001b[0m\n",
      "[M::main::0.132*1.01] loaded/built the index for 1977 target sequence(s)\n",
      "[M::mm_mapopt_update::0.132*1.01] mid_occ = 1000\n",
      "[M::mm_idx_stat] kmer size: 15; skip: 10; is_hpc: 0; #seq: 1977\n",
      "[M::mm_idx_stat::0.152*1.01] distinct minimizers: 1558062 (85.04% are singletons); average occurrences: 1.173; average spacing: 5.368\n",
      "[M::worker_pipeline::2.905*6.21] mapped 333334 sequences\n",
      "[M::worker_pipeline::3.790*5.94] mapped 324724 sequences\n",
      "[M::main] Version: 2.17-r941\n",
      "[M::main] CMD: /Users/pmccaffrey/anaconda3/envs/vamb_env/bin/minimap2 -t 10 -N 50 -ax sr example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/2021.03.13_09.59.34_sample_0/catalogue.mmi example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/2021.03.13_09.59.34_sample_0/reads/anonymous_reads.fq.gz\n",
      "[M::main] Real time: 3.800 sec; CPU: 22.516 sec; Peak RSS: 0.331 GB\n",
      "Processing: example_input_data/new_simulations/complexity_sim_1_genera_50_genomes/2021.03.13_10.06.07_sample_0\n",
      "[M::mm_idx_gen::0.258*1.04] collected minimizers\n",
      "[M::mm_idx_gen::0.328*1.45] sorted minimizers\n",
      "[M::main::0.441*1.33] loaded/built the index for 2741 target sequence(s)\n",
      "[M::mm_idx_stat] kmer size: 15; skip: 10; is_hpc: 0; #seq: 2741\n",
      "[M::mm_idx_stat::0.460*1.32] distinct minimizers: 1185349 (63.60% are singletons); average occurrences: 1.492; average spacing: 5.379\n",
      "[M::main] Version: 2.17-r941\n",
      "[M::main] CMD: /Users/pmccaffrey/anaconda3/envs/vamb_env/bin/minimap2 -d example_input_data/new_simulations/complexity_sim_1_genera_50_genomes/2021.03.13_10.06.07_sample_0/catalogue.mmi /Users/pmccaffrey/jupyter_projects/CS230_Stanford_Project/example_input_data/new_simulations/complexity_sim_1_genera_50_genomes/2021.03.13_10.06.07_sample_0/contigs/anonymous_gsa_filtered.fasta.gz\n",
      "[M::main] Real time: 0.471 sec; CPU: 0.618 sec; Peak RSS: 0.090 GB\n",
      "minimap index location example_input_data/new_simulations/complexity_sim_1_genera_50_genomes/2021.03.13_10.06.07_sample_0/catalogue.mmi\n",
      "[WARNING]\u001b[1;31m Indexing parameters (-k, -w or -H) overridden by parameters used in the prebuilt index.\u001b[0m\n",
      "[M::main::0.107*1.02] loaded/built the index for 2741 target sequence(s)\n",
      "[M::mm_mapopt_update::0.107*1.02] mid_occ = 1000\n",
      "[M::mm_idx_stat] kmer size: 15; skip: 10; is_hpc: 0; #seq: 2741\n",
      "[M::mm_idx_stat::0.123*1.02] distinct minimizers: 1185349 (63.60% are singletons); average occurrences: 1.492; average spacing: 5.379\n",
      "[M::worker_pipeline::3.492*7.16] mapped 333334 sequences\n",
      "[M::worker_pipeline::4.524*6.72] mapped 329860 sequences\n",
      "[M::main] Version: 2.17-r941\n",
      "[M::main] CMD: /Users/pmccaffrey/anaconda3/envs/vamb_env/bin/minimap2 -t 10 -N 50 -ax sr example_input_data/new_simulations/complexity_sim_1_genera_50_genomes/2021.03.13_10.06.07_sample_0/catalogue.mmi example_input_data/new_simulations/complexity_sim_1_genera_50_genomes/2021.03.13_10.06.07_sample_0/reads/anonymous_reads.fq.gz\n",
      "[M::main] Real time: 4.535 sec; CPU: 30.430 sec; Peak RSS: 0.376 GB\n",
      "Processing: example_input_data/new_simulations/complexity_sim_10_genera_50_genomes/2021.03.13_10.16.05_sample_0\n",
      "[M::mm_idx_gen::0.293*1.04] collected minimizers\n",
      "[M::mm_idx_gen::0.366*1.43] sorted minimizers\n",
      "[M::main::0.514*1.30] loaded/built the index for 2887 target sequence(s)\n",
      "[M::mm_idx_stat] kmer size: 15; skip: 10; is_hpc: 0; #seq: 2887\n",
      "[M::mm_idx_stat::0.541*1.29] distinct minimizers: 2010190 (96.61% are singletons); average occurrences: 1.042; average spacing: 5.377\n",
      "[M::main] Version: 2.17-r941\n",
      "[M::main] CMD: /Users/pmccaffrey/anaconda3/envs/vamb_env/bin/minimap2 -d example_input_data/new_simulations/complexity_sim_10_genera_50_genomes/2021.03.13_10.16.05_sample_0/catalogue.mmi /Users/pmccaffrey/jupyter_projects/CS230_Stanford_Project/example_input_data/new_simulations/complexity_sim_10_genera_50_genomes/2021.03.13_10.16.05_sample_0/contigs/anonymous_gsa_filtered.fasta.gz\n",
      "[M::main] Real time: 0.555 sec; CPU: 0.709 sec; Peak RSS: 0.138 GB\n",
      "minimap index location example_input_data/new_simulations/complexity_sim_10_genera_50_genomes/2021.03.13_10.16.05_sample_0/catalogue.mmi\n",
      "[WARNING]\u001b[1;31m Indexing parameters (-k, -w or -H) overridden by parameters used in the prebuilt index.\u001b[0m\n",
      "[M::main::0.173*1.01] loaded/built the index for 2887 target sequence(s)\n",
      "[M::mm_mapopt_update::0.173*1.01] mid_occ = 1000\n",
      "[M::mm_idx_stat] kmer size: 15; skip: 10; is_hpc: 0; #seq: 2887\n",
      "[M::mm_idx_stat::0.198*1.01] distinct minimizers: 2010190 (96.61% are singletons); average occurrences: 1.042; average spacing: 5.377\n",
      "[M::worker_pipeline::2.071*4.10] mapped 333334 sequences\n",
      "[M::worker_pipeline::3.047*4.76] mapped 322550 sequences\n",
      "[M::main] Version: 2.17-r941\n",
      "[M::main] CMD: /Users/pmccaffrey/anaconda3/envs/vamb_env/bin/minimap2 -t 10 -N 50 -ax sr example_input_data/new_simulations/complexity_sim_10_genera_50_genomes/2021.03.13_10.16.05_sample_0/catalogue.mmi example_input_data/new_simulations/complexity_sim_10_genera_50_genomes/2021.03.13_10.16.05_sample_0/reads/anonymous_reads.fq.gz\n",
      "[M::main] Real time: 3.058 sec; CPU: 14.528 sec; Peak RSS: 0.293 GB\n",
      "Processing: example_input_data/new_simulations/complexity_sim_5_genera_50_genomes/2021.03.13_10.23.50_sample_0\n",
      "[M::mm_idx_gen::0.316*1.04] collected minimizers\n",
      "[M::mm_idx_gen::0.391*1.41] sorted minimizers\n",
      "[M::main::0.552*1.28] loaded/built the index for 2425 target sequence(s)\n",
      "[M::mm_idx_stat] kmer size: 15; skip: 10; is_hpc: 0; #seq: 2425\n",
      "[M::mm_idx_stat::0.581*1.27] distinct minimizers: 2097622 (92.28% are singletons); average occurrences: 1.089; average spacing: 5.366\n",
      "[M::main] Version: 2.17-r941\n",
      "[M::main] CMD: /Users/pmccaffrey/anaconda3/envs/vamb_env/bin/minimap2 -d example_input_data/new_simulations/complexity_sim_5_genera_50_genomes/2021.03.13_10.23.50_sample_0/catalogue.mmi /Users/pmccaffrey/jupyter_projects/CS230_Stanford_Project/example_input_data/new_simulations/complexity_sim_5_genera_50_genomes/2021.03.13_10.23.50_sample_0/contigs/anonymous_gsa_filtered.fasta.gz\n",
      "[M::main] Real time: 0.598 sec; CPU: 0.754 sec; Peak RSS: 0.145 GB\n",
      "minimap index location example_input_data/new_simulations/complexity_sim_5_genera_50_genomes/2021.03.13_10.23.50_sample_0/catalogue.mmi\n",
      "[WARNING]\u001b[1;31m Indexing parameters (-k, -w or -H) overridden by parameters used in the prebuilt index.\u001b[0m\n",
      "[M::main::0.181*1.01] loaded/built the index for 2425 target sequence(s)\n",
      "[M::mm_mapopt_update::0.181*1.01] mid_occ = 1000\n",
      "[M::mm_idx_stat] kmer size: 15; skip: 10; is_hpc: 0; #seq: 2425\n",
      "[M::mm_idx_stat::0.209*1.01] distinct minimizers: 2097622 (92.28% are singletons); average occurrences: 1.089; average spacing: 5.366\n",
      "[M::worker_pipeline::2.602*5.28] mapped 333334 sequences\n",
      "[M::worker_pipeline::3.723*5.53] mapped 328388 sequences\n",
      "[M::main] Version: 2.17-r941\n",
      "[M::main] CMD: /Users/pmccaffrey/anaconda3/envs/vamb_env/bin/minimap2 -t 10 -N 50 -ax sr example_input_data/new_simulations/complexity_sim_5_genera_50_genomes/2021.03.13_10.23.50_sample_0/catalogue.mmi example_input_data/new_simulations/complexity_sim_5_genera_50_genomes/2021.03.13_10.23.50_sample_0/reads/anonymous_reads.fq.gz\n",
      "[M::main] Real time: 3.733 sec; CPU: 20.599 sec; Peak RSS: 0.330 GB\n",
      "Processing: example_input_data/new_simulations/complexity_sim_5_genera_250_genomes/2021.03.13_10.30.57_sample_0\n",
      "[M::mm_idx_gen::0.193*1.06] collected minimizers\n",
      "[M::mm_idx_gen::0.249*1.48] sorted minimizers\n",
      "[M::main::0.357*1.33] loaded/built the index for 1522 target sequence(s)\n",
      "[M::mm_idx_stat] kmer size: 15; skip: 10; is_hpc: 0; #seq: 1522\n",
      "[M::mm_idx_stat::0.370*1.32] distinct minimizers: 1261927 (98.22% are singletons); average occurrences: 1.020; average spacing: 5.365\n",
      "[M::main] Version: 2.17-r941\n",
      "[M::main] CMD: /Users/pmccaffrey/anaconda3/envs/vamb_env/bin/minimap2 -d example_input_data/new_simulations/complexity_sim_5_genera_250_genomes/2021.03.13_10.30.57_sample_0/catalogue.mmi /Users/pmccaffrey/jupyter_projects/CS230_Stanford_Project/example_input_data/new_simulations/complexity_sim_5_genera_250_genomes/2021.03.13_10.30.57_sample_0/contigs/anonymous_gsa_filtered.fasta.gz\n",
      "[M::main] Real time: 0.383 sec; CPU: 0.499 sec; Peak RSS: 0.074 GB\n",
      "minimap index location example_input_data/new_simulations/complexity_sim_5_genera_250_genomes/2021.03.13_10.30.57_sample_0/catalogue.mmi\n",
      "[WARNING]\u001b[1;31m Indexing parameters (-k, -w or -H) overridden by parameters used in the prebuilt index.\u001b[0m\n",
      "[M::main::0.102*1.02] loaded/built the index for 1522 target sequence(s)\n",
      "[M::mm_mapopt_update::0.102*1.02] mid_occ = 1000\n",
      "[M::mm_idx_stat] kmer size: 15; skip: 10; is_hpc: 0; #seq: 1522\n",
      "[M::mm_idx_stat::0.113*1.02] distinct minimizers: 1261927 (98.22% are singletons); average occurrences: 1.020; average spacing: 5.365\n",
      "[M::worker_pipeline::1.909*4.07] mapped 333334 sequences\n",
      "[M::worker_pipeline::2.978*4.64] mapped 330322 sequences\n",
      "[M::main] Version: 2.17-r941\n",
      "[M::main] CMD: /Users/pmccaffrey/anaconda3/envs/vamb_env/bin/minimap2 -t 10 -N 50 -ax sr example_input_data/new_simulations/complexity_sim_5_genera_250_genomes/2021.03.13_10.30.57_sample_0/catalogue.mmi example_input_data/new_simulations/complexity_sim_5_genera_250_genomes/2021.03.13_10.30.57_sample_0/reads/anonymous_reads.fq.gz\n",
      "[M::main] Real time: 2.989 sec; CPU: 13.815 sec; Peak RSS: 0.251 GB\n",
      "Processing: example_input_data/new_simulations/complexity_sim_10_genera_250_genomes/2021.03.13_10.42.02_sample_0\n",
      "[M::mm_idx_gen::0.195*1.07] collected minimizers\n",
      "[M::mm_idx_gen::0.242*1.44] sorted minimizers\n",
      "[M::main::0.355*1.28] loaded/built the index for 1602 target sequence(s)\n",
      "[M::mm_idx_stat] kmer size: 15; skip: 10; is_hpc: 0; #seq: 1602\n",
      "[M::mm_idx_stat::0.371*1.27] distinct minimizers: 1194737 (98.33% are singletons); average occurrences: 1.018; average spacing: 5.379\n",
      "[M::main] Version: 2.17-r941\n",
      "[M::main] CMD: /Users/pmccaffrey/anaconda3/envs/vamb_env/bin/minimap2 -d example_input_data/new_simulations/complexity_sim_10_genera_250_genomes/2021.03.13_10.42.02_sample_0/catalogue.mmi /Users/pmccaffrey/jupyter_projects/CS230_Stanford_Project/example_input_data/new_simulations/complexity_sim_10_genera_250_genomes/2021.03.13_10.42.02_sample_0/contigs/anonymous_gsa_filtered.fasta.gz\n",
      "[M::main] Real time: 0.385 sec; CPU: 0.486 sec; Peak RSS: 0.074 GB\n",
      "minimap index location example_input_data/new_simulations/complexity_sim_10_genera_250_genomes/2021.03.13_10.42.02_sample_0/catalogue.mmi\n",
      "[WARNING]\u001b[1;31m Indexing parameters (-k, -w or -H) overridden by parameters used in the prebuilt index.\u001b[0m\n",
      "[M::main::0.112*1.02] loaded/built the index for 1602 target sequence(s)\n",
      "[M::mm_mapopt_update::0.112*1.02] mid_occ = 1000\n",
      "[M::mm_idx_stat] kmer size: 15; skip: 10; is_hpc: 0; #seq: 1602\n",
      "[M::mm_idx_stat::0.127*1.01] distinct minimizers: 1194737 (98.33% are singletons); average occurrences: 1.018; average spacing: 5.379\n",
      "[M::worker_pipeline::2.078*3.65] mapped 333334 sequences\n",
      "[M::worker_pipeline::3.130*4.15] mapped 320314 sequences\n",
      "[M::main] Version: 2.17-r941\n",
      "[M::main] CMD: /Users/pmccaffrey/anaconda3/envs/vamb_env/bin/minimap2 -t 10 -N 50 -ax sr example_input_data/new_simulations/complexity_sim_10_genera_250_genomes/2021.03.13_10.42.02_sample_0/catalogue.mmi example_input_data/new_simulations/complexity_sim_10_genera_250_genomes/2021.03.13_10.42.02_sample_0/reads/anonymous_reads.fq.gz\n",
      "[M::main] Real time: 3.137 sec; CPU: 12.983 sec; Peak RSS: 0.244 GB\n"
     ]
    }
   ],
   "source": [
    "for SIM_FASTA_FILE in glob.glob('example_input_data/new_simulations/complexity_sim*/*sample_0*'):\n",
    "    print('Processing:', SIM_FASTA_FILE)\n",
    "    minimap2_input = f\"{SIM_FASTA_FILE}/reads/anonymous_reads.fq.gz\"\n",
    "    minimap2_output = f\"{SIM_FASTA_FILE}/assemblies/re_mapped.bam\"\n",
    "\n",
    "    minimap_index_location = f\"{SIM_FASTA_FILE}/catalogue.mmi\"\n",
    "    \n",
    "    output_filtered_contigs_fasta = os.path.join(BASE_DIR, f\"{SIM_FASTA_FILE}/contigs/anonymous_gsa_filtered.fasta.gz\")\n",
    "\n",
    "    !~/anaconda3/envs/vamb_env/bin/minimap2 -d $minimap_index_location $output_filtered_contigs_fasta \n",
    "    \n",
    "    print('minimap index location', minimap_index_location)\n",
    "\n",
    "    if not os.path.exists(os.path.join(SIM_FASTA_FILE,'assemblies')):\n",
    "        os.mkdir(os.path.join(SIM_FASTA_FILE,'assemblies'))\n",
    "\n",
    "    !~/anaconda3/envs/vamb_env/bin/minimap2 \\\n",
    "        -t 10 \\\n",
    "        -N 50 \\\n",
    "        -ax sr $minimap_index_location \\\n",
    "        $minimap2_input | samtools view -F 3584 -b --threads 8 > $minimap2_output"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "And we calculate the TNF and RPKM matrices"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 455,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bamfile for rpkm: ['example_input_data/new_simulations/complexity_sim_1_genera_250_genomes/2021.03.13_09.38.14_sample_0/assemblies/re_mapped.bam']\n",
      "Type of rpkms: <class 'numpy.ndarray'> of dtype float32\n",
      "Shape of rpkms (583, 1)\n",
      "tnfs shape: (583, 103)\n",
      "bamfile for rpkm: ['example_input_data/new_simulations/complexity_sim_1_genera_25_genomes/2021.03.13_09.21.48_sample_0/assemblies/re_mapped.bam']\n",
      "Type of rpkms: <class 'numpy.ndarray'> of dtype float32\n",
      "Shape of rpkms (2095, 1)\n",
      "tnfs shape: (2095, 103)\n",
      "bamfile for rpkm: ['example_input_data/new_simulations/complexity_sim_5_genera_25_genomes/2021.03.13_09.51.05_sample_0/assemblies/re_mapped.bam']\n",
      "Type of rpkms: <class 'numpy.ndarray'> of dtype float32\n",
      "Shape of rpkms (2071, 1)\n",
      "tnfs shape: (2071, 103)\n",
      "bamfile for rpkm: ['example_input_data/new_simulations/complexity_sim_10_genera_25_genomes/2021.03.13_09.59.34_sample_0/assemblies/re_mapped.bam']\n",
      "Type of rpkms: <class 'numpy.ndarray'> of dtype float32\n",
      "Shape of rpkms (1977, 1)\n",
      "tnfs shape: (1977, 103)\n",
      "bamfile for rpkm: ['example_input_data/new_simulations/complexity_sim_1_genera_50_genomes/2021.03.13_10.06.07_sample_0/assemblies/re_mapped.bam']\n",
      "Type of rpkms: <class 'numpy.ndarray'> of dtype float32\n",
      "Shape of rpkms (2741, 1)\n",
      "tnfs shape: (2741, 103)\n",
      "bamfile for rpkm: ['example_input_data/new_simulations/complexity_sim_10_genera_50_genomes/2021.03.13_10.16.05_sample_0/assemblies/re_mapped.bam']\n",
      "Type of rpkms: <class 'numpy.ndarray'> of dtype float32\n",
      "Shape of rpkms (2887, 1)\n",
      "tnfs shape: (2887, 103)\n",
      "bamfile for rpkm: ['example_input_data/new_simulations/complexity_sim_5_genera_50_genomes/2021.03.13_10.23.50_sample_0/assemblies/re_mapped.bam']\n",
      "Type of rpkms: <class 'numpy.ndarray'> of dtype float32\n",
      "Shape of rpkms (2425, 1)\n",
      "tnfs shape: (2425, 103)\n",
      "bamfile for rpkm: ['example_input_data/new_simulations/complexity_sim_5_genera_250_genomes/2021.03.13_10.30.57_sample_0/assemblies/re_mapped.bam']\n",
      "Type of rpkms: <class 'numpy.ndarray'> of dtype float32\n",
      "Shape of rpkms (1522, 1)\n",
      "tnfs shape: (1522, 103)\n",
      "bamfile for rpkm: ['example_input_data/new_simulations/complexity_sim_10_genera_250_genomes/2021.03.13_10.42.02_sample_0/assemblies/re_mapped.bam']\n",
      "Type of rpkms: <class 'numpy.ndarray'> of dtype float32\n",
      "Shape of rpkms (1602, 1)\n",
      "tnfs shape: (1602, 103)\n"
     ]
    }
   ],
   "source": [
    "for SIM_FASTA_FILE in glob.glob('example_input_data/new_simulations/complexity_sim*/*sample_0*'):\n",
    "    output_filtered_contigs_fasta = f\"{SIM_FASTA_FILE}/contigs/anonymous_gsa_filtered.fasta.gz\"\n",
    "    \n",
    "    # Use Reader to open plain or zipped files. File must be opened in binary mode\n",
    "    with vamb.vambtools.Reader(output_filtered_contigs_fasta, 'rb') as inputfile:\n",
    "        tnfs, contignames, lengths = vamb.parsecontigs.read_contigs(inputfile)\n",
    "        \n",
    "    sample_bamfile = glob.glob(f\"{SIM_FASTA_FILE}/assemblies/re_mapped.bam\")\n",
    "    print(\"bamfile for rpkm:\", sample_bamfile)\n",
    "    \n",
    "    rpkms = vamb.parsebam.read_bamfiles(sample_bamfile) \n",
    "    print('Type of rpkms:', type(rpkms), 'of dtype', rpkms.dtype)\n",
    "    print('Shape of rpkms', rpkms.shape)\n",
    "    \n",
    "    print(\"tnfs shape:\", tnfs.shape)\n",
    "    \n",
    "    \n",
    "    vamb_inputs_base = os.path.join(SIM_FASTA_FILE,'vamb_inputs')\n",
    "    \n",
    "    \n",
    "    if not os.path.exists(vamb_inputs_base):\n",
    "        os.mkdir(vamb_inputs_base)\n",
    "\n",
    "    with open(os.path.join(vamb_inputs_base, 'contignames.npz'), 'wb') as file:\n",
    "        vamb.vambtools.write_npz(file, np.array(contignames))\n",
    "\n",
    "    with open(os.path.join(vamb_inputs_base, 'lengths.npz'), 'wb') as file:\n",
    "        vamb.vambtools.write_npz(file, lengths)\n",
    "\n",
    "    with open(os.path.join(vamb_inputs_base, 'tnfs.npz'), 'wb') as file:\n",
    "        vamb.vambtools.write_npz(file, tnfs)\n",
    "\n",
    "    with open(os.path.join(vamb_inputs_base, 'rpkms.npz'), 'wb') as file:\n",
    "        vamb.vambtools.write_npz(file, rpkms)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 411,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "False\n",
      "Binary files example_input_data/new_simulations/complexity_sim_10_genera_50_genomes/2021.03.13_10.16.05_sample_0/assemblies/re_mapped.bam and example_input_data/new_simulations/complexity_sim_1_genera_250_genomes/2021.03.13_09.38.14_sample_0/assemblies/re_mapped.bam differ\n"
     ]
    }
   ],
   "source": [
    "f1, f2 = np.random.choice(glob.glob('example_input_data/new_simulations/complexity_sim*/*sample_0*/*assem*/*re_mapped*'), 2)\n",
    "\n",
    "print(f1 == f2)\n",
    "\n",
    "!diff $f1 $f2"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "vamb_env",
   "language": "python",
   "name": "vamb_env"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.5"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
